---
bibliography: references.bib
---

# Omics analysis

When you get the filtered ions, the next step is making annotations for them. Such annotations would be helpful for omics studies. Omics analysis try to combine the information from other 'omics' to answer one specific question. Since we have got the annotations, Omics analysis could be performed.Upload the data obtained from the **xcms** to other tools or databases.

You will get an updated database list [here](http://metabolomicssociety.org/resources/metabolomics-databases)

Right now, it is hard to connect different omics databases such as gene, protein and metabolites together for a whole scope of certain biological process. However, you might select few metabolites across those databases and find something interesting.

## From Bottom-up to Top-down

Bottom-up analysis mean the model for each metabolite. In this case, we could find out which metabolite will be affected by our experiment design. However, take care of multiple comparison issue.

$$
metabolite = f(control/treatment, co-variables)
$$

Top-down analysis mean the model for output. In this case, we could evaluate the contribution of each metabolites. You need variable selection to make a better model.

$$
control/treatment = f(metabolite 1,metabolite 2,...,metaboliteN,co-varuables)
$$

For omics study, you might need to integrate dataset from different sources.

$$
control/treatment = f(metabolites, proteins, genes, miRNA,co-varuables)
$$

## Pathway analysis

Pathwat analysis maps annotated data into known pathway and make statistical analysis to find the influenced pathway or the compounds with high inflences on certain pathway.

### Pathway Database

-   [SMPDB](http://smpdb.ca/view) (The Small Molecule Pathway Database) is an interactive, visual database containing more than 618 small molecule pathways found in humans. More than 70% of these pathways (\>433) are not found in any other pathway database. The pathways include metabolic, drug, and disease pathways.

-   [KEGG](https://www.genome.jp/kegg/) (Kyoto Encyclopedia of Genes and Genomes) is one of the most complete and widely used databases containing metabolic pathways (495 reference pathways) from a wide variety of organisms (\>4,700). These pathways are hyperlinked to metabolite and protein/enzyme information. Currently KEGG has \>17,000 compounds (from animals, plants and bacteria), 10,000 drugs (including different salt forms and drug carriers) and nearly 11,000 glycan structures.

-   [BioCyc](https://biocyc.org/) is a collection of 14558 Pathway/Genome Databases (PGDBs), plus software tools for exploring them.

-   [Reactome](https://reactome.org/what-is-reactome) is an open-source, open access, manually curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic and clinical research, genome analysis, modeling, systems biology and education.

-   [WikiPathway](https://www.wikipathways.org/index.php/WikiPathways) is a database of biological pathways maintained by and for the scientific community.

### Pathway software

-   [Pathway Commons](http://www.pathwaycommons.org/) online tools for pathway analysis

-   [RaMP](https://github.com/Mathelab/RaMP-DB) could make pathway analysis for batch search

-   [metabox](https://github.com/kwanjeeraw/metabox) could make pathway analysis

-   [impala](http://impala.molgen.mpg.de/) is used for pathway enrichment analysis

-   [Metscape](http://metscape.med.umich.edu/) based on Debiased Sparse Partial Correlation (DSPC) algorithm [@basu2017] to make annotation.

## Network analysis

[Mummichog](https://code.google.com/archive/p/atcg/wikis/mummichog_for_metabolomics.wiki) could make pathway and network analysis without annotation.

[MSS](http://web1.sph.emory.edu/users/tyu8/MSS): sequential feature screening procedure to select important sub-network and identify the optimal matching for metabolimics data [@cai2017].

## Omics integration

-   [Blast](https://blast.ncbi.nlm.nih.gov/Blast.cgi) finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.

-   [The Omics Discovery Index (OmicsDI)](https://www.omicsdi.org/) provides a knowledge discovery framework across heterogeneous omics data (genomics, proteomics, transcriptomics and metabolomics).

-   [Omics Data Integration Project](https://github.com/cran/mixOmics)
