---
bibliography: references.bib
---

# Exposome

Nature or nurture debate has a similar paradigm in environmental study: is the ecological system and human health risk dominated by heredity or environment? Twins and siblings study[@lakhani2019; @polderman2015] show that both heritability and environmental factors could explain the phenotypic variance among population. The contribution of environment among different disease functional domain such as hematological and endocrine could achieve almost half of the total variances [@polderman2015]. However, besides those epidemiology proof, little is known about the influences of overall environmental exposure process at molecular level. Conventional exposure study always investigate one or several specific compounds and their environmental fate or toxicology endpoint. Exposome, on the other hand, tries to access multiple exposure factors from biological or environmental samples as much as possible without a predefined compounds list. Those endogenous and exogenous molecules can reveal the exposure process in details. Exposome could not only help to investigate the comprehensive molecules level changes, but also the interactions among molecules in an non-targeted design. By following annotation of captured compounds, exposome can discover exposure markers for certain type of pollution, as well as biomarkers for certain exposure process and discuss related physiological process. The workflow for exposome is quite similar to metabolomics[@hu2021].

According to [CDC](https://www.cdc.gov/niosh/topics/exposome/default.html), The exposome can be defined as the measure of all the exposures of an individual in a lifetime and how those exposures relate to health. Exposomics is the study of the exposome and relies on the application of internal and external exposure assessment methods.

-   Internal exposure relies on fields of study such as genomics, metabolomics, lipidomics, transcriptomics and proteomics.

-   External exposure assessment relies on measuring environmental stresses.

Human Early Life Exposome (HELIX) project[@maitre2022a], a multi-centre cohort of 1301 mother-child pairs, associated individual exposomes consisting of >100 chemical, outdoor, social and lifestyle exposures assessed in pregnancy and childhood, with multi-omics profiles (methylome, transcriptome, proteins and metabolites) in childhood. The data could be found [online](https://helixomics.isglobal.org/).

“molecular gatekeepers”, key metabolites that link single or multiple exposure biomarkers with correlated clusters of endogenous metabolites, could be used to find health-relevant biological metabolites. [@yu2022c]

## Internal exposure

-   [Virtual Metabolic Human Database](https://www.vmh.life/) integrating human and gut microbiome metabolism with nutrition and disease.

## External exposure

### Environmental fate of compounds

#### QSPR

-   [Chemicalize](https://chemicalize.com/) is a powerful online platform for chemical calculations, search, and text processing.

-   [QSPR molecular descriptor generate tools list](http://www.moleculardescriptors.eu/resources/resources.htm)

-   [Spark](http://www.archemcalc.com/sparc.html) uses computational algorithms based on fundamental chemical structure theory to estimate a wide variety of reactivity parameters strictly from molecular structure.

-   [OPERA](https://jcheminf.biomedcentral.com/articles/10.1186/s13321-018-0263-1) OPERA models for predicting physicochemical properties and environmental fate endpoints[@mansouri2018].

LogP is important for analytical chemistry. Mannhold [@mannhold2009] report a comprehensive comparison of logP algorithms. Later, [Rajarshi Guha](http://blog.rguha.net/?p=896) make a comparison with logP algorithms with CDK based on logPstar dataset. Commercial software such as Spark, ACS Labs and ChemAxon might always claim a better performance on in-house dataset compared with public software like KowWIN within EPI Suite. However, we should be careful to evaluate the influence of logP accuracy on the metabolites or unknown compounds.

#### Fate

-   [Wania Group](https://www.utsc.utoronto.ca/labs/wania/downloads/) developed software tools to address various aspects of organic contaminant fate and behaviour.

-   [Trent University](http://www.trentu.ca/academic/aminss/envmodel/models/models.html) release models to predict environmental fate for pollutions such as Level 3.

-   [EAWAG-BBD](http://eawag-bbd.ethz.ch/index.html) could provide information on microbial enzyme-catalyzed reactions that are important for biotechnology.

### Exposure study database

-   The information system [PANGAEA](https://www.pangaea.de) is operated as an Open Access library aimed at archiving, publishing and distributing georeferenced data from earth system research.

-   [Environmental Health Criteria (EHC) Monographs](http://www.inchem.org/pages/ehc.html)

-   [CTD](http://ctdbase.org/) is a robust, publicly available database that aims to advance understanding about how environmental exposures affect human health.

-   [ODMOA](https://www.mass.gov/orgs/office-of-data-management-and-outcomes-assessment) facilitates and coordinates the collection, access to, and use of public health data in order to monitor and improve population health. This data is better for general public health research for Massachusetts.

-   [The Surveillance, Epidemiology, and End Results (SEER)](https://seer.cancer.gov/) Program provides information on cancer statistics in an effort to reduce the cancer burden among the U.S. population.
