<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 7 Annotation | Meta-Workflow</title>
  <meta name="description" content="This is a workflow for metabolomics studies." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 7 Annotation | Meta-Workflow" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="images/cover.png" />
  <meta property="og:description" content="This is a workflow for metabolomics studies." />
  <meta name="github-repo" content="yufree/metaworkflow" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 7 Annotation | Meta-Workflow" />
  
  <meta name="twitter:description" content="This is a workflow for metabolomics studies." />
  <meta name="twitter:image" content="images/cover.png" />

<meta name="author" content="Miao YU" />


<meta name="date" content="2021-01-08" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="raw-data-pretreatment.html"/>
<link rel="next" href="omics-analysis.html"/>
<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="libs/viz-1.8.2/viz.js"></script>
<link href="libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="libs/grViz-binding-1.0.6.1/grViz.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metabolomics Workflow</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#history"><i class="fa fa-check"></i><b>1.1</b> History</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#history-of-mass-spectrometry"><i class="fa fa-check"></i><b>1.1.1</b> History of Mass Spectrometry</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#history-of-metabolomcis"><i class="fa fa-check"></i><b>1.1.2</b> History of Metabolomcis</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#defination"><i class="fa fa-check"></i><b>1.1.3</b> Defination</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#reviews-and-tutorials"><i class="fa fa-check"></i><b>1.2</b> Reviews and tutorials</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#workflow"><i class="fa fa-check"></i><b>1.2.1</b> Workflow</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#data-analysis"><i class="fa fa-check"></i><b>1.2.2</b> Data analysis</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#application"><i class="fa fa-check"></i><b>1.2.3</b> Application</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#challenge"><i class="fa fa-check"></i><b>1.2.4</b> Challenge</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#trends-in-metabolomics"><i class="fa fa-check"></i><b>1.3</b> Trends in Metabolomics</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#workflow-1"><i class="fa fa-check"></i><b>1.4</b> Workflow</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html"><i class="fa fa-check"></i><b>2</b> Experimental design(DoE)</a>
<ul>
<li class="chapter" data-level="2.1" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#homogeneity-study"><i class="fa fa-check"></i><b>2.1</b> Homogeneity study</a></li>
<li class="chapter" data-level="2.2" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#heterogeneity-study"><i class="fa fa-check"></i><b>2.2</b> Heterogeneity study</a></li>
<li class="chapter" data-level="2.3" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#power-analysis"><i class="fa fa-check"></i><b>2.3</b> Power analysis</a></li>
<li class="chapter" data-level="2.4" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#optimization"><i class="fa fa-check"></i><b>2.4</b> Optimization</a></li>
<li class="chapter" data-level="2.5" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#pooled-qc"><i class="fa fa-check"></i><b>2.5</b> Pooled QC</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="pretreatment.html"><a href="pretreatment.html"><i class="fa fa-check"></i><b>3</b> Pretreatment</a>
<ul>
<li class="chapter" data-level="3.1" data-path="pretreatment.html"><a href="pretreatment.html#collection"><i class="fa fa-check"></i><b>3.1</b> Collection</a></li>
<li class="chapter" data-level="3.2" data-path="pretreatment.html"><a href="pretreatment.html#quenching"><i class="fa fa-check"></i><b>3.2</b> Quenching</a></li>
<li class="chapter" data-level="3.3" data-path="pretreatment.html"><a href="pretreatment.html#extraction"><i class="fa fa-check"></i><b>3.3</b> Extraction</a></li>
<li class="chapter" data-level="3.4" data-path="pretreatment.html"><a href="pretreatment.html#derivatization"><i class="fa fa-check"></i><b>3.4</b> Derivatization</a></li>
<li class="chapter" data-level="3.5" data-path="pretreatment.html"><a href="pretreatment.html#storage"><i class="fa fa-check"></i><b>3.5</b> Storage</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html"><i class="fa fa-check"></i><b>4</b> Instrumental analysis</a>
<ul>
<li class="chapter" data-level="4.1" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#column-and-gradient-selection"><i class="fa fa-check"></i><b>4.1</b> Column and gradient selection</a></li>
<li class="chapter" data-level="4.2" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#mass-resolution"><i class="fa fa-check"></i><b>4.2</b> Mass resolution</a></li>
<li class="chapter" data-level="4.3" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#matrix-effects"><i class="fa fa-check"></i><b>4.3</b> Matrix effects</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="workflow-2.html"><a href="workflow-2.html"><i class="fa fa-check"></i><b>5</b> Workflow</a>
<ul>
<li class="chapter" data-level="5.1" data-path="workflow-2.html"><a href="workflow-2.html#platform-for-metabolomics-data-analysis"><i class="fa fa-check"></i><b>5.1</b> Platform for metabolomics data analysis</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="workflow-2.html"><a href="workflow-2.html#xcms-xcms-online"><i class="fa fa-check"></i><b>5.1.1</b> XCMS &amp; XCMS online</a></li>
<li class="chapter" data-level="5.1.2" data-path="workflow-2.html"><a href="workflow-2.html#prime"><i class="fa fa-check"></i><b>5.1.2</b> PRIMe</a></li>
<li class="chapter" data-level="5.1.3" data-path="workflow-2.html"><a href="workflow-2.html#gnps"><i class="fa fa-check"></i><b>5.1.3</b> GNPS</a></li>
<li class="chapter" data-level="5.1.4" data-path="workflow-2.html"><a href="workflow-2.html#openms-sirius"><i class="fa fa-check"></i><b>5.1.4</b> OpenMS &amp; SIRIUS</a></li>
<li class="chapter" data-level="5.1.5" data-path="workflow-2.html"><a href="workflow-2.html#mzmine-2"><i class="fa fa-check"></i><b>5.1.5</b> MZmine 2</a></li>
<li class="chapter" data-level="5.1.6" data-path="workflow-2.html"><a href="workflow-2.html#emory-mahpic"><i class="fa fa-check"></i><b>5.1.6</b> Emory MaHPIC</a></li>
<li class="chapter" data-level="5.1.7" data-path="workflow-2.html"><a href="workflow-2.html#others"><i class="fa fa-check"></i><b>5.1.7</b> Others</a></li>
<li class="chapter" data-level="5.1.8" data-path="workflow-2.html"><a href="workflow-2.html#workflow-comparison"><i class="fa fa-check"></i><b>5.1.8</b> Workflow Comparison</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="workflow-2.html"><a href="workflow-2.html#project-setup"><i class="fa fa-check"></i><b>5.2</b> Project Setup</a></li>
<li class="chapter" data-level="5.3" data-path="workflow-2.html"><a href="workflow-2.html#data-sharing"><i class="fa fa-check"></i><b>5.3</b> Data sharing</a></li>
<li class="chapter" data-level="5.4" data-path="workflow-2.html"><a href="workflow-2.html#contest"><i class="fa fa-check"></i><b>5.4</b> Contest</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html"><i class="fa fa-check"></i><b>6</b> Raw data pretreatment</a>
<ul>
<li class="chapter" data-level="6.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#data-visualization"><i class="fa fa-check"></i><b>6.1</b> Data visualization</a></li>
<li class="chapter" data-level="6.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#peak-extraction"><i class="fa fa-check"></i><b>6.2</b> Peak extraction</a></li>
<li class="chapter" data-level="6.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#msms"><i class="fa fa-check"></i><b>6.3</b> MS/MS</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#mrm"><i class="fa fa-check"></i><b>6.3.1</b> MRM</a></li>
<li class="chapter" data-level="6.3.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dda"><i class="fa fa-check"></i><b>6.3.2</b> DDA</a></li>
<li class="chapter" data-level="6.3.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dia"><i class="fa fa-check"></i><b>6.3.3</b> DIA</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#retention-time-correction"><i class="fa fa-check"></i><b>6.4</b> Retention Time Correction</a></li>
<li class="chapter" data-level="6.5" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#filling-missing-values"><i class="fa fa-check"></i><b>6.5</b> Filling missing values</a></li>
<li class="chapter" data-level="6.6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#spectral-deconvolution"><i class="fa fa-check"></i><b>6.6</b> Spectral deconvolution</a></li>
<li class="chapter" data-level="6.7" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dynamic-range"><i class="fa fa-check"></i><b>6.7</b> Dynamic Range</a>
<ul>
<li class="chapter" data-level="6.7.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#non-detects"><i class="fa fa-check"></i><b>6.7.1</b> Non-detects</a></li>
<li class="chapter" data-level="6.7.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#over-detection-limit"><i class="fa fa-check"></i><b>6.7.2</b> Over Detection Limit</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#rsdfold-change-filter"><i class="fa fa-check"></i><b>6.8</b> RSD/fold change Filter</a></li>
<li class="chapter" data-level="6.9" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#power-analysis-filter"><i class="fa fa-check"></i><b>6.9</b> Power Analysis Filter</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="annotation.html"><a href="annotation.html"><i class="fa fa-check"></i><b>7</b> Annotation</a>
<ul>
<li class="chapter" data-level="7.1" data-path="annotation.html"><a href="annotation.html#issues-in-annotation"><i class="fa fa-check"></i><b>7.1</b> Issues in annotation</a></li>
<li class="chapter" data-level="7.2" data-path="annotation.html"><a href="annotation.html#peak-misidentification"><i class="fa fa-check"></i><b>7.2</b> Peak misidentification</a></li>
<li class="chapter" data-level="7.3" data-path="annotation.html"><a href="annotation.html#annotation-v.s.-identification"><i class="fa fa-check"></i><b>7.3</b> Annotation v.s. identification</a></li>
<li class="chapter" data-level="7.4" data-path="annotation.html"><a href="annotation.html#molecular-formula-assignment"><i class="fa fa-check"></i><b>7.4</b> Molecular Formula Assignment</a></li>
<li class="chapter" data-level="7.5" data-path="annotation.html"><a href="annotation.html#redundant-peaks"><i class="fa fa-check"></i><b>7.5</b> Redundant peaks</a>
<ul>
<li class="chapter" data-level="7.5.1" data-path="annotation.html"><a href="annotation.html#adducts-list"><i class="fa fa-check"></i><b>7.5.1</b> Adducts list</a></li>
<li class="chapter" data-level="7.5.2" data-path="annotation.html"><a href="annotation.html#isotope"><i class="fa fa-check"></i><b>7.5.2</b> Isotope</a></li>
<li class="chapter" data-level="7.5.3" data-path="annotation.html"><a href="annotation.html#camera"><i class="fa fa-check"></i><b>7.5.3</b> CAMERA</a></li>
<li class="chapter" data-level="7.5.4" data-path="annotation.html"><a href="annotation.html#ramclustr"><i class="fa fa-check"></i><b>7.5.4</b> RAMClustR</a></li>
<li class="chapter" data-level="7.5.5" data-path="annotation.html"><a href="annotation.html#biocan"><i class="fa fa-check"></i><b>7.5.5</b> BioCAn</a></li>
<li class="chapter" data-level="7.5.6" data-path="annotation.html"><a href="annotation.html#mzmatch"><i class="fa fa-check"></i><b>7.5.6</b> mzMatch</a></li>
<li class="chapter" data-level="7.5.7" data-path="annotation.html"><a href="annotation.html#xmsannotator"><i class="fa fa-check"></i><b>7.5.7</b> xMSannotator</a></li>
<li class="chapter" data-level="7.5.8" data-path="annotation.html"><a href="annotation.html#mait"><i class="fa fa-check"></i><b>7.5.8</b> MAIT</a></li>
<li class="chapter" data-level="7.5.9" data-path="annotation.html"><a href="annotation.html#pmd"><i class="fa fa-check"></i><b>7.5.9</b> pmd</a></li>
<li class="chapter" data-level="7.5.10" data-path="annotation.html"><a href="annotation.html#nontarget"><i class="fa fa-check"></i><b>7.5.10</b> nontarget</a></li>
<li class="chapter" data-level="7.5.11" data-path="annotation.html"><a href="annotation.html#binner"><i class="fa fa-check"></i><b>7.5.11</b> Binner</a></li>
<li class="chapter" data-level="7.5.12" data-path="annotation.html"><a href="annotation.html#mz.unity"><i class="fa fa-check"></i><b>7.5.12</b> mz.unity</a></li>
<li class="chapter" data-level="7.5.13" data-path="annotation.html"><a href="annotation.html#ms-flo"><i class="fa fa-check"></i><b>7.5.13</b> MS-FLO</a></li>
<li class="chapter" data-level="7.5.14" data-path="annotation.html"><a href="annotation.html#cliquems"><i class="fa fa-check"></i><b>7.5.14</b> CliqueMS</a></li>
<li class="chapter" data-level="7.5.15" data-path="annotation.html"><a href="annotation.html#interpretmsspectrum"><i class="fa fa-check"></i><b>7.5.15</b> InterpretMSSpectrum</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="annotation.html"><a href="annotation.html#msms-annotation"><i class="fa fa-check"></i><b>7.6</b> MS/MS annotation</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="annotation.html"><a href="annotation.html#metdna"><i class="fa fa-check"></i><b>7.6.1</b> MetDNA</a></li>
<li class="chapter" data-level="7.6.2" data-path="annotation.html"><a href="annotation.html#metfusion"><i class="fa fa-check"></i><b>7.6.2</b> MetFusion</a></li>
<li class="chapter" data-level="7.6.3" data-path="annotation.html"><a href="annotation.html#ms2analyzer"><i class="fa fa-check"></i><b>7.6.3</b> MS2Analyzer</a></li>
<li class="chapter" data-level="7.6.4" data-path="annotation.html"><a href="annotation.html#metfrag"><i class="fa fa-check"></i><b>7.6.4</b> MetFrag</a></li>
<li class="chapter" data-level="7.6.5" data-path="annotation.html"><a href="annotation.html#cfm-id"><i class="fa fa-check"></i><b>7.6.5</b> CFM-ID</a></li>
<li class="chapter" data-level="7.6.6" data-path="annotation.html"><a href="annotation.html#lipidfrag"><i class="fa fa-check"></i><b>7.6.6</b> LipidFrag</a></li>
<li class="chapter" data-level="7.6.7" data-path="annotation.html"><a href="annotation.html#lipidmatch"><i class="fa fa-check"></i><b>7.6.7</b> Lipidmatch</a></li>
<li class="chapter" data-level="7.6.8" data-path="annotation.html"><a href="annotation.html#barcoding"><i class="fa fa-check"></i><b>7.6.8</b> BarCoding</a></li>
<li class="chapter" data-level="7.6.9" data-path="annotation.html"><a href="annotation.html#imet"><i class="fa fa-check"></i><b>7.6.9</b> iMet</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="annotation.html"><a href="annotation.html#knowledge-based-annotation"><i class="fa fa-check"></i><b>7.7</b> Knowledge based annotation</a>
<ul>
<li class="chapter" data-level="7.7.1" data-path="annotation.html"><a href="annotation.html#chromatographic-retention-related-criteria"><i class="fa fa-check"></i><b>7.7.1</b> Chromatographic retention-related criteria</a></li>
<li class="chapter" data-level="7.7.2" data-path="annotation.html"><a href="annotation.html#probmetab"><i class="fa fa-check"></i><b>7.7.2</b> ProbMetab</a></li>
<li class="chapter" data-level="7.7.3" data-path="annotation.html"><a href="annotation.html#mi-pack"><i class="fa fa-check"></i><b>7.7.3</b> MI-Pack</a></li>
<li class="chapter" data-level="7.7.4" data-path="annotation.html"><a href="annotation.html#metexpert"><i class="fa fa-check"></i><b>7.7.4</b> MetExpert</a></li>
<li class="chapter" data-level="7.7.5" data-path="annotation.html"><a href="annotation.html#mycompoundid"><i class="fa fa-check"></i><b>7.7.5</b> MycompoundID</a></li>
<li class="chapter" data-level="7.7.6" data-path="annotation.html"><a href="annotation.html#metfamily"><i class="fa fa-check"></i><b>7.7.6</b> MetFamily</a></li>
<li class="chapter" data-level="7.7.7" data-path="annotation.html"><a href="annotation.html#plantmat"><i class="fa fa-check"></i><b>7.7.7</b> Plantmat</a></li>
</ul></li>
<li class="chapter" data-level="7.8" data-path="annotation.html"><a href="annotation.html#ms-database-for-annotation"><i class="fa fa-check"></i><b>7.8</b> MS Database for annotation</a>
<ul>
<li class="chapter" data-level="7.8.1" data-path="annotation.html"><a href="annotation.html#ms"><i class="fa fa-check"></i><b>7.8.1</b> MS</a></li>
<li class="chapter" data-level="7.8.2" data-path="annotation.html"><a href="annotation.html#msms-1"><i class="fa fa-check"></i><b>7.8.2</b> MS/MS</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="annotation.html"><a href="annotation.html#compounds-database"><i class="fa fa-check"></i><b>7.9</b> Compounds Database</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="omics-analysis.html"><a href="omics-analysis.html"><i class="fa fa-check"></i><b>8</b> Omics analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="omics-analysis.html"><a href="omics-analysis.html#from-bottom-up-to-top-down"><i class="fa fa-check"></i><b>8.1</b> From Bottom-up to Top-down</a></li>
<li class="chapter" data-level="8.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-analysis"><i class="fa fa-check"></i><b>8.2</b> Pathway analysis</a>
<ul>
<li class="chapter" data-level="8.2.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-database"><i class="fa fa-check"></i><b>8.2.1</b> Pathway Database</a></li>
<li class="chapter" data-level="8.2.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-software"><i class="fa fa-check"></i><b>8.2.2</b> Pathway software</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="omics-analysis.html"><a href="omics-analysis.html#network-analysis"><i class="fa fa-check"></i><b>8.3</b> Network analysis</a></li>
<li class="chapter" data-level="8.4" data-path="omics-analysis.html"><a href="omics-analysis.html#omics-integration"><i class="fa fa-check"></i><b>8.4</b> Omics integration</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="peaks-normalization.html"><a href="peaks-normalization.html"><i class="fa fa-check"></i><b>9</b> Peaks normalization</a>
<ul>
<li class="chapter" data-level="9.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#batch-effects-classification"><i class="fa fa-check"></i><b>9.1</b> Batch effects classification</a></li>
<li class="chapter" data-level="9.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#batch-effects-visualization"><i class="fa fa-check"></i><b>9.2</b> Batch effects visualization</a></li>
<li class="chapter" data-level="9.3" data-path="peaks-normalization.html"><a href="peaks-normalization.html#source-of-batch-effects"><i class="fa fa-check"></i><b>9.3</b> Source of batch effects</a></li>
<li class="chapter" data-level="9.4" data-path="peaks-normalization.html"><a href="peaks-normalization.html#avoid-batch-effects-by-doe"><i class="fa fa-check"></i><b>9.4</b> Avoid batch effects by DoE</a></li>
<li class="chapter" data-level="9.5" data-path="peaks-normalization.html"><a href="peaks-normalization.html#post-hoc-data-normalization"><i class="fa fa-check"></i><b>9.5</b> <em>post hoc</em> data normalization</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#unsupervised-methods"><i class="fa fa-check"></i><b>9.5.1</b> Unsupervised methods</a></li>
<li class="chapter" data-level="9.5.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#supervised-methods"><i class="fa fa-check"></i><b>9.5.2</b> Supervised methods</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="peaks-normalization.html"><a href="peaks-normalization.html#method-to-validate-the-normalization"><i class="fa fa-check"></i><b>9.6</b> Method to validate the normalization</a></li>
<li class="chapter" data-level="9.7" data-path="peaks-normalization.html"><a href="peaks-normalization.html#software"><i class="fa fa-check"></i><b>9.7</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="statistical-analysis.html"><a href="statistical-analysis.html"><i class="fa fa-check"></i><b>10</b> Statistical analysis</a>
<ul>
<li class="chapter" data-level="10.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#basic-statistical-analysis"><i class="fa fa-check"></i><b>10.1</b> Basic Statistical Analysis</a></li>
<li class="chapter" data-level="10.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#differences-analysis"><i class="fa fa-check"></i><b>10.2</b> Differences analysis</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#t-test-or-anova"><i class="fa fa-check"></i><b>10.2.1</b> T-test or ANOVA</a></li>
<li class="chapter" data-level="10.2.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#limma"><i class="fa fa-check"></i><b>10.2.2</b> LIMMA</a></li>
<li class="chapter" data-level="10.2.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#bayesian-mixture-model"><i class="fa fa-check"></i><b>10.2.3</b> Bayesian mixture model</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#pca"><i class="fa fa-check"></i><b>10.3</b> PCA</a></li>
<li class="chapter" data-level="10.4" data-path="statistical-analysis.html"><a href="statistical-analysis.html#cluster-analysis"><i class="fa fa-check"></i><b>10.4</b> Cluster Analysis</a></li>
<li class="chapter" data-level="10.5" data-path="statistical-analysis.html"><a href="statistical-analysis.html#plsda"><i class="fa fa-check"></i><b>10.5</b> PLSDA</a></li>
<li class="chapter" data-level="10.6" data-path="statistical-analysis.html"><a href="statistical-analysis.html#software-1"><i class="fa fa-check"></i><b>10.6</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="exposome.html"><a href="exposome.html"><i class="fa fa-check"></i><b>11</b> Exposome</a>
<ul>
<li class="chapter" data-level="11.1" data-path="exposome.html"><a href="exposome.html#internal-exposure"><i class="fa fa-check"></i><b>11.1</b> Internal exposure</a></li>
<li class="chapter" data-level="11.2" data-path="exposome.html"><a href="exposome.html#external-exposure"><i class="fa fa-check"></i><b>11.2</b> External exposure</a>
<ul>
<li class="chapter" data-level="11.2.1" data-path="exposome.html"><a href="exposome.html#environmental-fate-of-compounds"><i class="fa fa-check"></i><b>11.2.1</b> Environmental fate of compounds</a></li>
<li class="chapter" data-level="11.2.2" data-path="exposome.html"><a href="exposome.html#exposure-study-database"><i class="fa fa-check"></i><b>11.2.2</b> Exposure study database</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/yufree/metaworkflow" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Meta-Workflow</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="annotation" class="section level1" number="7">
<h1><span class="header-section-number">Chapter 7</span> Annotation</h1>
<p>When you get the peaks table or features table, annotation of the peaks would help you. Check this review<span class="citation">(<a href="references.html#ref-domingo-almenara2018" role="doc-biblioref">Domingo-Almenara, Montenegro-Burke, Benton, et al. 2018a</a>)</span> or other reviews<span class="citation">(<a href="references.html#ref-chaleckis2019" role="doc-biblioref">Chaleckis et al. 2019</a>; <a href="references.html#ref-lai2018a" role="doc-biblioref">Lai et al. 2018b</a>; <a href="references.html#ref-nash2019" role="doc-biblioref">Nash and Dunn 2019</a>; <a href="references.html#ref-viant2017" role="doc-biblioref">Mark R. Viant et al. 2017</a>; <a href="references.html#ref-allard2017" role="doc-biblioref">Allard, Genta-Jouve, and Wolfender 2017</a>; <a href="references.html#ref-domingo-almenara2018b" role="doc-biblioref">Domingo-Almenara, Montenegro-Burke, Benton, et al. 2018b</a>)</span> for a detailed notes on annotation. The first paper proposed five levels regarding currently computational annotation strategies.</p>
<ul>
<li><p>Level 1: Peak Grouping: MS Psedospectra extraction based on peak shape similarity and peak abundance correlation</p></li>
<li><p>Level 2: Peak Annotation: Adducts, Neutral losses, isotopes, and other mass relationships based on mass distances</p></li>
<li><p>Level 3: Biochemical knowledge based on putative identification, potential biochemical reaction and related statistical analysis</p></li>
<li><p>Level 4: Use and integration of tandem MS data based on data dependent/independent acquisition mode or <strong>in silico</strong> prediction</p></li>
<li><p>Level 5: Retention time prediction based on library-available retention index or quantitative structure-retention relationships (QSRR) models.</p></li>
</ul>
<p>Most of the software are at level 1 or 2. If we only have compounds structure, we could guess ions under different ionization method. If we have mass spectrum, we could match the mass spectral by a similarity analysis to the database. In metabolomics, we only have mass spectrum or mass-to-charge ratios. Single mass-to-charge ratio is not enough for identification. That’s the one bottleneck for annotation. So prediction is always performed on MS/MS data.</p>
<div id="issues-in-annotation" class="section level2" number="7.1">
<h2><span class="header-section-number">7.1</span> Issues in annotation</h2>
<p>The major issue in annotation is the redundancy peaks from same metabolite. Unlike genomes, peaks or features from peak selection are not independent with each other. Adducts, in-source fragments and isotopes would lead to wrong annotation. A common solution is that use known adducts, neutral losses, molecular multimers or multiple charged ions to compare mass distances.</p>
<p>Another issue is about the MS/MS database. Only 10% of known metabolites in databases have experimental spectral data. Thus <em>in silico</em> prediction is required. Some works try to fill the gap between experimental data, theoretical values(from chemical database like chemspider) and prediction together. Here is a nice review about MS/MS prediction<span class="citation">(<a href="references.html#ref-hufsky2014" role="doc-biblioref">Hufsky, Scheubert, and Böcker 2014</a>)</span>.</p>
</div>
<div id="peak-misidentification" class="section level2" number="7.2">
<h2><span class="header-section-number">7.2</span> Peak misidentification</h2>
<ul>
<li>Isomer</li>
</ul>
<p>Use separation methods such as chromatography, ion mobility MS, MS/MS. Reversed-phase ion-pairing chromatography and HILIC is useful. Chemical derivatization is another option.</p>
<ul>
<li>Interfering compounds</li>
</ul>
<p>20ppm is the least exact mass accuracy for HRMS.</p>
<ul>
<li>In-source degradation products</li>
</ul>
</div>
<div id="annotation-v.s.-identification" class="section level2" number="7.3">
<h2><span class="header-section-number">7.3</span> Annotation v.s. identification</h2>
<p>According to the definition from the Chemical Analysis Working Group of the Metabolomics Standards Intitvative<span class="citation">(<a href="references.html#ref-sumner2007" role="doc-biblioref">Lloyd W. Sumner et al. 2007</a>; <a href="references.html#ref-viant2017" role="doc-biblioref">Mark R. Viant et al. 2017</a>)</span>. Four levels of confidence could be assigned to identification:</p>
<ul>
<li>Level 1 ‘identified metabolites’</li>
<li>Level 2 ‘Putatively annotated compounds’</li>
<li>Level 3 ‘Putatively characterised compound classes’</li>
<li>Level 4 ‘Unknown’</li>
</ul>
<p>In practice, data analysis based annotation could reach level 2. For level 1, we need at extra methods such as MS/MS, retention time, accurate mass, 2D NMR spectra, and so on to confirm the compounds. However, standards are always required for solid proof.</p>
<p>Through MS/MS seemed a required step for identification, recent study found ESI might also generate fragments ions for structure identification <span class="citation">(<a href="references.html#ref-xue2020a" role="doc-biblioref">Xue, Domingo-Almenara, et al. 2020</a>)</span>.</p>
</div>
<div id="molecular-formula-assignment" class="section level2" number="7.4">
<h2><span class="header-section-number">7.4</span> Molecular Formula Assignment</h2>
<p>Cheminformatics will help for MS annotation. The first task is molecular formula assignment. For a given accurate mass, the formula should be constrained by predefined element type and atom number, mass error window and rules of chemical bonding, such as double bond equivalent (DBE) and the nitrogen rule. The nitrogen rule is that an odd nominal molecular mass implies also an odd number of nitrogen. This rule should only be used with nominal (integer) masses. Degree of unsaturation or DBE use rings-plus-double-bonds equivalent (RDBE) values, which should be interger. The elements oxygen and sulphur were not taken into account. Otherwise the molecular formula will not be true.</p>
<p><span class="math display">\[RDBE = C+Si - 1/2(H+F+Cl+Br+I) + 1/2(N+P)+1 \]</span></p>
<p>To assign molecular formula to a mass to charge ratio, Seven Golden Rules <span class="citation">(<a href="references.html#ref-kind2007" role="doc-biblioref">Kind and Fiehn 2007</a>)</span> for heuristic filtering of molecular formulas should be considered:</p>
<ul>
<li>Apply heuristic restrictions for number of elements during formula generation. This is the table for known compounds:</li>
</ul>
<pre><code>##   Mass.Range.[Da] Library C.max H.max N.max O.max P.max
## 1           &lt; 500     DNP    29    72    10    18     4
## 2            &lt;NA&gt;   Wiley    39    72    20    20     9
## 3          &lt; 1000     DNP    66   126    25    27     6
## 4            &lt;NA&gt;   Wiley    78   126    20    27     9
## 5          &lt; 2000     DNP   115   236    32    63     6
## 6            &lt;NA&gt;   Wiley   156   180    20    40     9
## 7          &lt; 3000     DNP   162   208    48    78     6
##   S.max F.max Cl.max Br.max Si.max
## 1     7    15      8      5     NA
## 2    10    16     10      4      8
## 3     8    16     11      8     NA
## 4    14    34     12      8     14
## 5     8    16     11      8     NA
## 6    14    48     12     10     15
## 7     9    16     11      8     NA</code></pre>
<ul>
<li><p>Perform LEWIS and SENIOR check. The LEWIS rule demands that molecules consisting of main group elements, especially carbon, nitrogen and oxygen, share electrons in a way that all atoms have completely filled s, p-valence shells (‘<a href="https://en.wikipedia.org/wiki/Octet_rule">octet rule</a>’). Senior’s theorem requires three essential conditions for the existence of molecular graphs</p>
<ul>
<li><p>The sum of valences or the total number of atoms having odd valences is even;</p></li>
<li><p>The sum of valences is greater than or equal to twice the maximum valence;</p></li>
<li><p>The sum of valences is greater than or equal to twice the number of atoms minus 1.</p></li>
</ul></li>
<li><p>Perform isotopic pattern filter. Isotope ratio abundance was included in the algorithm as an additional orthogonal constraint, assuming high quality data acquisitions, specifically sufficient ion statistics and high signal/noise ratio for the detection of the M+1 and M+2 abundances. For monoisotopic elements (F, Na, P, I) this rule has no impact. isotope pattern will be useful for brominated, chlorinated small molecules and sulphur-containing peptides.</p></li>
<li><p>Perform H/C ratio check (hydrogen/carbon ratio). In most cases the hydrogen/carbon ratio does not exceed H/C &gt; 3 with rare exception such as in methylhydrazine (CH6N2). Conversely, the H/C ratio is usually smaller than 2, and should not be less than 0.125 like in the case of tetracyanopyrrole (C8HN5).</p></li>
<li><p>Perform NOPS ratio check (N, O, P, S/C ratios).</p></li>
</ul>
<pre><code>##   Element.ratios Common.range.(covering.99.7%)
## 1            H/C                       0.2–3.1
## 2            F/C                         0–1.5
## 3           Cl/C                         0–0.8
## 4           Br/C                         0–0.8
## 5            N/C                         0–1.3
## 6            O/C                         0–1.2
## 7            P/C                         0–0.3
## 8            S/C                         0–0.8
## 9           Si/C                         0–0.5
##   Extended.range.(covering.99.99%)
## 1                            0.1–6
## 2                              0–6
## 3                              0–2
## 4                              0–2
## 5                              0–4
## 6                              0–3
## 7                              0–2
## 8                              0–3
## 9                              0–1
##   Extreme.range.(beyond.99.99%)
## 1                 &lt; 0.1 and 6–9
## 2                         &gt; 1.5
## 3                         &gt; 0.8
## 4                         &gt; 0.8
## 5                         &gt; 1.3
## 6                         &gt; 1.2
## 7                         &gt; 0.3
## 8                         &gt; 0.8
## 9                         &gt; 0.5</code></pre>
<ul>
<li>Perform heuristic HNOPS probability check (H, N, O, P, S/C high probability ratios)</li>
</ul>
<div class="sourceCode" id="cb12"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb12-1"><a href="annotation.html#cb12-1" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">data.frame</span>(</span>
<span id="cb12-2"><a href="annotation.html#cb12-2" aria-hidden="true" tabindex="-1"></a>                <span class="at">stringsAsFactors =</span> <span class="cn">FALSE</span>,</span>
<span id="cb12-3"><a href="annotation.html#cb12-3" aria-hidden="true" tabindex="-1"></a>                  <span class="at">Element.counts =</span> <span class="fu">c</span>(<span class="st">&quot;NOPS all &gt; 1&quot;</span>,<span class="st">&quot;NOP all &gt; 3&quot;</span>,<span class="st">&quot;OPS all &gt; 1&quot;</span>,</span>
<span id="cb12-4"><a href="annotation.html#cb12-4" aria-hidden="true" tabindex="-1"></a>                                     <span class="st">&quot;PSN all &gt; 1&quot;</span>,<span class="st">&quot;NOS all &gt; 6&quot;</span>),</span>
<span id="cb12-5"><a href="annotation.html#cb12-5" aria-hidden="true" tabindex="-1"></a>                  <span class="at">Heuristic.Rule =</span> <span class="fu">c</span>(<span class="st">&quot;N&lt; 10, O &lt; 20, P &lt; 4, S &lt; 3&quot;</span>,</span>
<span id="cb12-6"><a href="annotation.html#cb12-6" aria-hidden="true" tabindex="-1"></a>                                     <span class="st">&quot;N &lt; 11, O &lt; 22, P &lt; 6&quot;</span>,<span class="st">&quot;O &lt; 14, P &lt; 3, S &lt; 3&quot;</span>,</span>
<span id="cb12-7"><a href="annotation.html#cb12-7" aria-hidden="true" tabindex="-1"></a>                                     <span class="st">&quot;P &lt; 3, S &lt; 3, N &lt; 4&quot;</span>,<span class="st">&quot;N &lt; 19 O &lt; 14 S &lt; 8&quot;</span>),</span>
<span id="cb12-8"><a href="annotation.html#cb12-8" aria-hidden="true" tabindex="-1"></a>  <span class="at">DB.examples.for.maximum.values =</span> <span class="fu">c</span>(<span class="st">&quot;C15H34N9O8PS, C22H44N4O14P2S2, C24H38N7O19P3S&quot;</span>,<span class="st">&quot;C20H28N10O21P4, C10H18N5O20P5&quot;</span>,</span>
<span id="cb12-9"><a href="annotation.html#cb12-9" aria-hidden="true" tabindex="-1"></a>                                     <span class="st">&quot;C22H44N4O14P2S2, C16H36N4O4P2S2&quot;</span>,</span>
<span id="cb12-10"><a href="annotation.html#cb12-10" aria-hidden="true" tabindex="-1"></a>                                     <span class="st">&quot;C22H44N4O14P2S2, C16H36N4O4P2S2&quot;</span>,<span class="st">&quot;C59H64N18O14S7&quot;</span>)</span>
<span id="cb12-11"><a href="annotation.html#cb12-11" aria-hidden="true" tabindex="-1"></a>)</span>
<span id="cb12-12"><a href="annotation.html#cb12-12" aria-hidden="true" tabindex="-1"></a>df</span></code></pre></div>
<pre><code>##   Element.counts              Heuristic.Rule
## 1   NOPS all &gt; 1 N&lt; 10, O &lt; 20, P &lt; 4, S &lt; 3
## 2    NOP all &gt; 3       N &lt; 11, O &lt; 22, P &lt; 6
## 3    OPS all &gt; 1        O &lt; 14, P &lt; 3, S &lt; 3
## 4    PSN all &gt; 1         P &lt; 3, S &lt; 3, N &lt; 4
## 5    NOS all &gt; 6         N &lt; 19 O &lt; 14 S &lt; 8
##                  DB.examples.for.maximum.values
## 1 C15H34N9O8PS, C22H44N4O14P2S2, C24H38N7O19P3S
## 2                 C20H28N10O21P4, C10H18N5O20P5
## 3               C22H44N4O14P2S2, C16H36N4O4P2S2
## 4               C22H44N4O14P2S2, C16H36N4O4P2S2
## 5                                C59H64N18O14S7</code></pre>
<ul>
<li>Perform TMS check (for GC-MS if a silylation step is involved). For TMS derivatized molecules detected in GC/MS analyses, the rules on element ratio checks and valence tests are hence best applied after TMS groups are subtracted, in a similar manner as adducts need to be first recognized and subtracted in LC/MS analyses.</li>
</ul>
<p>Seven Golden Rules were built for GC-MS and Hydrogen Rearrangement Rules were major designed for LC-CID-MS/MS<span class="citation">(<a href="references.html#ref-tsugawa2016" role="doc-biblioref">Tsugawa et al. 2016</a>)</span>. Based on extensively curated database records and enthalpy calculations, “hydrogen rearrangement (HR) rules” could be extending the even-electron rule for carbon (C) and heteroatoms, oxygen (O), nitrogen (N), phosphorus (P), and sulfur (S). They used high abundance MS/MS peaks that exceeded 10% of their base peaks to identify common features in terms of 4 HR rules for positive mode and 5 HR rules for negative mode.</p>
<p>Seven Golden Rules and Hydrogen Rearrangement Rules might also be captured by statistical models. However, such heuristic rules could reduce the searching space of possible formula.</p>
<p><a href="http://molgen.de">molgen</a> generating all structures (connectivity isomers, constitutions) that correspond to a given molecular formula, with optional further restrictions, e.g. presence or absence of particular substructures <span class="citation">(<a href="references.html#ref-gugisch2015" role="doc-biblioref">Gugisch et al. 2015</a>)</span>.</p>
<p><a href="http://www.chemcalc.org/mf_finder/mfFinder_em_new">mfFinder</a> can predict formula based on accurate mass <span class="citation">(<a href="references.html#ref-patiny2013" role="doc-biblioref">Patiny and Borel 2013</a>)</span>.</p>
<p>RAMSI is the robust automated mass spectra interpretation and chemical formula calculation method using mixed integer linear programming optimization <span class="citation">(<a href="references.html#ref-baran2013" role="doc-biblioref">Baran and Northen 2013</a>)</span>.</p>
<p>Here is some other Cheminformatics tools, which could be used to assign meaningful formula or structures for mass spectra.</p>
<ul>
<li><a href="https://www.rdkit.org/">RDKit</a> Open-Source Cheminformatics Software</li>
<li><a href="https://sourceforge.net/projects/cdk/">cdk</a> The Chemistry Development Kit (CDK) is a scientific, LGPL-ed library for bio- and cheminformatics and computational chemistry written in Java <span class="citation">(<a href="references.html#ref-guha2007" role="doc-biblioref">Guha 2007</a>)</span>.</li>
<li><a href="http://openbabel.org/wiki/Main_Page">Open Babel</a> Open Babel is a chemical toolbox designed to speak the many languages of chemical data <span class="citation">(<a href="references.html#ref-oboyle2011" role="doc-biblioref">O’Boyle et al. 2011</a>)</span>.</li>
<li><a href="http://classyfire.wishartlab.com/">ClassyFire</a> is a tool for automated chemical classification with a comprehensive, computable taxonomy <span class="citation">(<a href="references.html#ref-djoumboufeunang2016" role="doc-biblioref">Djoumbou Feunang et al. 2016</a>)</span>.</li>
</ul>
</div>
<div id="redundant-peaks" class="section level2" number="7.5">
<h2><span class="header-section-number">7.5</span> Redundant peaks</h2>
<p>Full scan mass spectra always contain lots of redundant peaks such as adducts, isotope, fragments, multiple charged ions and other oligomers. Such peaks dominated the features table<span class="citation">(<a href="references.html#ref-xu2015" role="doc-biblioref">Xu, Lu, and Rabinowitz 2015</a>; <a href="references.html#ref-sindelar2020" role="doc-biblioref">Sindelar and Patti 2020</a>; <a href="references.html#ref-mahieu2017" role="doc-biblioref">Nathaniel G. Mahieu and Patti 2017</a>)</span>. Annotation tools could label those peaks either by known list or frequency analysis of the paired mass distances<span class="citation">(<a href="references.html#ref-ju2020" role="doc-biblioref">Ju et al. 2020</a>; <a href="references.html#ref-kouil" role="doc-biblioref">Kouřil et al., n.d.</a>)</span>.</p>
<div id="adducts-list" class="section level3" number="7.5.1">
<h3><span class="header-section-number">7.5.1</span> Adducts list</h3>
<p>You could find adducts list <a href="https://github.com/stanstrup/commonMZ">here</a> from commonMZ project.</p>
</div>
<div id="isotope" class="section level3" number="7.5.2">
<h3><span class="header-section-number">7.5.2</span> Isotope</h3>
<p>Here is <a href="https://www.envipat.eawag.ch/index.php">Isotope</a> pattern prediction.</p>
</div>
<div id="camera" class="section level3" number="7.5.3">
<h3><span class="header-section-number">7.5.3</span> CAMERA</h3>
<p>Common <a href="https://bioconductor.org/packages/release/bioc/html/CAMERA.html">annotation</a> for xcms workflow<span class="citation">(<a href="references.html#ref-kuhl2012" role="doc-biblioref">Kuhl et al. 2012</a>)</span>.</p>
</div>
<div id="ramclustr" class="section level3" number="7.5.4">
<h3><span class="header-section-number">7.5.4</span> RAMClustR</h3>
<p>The software could be found <a href="https://github.com/cbroeckl/RAMClustR">here</a> <span class="citation">(<a href="references.html#ref-broeckling2014" role="doc-biblioref">C. D. Broeckling et al. 2014</a>; <a href="references.html#ref-broeckling2016" role="doc-biblioref">Corey D. Broeckling et al. 2016</a>)</span>. The package included a vignette to follow.</p>
</div>
<div id="biocan" class="section level3" number="7.5.5">
<h3><span class="header-section-number">7.5.5</span> BioCAn</h3>
<p>BioCAn combines the results from database searches and in silico fragmentation analyses and places these results into a relevant biological context for the sample as captured by a metabolic model <span class="citation">(<a href="references.html#ref-alden2017" role="doc-biblioref">Alden et al. 2017</a>)</span>.</p>
</div>
<div id="mzmatch" class="section level3" number="7.5.6">
<h3><span class="header-section-number">7.5.6</span> mzMatch</h3>
<p><a href="https://github.com/andzajan/mzmatch.R">mzMatch</a> is a modular, open source and platform independent data processing pipeline for metabolomics LC/MS data written in the Java language. <span class="citation">(<a href="references.html#ref-chokkathukalam2013" role="doc-biblioref">Chokkathukalam et al. 2013</a>; <a href="references.html#ref-scheltema2011" role="doc-biblioref">Scheltema et al. 2011b</a>)</span> and MetAssign is a probabilistic annotation method using a Bayesian clustering approach, which is part of mzMatch<span class="citation">(<a href="references.html#ref-daly2014a" role="doc-biblioref">Daly et al. 2014b</a>)</span>.</p>
</div>
<div id="xmsannotator" class="section level3" number="7.5.7">
<h3><span class="header-section-number">7.5.7</span> xMSannotator</h3>
<p>The software could be found <a href="https://github.com/yufree/xMSannotator">here</a><span class="citation">(<a href="references.html#ref-uppal2017" role="doc-biblioref">Uppal, Walker, and Jones 2017</a>)</span>.</p>
</div>
<div id="mait" class="section level3" number="7.5.8">
<h3><span class="header-section-number">7.5.8</span> MAIT</h3>
<p>You could find source code <a href="https://github.com/jpgroup/MAIT">here</a><span class="citation">(<a href="references.html#ref-fernandez-albert2014a" role="doc-biblioref">Fernández-Albert et al. 2014</a>)</span>.</p>
</div>
<div id="pmd" class="section level3" number="7.5.9">
<h3><span class="header-section-number">7.5.9</span> pmd</h3>
<p><a href="https://github.com/yufree/pmd">Paired Mass Distance(PMD)</a> analysis for GC/LC-MS based nontarget analysis to remove redundant peaks<span class="citation">(<a href="references.html#ref-yu2019b" role="doc-biblioref">M. Yu, Olkowicz, and Pawliszyn 2019b</a>)</span>.</p>
</div>
<div id="nontarget" class="section level3" number="7.5.10">
<h3><span class="header-section-number">7.5.10</span> nontarget</h3>
<p><a href="https://github.com/blosloos/nontarget">nontarget</a> could find Isotope &amp; adduct peak grouping, and perform homologue series detection <span class="citation">(<a href="references.html#ref-loos2017" role="doc-biblioref">Loos and Singer 2017</a>)</span>.</p>
</div>
<div id="binner" class="section level3" number="7.5.11">
<h3><span class="header-section-number">7.5.11</span> Binner</h3>
<p><a href="https://binner.med.umich.edu/">Binner</a> Deep annotation of untargeted LC-MS metabolomics data <span class="citation">(<a href="references.html#ref-kachman2020" role="doc-biblioref">Kachman et al. 2020</a>)</span></p>
</div>
<div id="mz.unity" class="section level3" number="7.5.12">
<h3><span class="header-section-number">7.5.12</span> mz.unity</h3>
<p>You could find source code <a href="https://github.com/nathaniel-mahieu/mz.unity">here</a> <span class="citation">(<a href="references.html#ref-mahieu2016aa" role="doc-biblioref">Nathaniel G. Mahieu et al. 2016</a>)</span> and it’s for detecting and exploring complex relationships in accurate-mass mass spectrometry data.</p>
</div>
<div id="ms-flo" class="section level3" number="7.5.13">
<h3><span class="header-section-number">7.5.13</span> MS-FLO</h3>
<p><a href="https://bitbucket.org/fiehnlab/ms-flo/src/657d85ec7bdd?at=master">ms-flo</a> A Tool To Minimize False Positive Peak Reports in Untargeted Liquid Chromatography–Mass Spectroscopy (LC-MS) Data Processing <span class="citation">(<a href="references.html#ref-defelice2017" role="doc-biblioref">DeFelice et al. 2017</a>)</span>.</p>
</div>
<div id="cliquems" class="section level3" number="7.5.14">
<h3><span class="header-section-number">7.5.14</span> CliqueMS</h3>
<p>CliqueMS is a computational tool for annotating in-source metabolite ions from LC-MS untargeted metabolomics data based on a coelution similarity network <span class="citation">(<a href="references.html#ref-senan2019" role="doc-biblioref">Senan et al. 2019</a>)</span>.</p>
</div>
<div id="interpretmsspectrum" class="section level3" number="7.5.15">
<h3><span class="header-section-number">7.5.15</span> InterpretMSSpectrum</h3>
<p>This <a href="https://github.com/cran/InterpretMSSpectrum">package</a> is for annotate and interpret deconvoluted mass spectra (mass*intensity pairs) from high resolution mass spectrometry devices. You could use this package to find molecular ions for GC-MS <span class="citation">(<a href="references.html#ref-jaeger2016" role="doc-biblioref">Jaeger et al. 2016</a>)</span>.</p>
</div>
</div>
<div id="msms-annotation" class="section level2" number="7.6">
<h2><span class="header-section-number">7.6</span> MS/MS annotation</h2>
<p>You could check <a href="workflow-2.html#workflow-2">Workflow</a> section for popular platform. Here are some stand-alone annotation software:</p>
<div id="metdna" class="section level3" number="7.6.1">
<h3><span class="header-section-number">7.6.1</span> MetDNA</h3>
<p>MetDNA is the Metabolic reaction network-based recursive metabolite annotation for untargeted metabolomics <span class="citation">(<a href="references.html#ref-shen2019" role="doc-biblioref">Shen et al. 2019</a>)</span>.</p>
</div>
<div id="metfusion" class="section level3" number="7.6.2">
<h3><span class="header-section-number">7.6.2</span> MetFusion</h3>
<p>Java based <a href="https://github.com/mgerlich/MetFusion">integration</a> of compound identiﬁcation strategies. You could access the application <a href="https://msbi.ipb-halle.de/MetFusion/">here</a> <span class="citation">(<a href="references.html#ref-gerlich2013" role="doc-biblioref">Gerlich and Neumann 2013</a>)</span>.</p>
</div>
<div id="ms2analyzer" class="section level3" number="7.6.3">
<h3><span class="header-section-number">7.6.3</span> MS2Analyzer</h3>
<p>MS2Analyzer could annotate small molecule substructure from accurate tandem mass spectra. <span class="citation">(<a href="references.html#ref-ma2014" role="doc-biblioref">Ma et al. 2014</a>)</span></p>
</div>
<div id="metfrag" class="section level3" number="7.6.4">
<h3><span class="header-section-number">7.6.4</span> MetFrag</h3>
<p><a href="http://c-ruttkies.github.io/MetFrag/">MetFrag</a> could be used to make <em>in silico</em> prediction/match of MS/MS data<span class="citation">(<a href="references.html#ref-ruttkies2016" role="doc-biblioref">Ruttkies et al. 2016</a>; <a href="references.html#ref-wolf2010" role="doc-biblioref">Wolf et al. 2010</a>)</span>.</p>
</div>
<div id="cfm-id" class="section level3" number="7.6.5">
<h3><span class="header-section-number">7.6.5</span> CFM-ID</h3>
<p><a href="https://sourceforge.net/projects/cfm-id/">CFM-ID</a> use Metlin’s data to make prediction <span class="citation">(<a href="references.html#ref-allen2014" role="doc-biblioref">Allen et al. 2014</a>)</span>.</p>
</div>
<div id="lipidfrag" class="section level3" number="7.6.6">
<h3><span class="header-section-number">7.6.6</span> LipidFrag</h3>
<p><a href="https://msbi.ipb-halle.de/msbi/lipidfrag">LipidFrag</a> could be used to make <em>in silico</em> prediction/match of lipid related MS/MS data <span class="citation">(<a href="references.html#ref-witting2017" role="doc-biblioref">Witting et al. 2017</a>)</span>.</p>
</div>
<div id="lipidmatch" class="section level3" number="7.6.7">
<h3><span class="header-section-number">7.6.7</span> Lipidmatch</h3>
<p><a href="http://secim.ufl.edu/secim-tools/lipidmatch/">in silico</a>: <em>in silico</em> lipid mass spectrum search <span class="citation">(<a href="references.html#ref-koelmel2017" role="doc-biblioref">Koelmel et al. 2017</a>)</span>.</p>
</div>
<div id="barcoding" class="section level3" number="7.6.8">
<h3><span class="header-section-number">7.6.8</span> BarCoding</h3>
<p>Bar coding select mass-to-charge regions containing the most informative metabolite fragments and designate them as bins. Then translate each metabolite fragmentation pattern into a binary code by assigning 1’s to bins containing fragments and 0’s to bins without fragments. Such coding annotation could be used for MRM data <span class="citation">(<a href="references.html#ref-spalding2016" role="doc-biblioref">Spalding et al. 2016</a>)</span>.</p>
</div>
<div id="imet" class="section level3" number="7.6.9">
<h3><span class="header-section-number">7.6.9</span> iMet</h3>
<p>This online <a href="http://imet.seeslab.net/">application</a> is a network-based computation method for annotation <span class="citation">(<a href="references.html#ref-aguilar-mogas2017" role="doc-biblioref">Aguilar-Mogas et al. 2017</a>)</span>.</p>
</div>
</div>
<div id="knowledge-based-annotation" class="section level2" number="7.7">
<h2><span class="header-section-number">7.7</span> Knowledge based annotation</h2>
<div id="chromatographic-retention-related-criteria" class="section level3" number="7.7.1">
<h3><span class="header-section-number">7.7.1</span> Chromatographic retention-related criteria</h3>
<p>For targeted analysis, chromatographic retention time could be the qualitative method for certain compounds with a carefully designed pre-treatment. For untargeted analysis, such information could also be used for annotation. GC-MS usually use retention index for certain column while LC-MS might not show enough reproducible results as GC. Such method could be tracked back to quantitative structure-retention relationship (QSRR) models or linear solvation energy relationship (LSER). However, such methods need molecular descriptors as much as possible. For untargeted analysis, retention time and mass to charge ratio could not generate enough molecular descriptors to build QSPR models. In this case, such criteria might be usefully for validation instead of annotation unless we could measure or extract more information such as ion mobility from unknown compounds.</p>
<ul>
<li><p><a href="https://www.retip.app/">Retip</a> Retention Time Prediction for Compound Annotation in Untargeted Metabolomics <span class="citation">(<a href="references.html#ref-bonini2020" role="doc-biblioref">Bonini et al. 2020</a>)</span>.</p></li>
<li><p>JAVA based <a href="http://metabolomics.pharm.uconn.edu/?q=Software.html">MolFind</a> could make annotation for unknown chemical structure by prediction based on RI, ECOM50, drift time and CID spectra <span class="citation">(<a href="references.html#ref-menikarachchi2012" role="doc-biblioref">Menikarachchi et al. 2012</a>)</span>.</p></li>
<li><p><a href="https://water.for-ident.org/#!search">For-ident</a> could give a score for identification with the help of logD(relative retention time) and/or MS/MS.</p></li>
</ul>
</div>
<div id="probmetab" class="section level3" number="7.7.2">
<h3><span class="header-section-number">7.7.2</span> ProbMetab</h3>
<p>Provides probability ranking to candidate compounds assigned to masses, with the prior assumption of connected sample and additional previous and spectral information modeled by the user. You could find source code <a href="https://github.com/rsilvabioinfo/ProbMetab">here</a> <span class="citation">(<a href="references.html#ref-silva2014" role="doc-biblioref">R. R. Silva et al. 2014</a>)</span>.</p>
</div>
<div id="mi-pack" class="section level3" number="7.7.3">
<h3><span class="header-section-number">7.7.3</span> MI-Pack</h3>
<p>You could find python software <a href="http://www.biosciences-labs.bham.ac.uk/viant/mipack/">here</a> <span class="citation">(<a href="references.html#ref-weber2010" role="doc-biblioref">Weber and Viant 2010</a>)</span>.</p>
</div>
<div id="metexpert" class="section level3" number="7.7.4">
<h3><span class="header-section-number">7.7.4</span> MetExpert</h3>
<p><a href="https://sourceforge.net/projects/metexpert/">MetExpert</a> is an expert system to assist users with limited expertise in informatics to interpret GCMS data for metabolite identification without querying spectral databases <span class="citation">(<a href="references.html#ref-qiu2018" role="doc-biblioref">Qiu, Lei, and Sumner 2018</a>)</span>.</p>
</div>
<div id="mycompoundid" class="section level3" number="7.7.5">
<h3><span class="header-section-number">7.7.5</span> MycompoundID</h3>
<p><a href="http://www.mycompoundid.org/mycompoundid_IsoMS/single.jsp">MycompoundID</a> could be used to search known and unknown metabolites online <span class="citation">(<a href="references.html#ref-li2013a" role="doc-biblioref">Liang Li et al. 2013</a>)</span>.</p>
</div>
<div id="metfamily" class="section level3" number="7.7.6">
<h3><span class="header-section-number">7.7.6</span> MetFamily</h3>
<p><a href="https://msbi.ipb-halle.de/MetFamily/">Shiny app</a> for MS and MS/MS data annotation <span class="citation">(<a href="references.html#ref-treutler2016" role="doc-biblioref">Treutler et al. 2016a</a>)</span>.</p>
</div>
<div id="plantmat" class="section level3" number="7.7.7">
<h3><span class="header-section-number">7.7.7</span> Plantmat</h3>
<p><a href="https://sourceforge.net/projects/plantmat/">excel library</a> based prediction for plant metabolites<span class="citation">(<a href="references.html#ref-qiu2016" role="doc-biblioref">Qiu et al. 2016</a>)</span>.</p>
</div>
</div>
<div id="ms-database-for-annotation" class="section level2" number="7.8">
<h2><span class="header-section-number">7.8</span> MS Database for annotation</h2>
<div id="ms" class="section level3" number="7.8.1">
<h3><span class="header-section-number">7.8.1</span> MS</h3>
<ul>
<li><p><a href="http://fiehnlab.ucdavis.edu/projects/binbase-setup">Fiehn Lab</a></p></li>
<li><p><a href="https://www.nist.gov/srd/nist-standard-reference-database-1a-v17">NIST</a>: No free</p></li>
<li><p><a href="https://sdbs.db.aist.go.jp/sdbs/cgi-bin/cre_index.cgi?lang=eng">Spectral Database for Organic Compounds, SDBS</a></p></li>
<li><p><a href="http://minedatabase.mcs.anl.gov/#/faq">MINE</a> is an open access database of computationally predicted enzyme promiscuity products for untargeted metabolomics</p></li>
<li><p><a href="https://www.sciencedirect.com/science/article/abs/pii/S0039914019311786?via%3Dihub">Database</a> of free solution mobilities for 276 metabolites for capillary zone electrophoresis / mass spectrometry <span class="citation">(<a href="references.html#ref-petrov2020" role="doc-biblioref">Petrov et al. 2020</a>)</span>.</p></li>
<li><p><a href="http://minedatabase.mcs.anl.gov/">MINE</a> is an open access database of computationally predicted enzyme promiscuity products for untargeted metabolomics. The annotation would be accurate for general compounds database.</p></li>
</ul>
</div>
<div id="msms-1" class="section level3" number="7.8.2">
<h3><span class="header-section-number">7.8.2</span> MS/MS</h3>
<ul>
<li><p><a href="http://mona.fiehnlab.ucdavis.edu/">MoNA</a> Platform to collect all other open source database</p></li>
<li><p><a href="http://www.massbank.jp/?lang=en">MassBank</a></p></li>
<li><p><a href="https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp">GNPS</a> use inner correlationship in the data and make network analysis at peaks’ level instand of annotated compounds to annotate the data.</p></li>
<li><p><a href="http://spectra.psc.riken.jp/">ReSpect</a>: phytochemicals</p></li>
<li><p><a href="https://metlin.scripps.edu/">Metlin</a> is another useful online application for annotation<span class="citation">(<a href="references.html#ref-guijas2018" role="doc-biblioref">Guijas et al. 2018</a>)</span>.</p></li>
<li><p><a href="http://fiehnlab.ucdavis.edu/projects/LipidBlast">LipidBlast</a>: <em>in silico</em> prediction</p></li>
<li><p><a href="http://www.lipidmaps.org/">Lipid Maps</a></p></li>
<li><p><a href="https://www.mzcloud.org/">MZcloud</a></p></li>
<li><p><a href="https://www.nist.gov/srd/nist-standard-reference-database-1a-v17">NIST</a>: Not free</p></li>
<li><p><a href="https://jcggdb.jp/rcmg/glycodb/Ms_ResultSearch">GMDB</a> a multistage tandem mass spectral database using a variety of structurally defined glycans.</p></li>
<li><p><a href="http://www.hmdb.ca/">HMDB</a> is a freely available electronic database containing detailed information about small molecule metabolites found in the human body.</p></li>
<li><p><a href="https://www.genome.jp/kegg/compound/">KEGG</a> is a collection of small molecules, biopolymers, and other chemical substances that are relevant to biological systems.</p></li>
</ul>
</div>
</div>
<div id="compounds-database" class="section level2" number="7.9">
<h2><span class="header-section-number">7.9</span> Compounds Database</h2>
<ul>
<li><p><a href="https://pubchem.ncbi.nlm.nih.gov/">PubChem</a> is an open chemistry database at the National Institutes of Health (NIH).</p></li>
<li><p><a href="http://www.chemspider.com/">Chemspider</a> is a free chemical structure database providing fast text and structure search access to over 67 million structures from hundreds of data sources.</p></li>
<li><p><a href="https://www.ebi.ac.uk/chebi/">ChEBI</a> is a freely available dictionary of molecular entities focused on ‘small’ chemical compounds.</p></li>
<li><p><a href="http://www.metabolomicsworkbench.org/databases/refmet/index.php">RefMet</a> A Reference list of Metabolite names.</p></li>
<li><p><a href="https://www.cas.org/support/documentation/chemical-substances/cas-registry-100-millionth-fun-facts">CAS</a> Largest substance database</p></li>
<li><p><a href="https://comptox.epa.gov/dashboard">CompTox</a> compounds, exposure and toxicity database. <a href="https://www.epa.gov/chemical-research/downloadable-computational-toxicology-data">Here</a> is related data.</p></li>
<li><p><a href="http://www.t3db.ca/">T3DB</a> is a unique bioinformatics resource that combines detailed toxin data with comprehensive toxin target information.</p></li>
<li><p><a href="http://foodb.ca/">FooDB</a> is the world’s largest and most comprehensive resource on food constituents, chemistry and biology.</p></li>
<li><p><a href="http://phenol-explorer.eu">Phenol explorer</a> is the first comprehensive database on polyphenol content in foods.</p></li>
<li><p><a href="https://www.drugbank.ca/releases/latest">Drugbank</a> is a unique bioinformatics and cheminformatics resource that combines detailed drug data with comprehensive drug target information.</p></li>
<li><p><a href="http://lmdb.ca">LMDB</a> is a freely available electronic database containing detailed information about small molecule metabolites found in different livestock species.</p></li>
<li><p><a href="https://iaspub.epa.gov/oppthpv/hpv_hc_characterization.get_report?doctype=2">HPV</a> High Production Volume Information System</p></li>
</ul>

</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="raw-data-pretreatment.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="omics-analysis.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/yufree/metaworkflow/edit/master/07-annotation.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Metabolomics.pub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
