<!DOCTYPE html>
<html >

<head>

  <meta charset="UTF-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge">
  <title>Meta-Workflow</title>
  <meta name="description" content="This is a workflow for metabolomics studies.">
  <meta name="generator" content="bookdown 0.7 and GitBook 2.6.7">

  <meta property="og:title" content="Meta-Workflow" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="images/cover.png" />
  <meta property="og:description" content="This is a workflow for metabolomics studies." />
  <meta name="github-repo" content="yufree/metaworkflow" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Meta-Workflow" />
  
  <meta name="twitter:description" content="This is a workflow for metabolomics studies." />
  <meta name="twitter:image" content="images/cover.png" />

<meta name="author" content="Miao YU">


<meta name="date" content="2018-10-18">

  <meta name="viewport" content="width=device-width, initial-scale=1">
  <meta name="apple-mobile-web-app-capable" content="yes">
  <meta name="apple-mobile-web-app-status-bar-style" content="black">
  
  
<link rel="prev" href="annotation.html">
<link rel="next" href="common-analysis-methods-for-metabolomics.html">
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />







<script src="libs/htmlwidgets-1.3/htmlwidgets.js"></script>
<script src="libs/viz-0.3/viz.js"></script>
<link href="libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="libs/grViz-binding-1.0.0/grViz.js"></script>


<style type="text/css">
div.sourceCode { overflow-x: auto; }
table.sourceCode, tr.sourceCode, td.lineNumbers, td.sourceCode {
  margin: 0; padding: 0; vertical-align: baseline; border: none; }
table.sourceCode { width: 100%; line-height: 100%; }
td.lineNumbers { text-align: right; padding-right: 4px; padding-left: 4px; color: #aaaaaa; border-right: 1px solid #aaaaaa; }
td.sourceCode { padding-left: 5px; }
code > span.kw { color: #007020; font-weight: bold; } /* Keyword */
code > span.dt { color: #902000; } /* DataType */
code > span.dv { color: #40a070; } /* DecVal */
code > span.bn { color: #40a070; } /* BaseN */
code > span.fl { color: #40a070; } /* Float */
code > span.ch { color: #4070a0; } /* Char */
code > span.st { color: #4070a0; } /* String */
code > span.co { color: #60a0b0; font-style: italic; } /* Comment */
code > span.ot { color: #007020; } /* Other */
code > span.al { color: #ff0000; font-weight: bold; } /* Alert */
code > span.fu { color: #06287e; } /* Function */
code > span.er { color: #ff0000; font-weight: bold; } /* Error */
code > span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
code > span.cn { color: #880000; } /* Constant */
code > span.sc { color: #4070a0; } /* SpecialChar */
code > span.vs { color: #4070a0; } /* VerbatimString */
code > span.ss { color: #bb6688; } /* SpecialString */
code > span.im { } /* Import */
code > span.va { color: #19177c; } /* Variable */
code > span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code > span.op { color: #666666; } /* Operator */
code > span.bu { } /* BuiltIn */
code > span.ex { } /* Extension */
code > span.pp { color: #bc7a00; } /* Preprocessor */
code > span.at { color: #7d9029; } /* Attribute */
code > span.do { color: #ba2121; font-style: italic; } /* Documentation */
code > span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code > span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code > span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metabolomics Workflow</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a><ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#history"><i class="fa fa-check"></i><b>1.1</b> History</a><ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#history-of-mass-spectrometry"><i class="fa fa-check"></i><b>1.1.1</b> History of Mass Spectrometry</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#history-of-metabolomcis"><i class="fa fa-check"></i><b>1.1.2</b> History of Metabolomcis</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#reviews-and-tutorials"><i class="fa fa-check"></i><b>1.2</b> Reviews and tutorials</a></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#trends-in-metabolomics"><i class="fa fa-check"></i><b>1.3</b> Trends in Metabolomics</a><ul>
<li class="chapter" data-level="1.3.1" data-path="introduction.html"><a href="introduction.html#quantitative-metabolomics"><i class="fa fa-check"></i><b>1.3.1</b> Quantitative Metabolomics</a></li>
<li class="chapter" data-level="1.3.2" data-path="introduction.html"><a href="introduction.html#high-throughput-metabolomics"><i class="fa fa-check"></i><b>1.3.2</b> High throughput Metabolomics</a></li>
</ul></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#workflow"><i class="fa fa-check"></i><b>1.4</b> Workflow</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="exprimental-designdoe.html"><a href="exprimental-designdoe.html"><i class="fa fa-check"></i><b>2</b> Exprimental design(DoE)</a><ul>
<li class="chapter" data-level="2.1" data-path="exprimental-designdoe.html"><a href="exprimental-designdoe.html#software"><i class="fa fa-check"></i><b>2.1</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="pretreatment.html"><a href="pretreatment.html"><i class="fa fa-check"></i><b>3</b> Pretreatment</a><ul>
<li class="chapter" data-level="3.1" data-path="pretreatment.html"><a href="pretreatment.html#quenching"><i class="fa fa-check"></i><b>3.1</b> Quenching</a></li>
<li class="chapter" data-level="3.2" data-path="pretreatment.html"><a href="pretreatment.html#extraction"><i class="fa fa-check"></i><b>3.2</b> Extraction</a></li>
<li class="chapter" data-level="3.3" data-path="pretreatment.html"><a href="pretreatment.html#instrumental-analysis"><i class="fa fa-check"></i><b>3.3</b> Instrumental analysis</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html"><i class="fa fa-check"></i><b>4</b> Raw data pretreatment</a><ul>
<li class="chapter" data-level="4.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#peak-extraction"><i class="fa fa-check"></i><b>4.1</b> Peak extraction</a></li>
<li class="chapter" data-level="4.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#retention-time-correction"><i class="fa fa-check"></i><b>4.2</b> Retention Time Correction</a></li>
<li class="chapter" data-level="4.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#filling-missing-values"><i class="fa fa-check"></i><b>4.3</b> Filling missing values</a></li>
<li class="chapter" data-level="4.4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#spectral-deconvolution"><i class="fa fa-check"></i><b>4.4</b> Spectral deconvolution</a></li>
<li class="chapter" data-level="4.5" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dynamic-range"><i class="fa fa-check"></i><b>4.5</b> Dynamic Range</a><ul>
<li class="chapter" data-level="4.5.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#non-detects"><i class="fa fa-check"></i><b>4.5.1</b> Non-detects</a></li>
<li class="chapter" data-level="4.5.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#over-detection-limit"><i class="fa fa-check"></i><b>4.5.2</b> Over Detection Limit</a></li>
</ul></li>
<li class="chapter" data-level="4.6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#rsd-filter"><i class="fa fa-check"></i><b>4.6</b> RSD Filter</a></li>
<li class="chapter" data-level="4.7" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#power-analysis-filter"><i class="fa fa-check"></i><b>4.7</b> Power Analysis Filter</a></li>
<li class="chapter" data-level="4.8" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#normalization"><i class="fa fa-check"></i><b>4.8</b> Normalization</a><ul>
<li class="chapter" data-level="4.8.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#unsupervised-methods"><i class="fa fa-check"></i><b>4.8.1</b> Unsupervised methods</a></li>
<li class="chapter" data-level="4.8.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#supervised-methods"><i class="fa fa-check"></i><b>4.8.2</b> Supervised methods</a></li>
</ul></li>
<li class="chapter" data-level="4.9" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#software-1"><i class="fa fa-check"></i><b>4.9</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="peaks-selection.html"><a href="peaks-selection.html"><i class="fa fa-check"></i><b>5</b> Peaks selection</a><ul>
<li class="chapter" data-level="5.1" data-path="peaks-selection.html"><a href="peaks-selection.html#peak-misidentification"><i class="fa fa-check"></i><b>5.1</b> Peak misidentification</a></li>
<li class="chapter" data-level="5.2" data-path="peaks-selection.html"><a href="peaks-selection.html#software-2"><i class="fa fa-check"></i><b>5.2</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="annotation.html"><a href="annotation.html"><i class="fa fa-check"></i><b>6</b> Annotation</a><ul>
<li class="chapter" data-level="6.1" data-path="annotation.html"><a href="annotation.html#issues-in-annotation"><i class="fa fa-check"></i><b>6.1</b> Issues in annotation</a></li>
<li class="chapter" data-level="6.2" data-path="annotation.html"><a href="annotation.html#annotation-v.s.-identification"><i class="fa fa-check"></i><b>6.2</b> Annotation v.s. identification</a></li>
<li class="chapter" data-level="6.3" data-path="annotation.html"><a href="annotation.html#ms-database-for-annotation"><i class="fa fa-check"></i><b>6.3</b> MS Database for annotation</a><ul>
<li class="chapter" data-level="6.3.1" data-path="annotation.html"><a href="annotation.html#msms"><i class="fa fa-check"></i><b>6.3.1</b> MS/MS</a></li>
<li class="chapter" data-level="6.3.2" data-path="annotation.html"><a href="annotation.html#ms"><i class="fa fa-check"></i><b>6.3.2</b> MS</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="annotation.html"><a href="annotation.html#compounds-database"><i class="fa fa-check"></i><b>6.4</b> Compounds Database</a></li>
<li class="chapter" data-level="6.5" data-path="annotation.html"><a href="annotation.html#software-3"><i class="fa fa-check"></i><b>6.5</b> Software</a><ul>
<li class="chapter" data-level="6.5.1" data-path="annotation.html"><a href="annotation.html#adducts-list"><i class="fa fa-check"></i><b>6.5.1</b> Adducts list</a></li>
<li class="chapter" data-level="6.5.2" data-path="annotation.html"><a href="annotation.html#molgen"><i class="fa fa-check"></i><b>6.5.2</b> Molgen</a></li>
<li class="chapter" data-level="6.5.3" data-path="annotation.html"><a href="annotation.html#isotope"><i class="fa fa-check"></i><b>6.5.3</b> Isotope</a></li>
<li class="chapter" data-level="6.5.4" data-path="annotation.html"><a href="annotation.html#mffinder"><i class="fa fa-check"></i><b>6.5.4</b> mfFinder</a></li>
<li class="chapter" data-level="6.5.5" data-path="annotation.html"><a href="annotation.html#camera"><i class="fa fa-check"></i><b>6.5.5</b> CAMERA</a></li>
<li class="chapter" data-level="6.5.6" data-path="annotation.html"><a href="annotation.html#ramclustr"><i class="fa fa-check"></i><b>6.5.6</b> RAMClustR</a></li>
<li class="chapter" data-level="6.5.7" data-path="annotation.html"><a href="annotation.html#nontarget"><i class="fa fa-check"></i><b>6.5.7</b> nontarget</a></li>
<li class="chapter" data-level="6.5.8" data-path="annotation.html"><a href="annotation.html#xmsannotator"><i class="fa fa-check"></i><b>6.5.8</b> xMSannotator</a></li>
<li class="chapter" data-level="6.5.9" data-path="annotation.html"><a href="annotation.html#mine"><i class="fa fa-check"></i><b>6.5.9</b> MINE</a></li>
<li class="chapter" data-level="6.5.10" data-path="annotation.html"><a href="annotation.html#interpretmsspectrum"><i class="fa fa-check"></i><b>6.5.10</b> InterpretMSSpectrum</a></li>
<li class="chapter" data-level="6.5.11" data-path="annotation.html"><a href="annotation.html#for-ident"><i class="fa fa-check"></i><b>6.5.11</b> For Ident</a></li>
<li class="chapter" data-level="6.5.12" data-path="annotation.html"><a href="annotation.html#mzmatch"><i class="fa fa-check"></i><b>6.5.12</b> mzmatch</a></li>
<li class="chapter" data-level="6.5.13" data-path="annotation.html"><a href="annotation.html#mz.unity"><i class="fa fa-check"></i><b>6.5.13</b> mz.unity</a></li>
<li class="chapter" data-level="6.5.14" data-path="annotation.html"><a href="annotation.html#mait"><i class="fa fa-check"></i><b>6.5.14</b> MAIT</a></li>
<li class="chapter" data-level="6.5.15" data-path="annotation.html"><a href="annotation.html#probmetab"><i class="fa fa-check"></i><b>6.5.15</b> ProbMetab</a></li>
<li class="chapter" data-level="6.5.16" data-path="annotation.html"><a href="annotation.html#ramsi"><i class="fa fa-check"></i><b>6.5.16</b> RAMSI</a></li>
<li class="chapter" data-level="6.5.17" data-path="annotation.html"><a href="annotation.html#sirius"><i class="fa fa-check"></i><b>6.5.17</b> Sirius</a></li>
<li class="chapter" data-level="6.5.18" data-path="annotation.html"><a href="annotation.html#mi-pack"><i class="fa fa-check"></i><b>6.5.18</b> MI-Pack</a></li>
<li class="chapter" data-level="6.5.19" data-path="annotation.html"><a href="annotation.html#plantmat"><i class="fa fa-check"></i><b>6.5.19</b> Plantmat</a></li>
<li class="chapter" data-level="6.5.20" data-path="annotation.html"><a href="annotation.html#metfamily"><i class="fa fa-check"></i><b>6.5.20</b> MetFamily</a></li>
<li class="chapter" data-level="6.5.21" data-path="annotation.html"><a href="annotation.html#lipidmatch"><i class="fa fa-check"></i><b>6.5.21</b> Lipidmatch</a></li>
<li class="chapter" data-level="6.5.22" data-path="annotation.html"><a href="annotation.html#molfind"><i class="fa fa-check"></i><b>6.5.22</b> MolFind</a></li>
<li class="chapter" data-level="6.5.23" data-path="annotation.html"><a href="annotation.html#metfusion"><i class="fa fa-check"></i><b>6.5.23</b> MetFusion</a></li>
<li class="chapter" data-level="6.5.24" data-path="annotation.html"><a href="annotation.html#imet"><i class="fa fa-check"></i><b>6.5.24</b> iMet</a></li>
<li class="chapter" data-level="6.5.25" data-path="annotation.html"><a href="annotation.html#metscape"><i class="fa fa-check"></i><b>6.5.25</b> Metscape</a></li>
<li class="chapter" data-level="6.5.26" data-path="annotation.html"><a href="annotation.html#metfrag"><i class="fa fa-check"></i><b>6.5.26</b> MetFrag</a></li>
<li class="chapter" data-level="6.5.27" data-path="annotation.html"><a href="annotation.html#lipidfrag"><i class="fa fa-check"></i><b>6.5.27</b> LipidFrag</a></li>
<li class="chapter" data-level="6.5.28" data-path="annotation.html"><a href="annotation.html#mycompoundid"><i class="fa fa-check"></i><b>6.5.28</b> MycompoundID</a></li>
<li class="chapter" data-level="6.5.29" data-path="annotation.html"><a href="annotation.html#cfm-id"><i class="fa fa-check"></i><b>6.5.29</b> CFM-ID</a></li>
<li class="chapter" data-level="6.5.30" data-path="annotation.html"><a href="annotation.html#magma"><i class="fa fa-check"></i><b>6.5.30</b> magma</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="7" data-path="omics-analysis.html"><a href="omics-analysis.html"><i class="fa fa-check"></i><b>7</b> Omics analysis</a><ul>
<li class="chapter" data-level="7.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-analysis"><i class="fa fa-check"></i><b>7.1</b> Pathway analysis</a><ul>
<li class="chapter" data-level="7.1.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-database"><i class="fa fa-check"></i><b>7.1.1</b> Pathway Database</a></li>
<li class="chapter" data-level="7.1.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-software"><i class="fa fa-check"></i><b>7.1.2</b> Pathway software</a></li>
</ul></li>
<li class="chapter" data-level="7.2" data-path="omics-analysis.html"><a href="omics-analysis.html#network-analysis"><i class="fa fa-check"></i><b>7.2</b> Network analysis</a></li>
<li class="chapter" data-level="7.3" data-path="omics-analysis.html"><a href="omics-analysis.html#omics-integration"><i class="fa fa-check"></i><b>7.3</b> Omics integration</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html"><i class="fa fa-check"></i><b>8</b> Common analysis methods for metabolomics</a><ul>
<li class="chapter" data-level="8.1" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#basic-statistical-analysis"><i class="fa fa-check"></i><b>8.1</b> Basic Statistical Analysis</a></li>
<li class="chapter" data-level="8.2" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#pca"><i class="fa fa-check"></i><b>8.2</b> PCA</a></li>
<li class="chapter" data-level="8.3" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#cluster-analysis"><i class="fa fa-check"></i><b>8.3</b> Cluster Analysis</a></li>
<li class="chapter" data-level="8.4" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#plsda"><i class="fa fa-check"></i><b>8.4</b> PLSDA</a></li>
<li class="chapter" data-level="8.5" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#self-organizing-map"><i class="fa fa-check"></i><b>8.5</b> Self-organizing map</a></li>
<li class="chapter" data-level="8.6" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#canonical-correlation-analysis"><i class="fa fa-check"></i><b>8.6</b> Canonical correlation analysis</a></li>
<li class="chapter" data-level="8.7" data-path="common-analysis-methods-for-metabolomics.html"><a href="common-analysis-methods-for-metabolomics.html#software-4"><i class="fa fa-check"></i><b>8.7</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="demo.html"><a href="demo.html"><i class="fa fa-check"></i><b>9</b> Demo</a><ul>
<li class="chapter" data-level="9.1" data-path="demo.html"><a href="demo.html#project-setup"><i class="fa fa-check"></i><b>9.1</b> Project Setup</a></li>
<li class="chapter" data-level="9.2" data-path="demo.html"><a href="demo.html#data-input"><i class="fa fa-check"></i><b>9.2</b> Data input</a></li>
<li class="chapter" data-level="9.3" data-path="demo.html"><a href="demo.html#optimization"><i class="fa fa-check"></i><b>9.3</b> Optimization</a></li>
<li class="chapter" data-level="9.4" data-path="demo.html"><a href="demo.html#wrap-function"><i class="fa fa-check"></i><b>9.4</b> Wrap function</a><ul>
<li class="chapter" data-level="9.4.1" data-path="demo.html"><a href="demo.html#peak-picking"><i class="fa fa-check"></i><b>9.4.1</b> Peak picking</a></li>
<li class="chapter" data-level="9.4.2" data-path="demo.html"><a href="demo.html#data-correction"><i class="fa fa-check"></i><b>9.4.2</b> Data correction</a></li>
<li class="chapter" data-level="9.4.3" data-path="demo.html"><a href="demo.html#peaks-filling"><i class="fa fa-check"></i><b>9.4.3</b> Peaks filling</a></li>
</ul></li>
<li class="chapter" data-level="9.5" data-path="demo.html"><a href="demo.html#peaks-list"><i class="fa fa-check"></i><b>9.5</b> Peaks list</a></li>
<li class="chapter" data-level="9.6" data-path="demo.html"><a href="demo.html#peaks-filtering"><i class="fa fa-check"></i><b>9.6</b> Peaks filtering</a></li>
<li class="chapter" data-level="9.7" data-path="demo.html"><a href="demo.html#normalization-optional"><i class="fa fa-check"></i><b>9.7</b> Normalization (Optional)</a></li>
<li class="chapter" data-level="9.8" data-path="demo.html"><a href="demo.html#statistic-analysis"><i class="fa fa-check"></i><b>9.8</b> Statistic analysis</a></li>
<li class="chapter" data-level="9.9" data-path="demo.html"><a href="demo.html#annotation-1"><i class="fa fa-check"></i><b>9.9</b> Annotation</a></li>
<li class="chapter" data-level="9.10" data-path="demo.html"><a href="demo.html#pathway-analysis-1"><i class="fa fa-check"></i><b>9.10</b> Pathway Analysis</a></li>
<li class="chapter" data-level="9.11" data-path="demo.html"><a href="demo.html#metaboanalyst"><i class="fa fa-check"></i><b>9.11</b> MetaboAnalyst</a></li>
<li class="chapter" data-level="9.12" data-path="demo.html"><a href="demo.html#summary"><i class="fa fa-check"></i><b>9.12</b> Summary</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html"><i class="fa fa-check"></i><b>10</b> Software/Web Application/Workflow</a><ul>
<li class="chapter" data-level="10.1" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#platform-for-metabolomics"><i class="fa fa-check"></i><b>10.1</b> Platform for metabolomics</a><ul>
<li class="chapter" data-level="10.1.1" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#xcms-online"><i class="fa fa-check"></i><b>10.1.1</b> XCMS online</a></li>
<li class="chapter" data-level="10.1.2" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#prime"><i class="fa fa-check"></i><b>10.1.2</b> PRIMe</a></li>
<li class="chapter" data-level="10.1.3" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#openms"><i class="fa fa-check"></i><b>10.1.3</b> OpenMS</a></li>
<li class="chapter" data-level="10.1.4" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#mzmine-2"><i class="fa fa-check"></i><b>10.1.4</b> MZmine 2</a></li>
<li class="chapter" data-level="10.1.5" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#xcms"><i class="fa fa-check"></i><b>10.1.5</b> XCMS</a></li>
<li class="chapter" data-level="10.1.6" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#emory-mahpic"><i class="fa fa-check"></i><b>10.1.6</b> Emory MaHPIC</a></li>
<li class="chapter" data-level="10.1.7" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#dia-data-analysis"><i class="fa fa-check"></i><b>10.1.7</b> DIA data analysis</a></li>
<li class="chapter" data-level="10.1.8" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#others"><i class="fa fa-check"></i><b>10.1.8</b> Others</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#data-sharing"><i class="fa fa-check"></i><b>10.2</b> Data sharing</a></li>
<li class="chapter" data-level="10.3" data-path="softwareweb-applicationworkflow.html"><a href="softwareweb-applicationworkflow.html#contest"><i class="fa fa-check"></i><b>10.3</b> Contest</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="exposome.html"><a href="exposome.html"><i class="fa fa-check"></i><b>11</b> Exposome</a><ul>
<li class="chapter" data-level="11.1" data-path="exposome.html"><a href="exposome.html#internal-exposure"><i class="fa fa-check"></i><b>11.1</b> Internal exposure</a></li>
<li class="chapter" data-level="11.2" data-path="exposome.html"><a href="exposome.html#external-exposure"><i class="fa fa-check"></i><b>11.2</b> External exposure</a><ul>
<li class="chapter" data-level="11.2.1" data-path="exposome.html"><a href="exposome.html#environmental-fate-of-compounds"><i class="fa fa-check"></i><b>11.2.1</b> Environmental fate of compounds</a></li>
<li class="chapter" data-level="11.2.2" data-path="exposome.html"><a href="exposome.html#exposure-study-database"><i class="fa fa-check"></i><b>11.2.2</b> Exposure study database</a></li>
</ul></li>
</ul></li>
<li class="divider"></li>
<li><a href="https://github.com/yufree/metaworkflow" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Meta-Workflow</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="omics-analysis" class="section level1">
<h1><span class="header-section-number">Chapter 7</span> Omics analysis</h1>
<p>When you get the filtered ions, the next step is making annotations for them. Such annotations would be helpful for omics studies.</p>
<p>Since we have got the annotations, Omics analysis could be performed.Upload the data obtained from the <strong>xcms</strong> to other tools or databases.</p>
<p>You will get an updated database list <a href="http://metabolomicssociety.org/resources/metabolomics-databases">here</a></p>
<p>Right now, it is hard to connect different omics databases such as gene, protein and metabolites together for a whole scope of certain biological process. However, you might select few metabolites across those databases and find something interesting.</p>
<div id="pathway-analysis" class="section level2">
<h2><span class="header-section-number">7.1</span> Pathway analysis</h2>
<p>Pathwat analysis maps annotated data into known pathway and make statistical analysis to find the influenced pathway or the compounds with high inflences on certain pathway.</p>
<div id="pathway-database" class="section level3">
<h3><span class="header-section-number">7.1.1</span> Pathway Database</h3>
<ul>
<li><p><a href="http://smpdb.ca/view">SMPDB</a> (The Small Molecule Pathway Database) is an interactive, visual database containing more than 618 small molecule pathways found in humans. More than 70% of these pathways (&gt;433) are not found in any other pathway database. The pathways include metabolic, drug, and disease pathways.</p></li>
<li><p><a href="https://www.genome.jp/kegg/">KEGG</a> (Kyoto Encyclopedia of Genes and Genomes) is one of the most complete and widely used databases containing metabolic pathways (495 reference pathways) from a wide variety of organisms (&gt;4,700). These pathways are hyperlinked to metabolite and protein/enzyme information. Currently KEGG has &gt;17,000 compounds (from animals, plants and bacteria), 10,000 drugs (including different salt forms and drug carriers) and nearly 11,000 glycan structures.</p></li>
<li><p><a href="https://biocyc.org/">BioCyc</a> is a collection of 14558 Pathway/Genome Databases (PGDBs), plus software tools for exploring them.</p></li>
<li><p><a href="https://reactome.org/what-is-reactome">Reactome</a> is an open-source, open access, manually curated and peer-reviewed pathway database. Our goal is to provide intuitive bioinformatics tools for the visualization, interpretation and analysis of pathway knowledge to support basic and clinical research, genome analysis, modeling, systems biology and education.</p></li>
<li><p><a href="https://www.wikipathways.org/index.php/WikiPathways">WikiPathway</a> is a database of biological pathways maintained by and for the scientific community.</p></li>
</ul>
</div>
<div id="pathway-software" class="section level3">
<h3><span class="header-section-number">7.1.2</span> Pathway software</h3>
<ul>
<li><p><a href="http://www.pathwaycommons.org/">Pathway Commons</a> online tools for pathway analysis</p></li>
<li><p><a href="https://github.com/Mathelab/RaMP-DB">RaMP</a> could make pathway analysis for batch search</p></li>
<li><p><a href="https://github.com/kwanjeeraw/metabox">metabox</a> could make pathway analysis</p></li>
<li><p><a href="http://impala.molgen.mpg.de/">impala</a> is used for pathway enrichment analysis</p></li>
</ul>
</div>
</div>
<div id="network-analysis" class="section level2">
<h2><span class="header-section-number">7.2</span> Network analysis</h2>
<p><a href="https://code.google.com/archive/p/atcg/wikis/mummichog_for_metabolomics.wiki">Mummichog</a> could make pathway and network analysis without annotation.</p>
<p><a href="http://web1.sph.emory.edu/users/tyu8/MSS">MSS</a>: sequential feature screening procedure to select important sub-network and identify the optimal matching for metabolimics data <span class="citation">(Cai et al. <a href="#ref-cai2017">2017</a>)</span></p>
</div>
<div id="omics-integration" class="section level2">
<h2><span class="header-section-number">7.3</span> Omics integration</h2>
<ul>
<li><p><a href="https://blast.ncbi.nlm.nih.gov/Blast.cgi">Blast</a> finds regions of similarity between biological sequences. The program compares nucleotide or protein sequences to sequence databases and calculates the statistical significance.</p></li>
<li><p><a href="https://www.omicsdi.org/">The Omics Discovery Index (OmicsDI)</a> provides a knowledge discovery framework across heterogeneous omics data (genomics, proteomics, transcriptomics and metabolomics).</p></li>
<li><p><a href="https://github.com/cran/mixOmics">Omics Data Integration Project</a></p></li>
</ul>

</div>
</div>
<h3>References</h3>
<div id="refs" class="references">
<div id="ref-cai2017">
<p>Cai, Qingpo, Jessica A. Alvarez, Jian Kang, and Tianwei Yu. 2017. “Network Marker Selection for Untargeted LC.” <em>J. Proteome Res.</em> 16 (3): 1261–9. doi:<a href="https://doi.org/10.1021/acs.jproteome.6b00861">10.1021/acs.jproteome.6b00861</a>.</p>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="annotation.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="common-analysis-methods-for-metabolomics.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"google": false,
"linkedin": false,
"weibo": false,
"instapper": false,
"vk": false,
"all": ["facebook", "google", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/yufree/metaworkflow/edit/master/07-omics.Rmd",
"text": "Edit"
},
"download": ["Metabolomics.pdf", "Metabolomics.epub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "";
    if (src === "" || src === "true") src = "https://cdn.bootcss.com/mathjax/2.7.1/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:" && /^https?:/.test(src))
      src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
