<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>Chapter 9 Peaks normalization | Meta-Workflow</title>
  <meta name="description" content="This is a workflow for metabolomics studies." />
  <meta name="generator" content="bookdown 0.32 and GitBook 2.6.7" />

  <meta property="og:title" content="Chapter 9 Peaks normalization | Meta-Workflow" />
  <meta property="og:type" content="book" />
  <meta property="og:image" content="/images/cover.png" />
  <meta property="og:description" content="This is a workflow for metabolomics studies." />
  <meta name="github-repo" content="yufree/metaworkflow" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="Chapter 9 Peaks normalization | Meta-Workflow" />
  
  <meta name="twitter:description" content="This is a workflow for metabolomics studies." />
  <meta name="twitter:image" content="/images/cover.png" />

<meta name="author" content="Miao YU" />


<meta name="date" content="2023-02-01" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="omics-analysis.html"/>
<link rel="next" href="statistical-analysis.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.6.1/htmlwidgets.js"></script>
<script src="libs/viz-1.8.2/viz.js"></script>
<link href="libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="libs/grViz-binding-1.0.9/grViz.js"></script>
<script src="libs/d3-3.3.8/d3.min.js"></script>
<script src="libs/dagre-0.4.0/dagre-d3.min.js"></script>
<link href="libs/mermaid-0.3.0/dist/mermaid.css" rel="stylesheet" />
<script src="libs/mermaid-0.3.0/dist/mermaid.slim.min.js"></script>
<script src="libs/chromatography-0.1/chromatography.js"></script>
<script src="libs/DiagrammeR-binding-1.0.9/DiagrammeR.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metabolomics Workflow</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#history"><i class="fa fa-check"></i><b>1.1</b> History</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#history-of-mass-spectrometry"><i class="fa fa-check"></i><b>1.1.1</b> History of Mass Spectrometry</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#history-of-metabolomcis"><i class="fa fa-check"></i><b>1.1.2</b> History of Metabolomcis</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#defination"><i class="fa fa-check"></i><b>1.1.3</b> Defination</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#reviews-and-tutorials"><i class="fa fa-check"></i><b>1.2</b> Reviews and tutorials</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#workflow"><i class="fa fa-check"></i><b>1.2.1</b> Workflow</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#data-analysis"><i class="fa fa-check"></i><b>1.2.2</b> Data analysis</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#application"><i class="fa fa-check"></i><b>1.2.3</b> Application</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#challenge"><i class="fa fa-check"></i><b>1.2.4</b> Challenge</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#trends-in-metabolomics"><i class="fa fa-check"></i><b>1.3</b> Trends in Metabolomics</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#workflow-1"><i class="fa fa-check"></i><b>1.4</b> Workflow</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html"><i class="fa fa-check"></i><b>2</b> Experimental design(DoE)</a>
<ul>
<li class="chapter" data-level="2.1" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#homogeneity-study"><i class="fa fa-check"></i><b>2.1</b> Homogeneity study</a></li>
<li class="chapter" data-level="2.2" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#heterogeneity-study"><i class="fa fa-check"></i><b>2.2</b> Heterogeneity study</a></li>
<li class="chapter" data-level="2.3" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#power-analysis"><i class="fa fa-check"></i><b>2.3</b> Power analysis</a></li>
<li class="chapter" data-level="2.4" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#optimization"><i class="fa fa-check"></i><b>2.4</b> Optimization</a></li>
<li class="chapter" data-level="2.5" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#pooled-qc"><i class="fa fa-check"></i><b>2.5</b> Pooled QC</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="pretreatment.html"><a href="pretreatment.html"><i class="fa fa-check"></i><b>3</b> Pretreatment</a>
<ul>
<li class="chapter" data-level="3.1" data-path="pretreatment.html"><a href="pretreatment.html#collection"><i class="fa fa-check"></i><b>3.1</b> Collection</a></li>
<li class="chapter" data-level="3.2" data-path="pretreatment.html"><a href="pretreatment.html#quenching"><i class="fa fa-check"></i><b>3.2</b> Quenching</a></li>
<li class="chapter" data-level="3.3" data-path="pretreatment.html"><a href="pretreatment.html#extraction"><i class="fa fa-check"></i><b>3.3</b> Extraction</a></li>
<li class="chapter" data-level="3.4" data-path="pretreatment.html"><a href="pretreatment.html#derivatization"><i class="fa fa-check"></i><b>3.4</b> Derivatization</a></li>
<li class="chapter" data-level="3.5" data-path="pretreatment.html"><a href="pretreatment.html#isotope-label"><i class="fa fa-check"></i><b>3.5</b> Isotope label</a></li>
<li class="chapter" data-level="3.6" data-path="pretreatment.html"><a href="pretreatment.html#storage"><i class="fa fa-check"></i><b>3.6</b> Storage</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html"><i class="fa fa-check"></i><b>4</b> Instrumental analysis</a>
<ul>
<li class="chapter" data-level="4.1" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#column-and-gradient-selection"><i class="fa fa-check"></i><b>4.1</b> Column and gradient selection</a></li>
<li class="chapter" data-level="4.2" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#mass-resolution"><i class="fa fa-check"></i><b>4.2</b> Mass resolution</a></li>
<li class="chapter" data-level="4.3" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#matrix-effects"><i class="fa fa-check"></i><b>4.3</b> Matrix effects</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="workflow-2.html"><a href="workflow-2.html"><i class="fa fa-check"></i><b>5</b> Workflow</a>
<ul>
<li class="chapter" data-level="5.1" data-path="workflow-2.html"><a href="workflow-2.html#platform-for-metabolomics-data-analysis"><i class="fa fa-check"></i><b>5.1</b> Platform for metabolomics data analysis</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="workflow-2.html"><a href="workflow-2.html#xcms-xcms-online"><i class="fa fa-check"></i><b>5.1.1</b> XCMS &amp; XCMS online</a></li>
<li class="chapter" data-level="5.1.2" data-path="workflow-2.html"><a href="workflow-2.html#prime"><i class="fa fa-check"></i><b>5.1.2</b> PRIMe</a></li>
<li class="chapter" data-level="5.1.3" data-path="workflow-2.html"><a href="workflow-2.html#gnps"><i class="fa fa-check"></i><b>5.1.3</b> GNPS</a></li>
<li class="chapter" data-level="5.1.4" data-path="workflow-2.html"><a href="workflow-2.html#openms-sirius"><i class="fa fa-check"></i><b>5.1.4</b> OpenMS &amp; SIRIUS</a></li>
<li class="chapter" data-level="5.1.5" data-path="workflow-2.html"><a href="workflow-2.html#mzmine-2"><i class="fa fa-check"></i><b>5.1.5</b> MZmine 2</a></li>
<li class="chapter" data-level="5.1.6" data-path="workflow-2.html"><a href="workflow-2.html#emory-mahpic"><i class="fa fa-check"></i><b>5.1.6</b> Emory MaHPIC</a></li>
<li class="chapter" data-level="5.1.7" data-path="workflow-2.html"><a href="workflow-2.html#others"><i class="fa fa-check"></i><b>5.1.7</b> Others</a></li>
<li class="chapter" data-level="5.1.8" data-path="workflow-2.html"><a href="workflow-2.html#workflow-comparison"><i class="fa fa-check"></i><b>5.1.8</b> Workflow Comparison</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="workflow-2.html"><a href="workflow-2.html#project-setup"><i class="fa fa-check"></i><b>5.2</b> Project Setup</a></li>
<li class="chapter" data-level="5.3" data-path="workflow-2.html"><a href="workflow-2.html#data-sharing"><i class="fa fa-check"></i><b>5.3</b> Data sharing</a></li>
<li class="chapter" data-level="5.4" data-path="workflow-2.html"><a href="workflow-2.html#contest"><i class="fa fa-check"></i><b>5.4</b> Contest</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html"><i class="fa fa-check"></i><b>6</b> Raw data pretreatment</a>
<ul>
<li class="chapter" data-level="6.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#data-visualization"><i class="fa fa-check"></i><b>6.1</b> Data visualization</a></li>
<li class="chapter" data-level="6.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#peak-extraction"><i class="fa fa-check"></i><b>6.2</b> Peak extraction</a></li>
<li class="chapter" data-level="6.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#msms"><i class="fa fa-check"></i><b>6.3</b> MS/MS</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#mrm"><i class="fa fa-check"></i><b>6.3.1</b> MRM</a></li>
<li class="chapter" data-level="6.3.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dda"><i class="fa fa-check"></i><b>6.3.2</b> DDA</a></li>
<li class="chapter" data-level="6.3.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dia"><i class="fa fa-check"></i><b>6.3.3</b> DIA</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#retention-time-correction"><i class="fa fa-check"></i><b>6.4</b> Retention Time Correction</a></li>
<li class="chapter" data-level="6.5" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#filling-missing-values"><i class="fa fa-check"></i><b>6.5</b> Filling missing values</a></li>
<li class="chapter" data-level="6.6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#spectral-deconvolution"><i class="fa fa-check"></i><b>6.6</b> Spectral deconvolution</a></li>
<li class="chapter" data-level="6.7" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dynamic-range"><i class="fa fa-check"></i><b>6.7</b> Dynamic Range</a>
<ul>
<li class="chapter" data-level="6.7.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#non-detects"><i class="fa fa-check"></i><b>6.7.1</b> Non-detects</a></li>
<li class="chapter" data-level="6.7.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#over-detection-limit"><i class="fa fa-check"></i><b>6.7.2</b> Over Detection Limit</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#rsdfold-change-filter"><i class="fa fa-check"></i><b>6.8</b> RSD/fold change Filter</a></li>
<li class="chapter" data-level="6.9" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#power-analysis-filter"><i class="fa fa-check"></i><b>6.9</b> Power Analysis Filter</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="annotation.html"><a href="annotation.html"><i class="fa fa-check"></i><b>7</b> Annotation</a>
<ul>
<li class="chapter" data-level="7.1" data-path="annotation.html"><a href="annotation.html#issues-in-annotation"><i class="fa fa-check"></i><b>7.1</b> Issues in annotation</a></li>
<li class="chapter" data-level="7.2" data-path="annotation.html"><a href="annotation.html#peak-misidentification"><i class="fa fa-check"></i><b>7.2</b> Peak misidentification</a></li>
<li class="chapter" data-level="7.3" data-path="annotation.html"><a href="annotation.html#annotation-v.s.-identification"><i class="fa fa-check"></i><b>7.3</b> Annotation v.s. identification</a></li>
<li class="chapter" data-level="7.4" data-path="annotation.html"><a href="annotation.html#molecular-formula-assignment"><i class="fa fa-check"></i><b>7.4</b> Molecular Formula Assignment</a></li>
<li class="chapter" data-level="7.5" data-path="annotation.html"><a href="annotation.html#redundant-peaks"><i class="fa fa-check"></i><b>7.5</b> Redundant peaks</a>
<ul>
<li class="chapter" data-level="7.5.1" data-path="annotation.html"><a href="annotation.html#adducts-list"><i class="fa fa-check"></i><b>7.5.1</b> Adducts list</a></li>
<li class="chapter" data-level="7.5.2" data-path="annotation.html"><a href="annotation.html#isotope"><i class="fa fa-check"></i><b>7.5.2</b> Isotope</a></li>
<li class="chapter" data-level="7.5.3" data-path="annotation.html"><a href="annotation.html#camera"><i class="fa fa-check"></i><b>7.5.3</b> CAMERA</a></li>
<li class="chapter" data-level="7.5.4" data-path="annotation.html"><a href="annotation.html#ramclustr"><i class="fa fa-check"></i><b>7.5.4</b> RAMClustR</a></li>
<li class="chapter" data-level="7.5.5" data-path="annotation.html"><a href="annotation.html#biocan"><i class="fa fa-check"></i><b>7.5.5</b> BioCAn</a></li>
<li class="chapter" data-level="7.5.6" data-path="annotation.html"><a href="annotation.html#mzmatch"><i class="fa fa-check"></i><b>7.5.6</b> mzMatch</a></li>
<li class="chapter" data-level="7.5.7" data-path="annotation.html"><a href="annotation.html#xmsannotator"><i class="fa fa-check"></i><b>7.5.7</b> xMSannotator</a></li>
<li class="chapter" data-level="7.5.8" data-path="annotation.html"><a href="annotation.html#mwise"><i class="fa fa-check"></i><b>7.5.8</b> mWise</a></li>
<li class="chapter" data-level="7.5.9" data-path="annotation.html"><a href="annotation.html#mait"><i class="fa fa-check"></i><b>7.5.9</b> MAIT</a></li>
<li class="chapter" data-level="7.5.10" data-path="annotation.html"><a href="annotation.html#pmd"><i class="fa fa-check"></i><b>7.5.10</b> pmd</a></li>
<li class="chapter" data-level="7.5.11" data-path="annotation.html"><a href="annotation.html#nontarget"><i class="fa fa-check"></i><b>7.5.11</b> nontarget</a></li>
<li class="chapter" data-level="7.5.12" data-path="annotation.html"><a href="annotation.html#binner"><i class="fa fa-check"></i><b>7.5.12</b> Binner</a></li>
<li class="chapter" data-level="7.5.13" data-path="annotation.html"><a href="annotation.html#mz.unity"><i class="fa fa-check"></i><b>7.5.13</b> mz.unity</a></li>
<li class="chapter" data-level="7.5.14" data-path="annotation.html"><a href="annotation.html#ms-flo"><i class="fa fa-check"></i><b>7.5.14</b> MS-FLO</a></li>
<li class="chapter" data-level="7.5.15" data-path="annotation.html"><a href="annotation.html#cliquems"><i class="fa fa-check"></i><b>7.5.15</b> CliqueMS</a></li>
<li class="chapter" data-level="7.5.16" data-path="annotation.html"><a href="annotation.html#interpretmsspectrum"><i class="fa fa-check"></i><b>7.5.16</b> InterpretMSSpectrum</a></li>
<li class="chapter" data-level="7.5.17" data-path="annotation.html"><a href="annotation.html#netid"><i class="fa fa-check"></i><b>7.5.17</b> NetID</a></li>
<li class="chapter" data-level="7.5.18" data-path="annotation.html"><a href="annotation.html#isfrag"><i class="fa fa-check"></i><b>7.5.18</b> ISfrag</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="annotation.html"><a href="annotation.html#ms1-ms2-connection"><i class="fa fa-check"></i><b>7.6</b> MS1 MS2 connection</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="annotation.html"><a href="annotation.html#pmdda"><i class="fa fa-check"></i><b>7.6.1</b> PMDDA</a></li>
<li class="chapter" data-level="7.6.2" data-path="annotation.html"><a href="annotation.html#hermes"><i class="fa fa-check"></i><b>7.6.2</b> HERMES</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="annotation.html"><a href="annotation.html#msms-annotation"><i class="fa fa-check"></i><b>7.7</b> MS/MS annotation</a>
<ul>
<li class="chapter" data-level="7.7.1" data-path="annotation.html"><a href="annotation.html#matchms"><i class="fa fa-check"></i><b>7.7.1</b> Matchms</a></li>
<li class="chapter" data-level="7.7.2" data-path="annotation.html"><a href="annotation.html#metdna"><i class="fa fa-check"></i><b>7.7.2</b> MetDNA</a></li>
<li class="chapter" data-level="7.7.3" data-path="annotation.html"><a href="annotation.html#metfusion"><i class="fa fa-check"></i><b>7.7.3</b> MetFusion</a></li>
<li class="chapter" data-level="7.7.4" data-path="annotation.html"><a href="annotation.html#ms2analyzer"><i class="fa fa-check"></i><b>7.7.4</b> MS2Analyzer</a></li>
<li class="chapter" data-level="7.7.5" data-path="annotation.html"><a href="annotation.html#metfrag"><i class="fa fa-check"></i><b>7.7.5</b> MetFrag</a></li>
<li class="chapter" data-level="7.7.6" data-path="annotation.html"><a href="annotation.html#cfm-id"><i class="fa fa-check"></i><b>7.7.6</b> CFM-ID</a></li>
<li class="chapter" data-level="7.7.7" data-path="annotation.html"><a href="annotation.html#lc-ms2struct"><i class="fa fa-check"></i><b>7.7.7</b> LC-MS2Struct</a></li>
<li class="chapter" data-level="7.7.8" data-path="annotation.html"><a href="annotation.html#lipidfrag"><i class="fa fa-check"></i><b>7.7.8</b> LipidFrag</a></li>
<li class="chapter" data-level="7.7.9" data-path="annotation.html"><a href="annotation.html#lipidmatch"><i class="fa fa-check"></i><b>7.7.9</b> Lipidmatch</a></li>
<li class="chapter" data-level="7.7.10" data-path="annotation.html"><a href="annotation.html#barcoding"><i class="fa fa-check"></i><b>7.7.10</b> BarCoding</a></li>
<li class="chapter" data-level="7.7.11" data-path="annotation.html"><a href="annotation.html#imet"><i class="fa fa-check"></i><b>7.7.11</b> iMet</a></li>
</ul></li>
<li class="chapter" data-level="7.8" data-path="annotation.html"><a href="annotation.html#knowledge-based-annotation"><i class="fa fa-check"></i><b>7.8</b> Knowledge based annotation</a>
<ul>
<li class="chapter" data-level="7.8.1" data-path="annotation.html"><a href="annotation.html#chromatographic-retention-related-criteria"><i class="fa fa-check"></i><b>7.8.1</b> Chromatographic retention-related criteria</a></li>
<li class="chapter" data-level="7.8.2" data-path="annotation.html"><a href="annotation.html#probmetab"><i class="fa fa-check"></i><b>7.8.2</b> ProbMetab</a></li>
<li class="chapter" data-level="7.8.3" data-path="annotation.html"><a href="annotation.html#mi-pack"><i class="fa fa-check"></i><b>7.8.3</b> MI-Pack</a></li>
<li class="chapter" data-level="7.8.4" data-path="annotation.html"><a href="annotation.html#metexpert"><i class="fa fa-check"></i><b>7.8.4</b> MetExpert</a></li>
<li class="chapter" data-level="7.8.5" data-path="annotation.html"><a href="annotation.html#mycompoundid"><i class="fa fa-check"></i><b>7.8.5</b> MycompoundID</a></li>
<li class="chapter" data-level="7.8.6" data-path="annotation.html"><a href="annotation.html#metfamily"><i class="fa fa-check"></i><b>7.8.6</b> MetFamily</a></li>
<li class="chapter" data-level="7.8.7" data-path="annotation.html"><a href="annotation.html#plantmat"><i class="fa fa-check"></i><b>7.8.7</b> Plantmat</a></li>
<li class="chapter" data-level="7.8.8" data-path="annotation.html"><a href="annotation.html#coa-blast"><i class="fa fa-check"></i><b>7.8.8</b> CoA-Blast</a></li>
<li class="chapter" data-level="7.8.9" data-path="annotation.html"><a href="annotation.html#kgmn"><i class="fa fa-check"></i><b>7.8.9</b> KGMN</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="annotation.html"><a href="annotation.html#ms-database-for-annotation"><i class="fa fa-check"></i><b>7.9</b> MS Database for annotation</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="annotation.html"><a href="annotation.html#ms"><i class="fa fa-check"></i><b>7.9.1</b> MS</a></li>
<li class="chapter" data-level="7.9.2" data-path="annotation.html"><a href="annotation.html#msms-1"><i class="fa fa-check"></i><b>7.9.2</b> MS/MS</a></li>
</ul></li>
<li class="chapter" data-level="7.10" data-path="annotation.html"><a href="annotation.html#compounds-database"><i class="fa fa-check"></i><b>7.10</b> Compounds Database</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="omics-analysis.html"><a href="omics-analysis.html"><i class="fa fa-check"></i><b>8</b> Omics analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="omics-analysis.html"><a href="omics-analysis.html#from-bottom-up-to-top-down"><i class="fa fa-check"></i><b>8.1</b> From Bottom-up to Top-down</a></li>
<li class="chapter" data-level="8.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-analysis"><i class="fa fa-check"></i><b>8.2</b> Pathway analysis</a>
<ul>
<li class="chapter" data-level="8.2.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-database"><i class="fa fa-check"></i><b>8.2.1</b> Pathway Database</a></li>
<li class="chapter" data-level="8.2.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-software"><i class="fa fa-check"></i><b>8.2.2</b> Pathway software</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="omics-analysis.html"><a href="omics-analysis.html#network-analysis"><i class="fa fa-check"></i><b>8.3</b> Network analysis</a></li>
<li class="chapter" data-level="8.4" data-path="omics-analysis.html"><a href="omics-analysis.html#omics-integration"><i class="fa fa-check"></i><b>8.4</b> Omics integration</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="peaks-normalization.html"><a href="peaks-normalization.html"><i class="fa fa-check"></i><b>9</b> Peaks normalization</a>
<ul>
<li class="chapter" data-level="9.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#batch-effects-classification"><i class="fa fa-check"></i><b>9.1</b> Batch effects classification</a></li>
<li class="chapter" data-level="9.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#batch-effects-visualization"><i class="fa fa-check"></i><b>9.2</b> Batch effects visualization</a></li>
<li class="chapter" data-level="9.3" data-path="peaks-normalization.html"><a href="peaks-normalization.html#source-of-batch-effects"><i class="fa fa-check"></i><b>9.3</b> Source of batch effects</a></li>
<li class="chapter" data-level="9.4" data-path="peaks-normalization.html"><a href="peaks-normalization.html#avoid-batch-effects-by-doe"><i class="fa fa-check"></i><b>9.4</b> Avoid batch effects by DoE</a></li>
<li class="chapter" data-level="9.5" data-path="peaks-normalization.html"><a href="peaks-normalization.html#post-hoc-data-normalization"><i class="fa fa-check"></i><b>9.5</b> <em>post hoc</em> data normalization</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#unsupervised-methods"><i class="fa fa-check"></i><b>9.5.1</b> Unsupervised methods</a></li>
<li class="chapter" data-level="9.5.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#supervised-methods"><i class="fa fa-check"></i><b>9.5.2</b> Supervised methods</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="peaks-normalization.html"><a href="peaks-normalization.html#method-to-validate-the-normalization"><i class="fa fa-check"></i><b>9.6</b> Method to validate the normalization</a></li>
<li class="chapter" data-level="9.7" data-path="peaks-normalization.html"><a href="peaks-normalization.html#software"><i class="fa fa-check"></i><b>9.7</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="statistical-analysis.html"><a href="statistical-analysis.html"><i class="fa fa-check"></i><b>10</b> Statistical analysis</a>
<ul>
<li class="chapter" data-level="10.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#basic-statistical-analysis"><i class="fa fa-check"></i><b>10.1</b> Basic Statistical Analysis</a></li>
<li class="chapter" data-level="10.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#differences-analysis"><i class="fa fa-check"></i><b>10.2</b> Differences analysis</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#t-test-or-anova"><i class="fa fa-check"></i><b>10.2.1</b> T-test or ANOVA</a></li>
<li class="chapter" data-level="10.2.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#limma"><i class="fa fa-check"></i><b>10.2.2</b> LIMMA</a></li>
<li class="chapter" data-level="10.2.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#bayesian-mixture-model"><i class="fa fa-check"></i><b>10.2.3</b> Bayesian mixture model</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#pca"><i class="fa fa-check"></i><b>10.3</b> PCA</a></li>
<li class="chapter" data-level="10.4" data-path="statistical-analysis.html"><a href="statistical-analysis.html#cluster-analysis"><i class="fa fa-check"></i><b>10.4</b> Cluster Analysis</a></li>
<li class="chapter" data-level="10.5" data-path="statistical-analysis.html"><a href="statistical-analysis.html#plsda"><i class="fa fa-check"></i><b>10.5</b> PLSDA</a></li>
<li class="chapter" data-level="10.6" data-path="statistical-analysis.html"><a href="statistical-analysis.html#network-analysis-1"><i class="fa fa-check"></i><b>10.6</b> Network analysis</a>
<ul>
<li class="chapter" data-level="10.6.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#vertex-and-edge"><i class="fa fa-check"></i><b>10.6.1</b> Vertex and edge</a></li>
<li class="chapter" data-level="10.6.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#build-the-network"><i class="fa fa-check"></i><b>10.6.2</b> Build the network</a></li>
<li class="chapter" data-level="10.6.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#network-attributes"><i class="fa fa-check"></i><b>10.6.3</b> Network attributes</a></li>
</ul></li>
<li class="chapter" data-level="10.7" data-path="statistical-analysis.html"><a href="statistical-analysis.html#software-1"><i class="fa fa-check"></i><b>10.7</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="exposome.html"><a href="exposome.html"><i class="fa fa-check"></i><b>11</b> Exposome</a>
<ul>
<li class="chapter" data-level="11.1" data-path="exposome.html"><a href="exposome.html#internal-exposure"><i class="fa fa-check"></i><b>11.1</b> Internal exposure</a></li>
<li class="chapter" data-level="11.2" data-path="exposome.html"><a href="exposome.html#external-exposure"><i class="fa fa-check"></i><b>11.2</b> External exposure</a>
<ul>
<li class="chapter" data-level="11.2.1" data-path="exposome.html"><a href="exposome.html#environmental-fate-of-compounds"><i class="fa fa-check"></i><b>11.2.1</b> Environmental fate of compounds</a></li>
<li class="chapter" data-level="11.2.2" data-path="exposome.html"><a href="exposome.html#exposure-study-database"><i class="fa fa-check"></i><b>11.2.2</b> Exposure study database</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/yufree/metaworkflow" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Meta-Workflow</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="peaks-normalization" class="section level1 hasAnchor" number="9">
<h1><span class="header-section-number">Chapter 9</span> Peaks normalization<a href="peaks-normalization.html#peaks-normalization" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<div id="batch-effects-classification" class="section level2 hasAnchor" number="9.1">
<h2><span class="header-section-number">9.1</span> Batch effects classification<a href="peaks-normalization.html#batch-effects-classification" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Variances among the samples across all the extracted peaks might be affected by factors other than the experiment design. There are three types of those batch effects: Monotone, Block and Mixed.</p>
<ul>
<li>Monotone would increase/decrease with the injection order or batches.</li>
</ul>
<p><img src="Metabolomics_files/figure-html/unnamed-chunk-9-1.png" width="61.8%" /></p>
<ul>
<li>Block would be system shift among different batches.</li>
</ul>
<p><img src="Metabolomics_files/figure-html/unnamed-chunk-10-1.png" width="61.8%" /></p>
<ul>
<li>Mixed would be the combination of monotone and block batch effects.</li>
</ul>
<p><img src="Metabolomics_files/figure-html/unnamed-chunk-11-1.png" width="61.8%" /></p>
<p>Meanwhile, different compounds would suffer different type of batch effects. In this case, the normalization or batch correction should be done peak by peak.</p>
<p><img src="Metabolomics_files/figure-html/bem-1.png" width="70%" /></p>
</div>
<div id="batch-effects-visualization" class="section level2 hasAnchor" number="9.2">
<h2><span class="header-section-number">9.2</span> Batch effects visualization<a href="peaks-normalization.html#batch-effects-visualization" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Any correction might introduce bias. We need to make sure there are patterns which different from our experimental design. Pooled QC samples should be clustered on PCA score plot.</p>
</div>
<div id="source-of-batch-effects" class="section level2 hasAnchor" number="9.3">
<h2><span class="header-section-number">9.3</span> Source of batch effects<a href="peaks-normalization.html#source-of-batch-effects" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li><p>Different Operators &amp; Dates &amp; Sequences</p></li>
<li><p>Different Instrumental Condition such as different instrumental parameters, poor quality control, sample contamination during the analysis, Column (Pooled QC) and sample matrix effects (ions suppression or/and enhancement)</p></li>
<li><p>Unknown Unknowns</p></li>
</ul>
</div>
<div id="avoid-batch-effects-by-doe" class="section level2 hasAnchor" number="9.4">
<h2><span class="header-section-number">9.4</span> Avoid batch effects by DoE<a href="peaks-normalization.html#avoid-batch-effects-by-doe" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>You could avoid batch effects from experimental design. Cap the sequence with Pooled QC and Randomized samples sequence. Some internal standards/Instrumental QC might Help to find the source of batch effects while it’s not practical for every compounds in non-targeted analysis.</p>
<p>Batch effects might not change the conclusion when the effect size is relatively small. Here is a simulation:</p>
<div class="sourceCode" id="cb15"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb15-1"><a href="peaks-normalization.html#cb15-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">30</span>)</span>
<span id="cb15-2"><a href="peaks-normalization.html#cb15-2" aria-hidden="true" tabindex="-1"></a><span class="co"># real peaks</span></span>
<span id="cb15-3"><a href="peaks-normalization.html#cb15-3" aria-hidden="true" tabindex="-1"></a>group <span class="ot">&lt;-</span> <span class="fu">factor</span>(<span class="fu">c</span>(<span class="fu">rep</span>(<span class="dv">1</span>,<span class="dv">5</span>),<span class="fu">rep</span>(<span class="dv">2</span>,<span class="dv">5</span>)))</span>
<span id="cb15-4"><a href="peaks-normalization.html#cb15-4" aria-hidden="true" tabindex="-1"></a>con <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">rnorm</span>(<span class="dv">5</span>,<span class="dv">5</span>),<span class="fu">rnorm</span>(<span class="dv">5</span>,<span class="dv">8</span>))</span>
<span id="cb15-5"><a href="peaks-normalization.html#cb15-5" aria-hidden="true" tabindex="-1"></a>re <span class="ot">&lt;-</span> <span class="fu">t.test</span>(con<span class="sc">~</span>group)</span>
<span id="cb15-6"><a href="peaks-normalization.html#cb15-6" aria-hidden="true" tabindex="-1"></a><span class="co"># real peaks</span></span>
<span id="cb15-7"><a href="peaks-normalization.html#cb15-7" aria-hidden="true" tabindex="-1"></a>group <span class="ot">&lt;-</span> <span class="fu">factor</span>(<span class="fu">c</span>(<span class="fu">rep</span>(<span class="dv">1</span>,<span class="dv">5</span>),<span class="fu">rep</span>(<span class="dv">2</span>,<span class="dv">5</span>)))</span>
<span id="cb15-8"><a href="peaks-normalization.html#cb15-8" aria-hidden="true" tabindex="-1"></a>con <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">rnorm</span>(<span class="dv">5</span>,<span class="dv">5</span>),<span class="fu">rnorm</span>(<span class="dv">5</span>,<span class="dv">8</span>))</span>
<span id="cb15-9"><a href="peaks-normalization.html#cb15-9" aria-hidden="true" tabindex="-1"></a>batch <span class="ot">&lt;-</span> <span class="fu">seq</span>(<span class="dv">0</span>,<span class="dv">5</span>,<span class="at">length.out =</span> <span class="dv">10</span>)</span>
<span id="cb15-10"><a href="peaks-normalization.html#cb15-10" aria-hidden="true" tabindex="-1"></a>ins <span class="ot">&lt;-</span> batch<span class="sc">+</span>con</span>
<span id="cb15-11"><a href="peaks-normalization.html#cb15-11" aria-hidden="true" tabindex="-1"></a>re <span class="ot">&lt;-</span> <span class="fu">t.test</span>(ins<span class="sc">~</span>group)</span>
<span id="cb15-12"><a href="peaks-normalization.html#cb15-12" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb15-13"><a href="peaks-normalization.html#cb15-13" aria-hidden="true" tabindex="-1"></a>index <span class="ot">&lt;-</span> <span class="fu">sample</span>(<span class="dv">10</span>)</span>
<span id="cb15-14"><a href="peaks-normalization.html#cb15-14" aria-hidden="true" tabindex="-1"></a>ins <span class="ot">&lt;-</span> batch<span class="sc">+</span>con[index]</span>
<span id="cb15-15"><a href="peaks-normalization.html#cb15-15" aria-hidden="true" tabindex="-1"></a>re <span class="ot">&lt;-</span> <span class="fu">t.test</span>(ins<span class="sc">~</span>group[index])</span></code></pre></div>
<p>Randomization could not guarantee the results. Here is a simulation.</p>
<div class="sourceCode" id="cb16"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb16-1"><a href="peaks-normalization.html#cb16-1" aria-hidden="true" tabindex="-1"></a><span class="co"># real peaks</span></span>
<span id="cb16-2"><a href="peaks-normalization.html#cb16-2" aria-hidden="true" tabindex="-1"></a>group <span class="ot">&lt;-</span> <span class="fu">factor</span>(<span class="fu">c</span>(<span class="fu">rep</span>(<span class="dv">1</span>,<span class="dv">5</span>),<span class="fu">rep</span>(<span class="dv">2</span>,<span class="dv">5</span>)))</span>
<span id="cb16-3"><a href="peaks-normalization.html#cb16-3" aria-hidden="true" tabindex="-1"></a>con <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">rnorm</span>(<span class="dv">5</span>,<span class="dv">5</span>),<span class="fu">rnorm</span>(<span class="dv">5</span>,<span class="dv">8</span>))</span>
<span id="cb16-4"><a href="peaks-normalization.html#cb16-4" aria-hidden="true" tabindex="-1"></a>batch <span class="ot">&lt;-</span> <span class="fu">seq</span>(<span class="dv">5</span>,<span class="dv">0</span>,<span class="at">length.out =</span> <span class="dv">10</span>)</span>
<span id="cb16-5"><a href="peaks-normalization.html#cb16-5" aria-hidden="true" tabindex="-1"></a>ins <span class="ot">&lt;-</span> batch<span class="sc">+</span>con</span>
<span id="cb16-6"><a href="peaks-normalization.html#cb16-6" aria-hidden="true" tabindex="-1"></a>re <span class="ot">&lt;-</span> <span class="fu">t.test</span>(ins<span class="sc">~</span>group)</span></code></pre></div>
</div>
<div id="post-hoc-data-normalization" class="section level2 hasAnchor" number="9.5">
<h2><span class="header-section-number">9.5</span> <em>post hoc</em> data normalization<a href="peaks-normalization.html#post-hoc-data-normalization" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>To make the samples comparable, normalization across the samples are always needed when the experiment part is done. Batch effect should have patterns other than experimental design, otherwise just noise. Correction is possible by data analysis/randomized experimental design. There are numerous methods to make normalization with their combination. We could divided those methods into two categories: unsupervised and supervised.</p>
<p>Unsupervised methods only consider the normalization peaks intensity distribution across the samples. For example, quantile calibration try to make the intensity distribution among the samples similar. Such methods are preferred to explore the inner structures of the samples. Internal standards or pool QC samples also belong to this category. However, it’s hard to take a few peaks standing for all peaks extracted.</p>
<p>Supervised methods will use the group information or batch information in experimental design to normalize the data. A linear model is always used to model the unwanted variances and remove them for further analysis.</p>
<p>Since the real batch effects are always unknown, it’s hard to make validation for different normalization methods. Li et.al developed NOREVA to make comparision among 25 correction method <span class="citation">(<a href="#ref-li2017a" role="doc-biblioref">B. Li et al. 2017</a>)</span> and a recently updates make this numbers to 168 <span class="citation">(<a href="#ref-yang2020" role="doc-biblioref">Qingxia Yang et al. 2020</a>)</span>. MetaboDrift also contain some methods for batch correction in excel <span class="citation">(<a href="#ref-thonusin2017" role="doc-biblioref">Thonusin et al. 2017</a>)</span>. Another idea is use spiked-in samples to validate the methods <span class="citation">(<a href="#ref-franceschi2012" role="doc-biblioref">Franceschi et al. 2012</a>)</span> , which might be good for targeted analysis instead of non-targeted analysis.</p>
<p>Relative log abundance (RLA) plots<span class="citation">(<a href="#ref-delivera2012" role="doc-biblioref">De Livera et al. 2012</a>)</span> and heatmap often used to show the variances among the samples.</p>
<div id="unsupervised-methods" class="section level3 hasAnchor" number="9.5.1">
<h3><span class="header-section-number">9.5.1</span> Unsupervised methods<a href="peaks-normalization.html#unsupervised-methods" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="distribution-of-intensity" class="section level4 hasAnchor" number="9.5.1.1">
<h4><span class="header-section-number">9.5.1.1</span> Distribution of intensity<a href="peaks-normalization.html#distribution-of-intensity" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Intensity collects from LC/GC-MS always showed a right-skewed distribution. Log transformation is often necessary for further statistical analysis.</p>
</div>
<div id="centering" class="section level4 hasAnchor" number="9.5.1.2">
<h4><span class="header-section-number">9.5.1.2</span> Centering<a href="peaks-normalization.html#centering" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>For peak p of sample s in batch b, the corrected abundance I is:</p>
<p><span class="math display">\[\hat I_{p,s,b} = I_{p,s,b} - mean(I_{p,b}) + median(I_{p,qc})\]</span></p>
<p>If no quality control samples used, the corrected abundance I would be:</p>
<p><span class="math display">\[\hat I_{p,s,b} = I_{p,s,b} - mean(I_{p,b})\]</span></p>
</div>
<div id="scaling" class="section level4 hasAnchor" number="9.5.1.3">
<h4><span class="header-section-number">9.5.1.3</span> Scaling<a href="peaks-normalization.html#scaling" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>For peak p of sample s in certain batch b, the corrected abundance I is:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} - mean(I_{p,b})}{std_{p,b}} * std_{p,qc,b} + mean(I_{p,qc,b})\]</span> If no quality control samples used, the corrected abundance I would be:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} - mean(I_{p,b})}{std_{p,b}}\]</span></p>
</div>
<div id="pareto-scaling" class="section level4 hasAnchor" number="9.5.1.4">
<h4><span class="header-section-number">9.5.1.4</span> Pareto Scaling<a href="peaks-normalization.html#pareto-scaling" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>For peak p of sample s in certain batch b, the corrected abundance I is:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} - mean(I_{p,b})}{Sqrt(std_{p,b})} * Sqrt(std_{p,qc,b}) + mean(I_{p,qc,b})\]</span></p>
<p>If no quality control samples used, the corrected abundance I would be:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} - mean(I_{p,b})}{Sqrt(std_{p,b})}\]</span></p>
</div>
<div id="range-scaling" class="section level4 hasAnchor" number="9.5.1.5">
<h4><span class="header-section-number">9.5.1.5</span> Range Scaling<a href="peaks-normalization.html#range-scaling" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>For peak p of sample s in certain batch b, the corrected abundance I is:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} - mean(I_{p,b})}{max(I_{p,b}) - min(I_{p,b})} * (max(I_{p,qc,b}) - min(I_{p,qc,b})) + mean(I_{p,qc,b})\]</span></p>
<p>If no quality control samples used, the corrected abundance I would be:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} - mean(I_{p,b})}{max(I_{p,b}) - min(I_{p,b})} \]</span></p>
</div>
<div id="level-scaling" class="section level4 hasAnchor" number="9.5.1.6">
<h4><span class="header-section-number">9.5.1.6</span> Level scaling<a href="peaks-normalization.html#level-scaling" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>For peak p of sample s in certain batch b, the corrected abundance I is:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} - mean(I_{p,b})}{mean(I_{p,b})} * mean(I_{p,qc,b}) + mean(I_{p,qc,b})\]</span></p>
<p>If no quality control samples used, the corrected abundance I would be:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} - mean(I_{p,b})}{mean(I_{p,b})} \]</span></p>
</div>
<div id="quantile" class="section level4 hasAnchor" number="9.5.1.7">
<h4><span class="header-section-number">9.5.1.7</span> Quantile<a href="peaks-normalization.html#quantile" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>The idea of quantile calibration is that alignment of the intensities in certain samples according to quantile in each sample.</p>
<p>Here is the demo:</p>
<div class="sourceCode" id="cb17"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb17-1"><a href="peaks-normalization.html#cb17-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">42</span>)</span>
<span id="cb17-2"><a href="peaks-normalization.html#cb17-2" aria-hidden="true" tabindex="-1"></a>a <span class="ot">&lt;-</span> <span class="fu">rnorm</span>(<span class="dv">1000</span>)</span>
<span id="cb17-3"><a href="peaks-normalization.html#cb17-3" aria-hidden="true" tabindex="-1"></a><span class="co"># b sufferred batch effect with a bias of 10</span></span>
<span id="cb17-4"><a href="peaks-normalization.html#cb17-4" aria-hidden="true" tabindex="-1"></a>b <span class="ot">&lt;-</span> <span class="fu">rnorm</span>(<span class="dv">1000</span>,<span class="dv">10</span>)</span>
<span id="cb17-5"><a href="peaks-normalization.html#cb17-5" aria-hidden="true" tabindex="-1"></a><span class="fu">hist</span>(a,<span class="at">xlim=</span><span class="fu">c</span>(<span class="sc">-</span><span class="dv">5</span>,<span class="dv">15</span>),<span class="at">breaks =</span> <span class="dv">50</span>)</span>
<span id="cb17-6"><a href="peaks-normalization.html#cb17-6" aria-hidden="true" tabindex="-1"></a><span class="fu">hist</span>(b,<span class="at">col =</span> <span class="st">&#39;black&#39;</span>, <span class="at">breaks =</span> <span class="dv">50</span>, <span class="at">add=</span>T)</span>
<span id="cb17-7"><a href="peaks-normalization.html#cb17-7" aria-hidden="true" tabindex="-1"></a><span class="co"># quantile normalized</span></span>
<span id="cb17-8"><a href="peaks-normalization.html#cb17-8" aria-hidden="true" tabindex="-1"></a>cor <span class="ot">&lt;-</span> (a[<span class="fu">order</span>(a)]<span class="sc">+</span>b[<span class="fu">order</span>(b)])<span class="sc">/</span><span class="dv">2</span></span>
<span id="cb17-9"><a href="peaks-normalization.html#cb17-9" aria-hidden="true" tabindex="-1"></a><span class="co"># reorder</span></span>
<span id="cb17-10"><a href="peaks-normalization.html#cb17-10" aria-hidden="true" tabindex="-1"></a>cor <span class="ot">&lt;-</span> cor[<span class="fu">order</span>(<span class="fu">order</span>(a))]</span>
<span id="cb17-11"><a href="peaks-normalization.html#cb17-11" aria-hidden="true" tabindex="-1"></a><span class="fu">hist</span>(cor,<span class="at">col =</span> <span class="st">&#39;red&#39;</span>, <span class="at">breaks =</span> <span class="dv">50</span>, <span class="at">add=</span>T)</span></code></pre></div>
<p><img src="Metabolomics_files/figure-html/quantile-1.png" width="672" /></p>
</div>
<div id="ratio-based-calibration" class="section level4 hasAnchor" number="9.5.1.8">
<h4><span class="header-section-number">9.5.1.8</span> Ratio based calibration<a href="peaks-normalization.html#ratio-based-calibration" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>This method calibrates samples by the ratio between qc samples in all samples and in certain batch.For peak p of sample s in certain batch b, the corrected abundance I is:</p>
<p><span class="math display">\[\hat I_{p,s,b} = \frac{I_{p,s,b} * median(I_{p,qc})}{mean_{p,qc,b}}\]</span></p>
<div class="sourceCode" id="cb18"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb18-1"><a href="peaks-normalization.html#cb18-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">42</span>)</span>
<span id="cb18-2"><a href="peaks-normalization.html#cb18-2" aria-hidden="true" tabindex="-1"></a><span class="co"># raw data</span></span>
<span id="cb18-3"><a href="peaks-normalization.html#cb18-3" aria-hidden="true" tabindex="-1"></a>I <span class="ot">=</span> <span class="fu">c</span>(<span class="fu">rnorm</span>(<span class="dv">10</span>,<span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="fl">0.3</span>),<span class="fu">rnorm</span>(<span class="dv">10</span>,<span class="at">mean =</span> <span class="dv">1</span>, <span class="at">sd =</span> <span class="fl">0.5</span>))</span>
<span id="cb18-4"><a href="peaks-normalization.html#cb18-4" aria-hidden="true" tabindex="-1"></a><span class="co"># batch</span></span>
<span id="cb18-5"><a href="peaks-normalization.html#cb18-5" aria-hidden="true" tabindex="-1"></a>B <span class="ot">=</span> <span class="fu">c</span>(<span class="fu">rep</span>(<span class="dv">0</span>,<span class="dv">10</span>),<span class="fu">rep</span>(<span class="dv">1</span>,<span class="dv">10</span>))</span>
<span id="cb18-6"><a href="peaks-normalization.html#cb18-6" aria-hidden="true" tabindex="-1"></a><span class="co"># qc</span></span>
<span id="cb18-7"><a href="peaks-normalization.html#cb18-7" aria-hidden="true" tabindex="-1"></a>Iqc <span class="ot">=</span> <span class="fu">c</span>(<span class="fu">rnorm</span>(<span class="dv">1</span>,<span class="at">mean =</span> <span class="dv">0</span>, <span class="at">sd =</span> <span class="fl">0.3</span>),<span class="fu">rnorm</span>(<span class="dv">1</span>,<span class="at">mean =</span> <span class="dv">1</span>, <span class="at">sd =</span> <span class="fl">0.5</span>))</span>
<span id="cb18-8"><a href="peaks-normalization.html#cb18-8" aria-hidden="true" tabindex="-1"></a><span class="co"># corrected data</span></span>
<span id="cb18-9"><a href="peaks-normalization.html#cb18-9" aria-hidden="true" tabindex="-1"></a>Icor <span class="ot">=</span> I <span class="sc">*</span> <span class="fu">median</span>(<span class="fu">c</span>(<span class="fu">rep</span>(Iqc[<span class="dv">1</span>],<span class="dv">10</span>),<span class="fu">rep</span>(Iqc[<span class="dv">2</span>],<span class="dv">10</span>)))<span class="sc">/</span><span class="fu">mean</span>(<span class="fu">c</span>(<span class="fu">rep</span>(Iqc[<span class="dv">1</span>],<span class="dv">10</span>),<span class="fu">rep</span>(Iqc[<span class="dv">2</span>],<span class="dv">10</span>)))</span>
<span id="cb18-10"><a href="peaks-normalization.html#cb18-10" aria-hidden="true" tabindex="-1"></a><span class="co"># plot the result</span></span>
<span id="cb18-11"><a href="peaks-normalization.html#cb18-11" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(I)</span></code></pre></div>
<p><img src="Metabolomics_files/figure-html/ratio-1.png" width="672" /></p>
<div class="sourceCode" id="cb19"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb19-1"><a href="peaks-normalization.html#cb19-1" aria-hidden="true" tabindex="-1"></a><span class="fu">plot</span>(Icor)</span></code></pre></div>
<p><img src="Metabolomics_files/figure-html/ratio-2.png" width="672" /></p>
</div>
<div id="linear-normalizer" class="section level4 hasAnchor" number="9.5.1.9">
<h4><span class="header-section-number">9.5.1.9</span> Linear Normalizer<a href="peaks-normalization.html#linear-normalizer" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>This method initially scales each sample so that the sum of all peak abundances equals one. In this study, by multiplying the median sum of all peak abundances across all samples,we got the corrected data.</p>
<div class="sourceCode" id="cb20"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb20-1"><a href="peaks-normalization.html#cb20-1" aria-hidden="true" tabindex="-1"></a><span class="fu">set.seed</span>(<span class="dv">42</span>)</span>
<span id="cb20-2"><a href="peaks-normalization.html#cb20-2" aria-hidden="true" tabindex="-1"></a><span class="co"># raw data</span></span>
<span id="cb20-3"><a href="peaks-normalization.html#cb20-3" aria-hidden="true" tabindex="-1"></a>peaksa <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">rnorm</span>(<span class="dv">10</span>,<span class="at">mean =</span> <span class="dv">10</span>, <span class="at">sd =</span> <span class="fl">0.3</span>),<span class="fu">rnorm</span>(<span class="dv">10</span>,<span class="at">mean =</span> <span class="dv">20</span>, <span class="at">sd =</span> <span class="fl">0.5</span>))</span>
<span id="cb20-4"><a href="peaks-normalization.html#cb20-4" aria-hidden="true" tabindex="-1"></a>peaksb <span class="ot">&lt;-</span> <span class="fu">c</span>(<span class="fu">rnorm</span>(<span class="dv">10</span>,<span class="at">mean =</span> <span class="dv">10</span>, <span class="at">sd =</span> <span class="fl">0.3</span>),<span class="fu">rnorm</span>(<span class="dv">10</span>,<span class="at">mean =</span> <span class="dv">20</span>, <span class="at">sd =</span> <span class="fl">0.5</span>))</span>
<span id="cb20-5"><a href="peaks-normalization.html#cb20-5" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-6"><a href="peaks-normalization.html#cb20-6" aria-hidden="true" tabindex="-1"></a>df <span class="ot">&lt;-</span> <span class="fu">rbind</span>(peaksa,peaksb)</span>
<span id="cb20-7"><a href="peaks-normalization.html#cb20-7" aria-hidden="true" tabindex="-1"></a>dfcor <span class="ot">&lt;-</span> df<span class="sc">/</span><span class="fu">apply</span>(df,<span class="dv">2</span>,sum)<span class="sc">*</span> <span class="fu">sum</span>(<span class="fu">apply</span>(df,<span class="dv">2</span>,median))</span>
<span id="cb20-8"><a href="peaks-normalization.html#cb20-8" aria-hidden="true" tabindex="-1"></a></span>
<span id="cb20-9"><a href="peaks-normalization.html#cb20-9" aria-hidden="true" tabindex="-1"></a><span class="fu">image</span>(df)</span></code></pre></div>
<p><img src="Metabolomics_files/figure-html/Linear-1.png" width="672" /></p>
<div class="sourceCode" id="cb21"><pre class="sourceCode r"><code class="sourceCode r"><span id="cb21-1"><a href="peaks-normalization.html#cb21-1" aria-hidden="true" tabindex="-1"></a><span class="fu">image</span>(dfcor)</span></code></pre></div>
<p><img src="Metabolomics_files/figure-html/Linear-2.png" width="672" /></p>
</div>
<div id="internal-standards" class="section level4 hasAnchor" number="9.5.1.10">
<h4><span class="header-section-number">9.5.1.10</span> Internal standards<a href="peaks-normalization.html#internal-standards" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p><span class="math display">\[\hat I_{p,s} = \frac{I_{p,s} * median(I_{IS})}{I_{IS,s}}\]</span></p>
<p>Some methods also use pooled calibration samples and multiple internal standard strategy to correct the data <span class="citation">(<a href="#ref-vanderkloet2009" role="doc-biblioref">van der Kloet et al. 2009</a>; <a href="#ref-sysi-aho2007" role="doc-biblioref">Sysi-Aho et al. 2007</a>)</span>. Also some methods only use QC samples to handle the data <span class="citation">(<a href="#ref-kuligowski2015" role="doc-biblioref">Kuligowski et al. 2015</a>)</span>.</p>
</div>
</div>
<div id="supervised-methods" class="section level3 hasAnchor" number="9.5.2">
<h3><span class="header-section-number">9.5.2</span> Supervised methods<a href="peaks-normalization.html#supervised-methods" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="regression-calibration" class="section level4 hasAnchor" number="9.5.2.1">
<h4><span class="header-section-number">9.5.2.1</span> Regression calibration<a href="peaks-normalization.html#regression-calibration" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Considering the batch effect of injection order, regress the data by a linear model to get the calibration.</p>
</div>
<div id="batch-normalizer" class="section level4 hasAnchor" number="9.5.2.2">
<h4><span class="header-section-number">9.5.2.2</span> Batch Normalizer<a href="peaks-normalization.html#batch-normalizer" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>Use the total abundance scale and then fit with the regression line <span class="citation">(<a href="#ref-wang2013" role="doc-biblioref">S.-Y. Wang, Kuo, and Tseng 2013</a>)</span>.</p>
</div>
<div id="surrogate-variable-analysissva" class="section level4 hasAnchor" number="9.5.2.3">
<h4><span class="header-section-number">9.5.2.3</span> Surrogate Variable Analysis(SVA)<a href="peaks-normalization.html#surrogate-variable-analysissva" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>We have a data matrix(M*N) with M stands for identity peaks from one sample and N stand for individual samples. For one sample, <span class="math inline">\(X = (x_{i1},...,x_{in})^T\)</span> stands for the normalized intensities of peaks. We use <span class="math inline">\(Y = (y_i,...,y_m)^T\)</span> stands for the group information of our data. Then we could build such models:</p>
<p><span class="math display">\[x_{ij} = \mu_i + f_i(y_i) + e_{ij}\]</span></p>
<p><span class="math inline">\(\mu_i\)</span> stands for the baseline of the peak intensities in a normal state. Then we have:</p>
<p><span class="math display">\[f_i(y_i) = E(x_{ij}|y_j) - \mu_i\]</span></p>
<p>stands for the biological variations caused by the our group, for example, whether treated by exposure or not.</p>
<p>However, considering the batch effects, the real model could be:</p>
<p><span class="math display">\[x_{ij} = \mu_i + f_i(y_i) + \sum_{l = 1}^L \gamma_{li}p_{lj} + e_{ij}^*\]</span> <span class="math inline">\(\gamma_{li}\)</span> stands for the peak-specific coefficient for potential factor <span class="math inline">\(l\)</span>. <span class="math inline">\(p_{lj}\)</span> stands for the potential factors across the samples. Actually, the error item <span class="math inline">\(e_{ij}\)</span> in real sample could always be decomposed as <span class="math inline">\(e_{ij} = \sum_{l = 1}^L \gamma_{li}p_{lj} + e_{ij}^*\)</span> with <span class="math inline">\(e_{ij}^*\)</span> standing for the real random error in certain sample for certain peak.</p>
<p>We could not get the potential factors directly. Since we don’t care the details of the unknown factors, we could estimate orthogonal vectors <span class="math inline">\(h_k\)</span> standing for such potential factors. Thus we have:</p>
<p><span class="math display">\[
x_{ij} = \mu_i + f_i(y_i) + \sum_{l = 1}^L \gamma_{li}p_{lj} + e_{ij}^*\\
= \mu_i + f_i(y_i) + \sum_{k = 1}^K \lambda_{ki}h_{kj} + e_{ij}
\]</span></p>
<p>Here is the details of the algorithm:</p>
<blockquote>
<p>The algorithm is decomposed into two parts: detection of unmodeled factors and construction of surrogate variables</p>
</blockquote>
<div id="detection-of-unmodeled-factors" class="section level5 hasAnchor" number="9.5.2.3.1">
<h5><span class="header-section-number">9.5.2.3.1</span> Detection of unmodeled factors<a href="peaks-normalization.html#detection-of-unmodeled-factors" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<ul>
<li><p>Estimate <span class="math inline">\(\hat\mu_i\)</span> and <span class="math inline">\(f_i\)</span> by fitting the model <span class="math inline">\(x_{ij} = \mu_i + f_i(y_i) + e_{ij}\)</span> and get the residual <span class="math inline">\(r_{ij} = x_{ij}-\hat\mu_i - \hat f_i(y_i)\)</span>. Then we have the residual matrix R.</p></li>
<li><p>Perform the singular value decompositon(SVD) of the residual matrix <span class="math inline">\(R = UDV^T\)</span></p></li>
<li><p>Let <span class="math inline">\(d_l\)</span> be the <span class="math inline">\(l\)</span>th eigenvalue of the diagonal matrix D for <span class="math inline">\(l = 1,...,n\)</span>. Set <span class="math inline">\(df\)</span> as the freedom of the model <span class="math inline">\(\hat\mu_i + \hat f_i(y_i)\)</span>. We could build a statistic <span class="math inline">\(T_k\)</span> as:</p></li>
</ul>
<p><span class="math display">\[T_k = \frac{d_k^2}{\sum_{l=1}^{n-df}d_l^2}\]</span></p>
<p>to show the variance explained by the <span class="math inline">\(k\)</span>th eigenvalue.</p>
<ul>
<li><p>Permute each row of R to remove the structure in the matrix and get <span class="math inline">\(R^*\)</span>.</p></li>
<li><p>Fit the model <span class="math inline">\(r_{ij}^* = \mu_i^* + f_i^*(y_i) + e^*_{ij}\)</span> and get <span class="math inline">\(r_{ij}^0 = r^*_{ij}-\hat\mu^*_i - \hat f^*_i(y_i)\)</span> as a null matrix <span class="math inline">\(R_0\)</span></p></li>
<li><p>Perform the singular value decompositon(SVD) of the residual matrix <span class="math inline">\(R_0 = U_0D_0V_0^T\)</span></p></li>
<li><p>Compute the null statistic:</p></li>
</ul>
<p><span class="math display">\[
T_k^0 = \frac{d_{0k}^2}{\sum_{l=1}^{n-df}d_{0l}^2}
\]</span></p>
<ul>
<li><p>Repeat permuting the row B times to get the null statistics <span class="math inline">\(T_k^{0b}\)</span></p></li>
<li><p>Get the p-value for eigengene:</p></li>
</ul>
<p><span class="math display">\[p_k = \frac{\#{T_k^{0b}\geq T_k;b=1,...,B }}{B}\]</span></p>
<ul>
<li>For a significance level <span class="math inline">\(\alpha\)</span>, treat k as a significant signature of residual R if <span class="math inline">\(p_k\leq\alpha\)</span></li>
</ul>
</div>
<div id="construction-of-surrogate-variables" class="section level5 hasAnchor" number="9.5.2.3.2">
<h5><span class="header-section-number">9.5.2.3.2</span> Construction of surrogate variables<a href="peaks-normalization.html#construction-of-surrogate-variables" class="anchor-section" aria-label="Anchor link to header"></a></h5>
<ul>
<li><p>Estimate <span class="math inline">\(\hat\mu_i\)</span> and <span class="math inline">\(f_i\)</span> by fitting the model <span class="math inline">\(x_{ij} = \mu_i + f_i(y_i) + e_{ij}\)</span> and get the residual <span class="math inline">\(r_{ij} = x_{ij}-\hat\mu_i - \hat f_i(y_i)\)</span>. Then we have the residual matrix R.</p></li>
<li><p>Perform the singular value decompositon(SVD) of the residual matrix <span class="math inline">\(R = UDV^T\)</span>. Let <span class="math inline">\(e_k = (e_{k1},...,e_{kn})^T\)</span> be the <span class="math inline">\(k\)</span>th column of V</p></li>
<li><p>Set <span class="math inline">\(\hat K\)</span> as the significant eigenvalues found by the first step.</p></li>
<li><p>Regress each <span class="math inline">\(e_k\)</span> on <span class="math inline">\(x_i\)</span>, get the p-value for the association.</p></li>
<li><p>Set <span class="math inline">\(\pi_0\)</span> as the proportion of the peak intensity <span class="math inline">\(x_i\)</span> not associate with <span class="math inline">\(e_k\)</span> and find the numbers <span class="math inline">\(\hat m =[1-\hat \pi_0 \times m]\)</span> and the index of the peaks associated with the eigenvalues</p></li>
<li><p>Form the matrix <span class="math inline">\(\hat m_1 \times N\)</span>, this matrix<span class="math inline">\(X_r\)</span> stand for the potential variables. As was done for R, get the eigengents of <span class="math inline">\(X_r\)</span> and denote these by <span class="math inline">\(e_j^r\)</span></p></li>
<li><p>Let <span class="math inline">\(j^* = argmax_{1\leq j \leq n}cor(e_k,e_j^r)\)</span> and set <span class="math inline">\(\hat h_k=e_j^r\)</span>. Set the estimate of the surrogate variable to be the eigenvalue of the reduced matrix most correlated with the corresponding residual eigenvalue. Since the reduced matrix is enriched for peaks associated with this residual eigenvalue, this is a principled choice for the estimated surrogate variable that allows for correlation with the primary variable.</p></li>
<li><p>Employ the <span class="math inline">\(\mu_i + f_i(y_i) + \sum_{k = 1}^K \gamma_{ki}\hat h_{kj} + e_{ij}\)</span> as the estimate of the ideal model <span class="math inline">\(\mu_i + f_i(y_i) + \sum_{k = 1}^K \gamma_{ki}h_{kj} + e_{ij}\)</span></p></li>
</ul>
<p>This method could found the potential unwanted variables for the data. SVA were introduced by Jeff Leek <span class="citation">(<a href="#ref-leek2008" role="doc-biblioref">Leek and Storey 2008</a>, <a href="#ref-leek2007" role="doc-biblioref">2007</a>; <a href="#ref-leek2012" role="doc-biblioref">Leek et al. 2012</a>)</span> and EigenMS package implement SVA with modifications including analysis of data with missing values that are typical in LC-MS experiments <span class="citation">(<a href="#ref-karpievitch2014a" role="doc-biblioref">Karpievitch et al. 2014</a>)</span>.</p>
</div>
</div>
<div id="ruv-remove-unwanted-variation" class="section level4 hasAnchor" number="9.5.2.4">
<h4><span class="header-section-number">9.5.2.4</span> RUV (Remove Unwanted Variation)<a href="peaks-normalization.html#ruv-remove-unwanted-variation" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>This method’s performance is similar to SVA. Instead find surrogate variable from the whole dataset. RUA use control or pool QC to find the unwanted variances and remove them to find the peaks related to experimental design. However, we could also empirically estimate the control peaks by linear mixed model. RUA-random <span class="citation">(<a href="#ref-livera2015" role="doc-biblioref">Livera et al. 2015</a>; <a href="#ref-delivera2012" role="doc-biblioref">De Livera et al. 2012</a>)</span> further use linear mixed model to estimate the variances of random error. A hierarchical approach RUV was recently proposed for metabolomics data<span class="citation">(<a href="#ref-kim2021" role="doc-biblioref">T. Kim et al. 2021</a>)</span>. This method could be used with suitable control, which is common in metabolomics DoE.</p>
</div>
<div id="rrmix" class="section level4 hasAnchor" number="9.5.2.5">
<h4><span class="header-section-number">9.5.2.5</span> RRmix<a href="peaks-normalization.html#rrmix" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>RRmix also use a latent factor models correct the data <span class="citation">(<a href="#ref-jr2017" role="doc-biblioref">Jr et al. 2017</a>)</span>. This method could be treated as linear mixed model version SVA. No control samples are required and the unwanted variances could be removed by factor analysis. This method might be the best choice to remove the unwanted variables with common experiment design.</p>
</div>
<div id="norm-iswsvr" class="section level4 hasAnchor" number="9.5.2.6">
<h4><span class="header-section-number">9.5.2.6</span> Norm ISWSVR<a href="peaks-normalization.html#norm-iswsvr" class="anchor-section" aria-label="Anchor link to header"></a></h4>
<p>It is a two-step approach via combining the best-performance internal standard correction with support vector regression normalization, comprehensively removing the systematic and random errors and matrix effects<span class="citation">(<a href="#ref-ding2022a" role="doc-biblioref">Ding et al. 2022</a>)</span>.</p>
</div>
</div>
</div>
<div id="method-to-validate-the-normalization" class="section level2 hasAnchor" number="9.6">
<h2><span class="header-section-number">9.6</span> Method to validate the normalization<a href="peaks-normalization.html#method-to-validate-the-normalization" class="anchor-section" aria-label="Anchor link to header"></a></h2>
</div>
<div id="software" class="section level2 hasAnchor" number="9.7">
<h2><span class="header-section-number">9.7</span> Software<a href="peaks-normalization.html#software" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<ul>
<li><p><a href="https://github.com/rwehrens/BatchCorrMetabolomics">BatchCorrMetabolomics</a> is for improved batch correction in untargeted MS-based metabolomics</p></li>
<li><p><a href="https://github.com/cran/MetNorm">MetNorm</a> show Statistical Methods for Normalizing Metabolomics Data.</p></li>
<li><p><a href="https://github.com/mani2012/BatchQC">BatchQC</a> could be used to make batch effect simulation.</p></li>
<li><p><a href="http://idrb.zju.edu.cn/noreva/">Noreva</a> could make online batch correction and comparison<span class="citation">(<a href="#ref-fu2021" role="doc-biblioref">J. Fu et al. 2021</a>)</span>.</p></li>
</ul>

</div>
</div>
<h3>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-delivera2012" class="csl-entry">
De Livera, Alysha M., Daniel A. Dias, David De Souza, Thusitha Rupasinghe, James Pyke, Dedreia Tull, Ute Roessner, Malcolm McConville, and Terence P. Speed. 2012. <span>“Normalizing and <span>Integrating Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 84 (24): 10768–76. <a href="https://doi.org/10.1021/ac302748b">https://doi.org/10.1021/ac302748b</a>.
</div>
<div id="ref-ding2022a" class="csl-entry">
Ding, Xian, Fen Yang, Yanhua Chen, Jing Xu, Jiuming He, Ruiping Zhang, and Zeper Abliz. 2022. <span>“Norm <span>ISWSVR</span>: <span>A Data Integration</span> and <span>Normalization Approach</span> for <span>Large-Scale Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 94 (21): 7500–7509. <a href="https://doi.org/10.1021/acs.analchem.1c05502">https://doi.org/10.1021/acs.analchem.1c05502</a>.
</div>
<div id="ref-franceschi2012" class="csl-entry">
Franceschi, Pietro, Domenico Masuero, Urska Vrhovsek, Fulvio Mattivi, and Ron Wehrens. 2012. <span>“A Benchmark Spike-in Data Set for Biomarker Identification in Metabolomics.”</span> <em>Journal of Chemometrics</em> 26 (1-2): 16–24. <a href="https://doi.org/10.1002/cem.1420">https://doi.org/10.1002/cem.1420</a>.
</div>
<div id="ref-fu2021" class="csl-entry">
Fu, Jianbo, Ying Zhang, Yunxia Wang, Hongning Zhang, Jin Liu, Jing Tang, Qingxia Yang, et al. 2021. <span>“Optimization of Metabolomic Data Processing Using <span>NOREVA</span>.”</span> <em>Nature Protocols</em>, December, 1–23. <a href="https://doi.org/10.1038/s41596-021-00636-9">https://doi.org/10.1038/s41596-021-00636-9</a>.
</div>
<div id="ref-jr2017" class="csl-entry">
Jr, Stephen Salerno, Mahya Mehrmohamadi, Maria V. Liberti, Muting Wan, Martin T. Wells, James G. Booth, and Jason W. Locasale. 2017. <span>“<span>RRmix</span>: <span>A</span> Method for Simultaneous Batch Effect Correction and Analysis of Metabolomics Data in the Absence of Internal Standards.”</span> <em>PLOS ONE</em> 12 (6): e0179530. <a href="https://doi.org/10.1371/journal.pone.0179530">https://doi.org/10.1371/journal.pone.0179530</a>.
</div>
<div id="ref-karpievitch2014a" class="csl-entry">
Karpievitch, Yuliya V., Sonja B. Nikolic, Richard Wilson, James E. Sharman, and Lindsay M. Edwards. 2014. <span>“Metabolomics <span>Data Normalization</span> with <span>EigenMS</span>.”</span> <em>PLOS ONE</em> 9 (12): e116221. <a href="https://doi.org/10.1371/journal.pone.0116221">https://doi.org/10.1371/journal.pone.0116221</a>.
</div>
<div id="ref-kim2021" class="csl-entry">
Kim, Taiyun, Owen Tang, Stephen T. Vernon, Katharine A. Kott, Yen Chin Koay, John Park, David E. James, et al. 2021. <span>“A Hierarchical Approach to Removal of Unwanted Variation for Large-Scale Metabolomics Data.”</span> <em>Nature Communications</em> 12 (1): 4992. <a href="https://doi.org/10.1038/s41467-021-25210-5">https://doi.org/10.1038/s41467-021-25210-5</a>.
</div>
<div id="ref-kuligowski2015" class="csl-entry">
Kuligowski, Julia, Ángel Sánchez-Illana, Daniel Sanjuán-Herráez, Máximo Vento, and Guillermo Quintás. 2015. <span>“Intra-Batch Effect Correction in Liquid Chromatography-Mass Spectrometry Using Quality Control Samples and Support Vector Regression (<span>QC-SVRC</span>).”</span> <em>Analyst</em> 140 (22): 7810–17. <a href="https://doi.org/10.1039/C5AN01638J">https://doi.org/10.1039/C5AN01638J</a>.
</div>
<div id="ref-leek2012" class="csl-entry">
Leek, Jeffrey T., W. Evan Johnson, Hilary S. Parker, Andrew E. Jaffe, and John D. Storey. 2012. <span>“The Sva Package for Removing Batch Effects and Other Unwanted Variation in High-Throughput Experiments.”</span> <em>Bioinformatics</em> 28 (6): 882–83. <a href="https://doi.org/10.1093/bioinformatics/bts034">https://doi.org/10.1093/bioinformatics/bts034</a>.
</div>
<div id="ref-leek2007" class="csl-entry">
Leek, Jeffrey T., and John D. Storey. 2007. <span>“Capturing <span>Heterogeneity</span> in <span>Gene Expression Studies</span> by <span>Surrogate Variable Analysis</span>.”</span> <em>PLOS Genet</em> 3 (9): e161. <a href="https://doi.org/10.1371/journal.pgen.0030161">https://doi.org/10.1371/journal.pgen.0030161</a>.
</div>
<div id="ref-leek2008" class="csl-entry">
———. 2008. <span>“A General Framework for Multiple Testing Dependence.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (48): 18718–23. <a href="https://doi.org/10.1073/pnas.0808709105">https://doi.org/10.1073/pnas.0808709105</a>.
</div>
<div id="ref-li2017a" class="csl-entry">
Li, Bo, Jing Tang, Qingxia Yang, Shuang Li, Xuejiao Cui, Yinghong Li, Yuzong Chen, Weiwei Xue, Xiaofeng Li, and Feng Zhu. 2017. <span>“<span>NOREVA</span>: Normalization and Evaluation of <span class="nocase">MS-based</span> Metabolomics Data.”</span> <em>Nucleic Acids Research</em> 45 (W1): W162–70. <a href="https://doi.org/10.1093/nar/gkx449">https://doi.org/10.1093/nar/gkx449</a>.
</div>
<div id="ref-livera2015" class="csl-entry">
Livera, Alysha M. De, Marko Sysi-Aho, Laurent Jacob, Johann A. Gagnon-Bartsch, Sandra Castillo, Julie A. Simpson, and Terence P. Speed. 2015. <span>“Statistical <span>Methods</span> for <span>Handling Unwanted Variation</span> in <span>Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 87 (7): 3606–15. <a href="https://doi.org/10.1021/ac502439y">https://doi.org/10.1021/ac502439y</a>.
</div>
<div id="ref-sysi-aho2007" class="csl-entry">
Sysi-Aho, Marko, Mikko Katajamaa, Laxman Yetukuri, and Matej Orešič. 2007. <span>“Normalization Method for Metabolomics Data Using Optimal Selection of Multiple Internal Standards.”</span> <em>BMC Bioinformatics</em> 8 (March): 93. <a href="https://doi.org/10.1186/1471-2105-8-93">https://doi.org/10.1186/1471-2105-8-93</a>.
</div>
<div id="ref-thonusin2017" class="csl-entry">
Thonusin, Chanisa, Heidi B. IglayReger, Tanu Soni, Amy E. Rothberg, Charles F. Burant, and Charles R. Evans. 2017. <span>“Evaluation of Intensity Drift Correction Strategies Using <span>MetaboDrift</span>, a Normalization Tool for Multi-Batch Metabolomics Data.”</span> <em>Journal of Chromatography A</em>, Pushing the <span>Boundaries</span> of <span>Chromatography</span> and <span>Electrophoresis</span>, 1523 (Supplement C): 265–74. <a href="https://doi.org/10.1016/j.chroma.2017.09.023">https://doi.org/10.1016/j.chroma.2017.09.023</a>.
</div>
<div id="ref-vanderkloet2009" class="csl-entry">
van der Kloet, Frans M., Ivana Bobeldijk, Elwin R. Verheij, and Renger H. Jellema. 2009. <span>“Analytical <span>Error Reduction Using Single Point Calibration</span> for <span>Accurate</span> and <span>Precise Metabolomic Phenotyping</span>.”</span> <em>Journal of Proteome Research</em> 8 (11): 5132–41. <a href="https://doi.org/10.1021/pr900499r">https://doi.org/10.1021/pr900499r</a>.
</div>
<div id="ref-wang2013" class="csl-entry">
Wang, San-Yuan, Ching-Hua Kuo, and Yufeng J. Tseng. 2013. <span>“Batch <span>Normalizer</span>: <span>A Fast Total Abundance Regression Calibration Method</span> to <span>Simultaneously Adjust Batch</span> and <span>Injection Order Effects</span> in <span>Liquid Chromatography</span>/<span class="nocase">Time-of-Flight Mass Spectrometry-Based Metabolomics Data</span> and <span>Comparison</span> with <span>Current Calibration Methods</span>.”</span> <em>Analytical Chemistry</em> 85 (2): 1037–46. <a href="https://doi.org/10.1021/ac302877x">https://doi.org/10.1021/ac302877x</a>.
</div>
<div id="ref-yang2020" class="csl-entry">
Yang, Qingxia, Yunxia Wang, Ying Zhang, Fengcheng Li, Weiqi Xia, Ying Zhou, Yunqing Qiu, Honglin Li, and Feng Zhu. 2020. <span>“<span>NOREVA</span>: Enhanced Normalization and Evaluation of Time-Course and Multi-Class Metabolomic Data.”</span> <em>Nucleic Acids Research</em> 48 (W1): W436–48. <a href="https://doi.org/10.1093/nar/gkaa258">https://doi.org/10.1093/nar/gkaa258</a>.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="omics-analysis.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="statistical-analysis.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/yufree/metaworkflow/edit/master/09-normalization.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Metabolomics.pub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
