<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Meta-Workflow</title>
  <meta name="description" content="This is a workflow for metabolomics studies." />
  <meta name="generator" content="bookdown 0.21 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Meta-Workflow" />
  <meta property="og:type" content="book" />
  
  <meta property="og:image" content="images/cover.png" />
  <meta property="og:description" content="This is a workflow for metabolomics studies." />
  <meta name="github-repo" content="yufree/metaworkflow" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Meta-Workflow" />
  
  <meta name="twitter:description" content="This is a workflow for metabolomics studies." />
  <meta name="twitter:image" content="images/cover.png" />

<meta name="author" content="Miao YU" />


<meta name="date" content="2021-01-05" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="exposome.html"/>

<script src="libs/header-attrs-2.6/header-attrs.js"></script>
<script src="libs/jquery-2.2.3/jquery.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<script src="libs/htmlwidgets-1.5.3/htmlwidgets.js"></script>
<script src="libs/viz-1.8.2/viz.js"></script>
<link href="libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="libs/grViz-binding-1.0.6.1/grViz.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metabolomics Workflow</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#history"><i class="fa fa-check"></i><b>1.1</b> History</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#history-of-mass-spectrometry"><i class="fa fa-check"></i><b>1.1.1</b> History of Mass Spectrometry</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#history-of-metabolomcis"><i class="fa fa-check"></i><b>1.1.2</b> History of Metabolomcis</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#defination"><i class="fa fa-check"></i><b>1.1.3</b> Defination</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#reviews-and-tutorials"><i class="fa fa-check"></i><b>1.2</b> Reviews and tutorials</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#workflow"><i class="fa fa-check"></i><b>1.2.1</b> Workflow</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#data-analysis"><i class="fa fa-check"></i><b>1.2.2</b> Data analysis</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#application"><i class="fa fa-check"></i><b>1.2.3</b> Application</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#challenge"><i class="fa fa-check"></i><b>1.2.4</b> Challenge</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#trends-in-metabolomics"><i class="fa fa-check"></i><b>1.3</b> Trends in Metabolomics</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#workflow-1"><i class="fa fa-check"></i><b>1.4</b> Workflow</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html"><i class="fa fa-check"></i><b>2</b> Experimental design(DoE)</a>
<ul>
<li class="chapter" data-level="2.1" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#homogeneity-study"><i class="fa fa-check"></i><b>2.1</b> Homogeneity study</a></li>
<li class="chapter" data-level="2.2" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#heterogeneity-study"><i class="fa fa-check"></i><b>2.2</b> Heterogeneity study</a></li>
<li class="chapter" data-level="2.3" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#power-analysis"><i class="fa fa-check"></i><b>2.3</b> Power analysis</a></li>
<li class="chapter" data-level="2.4" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#optimization"><i class="fa fa-check"></i><b>2.4</b> Optimization</a></li>
<li class="chapter" data-level="2.5" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#pooled-qc"><i class="fa fa-check"></i><b>2.5</b> Pooled QC</a></li>
<li class="chapter" data-level="2.6" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#software"><i class="fa fa-check"></i><b>2.6</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="pretreatment.html"><a href="pretreatment.html"><i class="fa fa-check"></i><b>3</b> Pretreatment</a>
<ul>
<li class="chapter" data-level="3.1" data-path="pretreatment.html"><a href="pretreatment.html#collection"><i class="fa fa-check"></i><b>3.1</b> Collection</a></li>
<li class="chapter" data-level="3.2" data-path="pretreatment.html"><a href="pretreatment.html#quenching"><i class="fa fa-check"></i><b>3.2</b> Quenching</a></li>
<li class="chapter" data-level="3.3" data-path="pretreatment.html"><a href="pretreatment.html#extraction"><i class="fa fa-check"></i><b>3.3</b> Extraction</a></li>
<li class="chapter" data-level="3.4" data-path="pretreatment.html"><a href="pretreatment.html#derivatization"><i class="fa fa-check"></i><b>3.4</b> Derivatization</a></li>
<li class="chapter" data-level="3.5" data-path="pretreatment.html"><a href="pretreatment.html#storage"><i class="fa fa-check"></i><b>3.5</b> Storage</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html"><i class="fa fa-check"></i><b>4</b> Instrumental analysis</a>
<ul>
<li class="chapter" data-level="4.1" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#column-and-gradient-selection"><i class="fa fa-check"></i><b>4.1</b> Column and gradient selection</a></li>
<li class="chapter" data-level="4.2" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#mass-resolution"><i class="fa fa-check"></i><b>4.2</b> Mass resolution</a></li>
<li class="chapter" data-level="4.3" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#matrix-effects"><i class="fa fa-check"></i><b>4.3</b> Matrix effects</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="workflow-2.html"><a href="workflow-2.html"><i class="fa fa-check"></i><b>5</b> Workflow</a>
<ul>
<li class="chapter" data-level="5.1" data-path="workflow-2.html"><a href="workflow-2.html#platform-for-metabolomics-data-analysis"><i class="fa fa-check"></i><b>5.1</b> Platform for metabolomics data analysis</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="workflow-2.html"><a href="workflow-2.html#xcms-xcms-online"><i class="fa fa-check"></i><b>5.1.1</b> XCMS &amp; XCMS online</a></li>
<li class="chapter" data-level="5.1.2" data-path="workflow-2.html"><a href="workflow-2.html#prime"><i class="fa fa-check"></i><b>5.1.2</b> PRIMe</a></li>
<li class="chapter" data-level="5.1.3" data-path="workflow-2.html"><a href="workflow-2.html#gnps"><i class="fa fa-check"></i><b>5.1.3</b> GNPS</a></li>
<li class="chapter" data-level="5.1.4" data-path="workflow-2.html"><a href="workflow-2.html#openms-sirius"><i class="fa fa-check"></i><b>5.1.4</b> OpenMS &amp; SIRIUS</a></li>
<li class="chapter" data-level="5.1.5" data-path="workflow-2.html"><a href="workflow-2.html#mzmine-2"><i class="fa fa-check"></i><b>5.1.5</b> MZmine 2</a></li>
<li class="chapter" data-level="5.1.6" data-path="workflow-2.html"><a href="workflow-2.html#emory-mahpic"><i class="fa fa-check"></i><b>5.1.6</b> Emory MaHPIC</a></li>
<li class="chapter" data-level="5.1.7" data-path="workflow-2.html"><a href="workflow-2.html#others"><i class="fa fa-check"></i><b>5.1.7</b> Others</a></li>
<li class="chapter" data-level="5.1.8" data-path="workflow-2.html"><a href="workflow-2.html#workflow-comparison"><i class="fa fa-check"></i><b>5.1.8</b> Workflow Comparison</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="workflow-2.html"><a href="workflow-2.html#project-setup"><i class="fa fa-check"></i><b>5.2</b> Project Setup</a></li>
<li class="chapter" data-level="5.3" data-path="workflow-2.html"><a href="workflow-2.html#data-sharing"><i class="fa fa-check"></i><b>5.3</b> Data sharing</a></li>
<li class="chapter" data-level="5.4" data-path="workflow-2.html"><a href="workflow-2.html#contest"><i class="fa fa-check"></i><b>5.4</b> Contest</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html"><i class="fa fa-check"></i><b>6</b> Raw data pretreatment</a>
<ul>
<li class="chapter" data-level="6.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#data-visualization"><i class="fa fa-check"></i><b>6.1</b> Data visualization</a></li>
<li class="chapter" data-level="6.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#peak-extraction"><i class="fa fa-check"></i><b>6.2</b> Peak extraction</a></li>
<li class="chapter" data-level="6.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#msms"><i class="fa fa-check"></i><b>6.3</b> MS/MS</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#mrm"><i class="fa fa-check"></i><b>6.3.1</b> MRM</a></li>
<li class="chapter" data-level="6.3.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dda"><i class="fa fa-check"></i><b>6.3.2</b> DDA</a></li>
<li class="chapter" data-level="6.3.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dia"><i class="fa fa-check"></i><b>6.3.3</b> DIA</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#retention-time-correction"><i class="fa fa-check"></i><b>6.4</b> Retention Time Correction</a></li>
<li class="chapter" data-level="6.5" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#filling-missing-values"><i class="fa fa-check"></i><b>6.5</b> Filling missing values</a></li>
<li class="chapter" data-level="6.6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#spectral-deconvolution"><i class="fa fa-check"></i><b>6.6</b> Spectral deconvolution</a></li>
<li class="chapter" data-level="6.7" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dynamic-range"><i class="fa fa-check"></i><b>6.7</b> Dynamic Range</a>
<ul>
<li class="chapter" data-level="6.7.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#non-detects"><i class="fa fa-check"></i><b>6.7.1</b> Non-detects</a></li>
<li class="chapter" data-level="6.7.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#over-detection-limit"><i class="fa fa-check"></i><b>6.7.2</b> Over Detection Limit</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#rsdfold-change-filter"><i class="fa fa-check"></i><b>6.8</b> RSD/fold change Filter</a></li>
<li class="chapter" data-level="6.9" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#power-analysis-filter"><i class="fa fa-check"></i><b>6.9</b> Power Analysis Filter</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="annotation.html"><a href="annotation.html"><i class="fa fa-check"></i><b>7</b> Annotation</a>
<ul>
<li class="chapter" data-level="7.1" data-path="annotation.html"><a href="annotation.html#issues-in-annotation"><i class="fa fa-check"></i><b>7.1</b> Issues in annotation</a></li>
<li class="chapter" data-level="7.2" data-path="annotation.html"><a href="annotation.html#peak-misidentification"><i class="fa fa-check"></i><b>7.2</b> Peak misidentification</a></li>
<li class="chapter" data-level="7.3" data-path="annotation.html"><a href="annotation.html#annotation-v.s.-identification"><i class="fa fa-check"></i><b>7.3</b> Annotation v.s. identification</a></li>
<li class="chapter" data-level="7.4" data-path="annotation.html"><a href="annotation.html#molecular-formula-assignment"><i class="fa fa-check"></i><b>7.4</b> Molecular Formula Assignment</a></li>
<li class="chapter" data-level="7.5" data-path="annotation.html"><a href="annotation.html#redundant-peaks"><i class="fa fa-check"></i><b>7.5</b> Redundant peaks</a>
<ul>
<li class="chapter" data-level="7.5.1" data-path="annotation.html"><a href="annotation.html#adducts-list"><i class="fa fa-check"></i><b>7.5.1</b> Adducts list</a></li>
<li class="chapter" data-level="7.5.2" data-path="annotation.html"><a href="annotation.html#isotope"><i class="fa fa-check"></i><b>7.5.2</b> Isotope</a></li>
<li class="chapter" data-level="7.5.3" data-path="annotation.html"><a href="annotation.html#camera"><i class="fa fa-check"></i><b>7.5.3</b> CAMERA</a></li>
<li class="chapter" data-level="7.5.4" data-path="annotation.html"><a href="annotation.html#ramclustr"><i class="fa fa-check"></i><b>7.5.4</b> RAMClustR</a></li>
<li class="chapter" data-level="7.5.5" data-path="annotation.html"><a href="annotation.html#biocan"><i class="fa fa-check"></i><b>7.5.5</b> BioCAn</a></li>
<li class="chapter" data-level="7.5.6" data-path="annotation.html"><a href="annotation.html#mzmatch"><i class="fa fa-check"></i><b>7.5.6</b> mzMatch</a></li>
<li class="chapter" data-level="7.5.7" data-path="annotation.html"><a href="annotation.html#xmsannotator"><i class="fa fa-check"></i><b>7.5.7</b> xMSannotator</a></li>
<li class="chapter" data-level="7.5.8" data-path="annotation.html"><a href="annotation.html#mait"><i class="fa fa-check"></i><b>7.5.8</b> MAIT</a></li>
<li class="chapter" data-level="7.5.9" data-path="annotation.html"><a href="annotation.html#pmd"><i class="fa fa-check"></i><b>7.5.9</b> pmd</a></li>
<li class="chapter" data-level="7.5.10" data-path="annotation.html"><a href="annotation.html#nontarget"><i class="fa fa-check"></i><b>7.5.10</b> nontarget</a></li>
<li class="chapter" data-level="7.5.11" data-path="annotation.html"><a href="annotation.html#binner"><i class="fa fa-check"></i><b>7.5.11</b> Binner</a></li>
<li class="chapter" data-level="7.5.12" data-path="annotation.html"><a href="annotation.html#mz.unity"><i class="fa fa-check"></i><b>7.5.12</b> mz.unity</a></li>
<li class="chapter" data-level="7.5.13" data-path="annotation.html"><a href="annotation.html#ms-flo"><i class="fa fa-check"></i><b>7.5.13</b> MS-FLO</a></li>
<li class="chapter" data-level="7.5.14" data-path="annotation.html"><a href="annotation.html#cliquems"><i class="fa fa-check"></i><b>7.5.14</b> CliqueMS</a></li>
<li class="chapter" data-level="7.5.15" data-path="annotation.html"><a href="annotation.html#interpretmsspectrum"><i class="fa fa-check"></i><b>7.5.15</b> InterpretMSSpectrum</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="annotation.html"><a href="annotation.html#msms-annotation"><i class="fa fa-check"></i><b>7.6</b> MS/MS annotation</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="annotation.html"><a href="annotation.html#metdna"><i class="fa fa-check"></i><b>7.6.1</b> MetDNA</a></li>
<li class="chapter" data-level="7.6.2" data-path="annotation.html"><a href="annotation.html#metfusion"><i class="fa fa-check"></i><b>7.6.2</b> MetFusion</a></li>
<li class="chapter" data-level="7.6.3" data-path="annotation.html"><a href="annotation.html#ms2analyzer"><i class="fa fa-check"></i><b>7.6.3</b> MS2Analyzer</a></li>
<li class="chapter" data-level="7.6.4" data-path="annotation.html"><a href="annotation.html#metfrag"><i class="fa fa-check"></i><b>7.6.4</b> MetFrag</a></li>
<li class="chapter" data-level="7.6.5" data-path="annotation.html"><a href="annotation.html#cfm-id"><i class="fa fa-check"></i><b>7.6.5</b> CFM-ID</a></li>
<li class="chapter" data-level="7.6.6" data-path="annotation.html"><a href="annotation.html#lipidfrag"><i class="fa fa-check"></i><b>7.6.6</b> LipidFrag</a></li>
<li class="chapter" data-level="7.6.7" data-path="annotation.html"><a href="annotation.html#lipidmatch"><i class="fa fa-check"></i><b>7.6.7</b> Lipidmatch</a></li>
<li class="chapter" data-level="7.6.8" data-path="annotation.html"><a href="annotation.html#barcoding"><i class="fa fa-check"></i><b>7.6.8</b> BarCoding</a></li>
<li class="chapter" data-level="7.6.9" data-path="annotation.html"><a href="annotation.html#imet"><i class="fa fa-check"></i><b>7.6.9</b> iMet</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="annotation.html"><a href="annotation.html#knowledge-based-annotation"><i class="fa fa-check"></i><b>7.7</b> Knowledge based annotation</a>
<ul>
<li class="chapter" data-level="7.7.1" data-path="annotation.html"><a href="annotation.html#chromatographic-retention-related-criteria"><i class="fa fa-check"></i><b>7.7.1</b> Chromatographic retention-related criteria</a></li>
<li class="chapter" data-level="7.7.2" data-path="annotation.html"><a href="annotation.html#probmetab"><i class="fa fa-check"></i><b>7.7.2</b> ProbMetab</a></li>
<li class="chapter" data-level="7.7.3" data-path="annotation.html"><a href="annotation.html#mi-pack"><i class="fa fa-check"></i><b>7.7.3</b> MI-Pack</a></li>
<li class="chapter" data-level="7.7.4" data-path="annotation.html"><a href="annotation.html#metexpert"><i class="fa fa-check"></i><b>7.7.4</b> MetExpert</a></li>
<li class="chapter" data-level="7.7.5" data-path="annotation.html"><a href="annotation.html#mycompoundid"><i class="fa fa-check"></i><b>7.7.5</b> MycompoundID</a></li>
<li class="chapter" data-level="7.7.6" data-path="annotation.html"><a href="annotation.html#metfamily"><i class="fa fa-check"></i><b>7.7.6</b> MetFamily</a></li>
<li class="chapter" data-level="7.7.7" data-path="annotation.html"><a href="annotation.html#plantmat"><i class="fa fa-check"></i><b>7.7.7</b> Plantmat</a></li>
</ul></li>
<li class="chapter" data-level="7.8" data-path="annotation.html"><a href="annotation.html#ms-database-for-annotation"><i class="fa fa-check"></i><b>7.8</b> MS Database for annotation</a>
<ul>
<li class="chapter" data-level="7.8.1" data-path="annotation.html"><a href="annotation.html#ms"><i class="fa fa-check"></i><b>7.8.1</b> MS</a></li>
<li class="chapter" data-level="7.8.2" data-path="annotation.html"><a href="annotation.html#msms-1"><i class="fa fa-check"></i><b>7.8.2</b> MS/MS</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="annotation.html"><a href="annotation.html#compounds-database"><i class="fa fa-check"></i><b>7.9</b> Compounds Database</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="omics-analysis.html"><a href="omics-analysis.html"><i class="fa fa-check"></i><b>8</b> Omics analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="omics-analysis.html"><a href="omics-analysis.html#from-bottom-up-to-top-down"><i class="fa fa-check"></i><b>8.1</b> From Bottom-up to Top-down</a></li>
<li class="chapter" data-level="8.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-analysis"><i class="fa fa-check"></i><b>8.2</b> Pathway analysis</a>
<ul>
<li class="chapter" data-level="8.2.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-database"><i class="fa fa-check"></i><b>8.2.1</b> Pathway Database</a></li>
<li class="chapter" data-level="8.2.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-software"><i class="fa fa-check"></i><b>8.2.2</b> Pathway software</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="omics-analysis.html"><a href="omics-analysis.html#network-analysis"><i class="fa fa-check"></i><b>8.3</b> Network analysis</a></li>
<li class="chapter" data-level="8.4" data-path="omics-analysis.html"><a href="omics-analysis.html#omics-integration"><i class="fa fa-check"></i><b>8.4</b> Omics integration</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="peaks-normalization.html"><a href="peaks-normalization.html"><i class="fa fa-check"></i><b>9</b> Peaks normalization</a>
<ul>
<li class="chapter" data-level="9.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#batch-effects-classification"><i class="fa fa-check"></i><b>9.1</b> Batch effects classification</a></li>
<li class="chapter" data-level="9.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#batch-effects-visualization"><i class="fa fa-check"></i><b>9.2</b> Batch effects visualization</a></li>
<li class="chapter" data-level="9.3" data-path="peaks-normalization.html"><a href="peaks-normalization.html#source-of-batch-effects"><i class="fa fa-check"></i><b>9.3</b> Source of batch effects</a></li>
<li class="chapter" data-level="9.4" data-path="peaks-normalization.html"><a href="peaks-normalization.html#avoid-batch-effects-by-doe"><i class="fa fa-check"></i><b>9.4</b> Avoid batch effects by DoE</a></li>
<li class="chapter" data-level="9.5" data-path="peaks-normalization.html"><a href="peaks-normalization.html#post-hoc-data-normalization"><i class="fa fa-check"></i><b>9.5</b> <em>post hoc</em> data normalization</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#unsupervised-methods"><i class="fa fa-check"></i><b>9.5.1</b> Unsupervised methods</a></li>
<li class="chapter" data-level="9.5.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#supervised-methods"><i class="fa fa-check"></i><b>9.5.2</b> Supervised methods</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="peaks-normalization.html"><a href="peaks-normalization.html#method-to-validate-the-normalization"><i class="fa fa-check"></i><b>9.6</b> Method to validate the normalization</a></li>
<li class="chapter" data-level="9.7" data-path="peaks-normalization.html"><a href="peaks-normalization.html#software-1"><i class="fa fa-check"></i><b>9.7</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="statistical-analysis.html"><a href="statistical-analysis.html"><i class="fa fa-check"></i><b>10</b> Statistical analysis</a>
<ul>
<li class="chapter" data-level="10.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#basic-statistical-analysis"><i class="fa fa-check"></i><b>10.1</b> Basic Statistical Analysis</a></li>
<li class="chapter" data-level="10.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#differences-analysis"><i class="fa fa-check"></i><b>10.2</b> Differences analysis</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#t-test-or-anova"><i class="fa fa-check"></i><b>10.2.1</b> T-test or ANOVA</a></li>
<li class="chapter" data-level="10.2.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#limma"><i class="fa fa-check"></i><b>10.2.2</b> LIMMA</a></li>
<li class="chapter" data-level="10.2.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#bayesian-mixture-model"><i class="fa fa-check"></i><b>10.2.3</b> Bayesian mixture model</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#pca"><i class="fa fa-check"></i><b>10.3</b> PCA</a></li>
<li class="chapter" data-level="10.4" data-path="statistical-analysis.html"><a href="statistical-analysis.html#cluster-analysis"><i class="fa fa-check"></i><b>10.4</b> Cluster Analysis</a></li>
<li class="chapter" data-level="10.5" data-path="statistical-analysis.html"><a href="statistical-analysis.html#plsda"><i class="fa fa-check"></i><b>10.5</b> PLSDA</a></li>
<li class="chapter" data-level="10.6" data-path="statistical-analysis.html"><a href="statistical-analysis.html#software-2"><i class="fa fa-check"></i><b>10.6</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="exposome.html"><a href="exposome.html"><i class="fa fa-check"></i><b>11</b> Exposome</a>
<ul>
<li class="chapter" data-level="11.1" data-path="exposome.html"><a href="exposome.html#internal-exposure"><i class="fa fa-check"></i><b>11.1</b> Internal exposure</a></li>
<li class="chapter" data-level="11.2" data-path="exposome.html"><a href="exposome.html#external-exposure"><i class="fa fa-check"></i><b>11.2</b> External exposure</a>
<ul>
<li class="chapter" data-level="11.2.1" data-path="exposome.html"><a href="exposome.html#environmental-fate-of-compounds"><i class="fa fa-check"></i><b>11.2.1</b> Environmental fate of compounds</a></li>
<li class="chapter" data-level="11.2.2" data-path="exposome.html"><a href="exposome.html#exposure-study-database"><i class="fa fa-check"></i><b>11.2.2</b> Exposure study database</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/yufree/metaworkflow" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Meta-Workflow</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered">
<h1>References</h1>

<div id="refs" class="references csl-bib-body hanging-indent">
<div id="ref-adams2020" class="csl-entry">
Adams, Kendra J., Brian Pratt, Neelanjan Bose, Laura G. Dubois, Lisa St John-Williams, Kevin M. Perrott, Karina Ky, et al. 2020. <span>“Skyline for Small Molecules: A Unifying Software Package for Quantitative Metabolomics.”</span> <em>Journal of Proteome Research</em> 19 (4): 1447–58. <a href="https://doi.org/10.1021/acs.jproteome.9b00640">https://doi.org/10.1021/acs.jproteome.9b00640</a>.
</div>
<div id="ref-aguilar-mogas2017" class="csl-entry">
Aguilar-Mogas, Antoni, Marta Sales-Pardo, Miriam Navarro, Roger Guimerà, and Oscar Yanes. 2017. <span>“iMet: A Network-Based Computational Tool to Assist in the Annotation of Metabolites from Tandem Mass Spectra.”</span> <em>Analytical Chemistry</em> 89 (6): 3474–82. <a href="https://doi.org/10.1021/acs.analchem.6b04512">https://doi.org/10.1021/acs.analchem.6b04512</a>.
</div>
<div id="ref-alden2017" class="csl-entry">
Alden, Nicholas, Smitha Krishnan, Vladimir Porokhin, Ravali Raju, Kyle McElearney, Alan Gilbert, and Kyongbum Lee. 2017. <span>“Biologically Consistent Annotation of Metabolomics Data.”</span> <em>Analytical Chemistry</em>, November. <a href="https://doi.org/10.1021/acs.analchem.7b02162">https://doi.org/10.1021/acs.analchem.7b02162</a>.
</div>
<div id="ref-ali2019a" class="csl-entry">
Ali, Ahmed, Yasmine Abouleila, Yoshihiro Shimizu, Eiso Hiyama, Samy Emara, Alireza Mashaghi, and Thomas Hankemeier. 2019. <span>“Single-Cell Metabolomics by Mass Spectrometry: Advances, Challenges, and Future Applications.”</span> <em>TrAC Trends in Analytical Chemistry</em> 120 (November): 115436. <a href="https://doi.org/10.1016/j.trac.2019.02.033">https://doi.org/10.1016/j.trac.2019.02.033</a>.
</div>
<div id="ref-alka2020" class="csl-entry">
Alka, Oliver, Timo Sachsenberg, Leon Bichmann, Julianus Pfeuffer, Hendrik Weisser, Samuel Wein, Eugen Netz, Marc Rurik, Oliver Kohlbacher, and Hannes Röst. 2020a. <span>“CHAPTER 6:OpenMS and KNIME for Mass Spectrometry Data Processing.”</span> In, 201–31. <a href="https://doi.org/10.1039/9781788019880-00201">https://doi.org/10.1039/9781788019880-00201</a>.
</div>
<div id="ref-alka2020a" class="csl-entry">
———. 2020b. <span>“CHAPTER 6:OpenMS and KNIME for Mass Spectrometry Data Processing.”</span> In, 201–31. <a href="https://doi.org/10.1039/9781788019880-00201">https://doi.org/10.1039/9781788019880-00201</a>.
</div>
<div id="ref-allam-ndoul2016" class="csl-entry">
Allam-Ndoul, Bénédicte, Frédéric Guénard, Véronique Garneau, Hubert Cormier, Olivier Barbier, Louis Pérusse, and Marie-Claude Vohl. 2016. <span>“Association Between Metabolite Profiles, Metabolic Syndrome and Obesity Status.”</span> <em>Nutrients</em> 8 (6): 324. <a href="https://doi.org/10.3390/nu8060324">https://doi.org/10.3390/nu8060324</a>.
</div>
<div id="ref-allard2017" class="csl-entry">
Allard, Pierre-Marie, Grégory Genta-Jouve, and Jean-Luc Wolfender. 2017. <span>“Deep Metabolome Annotation in Natural Products Research: Towards a Virtuous Cycle in Metabolite Identification.”</span> <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 40–49. <a href="https://doi.org/10.1016/j.cbpa.2016.12.022">https://doi.org/10.1016/j.cbpa.2016.12.022</a>.
</div>
<div id="ref-allen2014" class="csl-entry">
Allen, Felicity, Allison Pon, Michael Wilson, Russ Greiner, and David Wishart. 2014. <span>“CFM-ID: A Web Server for Annotation, Spectrum Prediction and Metabolite Identification from Tandem Mass Spectra.”</span> <em>Nucleic Acids Research</em> 42 (W1): W94–99. <a href="https://doi.org/10.1093/nar/gku436">https://doi.org/10.1093/nar/gku436</a>.
</div>
<div id="ref-alonso2015" class="csl-entry">
Alonso, Arnald, Sara Marsal, and Antonio Julià. 2015. <span>“Analytical Methods in Untargeted Metabolomics: State of the Art in 2015.”</span> <em>Frontiers in Bioengineering and Biotechnology</em> 3 (March). <a href="https://doi.org/10.3389/fbioe.2015.00023">https://doi.org/10.3389/fbioe.2015.00023</a>.
</div>
<div id="ref-andersson2009" class="csl-entry">
Andersson, Martin. 2009. <span>“A Comparison of Nine Pls1 Algorithms.”</span> <em>Journal of Chemometrics</em> 23 (10): 518–29. <a href="https://doi.org/10.1002/cem.1248">https://doi.org/10.1002/cem.1248</a>.
</div>
<div id="ref-aron2020a" class="csl-entry">
Aron, Allegra T., Emily C. Gentry, Kerry L. McPhail, Louis-Félix Nothias, Mélissa Nothias-Esposito, Amina Bouslimani, Daniel Petras, et al. 2020. <span>“Reproducible Molecular Networking of Untargeted Mass Spectrometry Data Using GNPS.”</span> <em>Nature Protocols</em> 15 (6): 1954–91. <a href="https://doi.org/10.1038/s41596-020-0317-5">https://doi.org/10.1038/s41596-020-0317-5</a>.
</div>
<div id="ref-b.dunn2011" class="csl-entry">
B. Dunn, Warwick, David I. Broadhurst, Helen J. Atherton, Royston Goodacre, and Julian L. Griffin. 2011. <span>“Systems Level Studies of Mammalian Metabolomes: The Roles of Mass Spectrometry and Nuclear Magnetic Resonance Spectroscopy.”</span> <em>Chemical Society Reviews</em> 40 (1): 387–426. <a href="https://doi.org/10.1039/B906712B">https://doi.org/10.1039/B906712B</a>.
</div>
<div id="ref-baker2011" class="csl-entry">
Baker, Monya. 2011. <span>“Metabolomics: From Small Molecules to Big Ideas.”</span> <em>Nature Methods</em> 8 (2): 117–21. <a href="https://doi.org/10.1038/nmeth0211-117">https://doi.org/10.1038/nmeth0211-117</a>.
</div>
<div id="ref-baran2013" class="csl-entry">
Baran, Richard, and Trent R. Northen. 2013. <span>“Robust Automated Mass Spectra Interpretation and Chemical Formula Calculation Using Mixed Integer Linear Programming.”</span> <em>Analytical Chemistry</em> 85 (20): 9777–84. <a href="https://doi.org/10.1021/ac402180c">https://doi.org/10.1021/ac402180c</a>.
</div>
<div id="ref-barbiersainthilaire2018" class="csl-entry">
Barbier Saint Hilaire, Pierre, Ulli M. Hohenester, Benoit Colsch, Jean-Claude Tabet, Christophe Junot, and François Fenaille. 2018. <span>“Evaluation of the High-Field Orbitrap Fusion for Compound Annotation in Metabolomics.”</span> <em>Analytical Chemistry</em> 90 (5): 3030–35. <a href="https://doi.org/10.1021/acs.analchem.7b05372">https://doi.org/10.1021/acs.analchem.7b05372</a>.
</div>
<div id="ref-barnes2016a" class="csl-entry">
Barnes, Stephen, H. Paul Benton, Krista Casazza, Sara J. Cooper, Xiangqin Cui, Xiuxia Du, Jeffrey Engler, et al. 2016a. <span>“Training in Metabolomics Research. I. Designing the Experiment, Collecting and Extracting Samples and Generating Metabolomics Data.”</span> <em>Journal of Mass Spectrometry</em> 51 (7): 461–75. <a href="https://doi.org/10.1002/jms.3782">https://doi.org/10.1002/jms.3782</a>.
</div>
<div id="ref-barnes2016" class="csl-entry">
———, et al. 2016b. <span>“Training in Metabolomics Research. II. Processing and Statistical Analysis of Metabolomics Data, Metabolite Identification, Pathway Analysis, Applications of Metabolomics and Its Future.”</span> <em>Journal of Mass Spectrometry</em> 51 (8): 535–48. <a href="https://doi.org/10.1002/jms.3780">https://doi.org/10.1002/jms.3780</a>.
</div>
<div id="ref-basu2017" class="csl-entry">
Basu, Sumanta, William Duren, Charles R. Evans, Charles F. Burant, George Michailidis, and Alla Karnovsky. 2017. <span>“Sparse Network Modeling and Metscape-Based Visualization Methods for the Analysis of Large-Scale Metabolomics Data.”</span> <em>Bioinformatics</em> 33 (10): 1545–53. <a href="https://doi.org/10.1093/bioinformatics/btx012">https://doi.org/10.1093/bioinformatics/btx012</a>.
</div>
<div id="ref-beale2018" class="csl-entry">
Beale, David J., Farhana R. Pinu, Konstantinos A. Kouremenos, Mahesha M. Poojary, Vinod K. Narayana, Berin A. Boughton, Komal Kanojia, Saravanan Dayalan, Oliver A. H. Jones, and Daniel A. Dias. 2018. <span>“Review of Recent Developments in GC<span><span></span></span>MS Approaches to Metabolomics-Based Research.”</span> <em>Metabolomics</em> 14 (11): 152. <a href="https://doi.org/10.1007/s11306-018-1449-2">https://doi.org/10.1007/s11306-018-1449-2</a>.
</div>
<div id="ref-begou2017a" class="csl-entry">
Begou, O., H. G. Gika, I. D. Wilson, and G. Theodoridis. 2017a. <span>“Hyphenated MS-Based Targeted Approaches in Metabolomics.”</span> <em>Analyst</em> 142 (17): 3079–3100. <a href="https://doi.org/10.1039/C7AN00812K">https://doi.org/10.1039/C7AN00812K</a>.
</div>
<div id="ref-begou2017b" class="csl-entry">
———. 2017b. <span>“Hyphenated MS-Based Targeted Approaches in Metabolomics.”</span> <em>Analyst</em> 142 (17): 3079–3100. <a href="https://doi.org/10.1039/C7AN00812K">https://doi.org/10.1039/C7AN00812K</a>.
</div>
<div id="ref-benjamini1995" class="csl-entry">
Benjamini, Yoav, and Yosef Hochberg. 1995. <span>“Controlling the False Discovery Rate: A Practical and Powerful Approach to Multiple Testing.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 57 (1): 289–300. <a href="https://www.jstor.org/stable/2346101">https://www.jstor.org/stable/2346101</a>.
</div>
<div id="ref-bennett2009" class="csl-entry">
Bennett, Bryson D., Elizabeth H. Kimball, Melissa Gao, Robin Osterhout, Stephen J. Van Dien, and Joshua D. Rabinowitz. 2009. <span>“Absolute Metabolite Concentrations and Implied Enzyme Active Site Occupancy in Escherichia Coli.”</span> <em>Nature Chemical Biology</em> 5 (8): 593–99. <a href="https://doi.org/10.1038/nchembio.186">https://doi.org/10.1038/nchembio.186</a>.
</div>
<div id="ref-bertsch2011" class="csl-entry">
Bertsch, Andreas, Clemens Gröpl, Knut Reinert, and Oliver Kohlbacher. 2011. <span>“OpenMS and TOPP: Open Source Software for LC-MS Data Analysis.”</span> In, edited by Michael Hamacher, Martin Eisenacher, and Christian Stephan, 353–67. Methods in Molecular Biology. Totowa, NJ: Humana Press. <a href="https://doi.org/10.1007/978-1-60761-987-1_23">https://doi.org/10.1007/978-1-60761-987-1_23</a>.
</div>
<div id="ref-bijttebier2016" class="csl-entry">
Bijttebier, Sebastiaan, Anastasia Van der Auwera, Kenn Foubert, Stefan Voorspoels, Luc Pieters, and Sandra Apers. 2016. <span>“Bridging the Gap Between Comprehensive Extraction Protocols in Plant Metabolomics Studies and Method Validation.”</span> <em>Analytica Chimica Acta</em> 935 (September): 136–50. <a href="https://doi.org/10.1016/j.aca.2016.06.047">https://doi.org/10.1016/j.aca.2016.06.047</a>.
</div>
<div id="ref-bilbao2015" class="csl-entry">
Bilbao, Aivett, Emmanuel Varesio, Jeremy Luban, Caterina Strambio-De-Castillia, Gérard Hopfgartner, Markus Müller, and Frédérique Lisacek. 2015a. <span>“Processing Strategies and Software Solutions for Data-Independent Acquisition in Mass Spectrometry.”</span> <em>PROTEOMICS</em> 15 (5-6): 964–80. <a href="https://doi.org/10.1002/pmic.201400323">https://doi.org/10.1002/pmic.201400323</a>.
</div>
<div id="ref-bilbao2015a" class="csl-entry">
———. 2015b. <span>“Processing Strategies and Software Solutions for Data-Independent Acquisition in Mass Spectrometry.”</span> <em>PROTEOMICS</em> 15 (5-6): 964–80. <a href="https://doi.org/10.1002/pmic.201400323">https://doi.org/10.1002/pmic.201400323</a>.
</div>
<div id="ref-blaise2013" class="csl-entry">
Blaise, Benjamin J. 2013. <span>“Data-Driven Sample Size Determination for Metabolic Phenotyping Studies.”</span> <em>Analytical Chemistry</em> 85 (19): 8943–50. <a href="https://doi.org/10.1021/ac4022314">https://doi.org/10.1021/ac4022314</a>.
</div>
<div id="ref-blaise2016a" class="csl-entry">
Blaise, Benjamin J., Gonçalo Correia, Adrienne Tin, J. Hunter Young, Anne-Claire Vergnaud, Matthew Lewis, Jake T. M. Pearce, et al. 2016. <span>“Power Analysis and Sample Size Determination in Metabolic Phenotyping.”</span> <em>Analytical Chemistry</em> 88 (10): 5179–88. <a href="https://doi.org/10.1021/acs.analchem.6b00188">https://doi.org/10.1021/acs.analchem.6b00188</a>.
</div>
<div id="ref-blaenovi2017" class="csl-entry">
Blaženović, Ivana, Tobias Kind, Hrvoje Torbašinović, Slobodan Obrenović, Sajjan S. Mehta, Hiroshi Tsugawa, Tobias Wermuth, et al. 2017. <span>“Comprehensive Comparison of in Silico MS/MS Fragmentation Tools of the CASMI Contest: Database Boosting Is Needed to Achieve 93.”</span> <em>Journal of Cheminformatics</em> 9 (1): 32. <a href="https://doi.org/10.1186/s13321-017-0219-x">https://doi.org/10.1186/s13321-017-0219-x</a>.
</div>
<div id="ref-bonini2020" class="csl-entry">
Bonini, Paolo, Tobias Kind, Hiroshi Tsugawa, Dinesh Kumar Barupal, and Oliver Fiehn. 2020. <span>“Retip: Retention Time Prediction for Compound Annotation in Untargeted Metabolomics.”</span> <em>Analytical Chemistry</em> 92 (11): 7515–22. <a href="https://doi.org/10.1021/acs.analchem.9b05765">https://doi.org/10.1021/acs.analchem.9b05765</a>.
</div>
<div id="ref-bonner2018" class="csl-entry">
Bonner, Ron, and Gérard Hopfgartner. 2018. <span>“SWATH Data Independent Acquisition Mass Spectrometry for Metabolomics.”</span> <em>TrAC Trends in Analytical Chemistry</em>, October. <a href="https://doi.org/10.1016/j.trac.2018.10.014">https://doi.org/10.1016/j.trac.2018.10.014</a>.
</div>
<div id="ref-box2005" class="csl-entry">
Box, George E. P., J. Stuart Hunter, and William G. Hunter. 2005. <em>Statistics for Experimenters</em>. Wiley-Interscience. <a href="https://book.douban.com/subject/1775779/">https://book.douban.com/subject/1775779/</a>.
</div>
<div id="ref-brereton2018" class="csl-entry">
Brereton, Richard G., and Gavin R. Lloyd. 2018. <span>“Partial Least Squares Discriminant Analysis for Chemometrics and Metabolomics: How Scores, Loadings, and Weights Differ According to Two Common Algorithms.”</span> <em>Journal of Chemometrics</em> 32 (4): e3028. <a href="https://doi.org/10.1002/cem.3028">https://doi.org/10.1002/cem.3028</a>.
</div>
<div id="ref-broadhurst2018" class="csl-entry">
Broadhurst, David, Royston Goodacre, Stacey N. Reinke, Julia Kuligowski, Ian D. Wilson, Matthew R. Lewis, and Warwick B. Dunn. 2018. <span>“Guidelines and Considerations for the Use of System Suitability and Quality Control Samples in Mass Spectrometry Assays Applied in Untargeted Clinical Metabolomic Studies.”</span> <em>Metabolomics</em> 14 (6). <a href="https://doi.org/10.1007/s11306-018-1367-3">https://doi.org/10.1007/s11306-018-1367-3</a>.
</div>
<div id="ref-broeckling2014" class="csl-entry">
Broeckling, C. D., F. A. Afsar, S. Neumann, A. Ben-Hur, and J. E. Prenni. 2014. <span>“RAMClust: A Novel Feature Clustering Method Enables Spectral-Matching-Based Annotation for Metabolomics Data.”</span> <em>Analytical Chemistry</em> 86 (14): 6812–17. <a href="https://doi.org/10.1021/ac501530d">https://doi.org/10.1021/ac501530d</a>.
</div>
<div id="ref-broeckling2016" class="csl-entry">
Broeckling, Corey D., Andrea Ganna, Mark Layer, Kevin Brown, Ben Sutton, Erik Ingelsson, Graham Peers, and Jessica E. Prenni. 2016. <span>“Enabling Efficient and Confident Annotation of LC<span>-</span>MS Metabolomics Data Through Ms1 Spectrum and Time Prediction.”</span> <em>Analytical Chemistry</em> 88 (18): 9226–34. <a href="https://doi.org/10.1021/acs.analchem.6b02479">https://doi.org/10.1021/acs.analchem.6b02479</a>.
</div>
<div id="ref-bundy2009" class="csl-entry">
Bundy, Jacob G., Matthew P. Davey, and Mark R. Viant. 2009. <span>“Environmental Metabolomics: A Critical Review and Future Perspectives.”</span> <em>Metabolomics</em> 5 (1): 3. <a href="https://doi.org/10.1007/s11306-008-0152-0">https://doi.org/10.1007/s11306-008-0152-0</a>.
</div>
<div id="ref-cai2017" class="csl-entry">
Cai, Qingpo, Jessica A. Alvarez, Jian Kang, and Tianwei Yu. 2017. <span>“Network Marker Selection for Untargeted LC<span><span></span></span>MS Metabolomics Data.”</span> <em>Journal of Proteome Research</em> 16 (3): 1261–69. <a href="https://doi.org/10.1021/acs.jproteome.6b00861">https://doi.org/10.1021/acs.jproteome.6b00861</a>.
</div>
<div id="ref-cajka2016" class="csl-entry">
Cajka, Tomas, and Oliver Fiehn. 2016. <span>“Toward Merging Untargeted and Targeted Methods in Mass Spectrometry-Based Metabolomics and Lipidomics.”</span> <em>Analytical Chemistry</em> 88 (1): 524–45. <a href="https://doi.org/10.1021/acs.analchem.5b04491">https://doi.org/10.1021/acs.analchem.5b04491</a>.
</div>
<div id="ref-calbiani2006" class="csl-entry">
Calbiani, F., M. Careri, L. Elviri, A. Mangia, and I. Zagnoni. 2006. <span>“Matrix Effects on Accurate Mass Measurements of Low-Molecular Weight Compounds Using Liquid Chromatography-Electrospray-Quadrupole Time-of-Flight Mass Spectrometry.”</span> <em>Journal of Mass Spectrometry</em> 41 (3): 289–94. <a href="https://doi.org/10.1002/jms.984">https://doi.org/10.1002/jms.984</a>.
</div>
<div id="ref-carroll2010" class="csl-entry">
Carroll, Adam J., Murray R. Badger, and A. Harvey Millar. 2010. <span>“The MetabolomeExpress Project: Enabling Web-Based Processing, Analysis and Transparent Dissemination of GC/MS Metabolomics Datasets.”</span> <em>BMC Bioinformatics</em> 11 (1): 376. <a href="https://doi.org/10.1186/1471-2105-11-376">https://doi.org/10.1186/1471-2105-11-376</a>.
</div>
<div id="ref-castro-puyana2017" class="csl-entry">
Castro-Puyana, María, Raquel Pérez-Míguez, Lidia Montero, and Miguel Herrero. 2017. <span>“Application of Mass Spectrometry-Based Metabolomics Approaches for Food Safety, Quality and Traceability.”</span> <em>TrAC Trends in Analytical Chemistry</em> 93 (August): 102–18. <a href="https://doi.org/10.1016/j.trac.2017.05.004">https://doi.org/10.1016/j.trac.2017.05.004</a>.
</div>
<div id="ref-chaleckis2019" class="csl-entry">
Chaleckis, Romanas, Isabel Meister, Pei Zhang, and Craig E Wheelock. 2019. <span>“Challenges, Progress and Promises of Metabolite Annotation for LC<span><span></span></span>MS-Based Metabolomics.”</span> <em>Current Opinion in Biotechnology</em>, Analytical biotechnology, 55 (February): 44–50. <a href="https://doi.org/10.1016/j.copbio.2018.07.010">https://doi.org/10.1016/j.copbio.2018.07.010</a>.
</div>
<div id="ref-chambers2012" class="csl-entry">
Chambers, Matthew C., Brendan Maclean, Robert Burke, Dario Amodei, Daniel L. Ruderman, Steffen Neumann, Laurent Gatto, et al. 2012. <span>“A Cross-Platform Toolkit for Mass Spectrometry and Proteomics.”</span> <em>Nature Biotechnology</em> 30 (October): 918–20. <a href="https://doi.org/10.1038/nbt.2377">https://doi.org/10.1038/nbt.2377</a>.
</div>
<div id="ref-chang2016" class="csl-entry">
Chang, Hui-Yin, Ching-Tai Chen, T. Mamie Lih, Ke-Shiuan Lynn, Chiun-Gung Juo, Wen-Lian Hsu, and Ting-Yi Sung. 2016. <span>“iMet-Q: A User-Friendly Tool for Label-Free Metabolomics Quantitation Using Dynamic Peak-Width Determination.”</span> <em>PLOS ONE</em> 11 (1): e0146112. <a href="https://doi.org/10.1371/journal.pone.0146112">https://doi.org/10.1371/journal.pone.0146112</a>.
</div>
<div id="ref-chen2017a" class="csl-entry">
Chen, Gengbo, Scott Walmsley, Gemmy C. M. Cheung, Liyan Chen, Ching-Yu Cheng, Roger W. Beuerman, Tien Yin Wong, Lei Zhou, and Hyungwon Choi. 2017. <span>“Customized Consensus Spectral Library Building for Untargeted Quantitative Metabolomics Analysis with Data Independent Acquisition Mass Spectrometry and MetaboDIA Workflow.”</span> <em>Analytical Chemistry</em> 89 (9): 4897–4906. <a href="https://doi.org/10.1021/acs.analchem.6b05006">https://doi.org/10.1021/acs.analchem.6b05006</a>.
</div>
<div id="ref-chen2017" class="csl-entry">
Chen, Yanhua, Zhi Zhou, Wei Yang, Nan Bi, Jing Xu, Jiuming He, Ruiping Zhang, Lvhua Wang, and Zeper Abliz. 2017. <span>“Development of a Data-Independent Targeted Metabolomics Method for Relative Quantification Using Liquid Chromatography Coupled with Tandem Mass Spectrometry.”</span> <em>Analytical Chemistry</em> 89 (13): 6954–62. <a href="https://doi.org/10.1021/acs.analchem.6b04727">https://doi.org/10.1021/acs.analchem.6b04727</a>.
</div>
<div id="ref-cheng2017" class="csl-entry">
Cheng, Susan, Svati H. Shah, Elizabeth J. Corwin, Oliver Fiehn, Robert L. Fitzgerald, Robert E. Gerszten, Thomas Illig, et al. 2017. <span>“Potential Impact and Study Considerations of Metabolomics in Cardiovascular Health and Disease: A Scientific Statement From the American Heart Association.”</span> <em>Circulation: Cardiovascular Genetics</em> 10 (2): e000032. <a href="https://doi.org/10.1161/HCG.0000000000000032">https://doi.org/10.1161/HCG.0000000000000032</a>.
</div>
<div id="ref-choi2014" class="csl-entry">
Choi, Meena, Ching-Yun Chang, Timothy Clough, Daniel Broudy, Trevor Killeen, Brendan MacLean, and Olga Vitek. 2014a. <span>“MSstats: An r Package for Statistical Analysis of Quantitative Mass Spectrometry-Based Proteomic Experiments.”</span> <em>Bioinformatics</em> 30 (17): 2524–26. <a href="https://doi.org/10.1093/bioinformatics/btu305">https://doi.org/10.1093/bioinformatics/btu305</a>.
</div>
<div id="ref-choi2014a" class="csl-entry">
———. 2014b. <span>“MSstats: An r Package for Statistical Analysis of Quantitative Mass Spectrometry-Based Proteomic Experiments.”</span> <em>Bioinformatics</em> 30 (17): 2524–26. <a href="https://doi.org/10.1093/bioinformatics/btu305">https://doi.org/10.1093/bioinformatics/btu305</a>.
</div>
<div id="ref-chokkathukalam2013" class="csl-entry">
Chokkathukalam, Achuthanunni, Andris Jankevics, Darren J. Creek, Fiona Achcar, Michael P. Barrett, and Rainer Breitling. 2013. <span>“mzMatch<span><span></span></span>ISO: An R Tool for the Annotation and Relative Quantification of Isotope-Labelled Mass Spectrometry Data.”</span> <em>Bioinformatics</em> 29 (2): 281–83. <a href="https://doi.org/10.1093/bioinformatics/bts674">https://doi.org/10.1093/bioinformatics/bts674</a>.
</div>
<div id="ref-chong2019" class="csl-entry">
Chong, Jasmine, David S. Wishart, and Jianguo Xia. 2019. <span>“Using MetaboAnalyst 4.0 for Comprehensive and Integrative Metabolomics Data Analysis.”</span> <em>Current Protocols in Bioinformatics</em> 68 (1): e86. <a href="https://doi.org/10.1002/cpbi.86">https://doi.org/10.1002/cpbi.86</a>.
</div>
<div id="ref-clasquin2012" class="csl-entry">
Clasquin, Michelle F., Eugene Melamud, and Joshua D. Rabinowitz. 2012. <span>“LC-MS Data Processing with MAVEN: A Metabolomic Analysis and Visualization Engine.”</span> <em>Current Protocols in Bioinformatics</em> 37 (1): 14.11.1–23. https://doi.org/<a href="https://doi.org/10.1002/0471250953.bi1411s37">https://doi.org/10.1002/0471250953.bi1411s37</a>.
</div>
<div id="ref-considine2017" class="csl-entry">
Considine, E. C., G. Thomas, A. L. Boulesteix, A. S. Khashan, and L. C. Kenny. 2017. <span>“Critical Review of Reporting of the Data Analysis Step in Metabolomics.”</span> <em>Metabolomics</em> 14 (1): 7. <a href="https://doi.org/10.1007/s11306-017-1299-3">https://doi.org/10.1007/s11306-017-1299-3</a>.
</div>
<div id="ref-creek2012b" class="csl-entry">
Creek, Darren J., Andris Jankevics, Karl E. V. Burgess, Rainer Breitling, and Michael P. Barrett. 2012. <span>“IDEOM: An Excel Interface for Analysis of LC<span><span></span></span>MS-Based Metabolomics Data.”</span> <em>Bioinformatics</em> 28 (7): 1048–49. <a href="https://doi.org/10.1093/bioinformatics/bts069">https://doi.org/10.1093/bioinformatics/bts069</a>.
</div>
<div id="ref-daly2014" class="csl-entry">
Daly, Rónán, Simon Rogers, Joe Wandy, Andris Jankevics, Karl E. V. Burgess, and Rainer Breitling. 2014a. <span>“MetAssign: Probabilistic Annotation of Metabolites from LC<span><span></span></span>MS Data Using a Bayesian Clustering Approach.”</span> <em>Bioinformatics</em> 30 (19): 2764–71. <a href="https://doi.org/10.1093/bioinformatics/btu370">https://doi.org/10.1093/bioinformatics/btu370</a>.
</div>
<div id="ref-daly2014a" class="csl-entry">
———. 2014b. <span>“MetAssign: Probabilistic Annotation of Metabolites from LC<span><span></span></span>MS Data Using a Bayesian Clustering Approach.”</span> <em>Bioinformatics</em> 30 (19): 2764–71. <a href="https://doi.org/10.1093/bioinformatics/btu370">https://doi.org/10.1093/bioinformatics/btu370</a>.
</div>
<div id="ref-delivera2012" class="csl-entry">
De Livera, Alysha M., Daniel A. Dias, David De Souza, Thusitha Rupasinghe, James Pyke, Dedreia Tull, Ute Roessner, Malcolm McConville, and Terence P. Speed. 2012a. <span>“Normalizing and Integrating Metabolomics Data.”</span> <em>Analytical Chemistry</em> 84 (24): 10768–76. <a href="https://doi.org/10.1021/ac302748b">https://doi.org/10.1021/ac302748b</a>.
</div>
<div id="ref-delivera2012a" class="csl-entry">
———. 2012b. <span>“Normalizing and Integrating Metabolomics Data.”</span> <em>Analytical Chemistry</em> 84 (24): 10768–76. <a href="https://doi.org/10.1021/ac302748b">https://doi.org/10.1021/ac302748b</a>.
</div>
<div id="ref-deda2017" class="csl-entry">
Deda, Olga, Anastasia Chrysovalantou Chatziioannou, Stella Fasoula, Dimitris Palachanis, Nicolaos Raikos, Georgios A. Theodoridis, and Helen G. Gika. 2017. <span>“Sample Preparation Optimization in Fecal Metabolic Profiling.”</span> <em>Journal of Chromatography B</em>, Advances in mass spectrometry-based applications, 1047 (March): 115–23. <a href="https://doi.org/10.1016/j.jchromb.2016.06.047">https://doi.org/10.1016/j.jchromb.2016.06.047</a>.
</div>
<div id="ref-defelice2017" class="csl-entry">
DeFelice, Brian C., Sajjan Singh Mehta, Stephanie Samra, Tomáš Čajka, Benjamin Wancewicz, Johannes F. Fahrmann, and Oliver Fiehn. 2017. <span>“Mass Spectral Feature List Optimizer (MS-FLO): A Tool to Minimize False Positive Peak Reports in Untargeted Liquid Chromatography<span><span></span></span>mass Spectroscopy (LC-MS) Data Processing.”</span> <em>Analytical Chemistry</em> 89 (6): 3250–55. <a href="https://doi.org/10.1021/acs.analchem.6b04372">https://doi.org/10.1021/acs.analchem.6b04372</a>.
</div>
<div id="ref-domingo-almenara2016" class="csl-entry">
Domingo-Almenara, Xavier, Jesus Brezmes, Maria Vinaixa, Sara Samino, Noelia Ramirez, Marta Ramon-Krauel, Carles Lerin, et al. 2016. <span>“eRah: A Computational Tool Integrating Spectral Deconvolution and Alignment with Quantification and Identification of Metabolites in GC/MS-Based Metabolomics.”</span> <em>Analytical Chemistry</em> 88 (19): 9821–29. <a href="https://doi.org/10.1021/acs.analchem.6b02927">https://doi.org/10.1021/acs.analchem.6b02927</a>.
</div>
<div id="ref-domingo-almenara2018" class="csl-entry">
Domingo-Almenara, Xavier, J. Rafael Montenegro-Burke, H. Paul Benton, and Gary Siuzdak. 2018a. <span>“Annotation: A Computational Solution for Streamlining Metabolomics Analysis.”</span> <em>Analytical Chemistry</em> 90 (1): 480–89. <a href="https://doi.org/10.1021/acs.analchem.7b03929">https://doi.org/10.1021/acs.analchem.7b03929</a>.
</div>
<div id="ref-domingo-almenara2018b" class="csl-entry">
———. 2018b. <span>“Annotation: A Computational Solution for Streamlining Metabolomics Analysis.”</span> <em>Analytical Chemistry</em> 90 (1): 480–89. <a href="https://doi.org/10.1021/acs.analchem.7b03929">https://doi.org/10.1021/acs.analchem.7b03929</a>.
</div>
<div id="ref-domingo-almenara2018a" class="csl-entry">
Domingo-Almenara, Xavier, J. Rafael Montenegro-Burke, Julijana Ivanisevic, Aurelien Thomas, Jonathan Sidibé, Tony Teav, Carlos Guijas, et al. 2018. <span>“XCMS-MRM and METLIN-MRM: A Cloud Library and Public Resource for Targeted Analysis of Small Molecules.”</span> <em>Nature Methods</em> 15 (9): 681–84. <a href="https://doi.org/10.1038/s41592-018-0110-3">https://doi.org/10.1038/s41592-018-0110-3</a>.
</div>
<div id="ref-domingo-almenara2020" class="csl-entry">
Domingo-Almenara, Xavier, and Gary Siuzdak. 2020. <span>“Metabolomics Data Processing Using XCMS.”</span> In, edited by Shuzhao Li, 11–24. Methods in Molecular Biology. New York, NY: Springer US. <a href="https://doi.org/10.1007/978-1-0716-0239-3_2">https://doi.org/10.1007/978-1-0716-0239-3_2</a>.
</div>
<div id="ref-doppler2016" class="csl-entry">
Doppler, Maria, Bernhard Kluger, Christoph Bueschl, Christina Schneider, Rudolf Krska, Sylvie Delcambre, Karsten Hiller, Marc Lemmens, and Rainer Schuhmacher. 2016. <span>“Stable Isotope-Assisted Evaluation of Different Extraction Solvents for Untargeted Metabolomics of Plants.”</span> <em>International Journal of Molecular Sciences</em> 17 (7). <a href="https://doi.org/10.3390/ijms17071017">https://doi.org/10.3390/ijms17071017</a>.
</div>
<div id="ref-dryden2017" class="csl-entry">
Dryden, Michael D. M., Ryan Fobel, Christian Fobel, and Aaron R. Wheeler. 2017. <span>“Upon the Shoulders of Giants: Open-Source Hardware and Software in Analytical Chemistry.”</span> <em>Analytical Chemistry</em> 89 (8): 4330–38. <a href="https://doi.org/10.1021/acs.analchem.7b00485">https://doi.org/10.1021/acs.analchem.7b00485</a>.
</div>
<div id="ref-du2013a" class="csl-entry">
Du, Xiuxia, and Steven H Zeisel. 2013. <span>“SPECTRAL DECONVOLUTION FOR GAS CHROMATOGRAPHY MASS SPECTROMETRY-BASED METABOLOMICS: CURRENT STATUS AND FUTURE PERSPECTIVES.”</span> <em>Computational and Structural Biotechnology Journal</em> 4 (5): 1–10. <a href="https://doi.org/10.5936/csbj.201301013">https://doi.org/10.5936/csbj.201301013</a>.
</div>
<div id="ref-dudzik2018" class="csl-entry">
Dudzik, Danuta, Cecilia Barbas-Bernardos, Antonia García, and Coral Barbas. 2018. <span>“Quality Assurance Procedures for Mass Spectrometry Untargeted Metabolomics. A Review.”</span> <em>Journal of Pharmaceutical and Biomedical Analysis</em>, Review issue 2017, 147 (January): 149–73. <a href="https://doi.org/10.1016/j.jpba.2017.07.044">https://doi.org/10.1016/j.jpba.2017.07.044</a>.
</div>
<div id="ref-dunn2012" class="csl-entry">
Dunn, Warwick B, Ian D Wilson, Andrew W Nicholls, and David Broadhurst. 2012. <span>“The Importance of Experimental Design and QC Samples in Large-Scale and MS-Driven Untargeted Metabolomic Studies of Humans.”</span> <em>Bioanalysis</em> 4 (18): 2249–64. <a href="https://doi.org/10.4155/bio.12.204">https://doi.org/10.4155/bio.12.204</a>.
</div>
<div id="ref-dührkop2019" class="csl-entry">
Dührkop, Kai, Markus Fleischauer, Marcus Ludwig, Alexander A. Aksenov, Alexey V. Melnik, Marvin Meusel, Pieter C. Dorrestein, Juho Rousu, and Sebastian Böcker. 2019. <span>“SIRIUS 4: A Rapid Tool for Turning Tandem Mass Spectra into Metabolite Structure Information.”</span> <em>Nature Methods</em> 16 (4): 299–302. <a href="https://doi.org/10.1038/s41592-019-0344-8">https://doi.org/10.1038/s41592-019-0344-8</a>.
</div>
<div id="ref-dührkop2020" class="csl-entry">
Dührkop, Kai, Louis-Félix Nothias, Markus Fleischauer, Raphael Reher, Marcus Ludwig, Martin A. Hoffmann, Daniel Petras, et al. 2020. <span>“Systematic Classification of Unknown Metabolites Using High-Resolution Fragmentation Mass Spectra.”</span> <em>Nature Biotechnology</em>, November, 1–10. <a href="https://doi.org/10.1038/s41587-020-0740-8">https://doi.org/10.1038/s41587-020-0740-8</a>.
</div>
<div id="ref-edmands2015" class="csl-entry">
Edmands, William M. B., Dinesh K. Barupal, and Augustin Scalbert. 2015. <span>“MetMSLine: An Automated and Fully Integrated Pipeline for Rapid Processing of High-Resolution LC<span><span></span></span>MS Metabolomic Datasets.”</span> <em>Bioinformatics</em> 31 (5): 788–90. <a href="https://doi.org/10.1093/bioinformatics/btu705">https://doi.org/10.1093/bioinformatics/btu705</a>.
</div>
<div id="ref-edmands2018" class="csl-entry">
Edmands, William M. B., Josie Hayes, and Stephen M. Rappaport. 2018. <span>“SimExTargId: A Comprehensive Package for Real-Time LC-MS Data Acquisition and Analysis.”</span> <em>Bioinformatics</em> 34 (20): 3589–90. <a href="https://doi.org/10.1093/bioinformatics/bty218">https://doi.org/10.1093/bioinformatics/bty218</a>.
</div>
<div id="ref-edmands2017" class="csl-entry">
Edmands, William M. B., Lauren Petrick, Dinesh K. Barupal, Augustin Scalbert, Mark J. Wilson, Jeffrey K. Wickliffe, and Stephen M. Rappaport. 2017. <span>“compMS2Miner: An Automatable Metabolite Identification, Visualization, and Data-Sharing r Package for High-Resolution LC<span><span></span></span>MS Data Sets.”</span> <em>Analytical Chemistry</em> 89 (7): 3919–28. <a href="https://doi.org/10.1021/acs.analchem.6b02394">https://doi.org/10.1021/acs.analchem.6b02394</a>.
</div>
<div id="ref-fenaille2017" class="csl-entry">
Fenaille, François, Pierre Barbier Saint-Hilaire, Kathleen Rousseau, and Christophe Junot. 2017a. <span>“Data Acquisition Workflows in Liquid Chromatography Coupled to High Resolution Mass Spectrometry-Based Metabolomics: Where Do We Stand?”</span> <em>Journal of Chromatography A</em> 1526 (Supplement C): 1–12. <a href="https://doi.org/10.1016/j.chroma.2017.10.043">https://doi.org/10.1016/j.chroma.2017.10.043</a>.
</div>
<div id="ref-fenaille2017a" class="csl-entry">
———. 2017b. <span>“Data Acquisition Workflows in Liquid Chromatography Coupled to High Resolution Mass Spectrometry-Based Metabolomics: Where Do We Stand?”</span> <em>Journal of Chromatography A</em> 1526 (Supplement C): 1–12. <a href="https://doi.org/10.1016/j.chroma.2017.10.043">https://doi.org/10.1016/j.chroma.2017.10.043</a>.
</div>
<div id="ref-fernandez-albert2014a" class="csl-entry">
Fernández-Albert, Francesc, Rafael Llorach, Cristina Andrés-Lacueva, and Alexandre Perera. 2014. <span>“An r Package to Analyse LC/MS Metabolomic Data: MAIT (Metabolite Automatic Identification Toolkit).”</span> <em>Bioinformatics</em> 30 (13): 1937–39. <a href="https://doi.org/10.1093/bioinformatics/btu136">https://doi.org/10.1093/bioinformatics/btu136</a>.
</div>
<div id="ref-fessenden2016" class="csl-entry">
Fessenden, Marissa. 2016. <span>“Metabolomics: Small Molecules, Single Cells.”</span> <em>Nature</em> 540 (7631): 153–55. <a href="https://doi.org/10.1038/540153a">https://doi.org/10.1038/540153a</a>.
</div>
<div id="ref-fiehn2002" class="csl-entry">
Fiehn, Oliver. 2002. <span>“Metabolomics <span><span></span></span> the Link Between Genotypes and Phenotypes.”</span> <em>Plant Molecular Biology</em> 48 (1): 155–71. <a href="https://doi.org/10.1023/A:1013713905833">https://doi.org/10.1023/A:1013713905833</a>.
</div>
<div id="ref-forsberg2018" class="csl-entry">
Forsberg, Erica M., Tao Huan, Duane Rinehart, H. Paul Benton, Benedikt Warth, Brian Hilmers, and Gary Siuzdak. 2018. <span>“Data Processing, Multi-Omic Pathway Mapping, and Metabolite Activity Analysis Using XCMS Online.”</span> <em>Nature Protocols</em> 13 (4): 633–51. <a href="https://doi.org/10.1038/nprot.2017.151">https://doi.org/10.1038/nprot.2017.151</a>.
</div>
<div id="ref-franceschi2012" class="csl-entry">
Franceschi, Pietro, Domenico Masuero, Urska Vrhovsek, Fulvio Mattivi, and Ron Wehrens. 2012. <span>“A Benchmark Spike-in Data Set for Biomarker Identification in Metabolomics.”</span> <em>Journal of Chemometrics</em> 26 (1-2): 16–24. <a href="https://doi.org/10.1002/cem.1420">https://doi.org/10.1002/cem.1420</a>.
</div>
<div id="ref-fu2017" class="csl-entry">
Fu, Hai-Yan, Ou Hu, Yue-Ming Zhang, Li Zhang, Jing-Jing Song, Peang Lu, Qing-Xia Zheng, et al. 2017. <span>“Mass-Spectra-Based Peak Alignment for Automatic Nontargeted Metabolic Profiling Analysis for Biomarker Screening in Plant Samples.”</span> <em>Journal of Chromatography A</em> 1513 (Supplement C): 201–9. <a href="https://doi.org/10.1016/j.chroma.2017.07.044">https://doi.org/10.1016/j.chroma.2017.07.044</a>.
</div>
<div id="ref-gerlich2013" class="csl-entry">
Gerlich, Michael, and Steffen Neumann. 2013. <span>“MetFusion: Integration of Compound Identification Strategies.”</span> <em>Journal of Mass Spectrometry</em> 48 (3): 291–98. <a href="https://doi.org/10.1002/jms.3123">https://doi.org/10.1002/jms.3123</a>.
</div>
<div id="ref-ghaste2016" class="csl-entry">
Ghaste, Manoj, Robert Mistrik, and Vladimir Shulaev. 2016. <span>“Applications of Fourier Transform Ion Cyclotron Resonance (FT-ICR) and Orbitrap Based High Resolution Mass Spectrometry in Metabolomics and Lipidomics.”</span> <em>International Journal of Molecular Sciences</em> 17 (6). <a href="https://doi.org/10.3390/ijms17060816">https://doi.org/10.3390/ijms17060816</a>.
</div>
<div id="ref-ghosson2021" class="csl-entry">
Ghosson, Hikmat, Yann Guitton, Amani Ben Jrad, Chandrashekhar Patil, Delphine Raviglione, Marie-Virginie Salvia, and Cédric Bertrand. 2021. <span>“Electrospray Ionization and Heterogeneous Matrix Effects in Liquid Chromatography/Mass Spectrometry Based Meta-Metabolomics: A Biomarker or a Suppressed Ion?”</span> <em>Rapid Communications in Mass Spectrometry</em> 35 (2): e8977. https://doi.org/<a href="https://doi.org/10.1002/rcm.8977">https://doi.org/10.1002/rcm.8977</a>.
</div>
<div id="ref-giacomoni2015" class="csl-entry">
Giacomoni, Franck, Gildas Le Corguillé, Misharl Monsoor, Marion Landi, Pierre Pericard, Mélanie Pétéra, Christophe Duperier, et al. 2015. <span>“Workflow4Metabolomics: A Collaborative Research Infrastructure for Computational Metabolomics.”</span> <em>Bioinformatics</em> 31 (9): 1493–95. <a href="https://doi.org/10.1093/bioinformatics/btu813">https://doi.org/10.1093/bioinformatics/btu813</a>.
</div>
<div id="ref-gika2014" class="csl-entry">
Gika, Helen G., Georgios A. Theodoridis, Robert S. Plumb, and Ian D. Wilson. 2014. <span>“Current Practice of Liquid Chromatography<span><span></span></span>mass Spectrometry in Metabolomics and Metabonomics.”</span> <em>Journal of Pharmaceutical and Biomedical Analysis</em>, Review papers on pharmaceutical and biomedical analysis 2013, 87 (January): 12–25. <a href="https://doi.org/10.1016/j.jpba.2013.06.032">https://doi.org/10.1016/j.jpba.2013.06.032</a>.
</div>
<div id="ref-gil2015" class="csl-entry">
Gil, Andres, David Siegel, Hjalmar Permentier, Dirk-Jan Reijngoud, Frank Dekker, and Rainer Bischoff. 2015. <span>“Stability of Energy Metabolites<span><span></span></span>An Often Overlooked Issue in Metabolomics Studies: A Review.”</span> <em>ELECTROPHORESIS</em> 36 (18): 2156–69. <a href="https://doi.org/10.1002/elps.201500031">https://doi.org/10.1002/elps.201500031</a>.
</div>
<div id="ref-goldansaz2017" class="csl-entry">
Goldansaz, Seyed Ali, An Chi Guo, Tanvir Sajed, Michael A. Steele, Graham S. Plastow, and David S. Wishart. 2017. <span>“Livestock Metabolomics and the Livestock Metabolome: A Systematic Review.”</span> <em>PLOS ONE</em> 12 (5): e0177675. <a href="https://doi.org/10.1371/journal.pone.0177675">https://doi.org/10.1371/journal.pone.0177675</a>.
</div>
<div id="ref-gonzález-riano2020" class="csl-entry">
González-Riano, Carolina, Danuta Dudzik, Antonia Garcia, Alberto Gil-de-la-Fuente, Ana Gradillas, Joanna Godzien, Ángeles López-Gonzálvez, et al. 2020. <span>“Recent Developments Along the Analytical Process for Metabolomics Workflows.”</span> <em>Analytical Chemistry</em> 92 (1): 203–26. <a href="https://doi.org/10.1021/acs.analchem.9b04553">https://doi.org/10.1021/acs.analchem.9b04553</a>.
</div>
<div id="ref-griffiths2010a" class="csl-entry">
Griffiths, William J., Therese Koal, Yuqin Wang, Matthias Kohl, David P. Enot, and Hans-Peter Deigner. 2010. <span>“Targeted Metabolomics for Biomarker Discovery.”</span> <em>Angewandte Chemie International Edition</em> 49 (32): 5426–45. <a href="https://doi.org/10.1002/anie.200905579">https://doi.org/10.1002/anie.200905579</a>.
</div>
<div id="ref-gromski2015a" class="csl-entry">
Gromski, Piotr S., Howbeer Muhamadali, David I. Ellis, Yun Xu, Elon Correa, Michael L. Turner, and Royston Goodacre. 2015. <span>“A Tutorial Review: Metabolomics and Partial Least Squares-Discriminant Analysis <span><span></span></span> a Marriage of Convenience or a Shotgun Wedding.”</span> <em>Analytica Chimica Acta</em> 879 (June): 10–23. <a href="https://doi.org/10.1016/j.aca.2015.02.012">https://doi.org/10.1016/j.aca.2015.02.012</a>.
</div>
<div id="ref-gugisch2015" class="csl-entry">
Gugisch, Ralf, Adalbert Kerber, Axel Kohnert, Reinhard Laue, Markus Meringer, Christoph Rücker, and Alfred Wassermann. 2015. <span>“Chapter 6 - MOLGEN 5.0, A Molecular Structure Generator.”</span> In, edited by Subhash C. Basak, Guillermo Restrepo, and José L. Villaveces, 113–38. Bentham Science Publishers. <a href="https://doi.org/10.1016/B978-1-68108-198-4.50006-0">https://doi.org/10.1016/B978-1-68108-198-4.50006-0</a>.
</div>
<div id="ref-guijas2018" class="csl-entry">
Guijas, Carlos, J. Rafael Montenegro-Burke, Xavier Domingo-Almenara, Amelia Palermo, Benedikt Warth, Gerrit Hermann, Gunda Koellensperger, et al. 2018. <span>“METLIN: A Technology Platform for Identifying Knowns and Unknowns.”</span> <em>Analytical Chemistry</em> 90 (5): 3156–64. <a href="https://doi.org/10.1021/acs.analchem.7b04424">https://doi.org/10.1021/acs.analchem.7b04424</a>.
</div>
<div id="ref-hansen2018" class="csl-entry">
Hansen, Rebecca L., and Young Jin Lee. 2018b. <span>“High-Spatial Resolution Mass Spectrometry Imaging: Toward Single Cell Metabolomics in Plant Tissues.”</span> <em>The Chemical Record</em> 18 (1): 65–77. <a href="https://doi.org/10.1002/tcr.201700027">https://doi.org/10.1002/tcr.201700027</a>.
</div>
<div id="ref-hansen2018a" class="csl-entry">
———. 2018a. <span>“High-Spatial Resolution Mass Spectrometry Imaging: Toward Single Cell Metabolomics in Plant Tissues.”</span> <em>The Chemical Record</em> 18 (1): 65–77. <a href="https://doi.org/10.1002/tcr.201700027">https://doi.org/10.1002/tcr.201700027</a>.
</div>
<div id="ref-haug2017" class="csl-entry">
Haug, Kenneth, Reza M Salek, and Christoph Steinbeck. 2017. <span>“Global Open Data Management in Metabolomics.”</span> <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 58–63. <a href="https://doi.org/10.1016/j.cbpa.2016.12.024">https://doi.org/10.1016/j.cbpa.2016.12.024</a>.
</div>
<div id="ref-hernandes2017" class="csl-entry">
Hernandes, Vinicius Veri, Coral Barbas, and Danuta Dudzik. 2017. <span>“A Review of Blood Sample Handling and Pre-Processing for Metabolomics Studies.”</span> <em>ELECTROPHORESIS</em> 38 (18): 2232–41. <a href="https://doi.org/10.1002/elps.201700086">https://doi.org/10.1002/elps.201700086</a>.
</div>
<div id="ref-hiller2009" class="csl-entry">
Hiller, Karsten, Jasper Hangebrauk, Christian Jäger, Jana Spura, Kerstin Schreiber, and Dietmar Schomburg. 2009. <span>“MetaboliteDetector: Comprehensive Analysis Tool for Targeted and Nontargeted GC/MS Based Metabolome Analysis.”</span> <em>Analytical Chemistry</em> 81 (9): 3429–39. <a href="https://doi.org/10.1021/ac802689c">https://doi.org/10.1021/ac802689c</a>.
</div>
<div id="ref-hites2019" class="csl-entry">
Hites, Ronald A. 2019. <span>“Correcting for Censored Environmental Measurements.”</span> <em>Environmental Science &amp; Technology</em>, September. <a href="https://doi.org/10.1021/acs.est.9b05042">https://doi.org/10.1021/acs.est.9b05042</a>.
</div>
<div id="ref-hites2018" class="csl-entry">
Hites, Ronald A., and Karl J. Jobst. 2018. <span>“Is Nontargeted Screening Reproducible?”</span> <em>Environmental Science &amp; Technology</em> 52 (21): 11975–76. <a href="https://doi.org/10.1021/acs.est.8b05671">https://doi.org/10.1021/acs.est.8b05671</a>.
</div>
<div id="ref-hu2019" class="csl-entry">
Hu, Yaxi, Betty Cai, and Tao Huan. 2019. <span>“Enhancing Metabolome Coverage in Data-Dependent LC<span><span></span></span>MS/MS Analysis Through an Integrated Feature Extraction Strategy.”</span> <em>Analytical Chemistry</em> 91 (22): 14433–41. <a href="https://doi.org/10.1021/acs.analchem.9b02980">https://doi.org/10.1021/acs.analchem.9b02980</a>.
</div>
<div id="ref-huan2017" class="csl-entry">
Huan, Tao, Erica M. Forsberg, Duane Rinehart, Caroline H. Johnson, Julijana Ivanisevic, H. Paul Benton, Mingliang Fang, et al. 2017. <span>“Systems Biology Guided by XCMS Online Metabolomics.”</span> <em>Nature Methods</em> 14 (5): 461–62. <a href="https://doi.org/10.1038/nmeth.4260">https://doi.org/10.1038/nmeth.4260</a>.
</div>
<div id="ref-hufsky2014" class="csl-entry">
Hufsky, Franziska, Kerstin Scheubert, and Sebastian Böcker. 2014. <span>“Computational Mass Spectrometry for Small-Molecule Fragmentation.”</span> <em>TrAC Trends in Analytical Chemistry</em> 53 (January): 41–48. <a href="https://doi.org/10.1016/j.trac.2013.09.008">https://doi.org/10.1016/j.trac.2013.09.008</a>.
</div>
<div id="ref-ibanez2017" class="csl-entry">
Ibáñez, Clara, Carolina Simó, Mine Palazoglu, and Alejandro Cifuentes. 2017. <span>“GC-MS Based Metabolomics of Colon Cancer Cells Using Different Extraction Solvents.”</span> <em>Analytica Chimica Acta</em> 986 (September): 48–56. <a href="https://doi.org/10.1016/j.aca.2017.07.019">https://doi.org/10.1016/j.aca.2017.07.019</a>.
</div>
<div id="ref-jacyna2019" class="csl-entry">
Jacyna, Julia, Marta Kordalewska, and Michał J. Markuszewski. 2019. <span>“Design of Experiments in Metabolomics-Related Studies: An Overview.”</span> <em>Journal of Pharmaceutical and Biomedical Analysis</em> 164 (February): 598–606. <a href="https://doi.org/10.1016/j.jpba.2018.11.027">https://doi.org/10.1016/j.jpba.2018.11.027</a>.
</div>
<div id="ref-jaeger2016" class="csl-entry">
Jaeger, Carsten, Friederike Hoffmann, Clemens A. Schmitt, and Jan Lisec. 2016. <span>“Automated Annotation and Evaluation of in-Source Mass Spectra in GC/Atmospheric Pressure Chemical Ionization-MS-Based Metabolomics.”</span> <em>Analytical Chemistry</em> 88 (19): 9386–90. <a href="https://doi.org/10.1021/acs.analchem.6b02743">https://doi.org/10.1021/acs.analchem.6b02743</a>.
</div>
<div id="ref-jalili2020a" class="csl-entry">
Jalili, Vahid, Enis Afgan, Qiang Gu, Dave Clements, Daniel Blankenberg, Jeremy Goecks, James Taylor, and Anton Nekrutenko. 2020. <span>“The Galaxy Platform for Accessible, Reproducible and Collaborative Biomedical Analyses: 2020 Update.”</span> <em>Nucleic Acids Research</em> 48 (W1): W395–402. <a href="https://doi.org/10.1093/nar/gkaa434">https://doi.org/10.1093/nar/gkaa434</a>.
</div>
<div id="ref-jang2018" class="csl-entry">
Jang, Cholsoon, Li Chen, and Joshua D. Rabinowitz. 2018. <span>“Metabolomics and Isotope Tracing.”</span> <em>Cell</em> 173 (4): 822–37. <a href="https://doi.org/10.1016/j.cell.2018.03.055">https://doi.org/10.1016/j.cell.2018.03.055</a>.
</div>
<div id="ref-jones2012" class="csl-entry">
Jones, Dean P., Youngja Park, and Thomas R. Ziegler. 2012. <span>“Nutritional Metabolomics: Progress in Addressing Complexity in Diet and Health.”</span> <em>Annual Review of Nutrition</em> 32 (1): 183–202. <a href="https://doi.org/10.1146/annurev-nutr-072610-145159">https://doi.org/10.1146/annurev-nutr-072610-145159</a>.
</div>
<div id="ref-jorge2016a" class="csl-entry">
Jorge, Tiago F., Ana T. Mata, and Carla António. 2016a. <span>“Mass Spectrometry as a Quantitative Tool in Plant Metabolomics.”</span> <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150370. <a href="https://doi.org/10.1098/rsta.2015.0370">https://doi.org/10.1098/rsta.2015.0370</a>.
</div>
<div id="ref-jorge2016aa" class="csl-entry">
———. 2016b. <span>“Mass Spectrometry as a Quantitative Tool in Plant Metabolomics.”</span> <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150370. <a href="https://doi.org/10.1098/rsta.2015.0370">https://doi.org/10.1098/rsta.2015.0370</a>.
</div>
<div id="ref-jr2017" class="csl-entry">
Jr, Stephen Salerno, Mahya Mehrmohamadi, Maria V. Liberti, Muting Wan, Martin T. Wells, James G. Booth, and Jason W. Locasale. 2017. <span>“RRmix: A Method for Simultaneous Batch Effect Correction and Analysis of Metabolomics Data in the Absence of Internal Standards.”</span> <em>PLOS ONE</em> 12 (6): e0179530. <a href="https://doi.org/10.1371/journal.pone.0179530">https://doi.org/10.1371/journal.pone.0179530</a>.
</div>
<div id="ref-ju2020" class="csl-entry">
Ju, Ran, Xinyu Liu, Fujian Zheng, Xinjie Zhao, Xin Lu, Xiaohui Lin, Zhongda Zeng, and Guowang Xu. 2020. <span>“A Graph Density-Based Strategy for Features Fusion from Different Peak Extract Software to Achieve More Metabolites in Metabolic Profiling from High-Resolution Mass Spectrometry.”</span> <em>Analytica Chimica Acta</em> 1139 (December): 8–14. <a href="https://doi.org/10.1016/j.aca.2020.09.029">https://doi.org/10.1016/j.aca.2020.09.029</a>.
</div>
<div id="ref-kachman2020" class="csl-entry">
Kachman, Maureen, Hani Habra, William Duren, Janis Wigginton, Peter Sajjakulnukit, George Michailidis, Charles Burant, and Alla Karnovsky. 2020. <span>“Deep Annotation of Untargeted LC-MS Metabolomics Data with Binner.”</span> <em>Bioinformatics</em> 36 (6): 1801–6. <a href="https://doi.org/10.1093/bioinformatics/btz798">https://doi.org/10.1093/bioinformatics/btz798</a>.
</div>
<div id="ref-kapoore2016b" class="csl-entry">
Kapoore, Rahul Vijay, and Seetharaman Vaidyanathan. 2016. <span>“Towards Quantitative Mass Spectrometry-Based Metabolomics in Microbial and Mammalian Systems.”</span> <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150363. <a href="https://doi.org/10.1098/rsta.2015.0363">https://doi.org/10.1098/rsta.2015.0363</a>.
</div>
<div id="ref-karpievitch2014a" class="csl-entry">
Karpievitch, Yuliya V., Sonja B. Nikolic, Richard Wilson, James E. Sharman, and Lindsay M. Edwards. 2014. <span>“Metabolomics Data Normalization with EigenMS.”</span> <em>PLOS ONE</em> 9 (12): e116221. <a href="https://doi.org/10.1371/journal.pone.0116221">https://doi.org/10.1371/journal.pone.0116221</a>.
</div>
<div id="ref-kennedy2018" class="csl-entry">
Kennedy, Adam D., Bryan M. Wittmann, Anne M. Evans, Luke A. D. Miller, Douglas R. Toal, Shaun Lonergan, Sarah H. Elsea, and Kirk L. Pappan. 2018. <span>“Metabolomics in the Clinic: A Review of the Shared and Unique Features of Untargeted Metabolomics for Clinical Research and Clinical Testing.”</span> <em>Journal of Mass Spectrometry</em> 53 (11): 1143–54. <a href="https://doi.org/10.1002/jms.4292">https://doi.org/10.1002/jms.4292</a>.
</div>
<div id="ref-kew2017" class="csl-entry">
Kew, William, John W. T. Blackburn, David J. Clarke, and Dušan Uhrín. 2017. <span>“Interactive van Krevelen Diagrams <span><span></span></span> Advanced Visualisation of Mass Spectrometry Data of Complex Mixtures.”</span> <em>Rapid Communications in Mass Spectrometry</em> 31 (7): 658–62. <a href="https://doi.org/10.1002/rcm.7823">https://doi.org/10.1002/rcm.7823</a>.
</div>
<div id="ref-kim2020" class="csl-entry">
Kim, Jungyeon, Joong Kyong Ahn, Yu Eun Cheong, Sung-Joon Lee, Hoon-Suk Cha, and Kyoung Heon Kim. 2020. <span>“Systematic re-evaluation of the long-used standard protocol of urease-dependent metabolome sample preparation.”</span> <em>PloS One</em> 15 (3): e0230072. <a href="https://doi.org/10.1371/journal.pone.0230072">https://doi.org/10.1371/journal.pone.0230072</a>.
</div>
<div id="ref-kind2007" class="csl-entry">
Kind, Tobias, and Oliver Fiehn. 2007. <span>“Seven Golden Rules for Heuristic Filtering of Molecular Formulas Obtained by Accurate Mass Spectrometry.”</span> <em>BMC Bioinformatics</em> 8 (1): 105. <a href="https://doi.org/10.1186/1471-2105-8-105">https://doi.org/10.1186/1471-2105-8-105</a>.
</div>
<div id="ref-kind2018" class="csl-entry">
Kind, Tobias, Hiroshi Tsugawa, Tomas Cajka, Yan Ma, Zijuan Lai, Sajjan S. Mehta, Gert Wohlgemuth, et al. 2018. <span>“Identification of Small Molecules Using Accurate Mass MS/MS Search.”</span> <em>Mass Spectrometry Reviews</em> 37 (4): 513–32. https://doi.org/<a href="https://doi.org/10.1002/mas.21535">https://doi.org/10.1002/mas.21535</a>.
</div>
<div id="ref-koelmel2017" class="csl-entry">
Koelmel, Jeremy P., Nicholas M. Kroeger, Candice Z. Ulmer, John A. Bowden, Rainey E. Patterson, Jason A. Cochran, Christopher W. W. Beecher, Timothy J. Garrett, and Richard A. Yost. 2017. <span>“LipidMatch: An Automated Workflow for Rule-Based Lipid Identification Using Untargeted High-Resolution Tandem Mass Spectrometry Data.”</span> <em>BMC Bioinformatics</em> 18 (July): 331. <a href="https://doi.org/10.1186/s12859-017-1744-3">https://doi.org/10.1186/s12859-017-1744-3</a>.
</div>
<div id="ref-kouil" class="csl-entry">
Kouřil, Štěpán, Julie de Sousa, Jan Václavík, David Friedecký, and Tomáš Adam. n.d. <span>“CROP: Correlation-Based Reduction of Feature Multiplicities in Untargeted Metabolomic Data.”</span> <em>Bioinformatics</em>. <a href="https://doi.org/10.1093/bioinformatics/btaa012">https://doi.org/10.1093/bioinformatics/btaa012</a>.
</div>
<div id="ref-kuhl2012" class="csl-entry">
Kuhl, Carsten, Ralf Tautenhahn, Christoph Böttcher, Tony R. Larson, and Steffen Neumann. 2012. <span>“CAMERA: An Integrated Strategy for Compound Spectra Extraction and Annotation of Liquid Chromatography/Mass Spectrometry Data Sets.”</span> <em>Analytical Chemistry</em> 84 (1): 283–89. <a href="https://doi.org/10.1021/ac202450g">https://doi.org/10.1021/ac202450g</a>.
</div>
<div id="ref-kuligowski2015a" class="csl-entry">
Kuligowski, Julia, Ángel Sánchez-Illana, Daniel Sanjuán-Herráez, Máximo Vento, and Guillermo Quintás. 2015. <span>“Intra-Batch Effect Correction in Liquid Chromatography-Mass Spectrometry Using Quality Control Samples and Support Vector Regression (QC-SVRC).”</span> <em>Analyst</em> 140 (22): 7810–17. <a href="https://doi.org/10.1039/C5AN01638J">https://doi.org/10.1039/C5AN01638J</a>.
</div>
<div id="ref-kusonmano2016" class="csl-entry">
Kusonmano, Kanthida, Wanwipa Vongsangnak, and Pramote Chumnanpuen. 2016a. <span>“Informatics for Metabolomics.”</span> In, 91–115. Advances in Experimental Medicine and Biology. Springer, Singapore. <a href="https://link.springer.com/chapter/10.1007/978-981-10-1503-8_5">https://link.springer.com/chapter/10.1007/978-981-10-1503-8_5</a>.
</div>
<div id="ref-kusonmano2016a" class="csl-entry">
———. 2016b. <span>“Informatics for Metabolomics.”</span> In, 91–115. Advances in Experimental Medicine and Biology. Springer, Singapore. <a href="https://link.springer.com/chapter/10.1007/978-981-10-1503-8_5">https://link.springer.com/chapter/10.1007/978-981-10-1503-8_5</a>.
</div>
<div id="ref-lai2018" class="csl-entry">
Lai, Zijuan, Hiroshi Tsugawa, Gert Wohlgemuth, Sajjan Mehta, Matthew Mueller, Yuxuan Zheng, Atsushi Ogiwara, et al. 2018a. <span>“Identifying Metabolites by Integrating Metabolome Databases with Mass Spectrometry Cheminformatics.”</span> <em>Nature Methods</em> 15 (1): 53–56. <a href="https://doi.org/10.1038/nmeth.4512">https://doi.org/10.1038/nmeth.4512</a>.
</div>
<div id="ref-lai2018a" class="csl-entry">
———, et al. 2018b. <span>“Identifying Metabolites by Integrating Metabolome Databases with Mass Spectrometry Cheminformatics.”</span> <em>Nature Methods</em> 15 (1): 53–56. <a href="https://doi.org/10.1038/nmeth.4512">https://doi.org/10.1038/nmeth.4512</a>.
</div>
<div id="ref-lakhani2019" class="csl-entry">
Lakhani, Chirag M., Braden T. Tierney, Arjun K. Manrai, Jian Yang, Peter M. Visscher, and Chirag J. Patel. 2019. <span>“Repurposing Large Health Insurance Claims Data to Estimate Genetic and Environmental Contributions in 560 Phenotypes.”</span> <em>Nature Genetics</em>, January, 1. <a href="https://doi.org/10.1038/s41588-018-0313-7">https://doi.org/10.1038/s41588-018-0313-7</a>.
</div>
<div id="ref-laparre2017" class="csl-entry">
Laparre, Jérôme, Zied Kaabia, Mark Mooney, Tom Buckley, Mark Sherry, Bruno Le Bizec, and Gaud Dervilly-Pinel. 2017a. <span>“Impact of Storage Conditions on the Urinary Metabolomics Fingerprint.”</span> <em>Analytica Chimica Acta</em> 951 (January): 99–107. <a href="https://doi.org/10.1016/j.aca.2016.11.055">https://doi.org/10.1016/j.aca.2016.11.055</a>.
</div>
<div id="ref-laparre2017a" class="csl-entry">
———. 2017b. <span>“Impact of Storage Conditions on the Urinary Metabolomics Fingerprint.”</span> <em>Analytica Chimica Acta</em> 951 (January): 99–107. <a href="https://doi.org/10.1016/j.aca.2016.11.055">https://doi.org/10.1016/j.aca.2016.11.055</a>.
</div>
<div id="ref-larralde2017" class="csl-entry">
Larralde, Martin, Thomas N. Lawson, Ralf J. M. Weber, Pablo Moreno, Kenneth Haug, Philippe Rocca-Serra, Mark R. Viant, Christoph Steinbeck, and Reza M. Salek. 2017. <span>“mzML2ISA &amp; nmrML2ISA: Generating Enriched ISA-Tab Metadata Files from Metabolomics XML Data.”</span> <em>Bioinformatics</em> 33 (16): 2598–2600. <a href="https://doi.org/10.1093/bioinformatics/btx169">https://doi.org/10.1093/bioinformatics/btx169</a>.
</div>
<div id="ref-lawson2017" class="csl-entry">
Lawson, Thomas N., Ralf J. M. Weber, Martin R. Jones, Andrew J. Chetwynd, Giovanny Rodrıǵuez-Blanco, Riccardo Di Guida, Mark R. Viant, and Warwick B. Dunn. 2017. <span>“msPurity: Automated Evaluation of Precursor Ion Purity for Mass Spectrometry-Based Fragmentation in Metabolomics.”</span> <em>Analytical Chemistry</em> 89 (4): 2432–39. <a href="https://doi.org/10.1021/acs.analchem.6b04358">https://doi.org/10.1021/acs.analchem.6b04358</a>.
</div>
<div id="ref-leek2012" class="csl-entry">
Leek, Jeffrey T., W. Evan Johnson, Hilary S. Parker, Andrew E. Jaffe, and John D. Storey. 2012. <span>“The Sva Package for Removing Batch Effects and Other Unwanted Variation in High-Throughput Experiments.”</span> <em>Bioinformatics</em> 28 (6): 882–83. <a href="https://doi.org/10.1093/bioinformatics/bts034">https://doi.org/10.1093/bioinformatics/bts034</a>.
</div>
<div id="ref-leek2007" class="csl-entry">
Leek, Jeffrey T., and John D. Storey. 2007. <span>“Capturing Heterogeneity in Gene Expression Studies by Surrogate Variable Analysis.”</span> <em>PLOS Genet</em> 3 (9): e161. <a href="https://doi.org/10.1371/journal.pgen.0030161">https://doi.org/10.1371/journal.pgen.0030161</a>.
</div>
<div id="ref-leek2008" class="csl-entry">
———. 2008. <span>“A General Framework for Multiple Testing Dependence.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (48): 18718–23. <a href="https://doi.org/10.1073/pnas.0808709105">https://doi.org/10.1073/pnas.0808709105</a>.
</div>
<div id="ref-levy2019" class="csl-entry">
Levy, Allison J., Nicholas R. Oranzi, Atiye Ahmadireskety, Robin H. J. Kemperman, Michael S. Wei, and Richard A. Yost. 2019. <span>“Recent Progress in Metabolomics Using Ion Mobility-Mass Spectrometry.”</span> <em>TrAC Trends in Analytical Chemistry</em> 116 (July): 274–81. <a href="https://doi.org/10.1016/j.trac.2019.05.001">https://doi.org/10.1016/j.trac.2019.05.001</a>.
</div>
<div id="ref-lêcao2011" class="csl-entry">
Lê Cao, Kim-Anh, Simon Boitard, and Philippe Besse. 2011. <span>“Sparse PLS Discriminant Analysis: Biologically Relevant Feature Selection and Graphical Displays for Multiclass Problems.”</span> <em>BMC Bioinformatics</em> 12 (June): 253. <a href="https://doi.org/10.1186/1471-2105-12-253">https://doi.org/10.1186/1471-2105-12-253</a>.
</div>
<div id="ref-li2017a" class="csl-entry">
Li, Bo, Jing Tang, Qingxia Yang, Shuang Li, Xuejiao Cui, Yinghong Li, Yuzong Chen, Weiwei Xue, Xiaofeng Li, and Feng Zhu. 2017. <span>“NOREVA: Normalization and Evaluation of MS-Based Metabolomics Data.”</span> <em>Nucleic Acids Research</em> 45 (W1): W162–70. <a href="https://doi.org/10.1093/nar/gkx449">https://doi.org/10.1093/nar/gkx449</a>.
</div>
<div id="ref-li2016" class="csl-entry">
Li, Hao, Yuping Cai, Yuan Guo, Fangfang Chen, and Zheng-Jiang Zhu. 2016. <span>“MetDIA: Targeted Metabolite Extraction of Multiplexed MS/MS Spectra Generated by Data-Independent Acquisition.”</span> <em>Analytical Chemistry</em> 88 (17): 8757–64. <a href="https://doi.org/10.1021/acs.analchem.6b02122">https://doi.org/10.1021/acs.analchem.6b02122</a>.
</div>
<div id="ref-li2013a" class="csl-entry">
Li, Liang, Ronghong Li, Jianjun Zhou, Azeret Zuniga, Avalyn E. Stanislaus, Yiman Wu, Tao Huan, et al. 2013. <span>“MyCompoundID: Using an Evidence-Based Metabolome Library for Metabolite Identification.”</span> <em>Analytical Chemistry</em> 85 (6): 3401–8. <a href="https://doi.org/10.1021/ac400099b">https://doi.org/10.1021/ac400099b</a>.
</div>
<div id="ref-li2017" class="csl-entry">
Li, Lili, Weijie Ren, Hongwei Kong, Chunxia Zhao, Xinjie Zhao, Xiaohui Lin, Xin Lu, and Guowang Xu. 2017. <span>“An Alignment Algorithm for LC-MS-Based Metabolomics Dataset Assisted by MS/MS Information.”</span> <em>Analytica Chimica Acta</em> 990 (October): 96–102. <a href="https://doi.org/10.1016/j.aca.2017.07.058">https://doi.org/10.1016/j.aca.2017.07.058</a>.
</div>
<div id="ref-li2020" class="csl-entry">
Li, Shuzhao. 2020. <em>Computational Methods and Data Analysis for Metabolomics</em>. Springer.
</div>
<div id="ref-li2013" class="csl-entry">
Li, Shuzhao, Youngja Park, Sai Duraisingham, Frederick H. Strobel, Nooruddin Khan, Quinlyn A. Soltow, Dean P. Jones, and Bali Pulendran. 2013. <span>“Predicting Network Activity from High Throughput Metabolomics.”</span> <em>PLOS Computational Biology</em> 9 (7): e1003123. <a href="https://doi.org/10.1371/journal.pcbi.1003123">https://doi.org/10.1371/journal.pcbi.1003123</a>.
</div>
<div id="ref-li2018" class="csl-entry">
Li, Zhucui, Yan Lu, Yufeng Guo, Haijie Cao, Qinhong Wang, and Wenqing Shui. 2018. <span>“Comprehensive Evaluation of Untargeted Metabolomics Data Processing Software in Feature Detection, Quantification and Discriminating Marker Selection.”</span> <em>Analytica Chimica Acta</em> 1029 (October): 50–57. <a href="https://doi.org/10.1016/j.aca.2018.05.001">https://doi.org/10.1016/j.aca.2018.05.001</a>.
</div>
<div id="ref-libiseller2015" class="csl-entry">
Libiseller, Gunnar, Michaela Dvorzak, Ulrike Kleb, Edgar Gander, Tobias Eisenberg, Frank Madeo, Steffen Neumann, et al. 2015. <span>“IPO: A Tool for Automated Optimization of XCMS Parameters.”</span> <em>BMC Bioinformatics</em> 16 (April): 118. <a href="https://doi.org/10.1186/s12859-015-0562-8">https://doi.org/10.1186/s12859-015-0562-8</a>.
</div>
<div id="ref-lin2007" class="csl-entry">
Lin, Ching Yu, Huifeng Wu, Ronald S. Tjeerdema, and Mark R. Viant. 2007. <span>“Evaluation of Metabolite Extraction Strategies from Tissue Samples Using NMR Metabolomics.”</span> <em>Metabolomics</em> 3 (1): 55–67. <a href="https://doi.org/10.1007/s11306-006-0043-1">https://doi.org/10.1007/s11306-006-0043-1</a>.
</div>
<div id="ref-lisec2016" class="csl-entry">
Lisec, Jan, Friederike Hoffmann, Clemens Schmitt, and Carsten Jaeger. 2016. <span>“Extending the Dynamic Range in Metabolomics Experiments by Automatic Correction of Peaks Exceeding the Detection Limit.”</span> <em>Analytical Chemistry</em> 88 (15): 7487–92. <a href="https://doi.org/10.1021/acs.analchem.6b02515">https://doi.org/10.1021/acs.analchem.6b02515</a>.
</div>
<div id="ref-liu2020" class="csl-entry">
Liu, Qin, Douglas Walker, Karan Uppal, Zihe Liu, Chunyu Ma, ViLinh Tran, Shuzhao Li, Dean P. Jones, and Tianwei Yu. 2020. <span>“Addressing the Batch Effect Issue for LC/MS Metabolomics Data in Data Preprocessing.”</span> <em>Scientific Reports</em> 10 (1): 13856. <a href="https://doi.org/10.1038/s41598-020-70850-0">https://doi.org/10.1038/s41598-020-70850-0</a>.
</div>
<div id="ref-liu2019" class="csl-entry">
Liu, Xinyu, Lina Zhou, Xianzhe Shi, and Guowang Xu. 2019. <span>“New Advances in Analytical Methods for Mass Spectrometry-Based Large-Scale Metabolomics Study.”</span> <em>TrAC Trends in Analytical Chemistry</em> 121 (December): 115665. <a href="https://doi.org/10.1016/j.trac.2019.115665">https://doi.org/10.1016/j.trac.2019.115665</a>.
</div>
<div id="ref-livera2015" class="csl-entry">
Livera, Alysha M. De, Marko Sysi-Aho, Laurent Jacob, Johann A. Gagnon-Bartsch, Sandra Castillo, Julie A. Simpson, and Terence P. Speed. 2015. <span>“Statistical Methods for Handling Unwanted Variation in Metabolomics Data.”</span> <em>Analytical Chemistry</em> 87 (7): 3606–15. <a href="https://doi.org/10.1021/ac502439y">https://doi.org/10.1021/ac502439y</a>.
</div>
<div id="ref-loftfield2016" class="csl-entry">
Loftfield, Erikka, Emily Vogtmann, Joshua N. Sampson, Steven C. Moore, Heidi Nelson, Rob Knight, Nicholas Chia, and Rashmi Sinha. 2016. <span>“Comparison of Collection Methods for Fecal Samples for Discovery Metabolomics in Epidemiologic Studies.”</span> <em>Cancer Epidemiology and Prevention Biomarkers</em> 25 (11): 1483–90. <a href="https://doi.org/10.1158/1055-9965.EPI-16-0409">https://doi.org/10.1158/1055-9965.EPI-16-0409</a>.
</div>
<div id="ref-loos2017" class="csl-entry">
Loos, Martin, and Heinz Singer. 2017. <span>“Nontargeted Homologue Series Extraction from Hyphenated High Resolution Mass Spectrometry Data.”</span> <em>Journal of Cheminformatics</em> 9 (February). <a href="https://doi.org/10.1186/s13321-017-0197-z">https://doi.org/10.1186/s13321-017-0197-z</a>.
</div>
<div id="ref-lu2008a" class="csl-entry">
Lu, Wenyun, Bryson D. Bennett, and Joshua D. Rabinowitz. 2008. <span>“Analytical Strategies for LC<span><span></span></span>MS-Based Targeted Metabolomics.”</span> <em>Journal of Chromatography B</em>, Hyphenated techniques for global metabolite profiling, 871 (2): 236–42. <a href="https://doi.org/10.1016/j.jchromb.2008.04.031">https://doi.org/10.1016/j.jchromb.2008.04.031</a>.
</div>
<div id="ref-lu2017" class="csl-entry">
Lu, Wenyun, Xiaoyang Su, Matthias S. Klein, Ian A. Lewis, Oliver Fiehn, and Joshua D. Rabinowitz. 2017. <span>“Metabolite Measurement: Pitfalls to Avoid and Practices to Follow.”</span> <em>Annual Review of Biochemistry</em> 86 (1): 277–304. <a href="https://doi.org/10.1146/annurev-biochem-061516-044952">https://doi.org/10.1146/annurev-biochem-061516-044952</a>.
</div>
<div id="ref-lu2008" class="csl-entry">
Lu, Xin, and Guowang Xu. 2008. <span>“LC-MS Metabonomics Methodology in Biomarker Discovery.”</span> In, edited by Feng Wang, 291–315. Methods in Pharmacology and Toxicology<span><span></span></span>. Humana Press. <a href="http://dx.doi.org/10.1007/978-1-59745-463-6_14">http://dx.doi.org/10.1007/978-1-59745-463-6_14</a>.
</div>
<div id="ref-ludwig2020" class="csl-entry">
Ludwig, Marcus, Louis-Félix Nothias, Kai Dührkop, Irina Koester, Markus Fleischauer, Martin A. Hoffmann, Daniel Petras, et al. 2020. <span>“Database-Independent Molecular Formula Annotation Using Gibbs Sampling Through ZODIAC.”</span> <em>Nature Machine Intelligence</em> 2 (10): 629–41. <a href="https://doi.org/10.1038/s42256-020-00234-6">https://doi.org/10.1038/s42256-020-00234-6</a>.
</div>
<div id="ref-luo2017a" class="csl-entry">
Luo, Xian, and Liang Li. 2017. <span>“Metabolomics of Small Numbers of Cells: Metabolomic Profiling of 100, 1000, and 10000 Human Breast Cancer Cells.”</span> <em>Analytical Chemistry</em> 89 (21): 11664–71. <a href="https://doi.org/10.1021/acs.analchem.7b03100">https://doi.org/10.1021/acs.analchem.7b03100</a>.
</div>
<div id="ref-ma2014" class="csl-entry">
Ma, Yan, Tobias Kind, Dawei Yang, Carlos Leon, and Oliver Fiehn. 2014. <span>“MS2Analyzer: A Software for Small Molecule Substructure Annotations from Accurate Tandem Mass Spectra.”</span> <em>Analytical Chemistry</em> 86 (21): 10724–31. <a href="https://doi.org/10.1021/ac502818e">https://doi.org/10.1021/ac502818e</a>.
</div>
<div id="ref-madsen2010" class="csl-entry">
Madsen, Rasmus, Torbjörn Lundstedt, and Johan Trygg. 2010a. <span>“Chemometrics in Metabolomics<span><span></span></span>a Review in Human Disease Diagnosis.”</span> <em>Analytica Chimica Acta</em> 659 (1): 23–33. <a href="https://doi.org/10.1016/j.aca.2009.11.042">https://doi.org/10.1016/j.aca.2009.11.042</a>.
</div>
<div id="ref-madsen2010a" class="csl-entry">
———. 2010b. <span>“Chemometrics in Metabolomics<span><span></span></span>a Review in Human Disease Diagnosis.”</span> <em>Analytica Chimica Acta</em> 659 (1): 23–33. <a href="https://doi.org/10.1016/j.aca.2009.11.042">https://doi.org/10.1016/j.aca.2009.11.042</a>.
</div>
<div id="ref-mahieu2017" class="csl-entry">
Mahieu, Nathaniel G., and Gary J. Patti. 2017. <span>“Systems-Level Annotation of a Metabolomics Data Set Reduces 25?000 Features to Fewer Than 1000 Unique Metabolites.”</span> <em>Analytical Chemistry</em> 89 (19): 10397–406. <a href="https://doi.org/10.1021/acs.analchem.7b02380">https://doi.org/10.1021/acs.analchem.7b02380</a>.
</div>
<div id="ref-mahieu2016aa" class="csl-entry">
Mahieu, Nathaniel G., Jonathan L. Spalding, Susan J. Gelman, and Gary J. Patti. 2016. <span>“Defining and Detecting Complex Peak Relationships in Mass Spectral Data: The Mz.unity Algorithm.”</span> <em>Analytical Chemistry</em> 88 (18): 9037–46. <a href="https://doi.org/10.1021/acs.analchem.6b01702">https://doi.org/10.1021/acs.analchem.6b01702</a>.
</div>
<div id="ref-mahieu2016" class="csl-entry">
Mahieu, Nathaniel G, Jessica Lloyd Genenbacher, and Gary J Patti. 2016a. <span>“A Roadmap for the XCMS Family of Software Solutions in Metabolomics.”</span> <em>Current Opinion in Chemical Biology</em> 30 (February): 87–93. <a href="https://doi.org/10.1016/j.cbpa.2015.11.009">https://doi.org/10.1016/j.cbpa.2015.11.009</a>.
</div>
<div id="ref-mahieu2016a" class="csl-entry">
———. 2016b. <span>“A Roadmap for the XCMS Family of Software Solutions in Metabolomics.”</span> <em>Current Opinion in Chemical Biology</em> 30 (February): 87–93. <a href="https://doi.org/10.1016/j.cbpa.2015.11.009">https://doi.org/10.1016/j.cbpa.2015.11.009</a>.
</div>
<div id="ref-mahmud2017" class="csl-entry">
Mahmud, Iqbal, Sandi Sternberg, Michael Williams, and Timothy J. Garrett. 2017. <span>“Comparison of Global Metabolite Extraction Strategies for Soybeans Using UHPLC-HRMS.”</span> <em>Analytical and Bioanalytical Chemistry</em> 409 (26): 6173–80. <a href="https://doi.org/10.1007/s00216-017-0557-6">https://doi.org/10.1007/s00216-017-0557-6</a>.
</div>
<div id="ref-mangul2019" class="csl-entry">
Mangul, Serghei, Thiago Mosqueiro, Richard J. Abdill, Dat Duong, Keith Mitchell, Varuni Sarwal, Brian Hill, et al. 2019. <span>“Challenges and Recommendations to Improve the Installability and Archival Stability of Omics Computational Tools.”</span> <em>PLOS Biology</em> 17 (6): e3000333. <a href="https://doi.org/10.1371/journal.pbio.3000333">https://doi.org/10.1371/journal.pbio.3000333</a>.
</div>
<div id="ref-mannhold2009" class="csl-entry">
Mannhold, Raimund, Gennadiy I. Poda, Claude Ostermann, and Igor V. Tetko. 2009. <span>“Calculation of Molecular Lipophilicity: State-of-the-Art and Comparison of LogP Methods on More Than 96,000 Compounds.”</span> <em>Journal of Pharmaceutical Sciences</em> 98 (3): 861–93. <a href="https://doi.org/10.1002/jps.21494">https://doi.org/10.1002/jps.21494</a>.
</div>
<div id="ref-mansouri2018" class="csl-entry">
Mansouri, Kamel, Chris M. Grulke, Richard S. Judson, and Antony J. Williams. 2018. <span>“OPERA Models for Predicting Physicochemical Properties and Environmental Fate Endpoints.”</span> <em>Journal of Cheminformatics</em> 10 (1): 10. <a href="https://doi.org/10.1186/s13321-018-0263-1">https://doi.org/10.1186/s13321-018-0263-1</a>.
</div>
<div id="ref-martens2017" class="csl-entry">
Martens, Jonathan, Giel Berden, Rianne E. van Outersterp, Leo A. J. Kluijtmans, Udo F. Engelke, Clara D. M. van Karnebeek, Ron A. Wevers, and Jos Oomens. 2017. <span>“Molecular Identification in Metabolomics Using Infrared Ion Spectroscopy.”</span> <em>Scientific Reports</em> 7 (June). <a href="https://doi.org/10.1038/s41598-017-03387-4">https://doi.org/10.1038/s41598-017-03387-4</a>.
</div>
<div id="ref-matich2019a" class="csl-entry">
Matich, Eryn K., Nita G. Chavez Soria, Diana S. Aga, and G. Ekin Atilla-Gokcumen. 2019. <span>“Applications of Metabolomics in Assessing Ecological Effects of Emerging Contaminants and Pollutants on Plants.”</span> <em>Journal of Hazardous Materials</em> 373 (July): 527–35. <a href="https://doi.org/10.1016/j.jhazmat.2019.02.084">https://doi.org/10.1016/j.jhazmat.2019.02.084</a>.
</div>
<div id="ref-matsuo2017" class="csl-entry">
Matsuo, Teruko, Hiroshi Tsugawa, Hiromi Miyagawa, and Eiichiro Fukusaki. 2017. <span>“Integrated Strategy for Unknown EI<span><span></span></span>MS Identification Using Quality Control Calibration Curve, Multivariate Analysis, EI<span><span></span></span>MS Spectral Database, and Retention Index Prediction.”</span> <em>Analytical Chemistry</em> 89 (12): 6766–73. <a href="https://doi.org/10.1021/acs.analchem.7b01010">https://doi.org/10.1021/acs.analchem.7b01010</a>.
</div>
<div id="ref-mclean2020" class="csl-entry">
McLean, Craig, and Elizabeth B. Kujawinski. 2020. <span>“AutoTuner: High Fidelity and Robust Parameter Selection for Metabolomics Data Processing.”</span> <em>Analytical Chemistry</em> 92 (8): 5724–32. <a href="https://doi.org/10.1021/acs.analchem.9b04804">https://doi.org/10.1021/acs.analchem.9b04804</a>.
</div>
<div id="ref-melamud2010a" class="csl-entry">
Melamud, Eugene, Livia Vastag, and Joshua D. Rabinowitz. 2010. <span>“Metabolomic Analysis and Visualization Engine for LC<span>-</span>MS Data.”</span> <em>Analytical Chemistry</em> 82 (23): 9818–26. <a href="https://doi.org/10.1021/ac1021166">https://doi.org/10.1021/ac1021166</a>.
</div>
<div id="ref-menikarachchi2012" class="csl-entry">
Menikarachchi, Lochana C., Shannon Cawley, Dennis W. Hill, L. Mark Hall, Lowell Hall, Steven Lai, Janine Wilder, and David F. Grant. 2012. <span>“MolFind: A Software Package Enabling HPLC/MS-Based Identification of Unknown Chemical Structures.”</span> <em>Analytical Chemistry</em> 84 (21): 9388–94. <a href="https://doi.org/10.1021/ac302048x">https://doi.org/10.1021/ac302048x</a>.
</div>
<div id="ref-miggiels2019" class="csl-entry">
Miggiels, Paul, Bert Wouters, Gerard J. P. van Westen, Anne-Charlotte Dubbelman, and Thomas Hankemeier. 2019. <span>“Novel Technologies for Metabolomics: More for Less.”</span> <em>TrAC Trends in Analytical Chemistry</em> 120 (November): 115323. <a href="https://doi.org/10.1016/j.trac.2018.11.021">https://doi.org/10.1016/j.trac.2018.11.021</a>.
</div>
<div id="ref-misra2018" class="csl-entry">
Misra, Biswapriya B. 2018. <span>“New Tools and Resources in Metabolomics: 2016<span><span></span></span>2017.”</span> <em>ELECTROPHORESIS</em> 39 (7): 909–23. <a href="https://doi.org/10.1002/elps.201700441">https://doi.org/10.1002/elps.201700441</a>.
</div>
<div id="ref-misra2017" class="csl-entry">
Misra, Biswapriya B., Johannes F. Fahrmann, and Dmitry Grapov. 2017. <span>“Review of Emerging Metabolomic Tools and Resources: 2015<span><span></span></span>2016.”</span> <em>ELECTROPHORESIS</em> 38 (18): 2257–74. <a href="https://doi.org/10.1002/elps.201700110">https://doi.org/10.1002/elps.201700110</a>.
</div>
<div id="ref-misra2016" class="csl-entry">
Misra, Biswapriya B., and Justin J. J. van der Hooft. 2016. <span>“Updates in Metabolomics Tools and Resources: 2014<span><span></span></span>2015.”</span> <em>ELECTROPHORESIS</em> 37 (1): 86–110. <a href="https://doi.org/10.1002/elps.201500417">https://doi.org/10.1002/elps.201500417</a>.
</div>
<div id="ref-miyagawa2019" class="csl-entry">
Miyagawa, Hiromi, and Takeshi Bamba. 2019. <span>“Comparison of Sequential Derivatization with Concurrent Methods for GC/MS-Based Metabolomics.”</span> <em>Journal of Bioscience and Bioengineering</em> 127 (2): 160–68. <a href="https://doi.org/10.1016/j.jbiosc.2018.07.015">https://doi.org/10.1016/j.jbiosc.2018.07.015</a>.
</div>
<div id="ref-montenegro-burke2017" class="csl-entry">
Montenegro-Burke, J. Rafael, Aries E. Aisporna, H. Paul Benton, Duane Rinehart, Mingliang Fang, Tao Huan, Benedikt Warth, et al. 2017. <span>“Data Streaming for Metabolomics: Accelerating Data Processing and Analysis from Days to Minutes.”</span> <em>Analytical Chemistry</em> 89 (2): 1254–59. <a href="https://doi.org/10.1021/acs.analchem.6b03890">https://doi.org/10.1021/acs.analchem.6b03890</a>.
</div>
<div id="ref-müller2020a" class="csl-entry">
Müller, Manfred J., and Anja Bosy-Westphal. 2020. <span>“From a <span>“</span>Metabolomics Fashion<span>”</span> to a Sound Application of Metabolomics in Research on Human Nutrition.”</span> <em>European Journal of Clinical Nutrition</em> 74 (12): 1619–29. <a href="https://doi.org/10.1038/s41430-020-00781-6">https://doi.org/10.1038/s41430-020-00781-6</a>.
</div>
<div id="ref-myers2017" class="csl-entry">
Myers, Owen D., Susan J. Sumner, Shuzhao Li, Stephen Barnes, and Xiuxia Du. 2017. <span>“Detailed Investigation and Comparison of the XCMS and MZmine 2 Chromatogram Construction and Chromatographic Peak Detection Methods for Preprocessing Mass Spectrometry Metabolomics Data.”</span> <em>Analytical Chemistry</em> 89 (17): 8689–95. <a href="https://doi.org/10.1021/acs.analchem.7b01069">https://doi.org/10.1021/acs.analchem.7b01069</a>.
</div>
<div id="ref-najdekr2016" class="csl-entry">
Najdekr, Lukáš, David Friedecký, Ralf Tautenhahn, Tomáš Pluskal, Junhua Wang, Yingying Huang, and Tomáš Adam. 2016. <span>“Influence of Mass Resolving Power in Orbital Ion-Trap Mass Spectrometry-Based Metabolomics.”</span> <em>Analytical Chemistry</em> 88 (23): 11429–35. <a href="https://doi.org/10.1021/acs.analchem.6b02319">https://doi.org/10.1021/acs.analchem.6b02319</a>.
</div>
<div id="ref-nash2019" class="csl-entry">
Nash, William J., and Warwick B. Dunn. 2019. <span>“From Mass to Metabolite in Human Untargeted Metabolomics: Recent Advances in Annotation of Metabolites Applying Liquid Chromatography-Mass Spectrometry Data.”</span> <em>TrAC Trends in Analytical Chemistry</em> 120 (November): 115324. <a href="https://doi.org/10.1016/j.trac.2018.11.022">https://doi.org/10.1016/j.trac.2018.11.022</a>.
</div>
<div id="ref-ni2016" class="csl-entry">
Ni, Yan, Mingming Su, Yunping Qiu, Wei Jia, and Xiuxia Du. 2016. <span>“ADAP-GC 3.0: Improved Peak Detection and Deconvolution of Co-Eluting Metabolites from GC/TOF-MS Data for Metabolomics Studies.”</span> <em>Analytical Chemistry</em> 88 (17): 8802–11. <a href="https://doi.org/10.1021/acs.analchem.6b02222">https://doi.org/10.1021/acs.analchem.6b02222</a>.
</div>
<div id="ref-nikolskiy2013" class="csl-entry">
Nikolskiy, Igor, Nathaniel G. Mahieu, Ying-Jr Chen, Ralf Tautenhahn, and Gary J. Patti. 2013. <span>“An Untargeted Metabolomic Workflow to Improve Structural Characterization of Metabolites.”</span> <em>Analytical Chemistry</em> 85 (16): 7713–19. <a href="https://doi.org/10.1021/ac400751j">https://doi.org/10.1021/ac400751j</a>.
</div>
<div id="ref-nothias2020" class="csl-entry">
Nothias, Louis-Félix, Daniel Petras, Robin Schmid, Kai Dührkop, Johannes Rainer, Abinesh Sarvepalli, Ivan Protsyuk, et al. 2020. <span>“Feature-based molecular networking in the GNPS analysis environment.”</span> <em>Nature Methods</em> 17 (9): 905–8. <a href="https://doi.org/10.1038/s41592-020-0933-6">https://doi.org/10.1038/s41592-020-0933-6</a>.
</div>
<div id="ref-nyamundanda2013" class="csl-entry">
Nyamundanda, Gift, Isobel Claire Gormley, Yue Fan, William M. Gallagher, and Lorraine Brennan. 2013. <span>“MetSizeR: Selecting the Optimal Sample Size for Metabolomic Studies Using an Analysis Based Approach.”</span> <em>BMC Bioinformatics</em> 14: 338. <a href="https://doi.org/10.1186/1471-2105-14-338">https://doi.org/10.1186/1471-2105-14-338</a>.
</div>
<div id="ref-oberg2009" class="csl-entry">
Oberg, Ann L., and Olga Vitek. 2009. <span>“Statistical Design of Quantitative Mass Spectrometry-Based Proteomic Experiments.”</span> <em>Journal of Proteome Research</em> 8 (5): 2144–56. <a href="https://doi.org/10.1021/pr8010099">https://doi.org/10.1021/pr8010099</a>.
</div>
<div id="ref-ortmayr2016a" class="csl-entry">
Ortmayr, Karin, Verena Charwat, Cornelia Kasper, Stephan Hann, and Gunda Koellensperger. 2016. <span>“Uncertainty Budgeting in Fold Change Determination and Implications for Non-Targeted Metabolomics Studies in Model Systems”</span> 142 (1): 80–90. <a href="https://doi.org/10.1039/C6AN01342B">https://doi.org/10.1039/C6AN01342B</a>.
</div>
<div id="ref-palmer2017" class="csl-entry">
Palmer, Andrew, Prasad Phapale, Ilya Chernyavsky, Regis Lavigne, Dominik Fay, Artem Tarasov, Vitaly Kovalev, et al. 2017. <span>“FDR-Controlled Metabolite Annotation for High-Resolution Imaging Mass Spectrometry.”</span> <em>Nature Methods</em> 14 (1): 57–60. <a href="https://doi.org/10.1038/nmeth.4072">https://doi.org/10.1038/nmeth.4072</a>.
</div>
<div id="ref-pang2020" class="csl-entry">
Pang, Zhiqiang, Jasmine Chong, Shuzhao Li, and Jianguo Xia. 2020. <span>“MetaboAnalystR 3.0: Toward an Optimized Workflow for Global Metabolomics.”</span> <em>Metabolites</em> 10 (5): 186. <a href="https://doi.org/10.3390/metabo10050186">https://doi.org/10.3390/metabo10050186</a>.
</div>
<div id="ref-patiny2013" class="csl-entry">
Patiny, Luc, and Alain Borel. 2013. <span>“ChemCalc: A Building Block for Tomorrow<span>’</span>s Chemical Infrastructure.”</span> <em>Journal of Chemical Information and Modeling</em> 53 (5): 1223–28. <a href="https://doi.org/10.1021/ci300563h">https://doi.org/10.1021/ci300563h</a>.
</div>
<div id="ref-petrov2020" class="csl-entry">
Petrov, Alexander P., Lindy M. Sherman, Jon P. Camden, and Norman J. Dovichi. 2020. <span>“Database of Free Solution Mobilities for 276 Metabolites.”</span> <em>Talanta</em> 209 (March): 120545. <a href="https://doi.org/10.1016/j.talanta.2019.120545">https://doi.org/10.1016/j.talanta.2019.120545</a>.
</div>
<div id="ref-pezzatti2020" class="csl-entry">
Pezzatti, Julian, Julien Boccard, Santiago Codesido, Yoric Gagnebin, Abhinav Joshi, Didier Picard, Víctor González-Ruiz, and Serge Rudaz. 2020. <span>“Implementation of Liquid Chromatography<span><span></span></span>high Resolution Mass Spectrometry Methods for Untargeted Metabolomic Analyses of Biological Samples: A Tutorial.”</span> <em>Analytica Chimica Acta</em> 1105 (April): 28–44. <a href="https://doi.org/10.1016/j.aca.2019.12.062">https://doi.org/10.1016/j.aca.2019.12.062</a>.
</div>
<div id="ref-pfeuffer2017" class="csl-entry">
Pfeuffer, Julianus, Timo Sachsenberg, Oliver Alka, Mathias Walzer, Alexander Fillbrunn, Lars Nilse, Oliver Schilling, Knut Reinert, and Oliver Kohlbacher. 2017. <span>“OpenMS <span><span></span></span> A Platform for Reproducible Analysis of Mass Spectrometry Data.”</span> <em>Journal of Biotechnology</em>, Bioinformatics Solutions for Big Data Analysis in Life Sciences presented by the German Network for Bioinformatics Infrastructure, 261 (November): 142–48. <a href="https://doi.org/10.1016/j.jbiotec.2017.05.016">https://doi.org/10.1016/j.jbiotec.2017.05.016</a>.
</div>
<div id="ref-phapale2020" class="csl-entry">
Phapale, Prasad, Vineeta Rai, Ashok Kumar Mohanty, and Sanjeeva Srivastava. 2020. <span>“Untargeted Metabolomics Workshop Report: Quality Control Considerations from Sample Preparation to Data Analysis.”</span> <em>Journal of the American Society for Mass Spectrometry</em> 31 (9): 2006–10. <a href="https://doi.org/10.1021/jasms.0c00224">https://doi.org/10.1021/jasms.0c00224</a>.
</div>
<div id="ref-pluskal2010a" class="csl-entry">
Pluskal, Tomáš, Sandra Castillo, Alejandro Villar-Briones, and Matej Orešič. 2010. <span>“MZmine 2: Modular Framework for Processing, Visualizing, and Analyzing Mass Spectrometry-Based Molecular Profile Data.”</span> <em>BMC Bioinformatics</em> 11: 395. <a href="https://doi.org/10.1186/1471-2105-11-395">https://doi.org/10.1186/1471-2105-11-395</a>.
</div>
<div id="ref-pluskal2020" class="csl-entry">
Pluskal, Tomáš, Ansgar Korf, Aleksandr Smirnov, Robin Schmid, Timothy R. Fallon, Xiuxia Du, and Jing-Ke Weng. 2020. <span>“CHAPTER 7:Metabolomics Data Analysis Using MZmine.”</span> In, 232–54. <a href="https://doi.org/10.1039/9781788019880-00232">https://doi.org/10.1039/9781788019880-00232</a>.
</div>
<div id="ref-polderman2015" class="csl-entry">
Polderman, Tinca J. C., Beben Benyamin, Christiaan A. de Leeuw, Patrick F. Sullivan, Arjen van Bochoven, Peter M. Visscher, and Danielle Posthuma. 2015. <span>“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies.”</span> <em>Nature Genetics</em> 47 (7): 702–9. <a href="https://doi.org/10.1038/ng.3285">https://doi.org/10.1038/ng.3285</a>.
</div>
<div id="ref-qiu2016" class="csl-entry">
Qiu, Feng, Dennis D. Fine, Daniel J. Wherritt, Zhentian Lei, and Lloyd W. Sumner. 2016. <span>“PlantMAT: A Metabolomics Tool for Predicting the Specialized Metabolic Potential of a System and for Large-Scale Metabolite Identifications.”</span> <em>Analytical Chemistry</em> 88 (23): 11373–83. <a href="https://doi.org/10.1021/acs.analchem.6b00906">https://doi.org/10.1021/acs.analchem.6b00906</a>.
</div>
<div id="ref-qiu2018" class="csl-entry">
Qiu, Feng, Zhentian Lei, and Lloyd W. Sumner. 2018. <span>“MetExpert: An Expert System to Enhance Gas Chromatography?mass Spectrometry-Based Metabolite Identifications.”</span> <em>Analytica Chimica Acta</em>, Analytical metabolomics, 1037 (December): 316–26. <a href="https://doi.org/10.1016/j.aca.2018.03.052">https://doi.org/10.1016/j.aca.2018.03.052</a>.
</div>
<div id="ref-riquelme2020" class="csl-entry">
Riquelme, Gabriel, Nicolás Zabalegui, Pablo Marchi, Christina M. Jones, and María Eugenia Monge. 2020. <span>“A Python-Based Pipeline for Preprocessing LC<span><span></span></span>MS Data for Untargeted Metabolomics Workflows.”</span> <em>Metabolites</em> 10 (10): 416. <a href="https://doi.org/10.3390/metabo10100416">https://doi.org/10.3390/metabo10100416</a>.
</div>
<div id="ref-roszkowska2018" class="csl-entry">
Roszkowska, Anna, Miao Yu, Vincent Bessonneau, Leslie Bragg, Mark Servos, and Janusz Pawliszyn. 2018. <span>“Tissue Storage Affects Lipidome Profiling in Comparison to in Vivo Microsampling Approach.”</span> <em>Scientific Reports</em> 8 (1): 6980. <a href="https://doi.org/10.1038/s41598-018-25428-2">https://doi.org/10.1038/s41598-018-25428-2</a>.
</div>
<div id="ref-rost2016a" class="csl-entry">
Röst, Hannes L., Timo Sachsenberg, Stephan Aiche, Chris Bielow, Hendrik Weisser, Fabian Aicheler, Sandro Andreotti, et al. 2016. <span>“OpenMS: A Flexible Open-Source Software Platform for Mass Spectrometry Data Analysis.”</span> <em>Nature Methods</em> 13 (9): 741–48. <a href="https://doi.org/10.1038/nmeth.3959">https://doi.org/10.1038/nmeth.3959</a>.
</div>
<div id="ref-röst2014" class="csl-entry">
Röst, Hannes L., Uwe Schmitt, Ruedi Aebersold, and Lars Malmström. 2014. <span>“pyOpenMS: A Python-Based Interface to the OpenMS Mass-Spectrometry Algorithm Library.”</span> <em>PROTEOMICS</em> 14 (1): 74–77. https://doi.org/<a href="https://doi.org/10.1002/pmic.201300246">https://doi.org/10.1002/pmic.201300246</a>.
</div>
<div id="ref-rurik2020" class="csl-entry">
Rurik, Marc, Oliver Alka, Fabian Aicheler, and Oliver Kohlbacher. 2020. <span>“Metabolomics Data Processing Using OpenMS.”</span> In, edited by Shuzhao Li, 49–60. Methods in Molecular Biology. New York, NY: Springer US. <a href="https://doi.org/10.1007/978-1-0716-0239-3_4">https://doi.org/10.1007/978-1-0716-0239-3_4</a>.
</div>
<div id="ref-rusconi2019" class="csl-entry">
Rusconi, Filippo. 2019. <span>“<span class="nocase">mineXpert</span>: <span>Biological Mass Spectrometry Data Visualization</span> and <span>Mining</span> with <span>Full JavaScript Ability</span>.”</span> <em>Journal of Proteome Research</em> 18 (5): 2254–59. <a href="https://doi.org/10.1021/acs.jproteome.9b00099">https://doi.org/10.1021/acs.jproteome.9b00099</a>.
</div>
<div id="ref-ruttkies2016" class="csl-entry">
Ruttkies, Christoph, Emma L. Schymanski, Sebastian Wolf, Juliane Hollender, and Steffen Neumann. 2016. <span>“MetFrag Relaunched: Incorporating Strategies Beyond in Silico Fragmentation.”</span> <em>Journal of Cheminformatics</em> 8 (January): 3. <a href="https://doi.org/10.1186/s13321-016-0115-9">https://doi.org/10.1186/s13321-016-0115-9</a>.
</div>
<div id="ref-saccenti2016" class="csl-entry">
Saccenti, Edoardo, and Marieke E. Timmerman. 2016. <span>“Approaches to Sample Size Determination for Multivariate Data: Applications to PCA and PLS-DA of Omics Data.”</span> <em>Journal of Proteome Research</em> 15 (8): 2379–93. <a href="https://doi.org/10.1021/acs.jproteome.5b01029">https://doi.org/10.1021/acs.jproteome.5b01029</a>.
</div>
<div id="ref-samanipour2018" class="csl-entry">
Samanipour, Saer, Malcolm J. Reid, Kine Bæk, and Kevin V. Thomas. 2018. <span>“Combining a Deconvolution and a Universal Library Search Algorithm for the Nontarget Analysis of Data-Independent Acquisition Mode Liquid Chromatography<span>-</span>High-Resolution Mass Spectrometry Results.”</span> <em>Environmental Science &amp; Technology</em> 52 (8): 4694–4701. <a href="https://doi.org/10.1021/acs.est.8b00259">https://doi.org/10.1021/acs.est.8b00259</a>.
</div>
<div id="ref-sarpe2017" class="csl-entry">
Sarpe, Vladimir, and David C Schriemer. 2017. <span>“Supporting Metabolomics with Adaptable Software: Design Architectures for the End-User.”</span> <em>Current Opinion in Biotechnology</em>, Analytical biotechnology, 43 (February): 110–17. <a href="https://doi.org/10.1016/j.copbio.2016.11.001">https://doi.org/10.1016/j.copbio.2016.11.001</a>.
</div>
<div id="ref-scheltema2011" class="csl-entry">
Scheltema, Richard A., Andris Jankevics, Ritsert C. Jansen, Morris A. Swertz, and Rainer Breitling. 2011b. <span>“PeakML/mzMatch: A File Format, Java Library, r Library, and Tool-Chain for Mass Spectrometry Data Analysis.”</span> <em>Analytical Chemistry</em> 83 (7): 2786–93. <a href="https://doi.org/10.1021/ac2000994">https://doi.org/10.1021/ac2000994</a>.
</div>
<div id="ref-scheltema2011a" class="csl-entry">
———. 2011a. <span>“PeakML/mzMatch: A File Format, Java Library, r Library, and Tool-Chain for Mass Spectrometry Data Analysis.”</span> <em>Analytical Chemistry</em> 83 (7): 2786–93. <a href="https://doi.org/10.1021/ac2000994">https://doi.org/10.1021/ac2000994</a>.
</div>
<div id="ref-scheubert2017" class="csl-entry">
Scheubert, Kerstin, Franziska Hufsky, Daniel Petras, Mingxun Wang, Louis-Félix Nothias, Kai Dührkop, Nuno Bandeira, Pieter C. Dorrestein, and Sebastian Böcker. 2017. <span>“Significance Estimation for Large Scale Metabolomics Annotations by Spectral Matching.”</span> <em>Nature Communications</em> 8 (1): 1494. <a href="https://doi.org/10.1038/s41467-017-01318-5">https://doi.org/10.1038/s41467-017-01318-5</a>.
</div>
<div id="ref-schrimpe-rutledge2016" class="csl-entry">
Schrimpe-Rutledge, Alexandra C., Simona G. Codreanu, Stacy D. Sherrod, and John A. McLean. 2016. <span>“Untargeted Metabolomics Strategies<span><span></span></span>Challenges and Emerging Directions.”</span> <em>Journal of The American Society for Mass Spectrometry</em> 27 (12): 1897–1905. <a href="https://doi.org/10.1007/s13361-016-1469-y">https://doi.org/10.1007/s13361-016-1469-y</a>.
</div>
<div id="ref-schymanski2017" class="csl-entry">
Schymanski, Emma L., and Antony J. Williams. 2017. <span>“Open Science for Identifying <span>“</span>Known Unknown<span>”</span> Chemicals.”</span> <em>Environmental Science &amp; Technology</em> 51 (10): 5357–59. <a href="https://doi.org/10.1021/acs.est.7b01908">https://doi.org/10.1021/acs.est.7b01908</a>.
</div>
<div id="ref-senan2019" class="csl-entry">
Senan, Oriol, Antoni Aguilar-Mogas, Miriam Navarro, Jordi Capellades, Luke Noon, Deborah Burks, Oscar Yanes, Roger Guimerà, and Marta Sales-Pardo. 2019. <span>“CliqueMS: A Computational Tool for Annotating in-Source Metabolite Ions from LC-MS Untargeted Metabolomics Data Based on a Coelution Similarity Network.”</span> <em>Bioinformatics</em> 35 (20): 4089–97. <a href="https://doi.org/10.1093/bioinformatics/btz207">https://doi.org/10.1093/bioinformatics/btz207</a>.
</div>
<div id="ref-shen2019" class="csl-entry">
Shen, Xiaotao, Ruohong Wang, Xin Xiong, Yandong Yin, Yuping Cai, Zaijun Ma, Nan Liu, and Zheng-Jiang Zhu. 2019. <span>“Metabolic Reaction Network-Based Recursive Metabolite Annotation for Untargeted Metabolomics.”</span> <em>Nature Communications</em> 10 (1): 1–14. <a href="https://doi.org/10.1038/s41467-019-09550-x">https://doi.org/10.1038/s41467-019-09550-x</a>.
</div>
<div id="ref-silva2018" class="csl-entry">
Silva, Ricardo R. da, Mingxun Wang, Louis-Félix Nothias, Justin J. J. van der Hooft, Andrés Mauricio Caraballo-Rodríguez, Evan Fox, Marcy J. Balunas, Jonathan L. Klassen, Norberto Peporine Lopes, and Pieter C. Dorrestein. 2018. <span>“Propagating Annotations of Molecular Networks Using in Silico Fragmentation.”</span> <em>PLOS Computational Biology</em> 14 (4): e1006089. <a href="https://doi.org/10.1371/journal.pcbi.1006089">https://doi.org/10.1371/journal.pcbi.1006089</a>.
</div>
<div id="ref-silva2014" class="csl-entry">
Silva, Ricardo R., Fabien Jourdan, Diego M. Salvanha, Fabien Letisse, Emilien L. Jamin, Simone Guidetti-Gonzalez, Carlos A. Labate, and Ricardo Z. N. Vêncio. 2014. <span>“ProbMetab: An r Package for Bayesian Probabilistic Annotation of LC<span><span></span></span>MS-Based Metabolomics.”</span> <em>Bioinformatics</em> 30 (9): 1336–37. <a href="https://doi.org/10.1093/bioinformatics/btu019">https://doi.org/10.1093/bioinformatics/btu019</a>.
</div>
<div id="ref-sindelar2020" class="csl-entry">
Sindelar, Miriam, and Gary J. Patti. 2020. <span>“Chemical Discovery in the Era of Metabolomics.”</span> <em>Journal of the American Chemical Society</em>, April. <a href="https://doi.org/10.1021/jacs.9b13198">https://doi.org/10.1021/jacs.9b13198</a>.
</div>
<div id="ref-siskos2017" class="csl-entry">
Siskos, Alexandros P., Pooja Jain, Werner Römisch-Margl, Mark Bennett, David Achaintre, Yasmin Asad, Luke Marney, et al. 2017. <span>“Interlaboratory Reproducibility of a Targeted Metabolomics Platform for Analysis of Human Serum and Plasma.”</span> <em>Analytical Chemistry</em> 89 (1): 656–65. <a href="https://doi.org/10.1021/acs.analchem.6b02930">https://doi.org/10.1021/acs.analchem.6b02930</a>.
</div>
<div id="ref-sitnikov2016" class="csl-entry">
Sitnikov, Dmitri G., Cian S. Monnin, and Dajana Vuckovic. 2016. <span>“Systematic Assessment of Seven Solvent and Solid-Phase Extraction Methods for Metabolomics Analysis of Human Plasma by LC-MS.”</span> <em>Scientific Reports</em> 6 (December). <a href="https://doi.org/10.1038/srep38885">https://doi.org/10.1038/srep38885</a>.
</div>
<div id="ref-smirnov2019" class="csl-entry">
Smirnov, Kirill S., Sara Forcisi, Franco Moritz, Marianna Lucio, and Philippe Schmitt-Kopplin. 2019. <span>“Mass Difference Maps and Their Application for the Recalibration of Mass Spectrometric Data in Nontargeted Metabolomics.”</span> <em>Analytical Chemistry</em> 91 (5): 3350–58. <a href="https://doi.org/10.1021/acs.analchem.8b04555">https://doi.org/10.1021/acs.analchem.8b04555</a>.
</div>
<div id="ref-smirnov2016" class="csl-entry">
Smirnov, Kirill S., Tanja V. Maier, Alesia Walker, Silke S. Heinzmann, Sara Forcisi, Inés Martinez, Jens Walter, and Philippe Schmitt-Kopplin. 2016. <span>“Challenges of Metabolomics in Human Gut Microbiota Research.”</span> <em>International Journal of Medical Microbiology</em>, Intestinal microbiota - a microbial ecosystem at the edge between immune homeostasis and inflammation, 306 (5): 266–79. <a href="https://doi.org/10.1016/j.ijmm.2016.03.006">https://doi.org/10.1016/j.ijmm.2016.03.006</a>.
</div>
<div id="ref-smith2006" class="csl-entry">
Smith, Colin A., Elizabeth J. Want, Grace O’Maille, Ruben Abagyan, and Gary Siuzdak. 2006. <span>“<span>XCMS</span>:  <span>Processing Mass Spectrometry Data</span> for <span>Metabolite Profiling Using Nonlinear Peak Alignment</span>, <span>Matching</span>, and <span>Identification</span>.”</span> <em>Analytical Chemistry</em> 78 (3): 779–87. <a href="https://doi.org/10.1021/ac051437y">https://doi.org/10.1021/ac051437y</a>.
</div>
<div id="ref-spalding2016" class="csl-entry">
Spalding, Jonathan L., Kevin Cho, Nathaniel G. Mahieu, Igor Nikolskiy, Elizabeth M. Llufrio, Stephen L. Johnson, and Gary J. Patti. 2016. <span>“Bar Coding Ms2 Spectra for Metabolite Identification.”</span> <em>Analytical Chemistry</em> 88 (5): 2538–42. <a href="https://doi.org/10.1021/acs.analchem.5b04925">https://doi.org/10.1021/acs.analchem.5b04925</a>.
</div>
<div id="ref-spicer2017" class="csl-entry">
Spicer, Rachel, Reza M. Salek, Pablo Moreno, Daniel Cañueto, and Christoph Steinbeck. 2017. <span>“Navigating Freely-Available Software Tools for Metabolomics Analysis.”</span> <em>Metabolomics</em> 13 (9). <a href="https://doi.org/10.1007/s11306-017-1242-7">https://doi.org/10.1007/s11306-017-1242-7</a>.
</div>
<div id="ref-spratlin2009" class="csl-entry">
Spratlin, Jennifer L., Natalie J. Serkova, and S. Gail Eckhardt. 2009. <span>“Clinical Applications of Metabolomics in Oncology: A Review.”</span> <em>Clinical Cancer Research</em> 15 (2): 431–40. <a href="https://doi.org/10.1158/1078-0432.CCR-08-1059">https://doi.org/10.1158/1078-0432.CCR-08-1059</a>.
</div>
<div id="ref-styczynski2007a" class="csl-entry">
Styczynski, Mark P., Joel F. Moxley, Lily V. Tong, Jason L. Walther, Kyle L. Jensen, and Gregory N. Stephanopoulos. 2007. <span>“Systematic Identification of Conserved Metabolites in GC/MS Data for Metabolomics and Biomarker Discovery.”</span> <em>Analytical Chemistry</em> 79 (3): 966–73. <a href="https://doi.org/10.1021/ac0614846">https://doi.org/10.1021/ac0614846</a>.
</div>
<div id="ref-sumner2007" class="csl-entry">
Sumner, Lloyd W., Alexander Amberg, Dave Barrett, Michael H. Beale, Richard Beger, Clare A. Daykin, Teresa W.-M. Fan, et al. 2007. <span>“Proposed Minimum Reporting Standards for Chemical Analysis Chemical Analysis Working Group (CAWG) Metabolomics Standards Initiative (MSI).”</span> <em>Metabolomics : Official Journal of the Metabolomic Society</em> 3 (3): 211–21. <a href="https://doi.org/10.1007/s11306-007-0082-2">https://doi.org/10.1007/s11306-007-0082-2</a>.
</div>
<div id="ref-sumner2003" class="csl-entry">
Sumner, Lloyd W, Pedro Mendes, and Richard A Dixon. 2003. <span>“Plant Metabolomics: Large-Scale Phytochemistry in the Functional Genomics Era.”</span> <em>Phytochemistry</em>, Plant metabolomics, 62 (6): 817–36. <a href="https://doi.org/10.1016/S0031-9422(02)00708-2">https://doi.org/10.1016/S0031-9422(02)00708-2</a>.
</div>
<div id="ref-sysi-aho2007" class="csl-entry">
Sysi-Aho, Marko, Mikko Katajamaa, Laxman Yetukuri, and Matej Orešič. 2007. <span>“Normalization Method for Metabolomics Data Using Optimal Selection of Multiple Internal Standards.”</span> <em>BMC Bioinformatics</em> 8 (March): 93. <a href="https://doi.org/10.1186/1471-2105-8-93">https://doi.org/10.1186/1471-2105-8-93</a>.
</div>
<div id="ref-tang2020" class="csl-entry">
Tang, Yanan, Caley B. Craven, Nicholas J. P. Wawryk, Junlang Qiu, Feng Li, and Xing-Fang Li. 2020. <span>“Advances in Mass Spectrometry-Based Omics Analysis of Trace Organics in Water.”</span> <em>TrAC Trends in Analytical Chemistry</em> 128 (July): 115918. <a href="https://doi.org/10.1016/j.trac.2020.115918">https://doi.org/10.1016/j.trac.2020.115918</a>.
</div>
<div id="ref-tautenhahn2008" class="csl-entry">
Tautenhahn, Ralf, Christoph Böttcher, and Steffen Neumann. 2008. <span>“Highly Sensitive Feature Detection for High Resolution <span>LC</span>/<span>MS</span>.”</span> <em>BMC Bioinformatics</em> 9: 504. <a href="https://doi.org/10.1186/1471-2105-9-504">https://doi.org/10.1186/1471-2105-9-504</a>.
</div>
<div id="ref-tautenhahn2012" class="csl-entry">
Tautenhahn, Ralf, Kevin Cho, Winnie Uritboonthai, Zhengjiang Zhu, Gary J. Patti, and Gary Siuzdak. 2012. <span>“An Accelerated Workflow for Untargeted Metabolomics Using the METLIN Database.”</span> <em>Nature Biotechnology</em> 30 (9): 826–28. <a href="https://doi.org/10.1038/nbt.2348">https://doi.org/10.1038/nbt.2348</a>.
</div>
<div id="ref-theodoridis2012" class="csl-entry">
Theodoridis, Georgios A., Helen G. Gika, Elizabeth J. Want, and Ian D. Wilson. 2012. <span>“Liquid Chromatography<span><span></span></span>mass Spectrometry Based Global Metabolite Profiling: A Review.”</span> <em>Analytica Chimica Acta</em> 711 (January): 7–16. <a href="https://doi.org/10.1016/j.aca.2011.09.042">https://doi.org/10.1016/j.aca.2011.09.042</a>.
</div>
<div id="ref-thonusin2017" class="csl-entry">
Thonusin, Chanisa, Heidi B. IglayReger, Tanu Soni, Amy E. Rothberg, Charles F. Burant, and Charles R. Evans. 2017. <span>“Evaluation of Intensity Drift Correction Strategies Using MetaboDrift, a Normalization Tool for Multi-Batch Metabolomics Data.”</span> <em>Journal of Chromatography A</em>, Pushing the boundaries of chromatography and electrophoresis, 1523 (Supplement C): 265–74. <a href="https://doi.org/10.1016/j.chroma.2017.09.023">https://doi.org/10.1016/j.chroma.2017.09.023</a>.
</div>
<div id="ref-tian2016" class="csl-entry">
Tian, Tze-Feng, San-Yuan Wang, Tien-Chueh Kuo, Cheng-En Tan, Guan-Yuan Chen, Ching-Hua Kuo, Chi-Hsin Sally Chen, Chang-Chuan Chan, Olivia A. Lin, and Y. Jane Tseng. 2016a. <span>“Web Server for Peak Detection, Baseline Correction, and Alignment in Two-Dimensional Gas Chromatography Mass Spectrometry-Based Metabolomics Data.”</span> <em>Analytical Chemistry</em> 88 (21): 10395–403. <a href="https://doi.org/10.1021/acs.analchem.6b00755">https://doi.org/10.1021/acs.analchem.6b00755</a>.
</div>
<div id="ref-tian2016b" class="csl-entry">
———. 2016b. <span>“Web Server for Peak Detection, Baseline Correction, and Alignment in Two-Dimensional Gas Chromatography Mass Spectrometry-Based Metabolomics Data.”</span> <em>Analytical Chemistry</em> 88 (21): 10395–403. <a href="https://doi.org/10.1021/acs.analchem.6b00755">https://doi.org/10.1021/acs.analchem.6b00755</a>.
</div>
<div id="ref-treutler2016" class="csl-entry">
Treutler, Hendrik, Hiroshi Tsugawa, Andrea Porzel, Karin Gorzolka, Alain Tissier, Steffen Neumann, and Gerd Ulrich Balcke. 2016a. <span>“Discovering Regulated Metabolite Families in Untargeted Metabolomics Studies.”</span> <em>Analytical Chemistry</em> 88 (16): 8082–90. <a href="https://doi.org/10.1021/acs.analchem.6b01569">https://doi.org/10.1021/acs.analchem.6b01569</a>.
</div>
<div id="ref-treutler2016a" class="csl-entry">
———. 2016b. <span>“Discovering Regulated Metabolite Families in Untargeted Metabolomics Studies.”</span> <em>Analytical Chemistry</em> 88 (16): 8082–90. <a href="https://doi.org/10.1021/acs.analchem.6b01569">https://doi.org/10.1021/acs.analchem.6b01569</a>.
</div>
<div id="ref-tsou2015" class="csl-entry">
Tsou, Chih-Chiang, Dmitry Avtonomov, Brett Larsen, Monika Tucholska, Hyungwon Choi, Anne-Claude Gingras, and Alexey I. Nesvizhskii. 2015. <span>“DIA-Umpire: Comprehensive Computational Framework for Data-Independent Acquisition Proteomics.”</span> <em>Nature Methods</em> 12 (3): 258–64. <a href="https://doi.org/10.1038/nmeth.3255">https://doi.org/10.1038/nmeth.3255</a>.
</div>
<div id="ref-tsugawa2015a" class="csl-entry">
Tsugawa, Hiroshi, Tomas Cajka, Tobias Kind, Yan Ma, Brendan Higgins, Kazutaka Ikeda, Mitsuhiro Kanazawa, Jean VanderGheynst, Oliver Fiehn, and Masanori Arita. 2015a. <span>“MS-DIAL: Data-Independent MS/MS Deconvolution for Comprehensive Metabolome Analysis.”</span> <em>Nature Methods</em> 12 (6): 523–26. <a href="https://doi.org/10.1038/nmeth.3393">https://doi.org/10.1038/nmeth.3393</a>.
</div>
<div id="ref-tsugawa2015b" class="csl-entry">
———. 2015b. <span>“MS-DIAL: Data-Independent MS/MS Deconvolution for Comprehensive Metabolome Analysis.”</span> <em>Nature Methods</em> 12 (6): 523–26. <a href="https://doi.org/10.1038/nmeth.3393">https://doi.org/10.1038/nmeth.3393</a>.
</div>
<div id="ref-tsugawa2016" class="csl-entry">
Tsugawa, Hiroshi, Tobias Kind, Ryo Nakabayashi, Daichi Yukihira, Wataru Tanaka, Tomas Cajka, Kazuki Saito, Oliver Fiehn, and Masanori Arita. 2016. <span>“Hydrogen Rearrangement Rules: Computational MS/MS Fragmentation and Structure Elucidation Using MS-FINDER Software.”</span> <em>Analytical Chemistry</em> 88 (16): 7946–58. <a href="https://doi.org/10.1021/acs.analchem.6b00770">https://doi.org/10.1021/acs.analchem.6b00770</a>.
</div>
<div id="ref-uppal2013a" class="csl-entry">
Uppal, Karan, Quinlyn A. Soltow, Frederick H. Strobel, W. Stephen Pittard, Kim M. Gernert, Tianwei Yu, and Dean P. Jones. 2013. <span>“xMSanalyzer: Automated Pipeline for Improved Feature Detection and Downstream Analysis of Large-Scale, Non-Targeted Metabolomics Data.”</span> <em>BMC Bioinformatics</em> 14 (1): 15. <a href="https://doi.org/10.1186/1471-2105-14-15">https://doi.org/10.1186/1471-2105-14-15</a>.
</div>
<div id="ref-uppal2017" class="csl-entry">
Uppal, Karan, Douglas I. Walker, and Dean P. Jones. 2017. <span>“xMSannotator: An r Package for Network-Based Annotation of High-Resolution Metabolomics Data.”</span> <em>Analytical Chemistry</em> 89 (2): 1063–67. <a href="https://doi.org/10.1021/acs.analchem.6b01214">https://doi.org/10.1021/acs.analchem.6b01214</a>.
</div>
<div id="ref-uppal2016" class="csl-entry">
Uppal, Karan, Douglas I. Walker, Ken Liu, Shuzhao Li, Young-Mi Go, and Dean P. Jones. 2016b. <span>“Computational Metabolomics: A Framework for the Million Metabolome.”</span> <em>Chemical Research in Toxicology</em> 29 (12): 1956–75. <a href="https://doi.org/10.1021/acs.chemrestox.6b00179">https://doi.org/10.1021/acs.chemrestox.6b00179</a>.
</div>
<div id="ref-uppal2016a" class="csl-entry">
———. 2016a. <span>“Computational Metabolomics: A Framework for the Million Metabolome.”</span> <em>Chemical Research in Toxicology</em> 29 (12): 1956–75. <a href="https://doi.org/10.1021/acs.chemrestox.6b00179">https://doi.org/10.1021/acs.chemrestox.6b00179</a>.
</div>
<div id="ref-vanderkloet2009a" class="csl-entry">
van der Kloet, Frans M., Ivana Bobeldijk, Elwin R. Verheij, and Renger H. Jellema. 2009. <span>“Analytical Error Reduction Using Single Point Calibration for Accurate and Precise Metabolomic Phenotyping.”</span> <em>Journal of Proteome Research</em> 8 (11): 5132–41. <a href="https://doi.org/10.1021/pr900499r">https://doi.org/10.1021/pr900499r</a>.
</div>
<div id="ref-verhoeven2020" class="csl-entry">
Verhoeven, Aswin, Martin Giera, and Oleg A. Mayboroda. 2020. <span>“Scientific Workflow Managers in Metabolomics: An Overview.”</span> <em>Analyst</em> 145 (11): 3801–8. <a href="https://doi.org/10.1039/D0AN00272K">https://doi.org/10.1039/D0AN00272K</a>.
</div>
<div id="ref-viant2019" class="csl-entry">
Viant, Mark R., Timothy M. D. Ebbels, Richard D. Beger, Drew R. Ekman, David J. T. Epps, Hennicke Kamp, Pim E. G. Leonards, et al. 2019. <span>“Use Cases, Best Practice and Reporting Standards for Metabolomics in Regulatory Toxicology.”</span> <em>Nature Communications</em> 10 (1): 3041. <a href="https://doi.org/10.1038/s41467-019-10900-y">https://doi.org/10.1038/s41467-019-10900-y</a>.
</div>
<div id="ref-viant2017" class="csl-entry">
Viant, Mark R, Irwin J Kurland, Martin R Jones, and Warwick B Dunn. 2017. <span>“How Close Are We to Complete Annotation of Metabolomes?”</span> <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 64–69. <a href="https://doi.org/10.1016/j.cbpa.2017.01.001">https://doi.org/10.1016/j.cbpa.2017.01.001</a>.
</div>
<div id="ref-vinaixa2016" class="csl-entry">
Vinaixa, Maria, Emma L. Schymanski, Steffen Neumann, Miriam Navarro, Reza M. Salek, and Oscar Yanes. 2016. <span>“Mass Spectral Databases for LC/MS- and GC/MS-Based Metabolomics: State of the Field and Future Prospects.”</span> <em>TrAC Trends in Analytical Chemistry</em> 78 (April): 23–35. <a href="https://doi.org/10.1016/j.trac.2015.09.005">https://doi.org/10.1016/j.trac.2015.09.005</a>.
</div>
<div id="ref-wallach2018" class="csl-entry">
Wallach, Joshua D., Kevin W. Boyack, and John P. A. Ioannidis. 2018. <span>“Reproducible Research Practices, Transparency, and Open Access Data in the Biomedical Literature, 2015<span><span></span></span>2017.”</span> <em>PLOS Biology</em> 16 (11): e2006930. <a href="https://doi.org/10.1371/journal.pbio.2006930">https://doi.org/10.1371/journal.pbio.2006930</a>.
</div>
<div id="ref-wandro2017" class="csl-entry">
Wandro, Stephen, Lisa Carmody, Tara Gallagher, John J. LiPuma, and Katrine Whiteson. 2017. <span>“Making It Last: Storage Time and Temperature Have Differential Impacts on Metabolite Profiles of Airway Samples from Cystic Fibrosis Patients.”</span> <em>mSystems</em> 2 (6). <a href="https://doi.org/10.1128/mSystems.00100-17">https://doi.org/10.1128/mSystems.00100-17</a>.
</div>
<div id="ref-wang2016" class="csl-entry">
Wang, Mingxun, Jeremy J. Carver, Vanessa V. Phelan, Laura M. Sanchez, Neha Garg, Yao Peng, Don Duy Nguyen, et al. 2016. <span>“Sharing and Community Curation of Mass Spectrometry Data with Global Natural Products Social Molecular Networking.”</span> <em>Nature Biotechnology</em> 34 (8): 828–37. <a href="https://doi.org/10.1038/nbt.3597">https://doi.org/10.1038/nbt.3597</a>.
</div>
<div id="ref-wang2019" class="csl-entry">
Wang, Ruohong, Yandong Yin, and Zheng-Jiang Zhu. 2019. <span>“Advancing Untargeted Metabolomics Using Data-Independent Acquisition Mass Spectrometry Technology.”</span> <em>Analytical and Bioanalytical Chemistry</em> 411 (19): 4349–57. <a href="https://doi.org/10.1007/s00216-019-01709-1">https://doi.org/10.1007/s00216-019-01709-1</a>.
</div>
<div id="ref-wang2019" class="csl-entry">
———. 2019. <span>“Advancing Untargeted Metabolomics Using Data-Independent Acquisition Mass Spectrometry Technology.”</span> <em>Analytical and Bioanalytical Chemistry</em> 411 (19): 4349–57. <a href="https://doi.org/10.1007/s00216-019-01709-1">https://doi.org/10.1007/s00216-019-01709-1</a>.
</div>
<div id="ref-wang2013" class="csl-entry">
Wang, San-Yuan, Ching-Hua Kuo, and Yufeng J. Tseng. 2013. <span>“Batch Normalizer: A Fast Total Abundance Regression Calibration Method to Simultaneously Adjust Batch and Injection Order Effects in Liquid Chromatography/Time-of-Flight Mass Spectrometry-Based Metabolomics Data and Comparison with Current Calibration Methods.”</span> <em>Analytical Chemistry</em> 85 (2): 1037–46. <a href="https://doi.org/10.1021/ac302877x">https://doi.org/10.1021/ac302877x</a>.
</div>
<div id="ref-wang2016a" class="csl-entry">
Wang, Yang, Fang Liu, Peng Li, Chengwei He, Ruibing Wang, Huanxing Su, and Jian-Bo Wan. 2016. <span>“An improved pseudotargeted metabolomics approach using multiple ion monitoring with time-staggered ion lists based on ultra-high performance liquid chromatography/quadrupole time-of-flight mass spectrometry.”</span> <em>Analytica Chimica Acta</em> 927 (July): 82–88. <a href="https://doi.org/10.1016/j.aca.2016.05.008">https://doi.org/10.1016/j.aca.2016.05.008</a>.
</div>
<div id="ref-warth2017" class="csl-entry">
Warth, Benedikt, Scott Spangler, Mingliang Fang, Caroline H. Johnson, Erica M. Forsberg, Ana Granados, Richard L. Martin, et al. 2017. <span>“Exposome-Scale Investigations Guided by Global Metabolomics, Pathway Analysis, and Cognitive Computing.”</span> <em>Analytical Chemistry</em> 89 (21): 11505–13. <a href="https://doi.org/10.1021/acs.analchem.7b02759">https://doi.org/10.1021/acs.analchem.7b02759</a>.
</div>
<div id="ref-weber2017" class="csl-entry">
Weber, Ralf J. M., Thomas N. Lawson, Reza M. Salek, Timothy M. D. Ebbels, Robert C. Glen, Royston Goodacre, Julian L. Griffin, et al. 2017. <span>“Computational Tools and Workflows in Metabolomics: An International Survey Highlights the Opportunity for Harmonisation Through Galaxy.”</span> <em>Metabolomics</em> 13 (2). <a href="https://doi.org/10.1007/s11306-016-1147-x">https://doi.org/10.1007/s11306-016-1147-x</a>.
</div>
<div id="ref-weber2010" class="csl-entry">
Weber, Ralf J. M., and Mark R. Viant. 2010. <span>“MI-Pack: Increased Confidence of Metabolite Identification in Mass Spectra by Integrating Accurate Masses and Metabolic Pathways.”</span> <em>Chemometrics and Intelligent Laboratory Systems</em>, OMICS, 104 (1): 75–82. <a href="https://doi.org/10.1016/j.chemolab.2010.04.010">https://doi.org/10.1016/j.chemolab.2010.04.010</a>.
</div>
<div id="ref-wehrens2015" class="csl-entry">
Wehrens, Ron, Tom G. Bloemberg, and Paul H. C. Eilers. 2015. <span>“Fast Parametric Time Warping of Peak Lists.”</span> <em>Bioinformatics</em> 31 (18): 3063–65. <a href="https://doi.org/10.1093/bioinformatics/btv299">https://doi.org/10.1093/bioinformatics/btv299</a>.
</div>
<div id="ref-wehrens2015" class="csl-entry">
———. 2015. <span>“Fast Parametric Time Warping of Peak Lists.”</span> <em>Bioinformatics</em> 31 (18): 3063–65. <a href="https://doi.org/10.1093/bioinformatics/btv299">https://doi.org/10.1093/bioinformatics/btv299</a>.
</div>
<div id="ref-wei2018" class="csl-entry">
Wei, Runmin, Jingye Wang, Mingming Su, Erik Jia, Shaoqiu Chen, Tianlu Chen, and Yan Ni. 2018. <span>“Missing Value Imputation Approach for Mass Spectrometry-Based Metabolomics Data.”</span> <em>Scientific Reports</em> 8 (1): 663. <a href="https://doi.org/10.1038/s41598-017-19120-0">https://doi.org/10.1038/s41598-017-19120-0</a>.
</div>
<div id="ref-weljie2006" class="csl-entry">
Weljie, Aalim M., Jack Newton, Pascal Mercier, Erin Carlson, and Carolyn M. Slupsky. 2006. <span>“Targeted Profiling:<span><span>0.167em</span></span> Quantitative Analysis of 1h NMR Metabolomics Data.”</span> <em>Analytical Chemistry</em> 78 (13): 4430–42. <a href="https://doi.org/10.1021/ac060209g">https://doi.org/10.1021/ac060209g</a>.
</div>
<div id="ref-wen2017a" class="csl-entry">
Wen, Bo, Zhanlong Mei, Chunwei Zeng, and Siqi Liu. 2017. <span>“metaX: A Flexible and Comprehensive Software for Processing Metabolomics Data.”</span> <em>BMC Bioinformatics</em> 18 (March): 183. <a href="https://doi.org/10.1186/s12859-017-1579-y">https://doi.org/10.1186/s12859-017-1579-y</a>.
</div>
<div id="ref-wiklund2008" class="csl-entry">
Wiklund, Susanne, Erik Johansson, Lina Sjöström, Ewa J. Mellerowicz, Ulf Edlund, John P. Shockcor, Johan Gottfries, Thomas Moritz, and Johan Trygg. 2008. <span>“Visualization of GC/TOF-MS-Based Metabolomics Data for Identification of Biochemically Interesting Compounds Using OPLS Class Models.”</span> <em>Analytical Chemistry</em> 80 (1): 115–22. <a href="https://doi.org/10.1021/ac0713510">https://doi.org/10.1021/ac0713510</a>.
</div>
<div id="ref-wishart2016" class="csl-entry">
Wishart, David S. 2016. <span>“Emerging Applications of Metabolomics in Drug Discovery and Precision Medicine.”</span> <em>Nature Reviews Drug Discovery</em> 15 (7): 473–84. <a href="https://doi.org/10.1038/nrd.2016.32">https://doi.org/10.1038/nrd.2016.32</a>.
</div>
<div id="ref-witting2017" class="csl-entry">
Witting, Michael, Christoph Ruttkies, Steffen Neumann, and Philippe Schmitt-Kopplin. 2017. <span>“LipidFrag: Improving Reliability of in Silico Fragmentation of Lipids and Application to the Caenorhabditis Elegans Lipidome.”</span> <em>PLOS ONE</em> 12 (3): e0172311. <a href="https://doi.org/10.1371/journal.pone.0172311">https://doi.org/10.1371/journal.pone.0172311</a>.
</div>
<div id="ref-wolf2010" class="csl-entry">
Wolf, Sebastian, Stephan Schmidt, Matthias Müller-Hannemann, and Steffen Neumann. 2010. <span>“In Silico Fragmentation for Computer Assisted Identification of Metabolite Mass Spectra.”</span> <em>BMC Bioinformatics</em> 11 (March): 148. <a href="https://doi.org/10.1186/1471-2105-11-148">https://doi.org/10.1186/1471-2105-11-148</a>.
</div>
<div id="ref-wolfender2015" class="csl-entry">
Wolfender, Jean-Luc, Guillaume Marti, Aurélien Thomas, and Samuel Bertrand. 2015. <span>“Current Approaches and Challenges for the Metabolite Profiling of Complex Natural Extracts.”</span> <em>Journal of Chromatography A</em>, Editors’ choice IX, 1382 (February): 136–64. <a href="https://doi.org/10.1016/j.chroma.2014.10.091">https://doi.org/10.1016/j.chroma.2014.10.091</a>.
</div>
<div id="ref-wu2016" class="csl-entry">
Wu, Yiman, and Liang Li. 2016. <span>“Sample Normalization Methods in Quantitative Metabolomics.”</span> <em>Journal of Chromatography A</em>, Editors’ choice x, 1430 (January): 80–95. <a href="https://doi.org/10.1016/j.chroma.2015.12.007">https://doi.org/10.1016/j.chroma.2015.12.007</a>.
</div>
<div id="ref-xu2015" class="csl-entry">
Xu, Yi-Fan, Wenyun Lu, and Joshua D. Rabinowitz. 2015. <span>“Avoiding Misannotation of in-Source Fragmentation Products as Cellular Metabolites in Liquid Chromatography<span><span></span></span>mass Spectrometry-Based Metabolomics.”</span> <em>Analytical Chemistry</em> 87 (4): 2273–81. <a href="https://doi.org/10.1021/ac504118y">https://doi.org/10.1021/ac504118y</a>.
</div>
<div id="ref-xue2020" class="csl-entry">
Xue, Jingchuan, Carlos Guijas, H. Paul Benton, Benedikt Warth, and Gary Siuzdak. 2020. <span>“METLIN MS 2 Molecular Standards Database: A Broad Chemical and Biological Resource.”</span> <em>Nature Methods</em> 17 (10): 953–54. <a href="https://doi.org/10.1038/s41592-020-0942-5">https://doi.org/10.1038/s41592-020-0942-5</a>.
</div>
<div id="ref-yamamoto2014" class="csl-entry">
Yamamoto, Hiroyuki, Tamaki Fujimori, Hajime Sato, Gen Ishikawa, Kenjiro Kami, and Yoshiaki Ohashi. 2014. <span>“Statistical Hypothesis Testing of Factor Loading in Principal Component Analysis and Its Application to Metabolite Set Enrichment Analysis.”</span> <em>BMC Bioinformatics</em> 15 (February): 51. <a href="https://doi.org/10.1186/1471-2105-15-51">https://doi.org/10.1186/1471-2105-15-51</a>.
</div>
<div id="ref-yang2020" class="csl-entry">
Yang, Qingxia, Yunxia Wang, Ying Zhang, Fengcheng Li, Weiqi Xia, Ying Zhou, Yunqing Qiu, Honglin Li, and Feng Zhu. 2020. <span>“NOREVA: Enhanced Normalization and Evaluation of Time-Course and Multi-Class Metabolomic Data.”</span> <em>Nucleic Acids Research</em> 48 (W1): W436–48. <a href="https://doi.org/10.1093/nar/gkaa258">https://doi.org/10.1093/nar/gkaa258</a>.
</div>
<div id="ref-yang2017a" class="csl-entry">
Yang, Qin, Shan-Shan Lin, Jiang-Tao Yang, Li-Juan Tang, and Ru-Qin Yu. 2017. <span>“Detection of Inborn Errors of Metabolism Utilizing GC-MS Urinary Metabolomics Coupled with a Modified Orthogonal Partial Least Squares Discriminant Analysis.”</span> <em>Talanta</em> 165 (April): 545–52. <a href="https://doi.org/10.1016/j.talanta.2017.01.018">https://doi.org/10.1016/j.talanta.2017.01.018</a>.
</div>
<div id="ref-yang2019b" class="csl-entry">
Yang, Ruochen, Xi Chen, and Idoia Ochoa. 2019. <span>“<span>MassComp</span>, a Lossless Compressor for Mass Spectrometry Data.”</span> <em>BMC Bioinformatics</em> 20 (1): 368. <a href="https://doi.org/10.1186/s12859-019-2962-7">https://doi.org/10.1186/s12859-019-2962-7</a>.
</div>
<div id="ref-yatesiii2011" class="csl-entry">
Yates Iii, John R. 2011. <span>“A Century of Mass Spectrometry: From Atoms to Proteomes.”</span> <em>Nature Methods</em> 8 (8): 633–37. <a href="https://doi.org/10.1038/nmeth.1659">https://doi.org/10.1038/nmeth.1659</a>.
</div>
<div id="ref-yu2017" class="csl-entry">
Yu, Miao, Xingwang Hou, Qian Liu, Yawei Wang, Jiyan Liu, and Guibin Jiang. 2017a. <span>“Evaluation and Reduction of the Analytical Uncertainties in GC-MS Analysis Using a Boundary Regression Model.”</span> <em>Talanta</em> 164 (March): 141–47. <a href="https://doi.org/10.1016/j.talanta.2016.11.046">https://doi.org/10.1016/j.talanta.2016.11.046</a>.
</div>
<div id="ref-yu2017a" class="csl-entry">
———. 2017b. <span>“Evaluation and Reduction of the Analytical Uncertainties in GC-MS Analysis Using a Boundary Regression Model.”</span> <em>Talanta</em> 164 (March): 141–47. <a href="https://doi.org/10.1016/j.talanta.2016.11.046">https://doi.org/10.1016/j.talanta.2016.11.046</a>.
</div>
<div id="ref-yu2020" class="csl-entry">
Yu, Miao, Sofia Lendor, Anna Roszkowska, Mariola Olkowicz, Leslie Bragg, Mark Servos, and Janusz Pawliszyn. 2020. <span>“Metabolic Profile of Fish Muscle Tissue Changes with Sampling Method, Storage Strategy and Time.”</span> <em>Analytica Chimica Acta</em> 1136 (November): 42–50. <a href="https://doi.org/10.1016/j.aca.2020.08.050">https://doi.org/10.1016/j.aca.2020.08.050</a>.
</div>
<div id="ref-yu2019a" class="csl-entry">
Yu, Miao, Mariola Olkowicz, and Janusz Pawliszyn. 2019a. <span>“Structure/Reaction Directed Analysis for LC-MS Based Untargeted Analysis.”</span> <em>Analytica Chimica Acta</em> 1050 (March): 16–24. <a href="https://doi.org/10.1016/j.aca.2018.10.062">https://doi.org/10.1016/j.aca.2018.10.062</a>.
</div>
<div id="ref-yu2019b" class="csl-entry">
———. 2019b. <span>“Structure/Reaction Directed Analysis for LC-MS Based Untargeted Analysis.”</span> <em>Analytica Chimica Acta</em> 1050 (March): 16–24. <a href="https://doi.org/10.1016/j.aca.2018.10.062">https://doi.org/10.1016/j.aca.2018.10.062</a>.
</div>
<div id="ref-yu2020a" class="csl-entry">
Yu, Miao, and Lauren Petrick. 2020. <span>“Untargeted High-Resolution Paired Mass Distance Data Mining for Retrieving General Chemical Relationships.”</span> <em>Communications Chemistry</em> 3 (1): 1–6. <a href="https://doi.org/10.1038/s42004-020-00403-z">https://doi.org/10.1038/s42004-020-00403-z</a>.
</div>
<div id="ref-yu2009a" class="csl-entry">
Yu, Tianwei, Youngja Park, Jennifer M. Johnson, and Dean P. Jones. 2009. <span>“apLCMS<span><span></span></span>adaptive Processing of High-Resolution LC/MS Data.”</span> <em>Bioinformatics</em> 25 (15): 1930–36. <a href="https://doi.org/10.1093/bioinformatics/btp291">https://doi.org/10.1093/bioinformatics/btp291</a>.
</div>
<div id="ref-yu2019" class="csl-entry">
Yu, Yong-Jie, Qing-Xia Zheng, Yue-Ming Zhang, Qian Zhang, Yu-Ying Zhang, Ping-Ping Liu, Peng Lu, et al. 2019. <span>“Automatic Data Analysis Workflow for Ultra-High Performance Liquid Chromatography-High Resolution Mass Spectrometry-Based Metabolomics.”</span> <em>Journal of Chromatography A</em> 1585 (January): 172–81. <a href="https://doi.org/10.1016/j.chroma.2018.11.070">https://doi.org/10.1016/j.chroma.2018.11.070</a>.
</div>
<div id="ref-yuan2012" class="csl-entry">
Yuan, Min, Susanne B. Breitkopf, Xuemei Yang, and John M. Asara. 2012. <span>“A Positive/Negative Ion<span><span></span></span>switching, Targeted Mass Spectrometry<span><span></span></span>based Metabolomics Platform for Bodily Fluids, Cells, and Fresh and Fixed Tissue.”</span> <em>Nature Protocols</em> 7 (5): 872–81. <a href="https://doi.org/10.1038/nprot.2012.024">https://doi.org/10.1038/nprot.2012.024</a>.
</div>
<div id="ref-zenobi2013" class="csl-entry">
Zenobi, R. 2013. <span>“Single-Cell Metabolomics: Analytical and Biological Perspectives.”</span> <em>Science</em> 342 (6163): 1243259. <a href="https://doi.org/10.1126/science.1243259">https://doi.org/10.1126/science.1243259</a>.
</div>
<div id="ref-zha2018" class="csl-entry">
Zha, Haihong, Yuping Cai, Yandong Yin, Zhuozhong Wang, Kang Li, and Zheng-Jiang Zhu. 2018. <span>“SWATHtoMRM: Development of High-Coverage Targeted Metabolomics Method Using SWATH Technology for Biomarker Discovery.”</span> <em>Analytical Chemistry</em> 90 (6): 4062–70. <a href="https://doi.org/10.1021/acs.analchem.7b05318">https://doi.org/10.1021/acs.analchem.7b05318</a>.
</div>
<div id="ref-zhang2012" class="csl-entry">
Zhang, Aihua, Hui Sun, Ping Wang, Ying Han, and Xijun Wang. 2012. <span>“Modern Analytical Techniques in Metabolomics Analysis.”</span> <em>The Analyst</em> 137 (2): 293–300. <a href="https://doi.org/10.1039/C1AN15605E">https://doi.org/10.1039/C1AN15605E</a>.
</div>
<div id="ref-zhang2020" class="csl-entry">
Zhang, Yu-Ying, Qian Zhang, Yue-Ming Zhang, Wei-Wei Wang, Li Zhang, Yong-Jie Yu, Chang-Cai Bai, Ji-Zhao Guo, Hai-Yan Fu, and Yuanbin She. 2020. <span>“A Comprehensive Automatic Data Analysis Strategy for Gas Chromatography-Mass Spectrometry Based Untargeted Metabolomics.”</span> <em>Journal of Chromatography A</em> 1616 (April): 460787. <a href="https://doi.org/10.1016/j.chroma.2019.460787">https://doi.org/10.1016/j.chroma.2019.460787</a>.
</div>
<div id="ref-zhao2021" class="csl-entry">
Zhao, Fan, Shuai Huang, and Xiaozhe Zhang. 2021. <span>“High Sensitivity and Specificity Feature Detection in Liquid Chromatography<span><span></span></span>mass Spectrometry Data: A Deep Learning Framework.”</span> <em>Talanta</em> 222 (January): 121580. <a href="https://doi.org/10.1016/j.talanta.2020.121580">https://doi.org/10.1016/j.talanta.2020.121580</a>.
</div>
<div id="ref-zhao2020" class="csl-entry">
Zhao, Shuang, and Liang Li. 2020. <span>“Chemical Derivatization in LC-MS-Based Metabolomics Study.”</span> <em>TrAC Trends in Analytical Chemistry</em> 131 (October): 115988. <a href="https://doi.org/10.1016/j.trac.2020.115988">https://doi.org/10.1016/j.trac.2020.115988</a>.
</div>
<div id="ref-zheng2020" class="csl-entry">
Zheng, Fujian, Xinjie Zhao, Zhongda Zeng, Lichao Wang, Wangjie Lv, Qingqing Wang, and Guowang Xu. 2020. <span>“Development of a Plasma Pseudotargeted Metabolomics Method Based on Ultra-High-Performance Liquid Chromatography<span><span></span></span>mass Spectrometry.”</span> <em>Nature Protocols</em> 15 (8): 2519–37. <a href="https://doi.org/10.1038/s41596-020-0341-5">https://doi.org/10.1038/s41596-020-0341-5</a>.
</div>
<div id="ref-zhou2016b" class="csl-entry">
Zhou, Juntuo, and Yuxin Yin. 2016. <span>“Strategies for Large-Scale Targeted Metabolomics Quantification by Liquid Chromatography-Mass Spectrometry.”</span> <em>Analyst</em> 141 (23): 6362–73. <a href="https://doi.org/10.1039/C6AN01753C">https://doi.org/10.1039/C6AN01753C</a>.
</div>
<div id="ref-zhu2014" class="csl-entry">
Zhu, Xiaochun, Yuping Chen, and Raju Subramanian. 2014. <span>“Comparison of Information-Dependent Acquisition, SWATH, and MSAll Techniques in Metabolite Identification Study Employing Ultrahigh-Performance Liquid Chromatography<span><span></span></span>quadrupole Time-of-Flight Mass Spectrometry.”</span> <em>Analytical Chemistry</em> 86 (2): 1202–9. <a href="https://doi.org/10.1021/ac403385y">https://doi.org/10.1021/ac403385y</a>.
</div>
<div id="ref-zubeldia-varela2020" class="csl-entry">
Zubeldia-Varela, Elisa, Domingo Barber, Coral Barbas, Marina Perez-Gordo, and David Rojo. 2020. <span>“Sample Pre-Treatment Procedures for the Omics Analysis of Human Gut Microbiota: Turning Points, Tips and Tricks for Gene Sequencing and Metabolomics.”</span> <em>Journal of Pharmaceutical and Biomedical Analysis</em> 191 (November): 113592. <a href="https://doi.org/10.1016/j.jpba.2020.113592">https://doi.org/10.1016/j.jpba.2020.113592</a>.
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="exposome.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/yufree/metaworkflow/edit/master/references.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Metabolomics.pub"],
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
