<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>References | Meta-Workflow</title>
  <meta name="description" content="This is a workflow for metabolomics studies." />
  <meta name="generator" content="bookdown 0.32 and GitBook 2.6.7" />

  <meta property="og:title" content="References | Meta-Workflow" />
  <meta property="og:type" content="book" />
  <meta property="og:image" content="/images/cover.png" />
  <meta property="og:description" content="This is a workflow for metabolomics studies." />
  <meta name="github-repo" content="yufree/metaworkflow" />

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="References | Meta-Workflow" />
  
  <meta name="twitter:description" content="This is a workflow for metabolomics studies." />
  <meta name="twitter:image" content="/images/cover.png" />

<meta name="author" content="Miao YU" />


<meta name="date" content="2023-02-02" />

  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="exposome.html"/>

<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />








<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<script src="libs/htmlwidgets-1.6.1/htmlwidgets.js"></script>
<script src="libs/viz-1.8.2/viz.js"></script>
<link href="libs/DiagrammeR-styles-0.2/styles.css" rel="stylesheet" />
<script src="libs/grViz-binding-1.0.9/grViz.js"></script>
<script src="libs/d3-3.3.8/d3.min.js"></script>
<script src="libs/dagre-0.4.0/dagre-d3.min.js"></script>
<link href="libs/mermaid-0.3.0/dist/mermaid.css" rel="stylesheet" />
<script src="libs/mermaid-0.3.0/dist/mermaid.slim.min.js"></script>
<script src="libs/chromatography-0.1/chromatography.js"></script>
<script src="libs/DiagrammeR-binding-1.0.9/DiagrammeR.js"></script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {   }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ff0000; font-weight: bold; } /* Alert */
code span.an { color: #60a0b0; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #7d9029; } /* Attribute */
code span.bn { color: #40a070; } /* BaseN */
code span.bu { color: #008000; } /* BuiltIn */
code span.cf { color: #007020; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4070a0; } /* Char */
code span.cn { color: #880000; } /* Constant */
code span.co { color: #60a0b0; font-style: italic; } /* Comment */
code span.cv { color: #60a0b0; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #ba2121; font-style: italic; } /* Documentation */
code span.dt { color: #902000; } /* DataType */
code span.dv { color: #40a070; } /* DecVal */
code span.er { color: #ff0000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #40a070; } /* Float */
code span.fu { color: #06287e; } /* Function */
code span.im { color: #008000; font-weight: bold; } /* Import */
code span.in { color: #60a0b0; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #007020; font-weight: bold; } /* Keyword */
code span.op { color: #666666; } /* Operator */
code span.ot { color: #007020; } /* Other */
code span.pp { color: #bc7a00; } /* Preprocessor */
code span.sc { color: #4070a0; } /* SpecialChar */
code span.ss { color: #bb6688; } /* SpecialString */
code span.st { color: #4070a0; } /* String */
code span.va { color: #19177c; } /* Variable */
code span.vs { color: #4070a0; } /* VerbatimString */
code span.wa { color: #60a0b0; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li><a href="./">Metabolomics Workflow</a></li>

<li class="divider"></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>Preface</a></li>
<li class="chapter" data-level="1" data-path="introduction.html"><a href="introduction.html"><i class="fa fa-check"></i><b>1</b> Introduction</a>
<ul>
<li class="chapter" data-level="1.1" data-path="introduction.html"><a href="introduction.html#history"><i class="fa fa-check"></i><b>1.1</b> History</a>
<ul>
<li class="chapter" data-level="1.1.1" data-path="introduction.html"><a href="introduction.html#history-of-mass-spectrometry"><i class="fa fa-check"></i><b>1.1.1</b> History of Mass Spectrometry</a></li>
<li class="chapter" data-level="1.1.2" data-path="introduction.html"><a href="introduction.html#history-of-metabolomcis"><i class="fa fa-check"></i><b>1.1.2</b> History of Metabolomcis</a></li>
<li class="chapter" data-level="1.1.3" data-path="introduction.html"><a href="introduction.html#defination"><i class="fa fa-check"></i><b>1.1.3</b> Defination</a></li>
</ul></li>
<li class="chapter" data-level="1.2" data-path="introduction.html"><a href="introduction.html#reviews-and-tutorials"><i class="fa fa-check"></i><b>1.2</b> Reviews and tutorials</a>
<ul>
<li class="chapter" data-level="1.2.1" data-path="introduction.html"><a href="introduction.html#workflow"><i class="fa fa-check"></i><b>1.2.1</b> Workflow</a></li>
<li class="chapter" data-level="1.2.2" data-path="introduction.html"><a href="introduction.html#data-analysis"><i class="fa fa-check"></i><b>1.2.2</b> Data analysis</a></li>
<li class="chapter" data-level="1.2.3" data-path="introduction.html"><a href="introduction.html#application"><i class="fa fa-check"></i><b>1.2.3</b> Application</a></li>
<li class="chapter" data-level="1.2.4" data-path="introduction.html"><a href="introduction.html#challenge"><i class="fa fa-check"></i><b>1.2.4</b> Challenge</a></li>
</ul></li>
<li class="chapter" data-level="1.3" data-path="introduction.html"><a href="introduction.html#trends-in-metabolomics"><i class="fa fa-check"></i><b>1.3</b> Trends in Metabolomics</a></li>
<li class="chapter" data-level="1.4" data-path="introduction.html"><a href="introduction.html#workflow-1"><i class="fa fa-check"></i><b>1.4</b> Workflow</a></li>
</ul></li>
<li class="chapter" data-level="2" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html"><i class="fa fa-check"></i><b>2</b> Experimental design(DoE)</a>
<ul>
<li class="chapter" data-level="2.1" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#homogeneity-study"><i class="fa fa-check"></i><b>2.1</b> Homogeneity study</a></li>
<li class="chapter" data-level="2.2" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#heterogeneity-study"><i class="fa fa-check"></i><b>2.2</b> Heterogeneity study</a></li>
<li class="chapter" data-level="2.3" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#power-analysis"><i class="fa fa-check"></i><b>2.3</b> Power analysis</a></li>
<li class="chapter" data-level="2.4" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#optimization"><i class="fa fa-check"></i><b>2.4</b> Optimization</a></li>
<li class="chapter" data-level="2.5" data-path="experimental-designdoe.html"><a href="experimental-designdoe.html#pooled-qc"><i class="fa fa-check"></i><b>2.5</b> Pooled QC</a></li>
</ul></li>
<li class="chapter" data-level="3" data-path="pretreatment.html"><a href="pretreatment.html"><i class="fa fa-check"></i><b>3</b> Pretreatment</a>
<ul>
<li class="chapter" data-level="3.1" data-path="pretreatment.html"><a href="pretreatment.html#collection"><i class="fa fa-check"></i><b>3.1</b> Collection</a></li>
<li class="chapter" data-level="3.2" data-path="pretreatment.html"><a href="pretreatment.html#quenching"><i class="fa fa-check"></i><b>3.2</b> Quenching</a></li>
<li class="chapter" data-level="3.3" data-path="pretreatment.html"><a href="pretreatment.html#extraction"><i class="fa fa-check"></i><b>3.3</b> Extraction</a></li>
<li class="chapter" data-level="3.4" data-path="pretreatment.html"><a href="pretreatment.html#derivatization"><i class="fa fa-check"></i><b>3.4</b> Derivatization</a></li>
<li class="chapter" data-level="3.5" data-path="pretreatment.html"><a href="pretreatment.html#isotope-label"><i class="fa fa-check"></i><b>3.5</b> Isotope label</a></li>
<li class="chapter" data-level="3.6" data-path="pretreatment.html"><a href="pretreatment.html#storage"><i class="fa fa-check"></i><b>3.6</b> Storage</a></li>
</ul></li>
<li class="chapter" data-level="4" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html"><i class="fa fa-check"></i><b>4</b> Instrumental analysis</a>
<ul>
<li class="chapter" data-level="4.1" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#column-and-gradient-selection"><i class="fa fa-check"></i><b>4.1</b> Column and gradient selection</a></li>
<li class="chapter" data-level="4.2" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#mass-resolution"><i class="fa fa-check"></i><b>4.2</b> Mass resolution</a></li>
<li class="chapter" data-level="4.3" data-path="instrumental-analysis.html"><a href="instrumental-analysis.html#matrix-effects"><i class="fa fa-check"></i><b>4.3</b> Matrix effects</a></li>
</ul></li>
<li class="chapter" data-level="5" data-path="workflow-2.html"><a href="workflow-2.html"><i class="fa fa-check"></i><b>5</b> Workflow</a>
<ul>
<li class="chapter" data-level="5.1" data-path="workflow-2.html"><a href="workflow-2.html#platform-for-metabolomics-data-analysis"><i class="fa fa-check"></i><b>5.1</b> Platform for metabolomics data analysis</a>
<ul>
<li class="chapter" data-level="5.1.1" data-path="workflow-2.html"><a href="workflow-2.html#xcms-xcms-online"><i class="fa fa-check"></i><b>5.1.1</b> XCMS &amp; XCMS online</a></li>
<li class="chapter" data-level="5.1.2" data-path="workflow-2.html"><a href="workflow-2.html#prime"><i class="fa fa-check"></i><b>5.1.2</b> PRIMe</a></li>
<li class="chapter" data-level="5.1.3" data-path="workflow-2.html"><a href="workflow-2.html#gnps"><i class="fa fa-check"></i><b>5.1.3</b> GNPS</a></li>
<li class="chapter" data-level="5.1.4" data-path="workflow-2.html"><a href="workflow-2.html#openms-sirius"><i class="fa fa-check"></i><b>5.1.4</b> OpenMS &amp; SIRIUS</a></li>
<li class="chapter" data-level="5.1.5" data-path="workflow-2.html"><a href="workflow-2.html#mzmine-2"><i class="fa fa-check"></i><b>5.1.5</b> MZmine 2</a></li>
<li class="chapter" data-level="5.1.6" data-path="workflow-2.html"><a href="workflow-2.html#emory-mahpic"><i class="fa fa-check"></i><b>5.1.6</b> Emory MaHPIC</a></li>
<li class="chapter" data-level="5.1.7" data-path="workflow-2.html"><a href="workflow-2.html#others"><i class="fa fa-check"></i><b>5.1.7</b> Others</a></li>
<li class="chapter" data-level="5.1.8" data-path="workflow-2.html"><a href="workflow-2.html#workflow-comparison"><i class="fa fa-check"></i><b>5.1.8</b> Workflow Comparison</a></li>
</ul></li>
<li class="chapter" data-level="5.2" data-path="workflow-2.html"><a href="workflow-2.html#project-setup"><i class="fa fa-check"></i><b>5.2</b> Project Setup</a></li>
<li class="chapter" data-level="5.3" data-path="workflow-2.html"><a href="workflow-2.html#data-sharing"><i class="fa fa-check"></i><b>5.3</b> Data sharing</a></li>
<li class="chapter" data-level="5.4" data-path="workflow-2.html"><a href="workflow-2.html#contest"><i class="fa fa-check"></i><b>5.4</b> Contest</a></li>
</ul></li>
<li class="chapter" data-level="6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html"><i class="fa fa-check"></i><b>6</b> Raw data pretreatment</a>
<ul>
<li class="chapter" data-level="6.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#data-visualization"><i class="fa fa-check"></i><b>6.1</b> Data visualization</a></li>
<li class="chapter" data-level="6.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#peak-extraction"><i class="fa fa-check"></i><b>6.2</b> Peak extraction</a></li>
<li class="chapter" data-level="6.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#msms"><i class="fa fa-check"></i><b>6.3</b> MS/MS</a>
<ul>
<li class="chapter" data-level="6.3.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#mrm"><i class="fa fa-check"></i><b>6.3.1</b> MRM</a></li>
<li class="chapter" data-level="6.3.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dda"><i class="fa fa-check"></i><b>6.3.2</b> DDA</a></li>
<li class="chapter" data-level="6.3.3" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dia"><i class="fa fa-check"></i><b>6.3.3</b> DIA</a></li>
</ul></li>
<li class="chapter" data-level="6.4" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#retention-time-correction"><i class="fa fa-check"></i><b>6.4</b> Retention Time Correction</a></li>
<li class="chapter" data-level="6.5" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#filling-missing-values"><i class="fa fa-check"></i><b>6.5</b> Filling missing values</a></li>
<li class="chapter" data-level="6.6" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#spectral-deconvolution"><i class="fa fa-check"></i><b>6.6</b> Spectral deconvolution</a></li>
<li class="chapter" data-level="6.7" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#dynamic-range"><i class="fa fa-check"></i><b>6.7</b> Dynamic Range</a>
<ul>
<li class="chapter" data-level="6.7.1" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#non-detects"><i class="fa fa-check"></i><b>6.7.1</b> Non-detects</a></li>
<li class="chapter" data-level="6.7.2" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#over-detection-limit"><i class="fa fa-check"></i><b>6.7.2</b> Over Detection Limit</a></li>
</ul></li>
<li class="chapter" data-level="6.8" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#rsdfold-change-filter"><i class="fa fa-check"></i><b>6.8</b> RSD/fold change Filter</a></li>
<li class="chapter" data-level="6.9" data-path="raw-data-pretreatment.html"><a href="raw-data-pretreatment.html#power-analysis-filter"><i class="fa fa-check"></i><b>6.9</b> Power Analysis Filter</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="annotation.html"><a href="annotation.html"><i class="fa fa-check"></i><b>7</b> Annotation</a>
<ul>
<li class="chapter" data-level="7.1" data-path="annotation.html"><a href="annotation.html#issues-in-annotation"><i class="fa fa-check"></i><b>7.1</b> Issues in annotation</a></li>
<li class="chapter" data-level="7.2" data-path="annotation.html"><a href="annotation.html#peak-misidentification"><i class="fa fa-check"></i><b>7.2</b> Peak misidentification</a></li>
<li class="chapter" data-level="7.3" data-path="annotation.html"><a href="annotation.html#annotation-v.s.-identification"><i class="fa fa-check"></i><b>7.3</b> Annotation v.s. identification</a></li>
<li class="chapter" data-level="7.4" data-path="annotation.html"><a href="annotation.html#molecular-formula-assignment"><i class="fa fa-check"></i><b>7.4</b> Molecular Formula Assignment</a></li>
<li class="chapter" data-level="7.5" data-path="annotation.html"><a href="annotation.html#redundant-peaks"><i class="fa fa-check"></i><b>7.5</b> Redundant peaks</a>
<ul>
<li class="chapter" data-level="7.5.1" data-path="annotation.html"><a href="annotation.html#adducts-list"><i class="fa fa-check"></i><b>7.5.1</b> Adducts list</a></li>
<li class="chapter" data-level="7.5.2" data-path="annotation.html"><a href="annotation.html#isotope"><i class="fa fa-check"></i><b>7.5.2</b> Isotope</a></li>
<li class="chapter" data-level="7.5.3" data-path="annotation.html"><a href="annotation.html#camera"><i class="fa fa-check"></i><b>7.5.3</b> CAMERA</a></li>
<li class="chapter" data-level="7.5.4" data-path="annotation.html"><a href="annotation.html#ramclustr"><i class="fa fa-check"></i><b>7.5.4</b> RAMClustR</a></li>
<li class="chapter" data-level="7.5.5" data-path="annotation.html"><a href="annotation.html#biocan"><i class="fa fa-check"></i><b>7.5.5</b> BioCAn</a></li>
<li class="chapter" data-level="7.5.6" data-path="annotation.html"><a href="annotation.html#mzmatch"><i class="fa fa-check"></i><b>7.5.6</b> mzMatch</a></li>
<li class="chapter" data-level="7.5.7" data-path="annotation.html"><a href="annotation.html#xmsannotator"><i class="fa fa-check"></i><b>7.5.7</b> xMSannotator</a></li>
<li class="chapter" data-level="7.5.8" data-path="annotation.html"><a href="annotation.html#mwise"><i class="fa fa-check"></i><b>7.5.8</b> mWise</a></li>
<li class="chapter" data-level="7.5.9" data-path="annotation.html"><a href="annotation.html#mait"><i class="fa fa-check"></i><b>7.5.9</b> MAIT</a></li>
<li class="chapter" data-level="7.5.10" data-path="annotation.html"><a href="annotation.html#pmd"><i class="fa fa-check"></i><b>7.5.10</b> pmd</a></li>
<li class="chapter" data-level="7.5.11" data-path="annotation.html"><a href="annotation.html#nontarget"><i class="fa fa-check"></i><b>7.5.11</b> nontarget</a></li>
<li class="chapter" data-level="7.5.12" data-path="annotation.html"><a href="annotation.html#binner"><i class="fa fa-check"></i><b>7.5.12</b> Binner</a></li>
<li class="chapter" data-level="7.5.13" data-path="annotation.html"><a href="annotation.html#mz.unity"><i class="fa fa-check"></i><b>7.5.13</b> mz.unity</a></li>
<li class="chapter" data-level="7.5.14" data-path="annotation.html"><a href="annotation.html#ms-flo"><i class="fa fa-check"></i><b>7.5.14</b> MS-FLO</a></li>
<li class="chapter" data-level="7.5.15" data-path="annotation.html"><a href="annotation.html#cliquems"><i class="fa fa-check"></i><b>7.5.15</b> CliqueMS</a></li>
<li class="chapter" data-level="7.5.16" data-path="annotation.html"><a href="annotation.html#interpretmsspectrum"><i class="fa fa-check"></i><b>7.5.16</b> InterpretMSSpectrum</a></li>
<li class="chapter" data-level="7.5.17" data-path="annotation.html"><a href="annotation.html#netid"><i class="fa fa-check"></i><b>7.5.17</b> NetID</a></li>
<li class="chapter" data-level="7.5.18" data-path="annotation.html"><a href="annotation.html#isfrag"><i class="fa fa-check"></i><b>7.5.18</b> ISfrag</a></li>
</ul></li>
<li class="chapter" data-level="7.6" data-path="annotation.html"><a href="annotation.html#ms1-ms2-connection"><i class="fa fa-check"></i><b>7.6</b> MS1 MS2 connection</a>
<ul>
<li class="chapter" data-level="7.6.1" data-path="annotation.html"><a href="annotation.html#pmdda"><i class="fa fa-check"></i><b>7.6.1</b> PMDDA</a></li>
<li class="chapter" data-level="7.6.2" data-path="annotation.html"><a href="annotation.html#hermes"><i class="fa fa-check"></i><b>7.6.2</b> HERMES</a></li>
</ul></li>
<li class="chapter" data-level="7.7" data-path="annotation.html"><a href="annotation.html#msms-annotation"><i class="fa fa-check"></i><b>7.7</b> MS/MS annotation</a>
<ul>
<li class="chapter" data-level="7.7.1" data-path="annotation.html"><a href="annotation.html#matchms"><i class="fa fa-check"></i><b>7.7.1</b> Matchms</a></li>
<li class="chapter" data-level="7.7.2" data-path="annotation.html"><a href="annotation.html#metdna"><i class="fa fa-check"></i><b>7.7.2</b> MetDNA</a></li>
<li class="chapter" data-level="7.7.3" data-path="annotation.html"><a href="annotation.html#metfusion"><i class="fa fa-check"></i><b>7.7.3</b> MetFusion</a></li>
<li class="chapter" data-level="7.7.4" data-path="annotation.html"><a href="annotation.html#ms2analyzer"><i class="fa fa-check"></i><b>7.7.4</b> MS2Analyzer</a></li>
<li class="chapter" data-level="7.7.5" data-path="annotation.html"><a href="annotation.html#metfrag"><i class="fa fa-check"></i><b>7.7.5</b> MetFrag</a></li>
<li class="chapter" data-level="7.7.6" data-path="annotation.html"><a href="annotation.html#cfm-id"><i class="fa fa-check"></i><b>7.7.6</b> CFM-ID</a></li>
<li class="chapter" data-level="7.7.7" data-path="annotation.html"><a href="annotation.html#lc-ms2struct"><i class="fa fa-check"></i><b>7.7.7</b> LC-MS2Struct</a></li>
<li class="chapter" data-level="7.7.8" data-path="annotation.html"><a href="annotation.html#lipidfrag"><i class="fa fa-check"></i><b>7.7.8</b> LipidFrag</a></li>
<li class="chapter" data-level="7.7.9" data-path="annotation.html"><a href="annotation.html#lipidmatch"><i class="fa fa-check"></i><b>7.7.9</b> Lipidmatch</a></li>
<li class="chapter" data-level="7.7.10" data-path="annotation.html"><a href="annotation.html#barcoding"><i class="fa fa-check"></i><b>7.7.10</b> BarCoding</a></li>
<li class="chapter" data-level="7.7.11" data-path="annotation.html"><a href="annotation.html#imet"><i class="fa fa-check"></i><b>7.7.11</b> iMet</a></li>
</ul></li>
<li class="chapter" data-level="7.8" data-path="annotation.html"><a href="annotation.html#knowledge-based-annotation"><i class="fa fa-check"></i><b>7.8</b> Knowledge based annotation</a>
<ul>
<li class="chapter" data-level="7.8.1" data-path="annotation.html"><a href="annotation.html#chromatographic-retention-related-criteria"><i class="fa fa-check"></i><b>7.8.1</b> Chromatographic retention-related criteria</a></li>
<li class="chapter" data-level="7.8.2" data-path="annotation.html"><a href="annotation.html#probmetab"><i class="fa fa-check"></i><b>7.8.2</b> ProbMetab</a></li>
<li class="chapter" data-level="7.8.3" data-path="annotation.html"><a href="annotation.html#mi-pack"><i class="fa fa-check"></i><b>7.8.3</b> MI-Pack</a></li>
<li class="chapter" data-level="7.8.4" data-path="annotation.html"><a href="annotation.html#metexpert"><i class="fa fa-check"></i><b>7.8.4</b> MetExpert</a></li>
<li class="chapter" data-level="7.8.5" data-path="annotation.html"><a href="annotation.html#mycompoundid"><i class="fa fa-check"></i><b>7.8.5</b> MycompoundID</a></li>
<li class="chapter" data-level="7.8.6" data-path="annotation.html"><a href="annotation.html#metfamily"><i class="fa fa-check"></i><b>7.8.6</b> MetFamily</a></li>
<li class="chapter" data-level="7.8.7" data-path="annotation.html"><a href="annotation.html#plantmat"><i class="fa fa-check"></i><b>7.8.7</b> Plantmat</a></li>
<li class="chapter" data-level="7.8.8" data-path="annotation.html"><a href="annotation.html#coa-blast"><i class="fa fa-check"></i><b>7.8.8</b> CoA-Blast</a></li>
<li class="chapter" data-level="7.8.9" data-path="annotation.html"><a href="annotation.html#kgmn"><i class="fa fa-check"></i><b>7.8.9</b> KGMN</a></li>
</ul></li>
<li class="chapter" data-level="7.9" data-path="annotation.html"><a href="annotation.html#ms-database-for-annotation"><i class="fa fa-check"></i><b>7.9</b> MS Database for annotation</a>
<ul>
<li class="chapter" data-level="7.9.1" data-path="annotation.html"><a href="annotation.html#ms"><i class="fa fa-check"></i><b>7.9.1</b> MS</a></li>
<li class="chapter" data-level="7.9.2" data-path="annotation.html"><a href="annotation.html#msms-1"><i class="fa fa-check"></i><b>7.9.2</b> MS/MS</a></li>
</ul></li>
<li class="chapter" data-level="7.10" data-path="annotation.html"><a href="annotation.html#compounds-database"><i class="fa fa-check"></i><b>7.10</b> Compounds Database</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="omics-analysis.html"><a href="omics-analysis.html"><i class="fa fa-check"></i><b>8</b> Omics analysis</a>
<ul>
<li class="chapter" data-level="8.1" data-path="omics-analysis.html"><a href="omics-analysis.html#from-bottom-up-to-top-down"><i class="fa fa-check"></i><b>8.1</b> From Bottom-up to Top-down</a></li>
<li class="chapter" data-level="8.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-analysis"><i class="fa fa-check"></i><b>8.2</b> Pathway analysis</a>
<ul>
<li class="chapter" data-level="8.2.1" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-database"><i class="fa fa-check"></i><b>8.2.1</b> Pathway Database</a></li>
<li class="chapter" data-level="8.2.2" data-path="omics-analysis.html"><a href="omics-analysis.html#pathway-software"><i class="fa fa-check"></i><b>8.2.2</b> Pathway software</a></li>
</ul></li>
<li class="chapter" data-level="8.3" data-path="omics-analysis.html"><a href="omics-analysis.html#network-analysis"><i class="fa fa-check"></i><b>8.3</b> Network analysis</a></li>
<li class="chapter" data-level="8.4" data-path="omics-analysis.html"><a href="omics-analysis.html#omics-integration"><i class="fa fa-check"></i><b>8.4</b> Omics integration</a></li>
</ul></li>
<li class="chapter" data-level="9" data-path="peaks-normalization.html"><a href="peaks-normalization.html"><i class="fa fa-check"></i><b>9</b> Peaks normalization</a>
<ul>
<li class="chapter" data-level="9.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#batch-effects-classification"><i class="fa fa-check"></i><b>9.1</b> Batch effects classification</a></li>
<li class="chapter" data-level="9.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#batch-effects-visualization"><i class="fa fa-check"></i><b>9.2</b> Batch effects visualization</a></li>
<li class="chapter" data-level="9.3" data-path="peaks-normalization.html"><a href="peaks-normalization.html#source-of-batch-effects"><i class="fa fa-check"></i><b>9.3</b> Source of batch effects</a></li>
<li class="chapter" data-level="9.4" data-path="peaks-normalization.html"><a href="peaks-normalization.html#avoid-batch-effects-by-doe"><i class="fa fa-check"></i><b>9.4</b> Avoid batch effects by DoE</a></li>
<li class="chapter" data-level="9.5" data-path="peaks-normalization.html"><a href="peaks-normalization.html#post-hoc-data-normalization"><i class="fa fa-check"></i><b>9.5</b> <em>post hoc</em> data normalization</a>
<ul>
<li class="chapter" data-level="9.5.1" data-path="peaks-normalization.html"><a href="peaks-normalization.html#unsupervised-methods"><i class="fa fa-check"></i><b>9.5.1</b> Unsupervised methods</a></li>
<li class="chapter" data-level="9.5.2" data-path="peaks-normalization.html"><a href="peaks-normalization.html#supervised-methods"><i class="fa fa-check"></i><b>9.5.2</b> Supervised methods</a></li>
</ul></li>
<li class="chapter" data-level="9.6" data-path="peaks-normalization.html"><a href="peaks-normalization.html#method-to-validate-the-normalization"><i class="fa fa-check"></i><b>9.6</b> Method to validate the normalization</a></li>
<li class="chapter" data-level="9.7" data-path="peaks-normalization.html"><a href="peaks-normalization.html#software"><i class="fa fa-check"></i><b>9.7</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="10" data-path="statistical-analysis.html"><a href="statistical-analysis.html"><i class="fa fa-check"></i><b>10</b> Statistical analysis</a>
<ul>
<li class="chapter" data-level="10.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#basic-statistical-analysis"><i class="fa fa-check"></i><b>10.1</b> Basic Statistical Analysis</a></li>
<li class="chapter" data-level="10.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#differences-analysis"><i class="fa fa-check"></i><b>10.2</b> Differences analysis</a>
<ul>
<li class="chapter" data-level="10.2.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#t-test-or-anova"><i class="fa fa-check"></i><b>10.2.1</b> T-test or ANOVA</a></li>
<li class="chapter" data-level="10.2.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#limma"><i class="fa fa-check"></i><b>10.2.2</b> LIMMA</a></li>
<li class="chapter" data-level="10.2.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#bayesian-mixture-model"><i class="fa fa-check"></i><b>10.2.3</b> Bayesian mixture model</a></li>
</ul></li>
<li class="chapter" data-level="10.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#pca"><i class="fa fa-check"></i><b>10.3</b> PCA</a></li>
<li class="chapter" data-level="10.4" data-path="statistical-analysis.html"><a href="statistical-analysis.html#cluster-analysis"><i class="fa fa-check"></i><b>10.4</b> Cluster Analysis</a></li>
<li class="chapter" data-level="10.5" data-path="statistical-analysis.html"><a href="statistical-analysis.html#plsda"><i class="fa fa-check"></i><b>10.5</b> PLSDA</a></li>
<li class="chapter" data-level="10.6" data-path="statistical-analysis.html"><a href="statistical-analysis.html#network-analysis-1"><i class="fa fa-check"></i><b>10.6</b> Network analysis</a>
<ul>
<li class="chapter" data-level="10.6.1" data-path="statistical-analysis.html"><a href="statistical-analysis.html#vertex-and-edge"><i class="fa fa-check"></i><b>10.6.1</b> Vertex and edge</a></li>
<li class="chapter" data-level="10.6.2" data-path="statistical-analysis.html"><a href="statistical-analysis.html#build-the-network"><i class="fa fa-check"></i><b>10.6.2</b> Build the network</a></li>
<li class="chapter" data-level="10.6.3" data-path="statistical-analysis.html"><a href="statistical-analysis.html#network-attributes"><i class="fa fa-check"></i><b>10.6.3</b> Network attributes</a></li>
</ul></li>
<li class="chapter" data-level="10.7" data-path="statistical-analysis.html"><a href="statistical-analysis.html#software-1"><i class="fa fa-check"></i><b>10.7</b> Software</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="exposome.html"><a href="exposome.html"><i class="fa fa-check"></i><b>11</b> Exposome</a>
<ul>
<li class="chapter" data-level="11.1" data-path="exposome.html"><a href="exposome.html#internal-exposure"><i class="fa fa-check"></i><b>11.1</b> Internal exposure</a></li>
<li class="chapter" data-level="11.2" data-path="exposome.html"><a href="exposome.html#external-exposure"><i class="fa fa-check"></i><b>11.2</b> External exposure</a>
<ul>
<li class="chapter" data-level="11.2.1" data-path="exposome.html"><a href="exposome.html#environmental-fate-of-compounds"><i class="fa fa-check"></i><b>11.2.1</b> Environmental fate of compounds</a></li>
<li class="chapter" data-level="11.2.2" data-path="exposome.html"><a href="exposome.html#exposure-study-database"><i class="fa fa-check"></i><b>11.2.2</b> Exposure study database</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i>References</a></li>
<li class="divider"></li>
<li><a href="https://github.com/yufree/metaworkflow" target="blank">Published with bookdown</a></li>

</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">Meta-Workflow</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="references" class="section level1 unnumbered hasAnchor">
<h1>References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h1>

<div id="refs" class="references csl-bib-body hanging-indent">
<div class="csl-entry">
Adams, Kendra J., Brian Pratt, Neelanjan Bose, Laura G. Dubois, Lisa St John-Williams, Kevin M. Perrott, Karina Ky, et al. 2020. <span>“Skyline for <span>Small Molecules</span>: <span>A Unifying Software Package</span> for <span>Quantitative Metabolomics</span>.”</span> <em>Journal of Proteome Research</em> 19 (4): 1447–58. <a href="https://doi.org/10.1021/acs.jproteome.9b00640">https://doi.org/10.1021/acs.jproteome.9b00640</a>.
</div>
<div class="csl-entry">
Aguilar-Mogas, Antoni, Marta Sales-Pardo, Miriam Navarro, Roger Guimerà, and Oscar Yanes. 2017. <span>“<span class="nocase">iMet</span>: <span>A Network-Based Computational Tool To Assist</span> in the <span>Annotation</span> of <span>Metabolites</span> from <span>Tandem Mass Spectra</span>.”</span> <em>Analytical Chemistry</em> 89 (6): 3474–82. <a href="https://doi.org/10.1021/acs.analchem.6b04512">https://doi.org/10.1021/acs.analchem.6b04512</a>.
</div>
<div class="csl-entry">
Alden, Nicholas, Smitha Krishnan, Vladimir Porokhin, Ravali Raju, Kyle McElearney, Alan Gilbert, and Kyongbum Lee. 2017. <span>“Biologically <span>Consistent Annotation</span> of <span>Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em>, November. <a href="https://doi.org/10.1021/acs.analchem.7b02162">https://doi.org/10.1021/acs.analchem.7b02162</a>.
</div>
<div class="csl-entry">
Ali, Ahmed, Yasmine Abouleila, Yoshihiro Shimizu, Eiso Hiyama, Samy Emara, Alireza Mashaghi, and Thomas Hankemeier. 2019. <span>“Single-Cell Metabolomics by Mass Spectrometry: <span>Advances</span>, Challenges, and Future Applications.”</span> <em>TrAC Trends in Analytical Chemistry</em> 120 (November): 115436. <a href="https://doi.org/10.1016/j.trac.2019.02.033">https://doi.org/10.1016/j.trac.2019.02.033</a>.
</div>
<div class="csl-entry">
Alka, Oliver, Timo Sachsenberg, Leon Bichmann, Julianus Pfeuffer, Hendrik Weisser, Samuel Wein, Eugen Netz, Marc Rurik, Oliver Kohlbacher, and Hannes Röst. 2020. <span>“<span>CHAPTER</span> 6:<span>OpenMS</span> and <span>KNIME</span> for <span>Mass Spectrometry Data Processing</span>.”</span> In <em>Processing <span>Metabolomics</span> and <span>Proteomics Data</span> with <span>Open Software</span></em>, 201–31. <a href="https://doi.org/10.1039/9781788019880-00201">https://doi.org/10.1039/9781788019880-00201</a>.
</div>
<div class="csl-entry">
Alka, Oliver, Premy Shanthamoorthy, Michael Witting, Karin Kleigrewe, Oliver Kohlbacher, and Hannes L. Röst. 2022. <span>“<span>DIAMetAlyzer</span> Allows Automated False-Discovery Rate-Controlled Analysis for Data-Independent Acquisition in Metabolomics.”</span> <em>Nature Communications</em> 13 (1): 1347. <a href="https://doi.org/10.1038/s41467-022-29006-z">https://doi.org/10.1038/s41467-022-29006-z</a>.
</div>
<div class="csl-entry">
Allam-Ndoul, Bénédicte, Frédéric Guénard, Véronique Garneau, Hubert Cormier, Olivier Barbier, Louis Pérusse, and Marie-Claude Vohl. 2016. <span>“Association Between <span>Metabolite Profiles</span>, <span>Metabolic Syndrome</span> and <span>Obesity Status</span>.”</span> <em>Nutrients</em> 8 (6): 324. <a href="https://doi.org/10.3390/nu8060324">https://doi.org/10.3390/nu8060324</a>.
</div>
<div class="csl-entry">
Allard, Pierre-Marie, Grégory Genta-Jouve, and Jean-Luc Wolfender. 2017. <span>“Deep Metabolome Annotation in Natural Products Research: Towards a Virtuous Cycle in Metabolite Identification.”</span> <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 40–49. <a href="https://doi.org/10.1016/j.cbpa.2016.12.022">https://doi.org/10.1016/j.cbpa.2016.12.022</a>.
</div>
<div class="csl-entry">
Allen, Felicity, Allison Pon, Michael Wilson, Russ Greiner, and David Wishart. 2014. <span>“<span>CFM-ID</span>: A Web Server for Annotation, Spectrum Prediction and Metabolite Identification from Tandem Mass Spectra.”</span> <em>Nucleic Acids Research</em> 42 (W1): W94–99. <a href="https://doi.org/10.1093/nar/gku436">https://doi.org/10.1093/nar/gku436</a>.
</div>
<div class="csl-entry">
Alonso, Arnald, Sara Marsal, and Antonio Julià. 2015. <span>“Analytical <span>Methods</span> in <span>Untargeted Metabolomics</span>: <span>State</span> of the <span>Art</span> in 2015.”</span> <em>Frontiers in Bioengineering and Biotechnology</em> 3 (March). <a href="https://doi.org/10.3389/fbioe.2015.00023">https://doi.org/10.3389/fbioe.2015.00023</a>.
</div>
<div class="csl-entry">
Anderson, Brady G., Alexander Raskind, Hani Habra, Robert T. Kennedy, and Charles R. Evans. 2021. <span>“Modifying <span>Chromatography Conditions</span> for <span>Improved Unknown Feature Identification</span> in <span>Untargeted Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 93 (48): 15840–49. <a href="https://doi.org/10.1021/acs.analchem.1c02149">https://doi.org/10.1021/acs.analchem.1c02149</a>.
</div>
<div class="csl-entry">
Andersson, Martin. 2009. <span>“A Comparison of Nine <span>PLS1</span> Algorithms.”</span> <em>Journal of Chemometrics</em> 23 (10): 518–29. <a href="https://doi.org/10.1002/cem.1248">https://doi.org/10.1002/cem.1248</a>.
</div>
<div class="csl-entry">
Aron, Allegra T., Emily C. Gentry, Kerry L. McPhail, Louis-Félix Nothias, Mélissa Nothias-Esposito, Amina Bouslimani, Daniel Petras, et al. 2020. <span>“Reproducible Molecular Networking of Untargeted Mass Spectrometry Data Using <span>GNPS</span>.”</span> <em>Nature Protocols</em> 15 (6): 1954–91. <a href="https://doi.org/10.1038/s41596-020-0317-5">https://doi.org/10.1038/s41596-020-0317-5</a>.
</div>
<div class="csl-entry">
Bach, Eric, Emma L. Schymanski, and Juho Rousu. 2022. <span>“Joint Structural Annotation of Small Molecules Using Liquid Chromatography Retention Order and Tandem Mass Spectrometry Data.”</span> <em>Nature Machine Intelligence</em> 4 (12): 1224–37. <a href="https://doi.org/10.1038/s42256-022-00577-2">https://doi.org/10.1038/s42256-022-00577-2</a>.
</div>
<div class="csl-entry">
Bai, Caihong, Suyun Xu, Jingyi Tang, Yuxi Zhang, Jiahui Yang, and Kaifeng Hu. 2022. <span>“A <span>‘Shape-Orientated’</span> Algorithm Employing an Adapted <span>Marr</span> Wavelet and Shape Matching Index Improves the Performance of Continuous Wavelet Transform for Chromatographic Peak Detection and Quantification.”</span> <em>Journal of Chromatography A</em> 1673 (June): 463086. <a href="https://doi.org/10.1016/j.chroma.2022.463086">https://doi.org/10.1016/j.chroma.2022.463086</a>.
</div>
<div class="csl-entry">
Baker, Monya. 2011. <span>“Metabolomics: From Small Molecules to Big Ideas.”</span> <em>Nature Methods</em> 8 (2): 117–21. <a href="https://doi.org/10.1038/nmeth0211-117">https://doi.org/10.1038/nmeth0211-117</a>.
</div>
<div class="csl-entry">
Baran, Richard, and Trent R. Northen. 2013. <span>“Robust <span>Automated Mass Spectra Interpretation</span> and <span>Chemical Formula Calculation Using Mixed Integer Linear Programming</span>.”</span> <em>Analytical Chemistry</em> 85 (20): 9777–84. <a href="https://doi.org/10.1021/ac402180c">https://doi.org/10.1021/ac402180c</a>.
</div>
<div class="csl-entry">
Barbier Saint Hilaire, Pierre, Ulli M. Hohenester, Benoit Colsch, Jean-Claude Tabet, Christophe Junot, and François Fenaille. 2018. <span>“Evaluation of the <span>High-Field Orbitrap Fusion</span> for <span>Compound Annotation</span> in <span>Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 90 (5): 3030–35. <a href="https://doi.org/10.1021/acs.analchem.7b05372">https://doi.org/10.1021/acs.analchem.7b05372</a>.
</div>
<div class="csl-entry">
Barnes, Stephen, H. Paul Benton, Krista Casazza, Sara J. Cooper, Xiangqin Cui, Xiuxia Du, Jeffrey Engler, et al. 2016a. <span>“Training in Metabolomics Research. <span>I</span>. <span>Designing</span> the Experiment, Collecting and Extracting Samples and Generating Metabolomics Data.”</span> <em>Journal of Mass Spectrometry</em> 51 (7): 461–75. <a href="https://doi.org/10.1002/jms.3782">https://doi.org/10.1002/jms.3782</a>.
</div>
<div class="csl-entry">
———, et al. 2016b. <span>“Training in Metabolomics Research. <span>II</span>. <span>Processing</span> and Statistical Analysis of Metabolomics Data, Metabolite Identification, Pathway Analysis, Applications of Metabolomics and Its Future.”</span> <em>Journal of Mass Spectrometry</em> 51 (8): 535–48. <a href="https://doi.org/10.1002/jms.3780">https://doi.org/10.1002/jms.3780</a>.
</div>
<div class="csl-entry">
Barranco-Altirriba, Maria, Pol Solà-Santos, Sergio Picart-Armada, Samir Kanaan-Izquierdo, Jordi Fonollosa, and Alexandre Perera-Lluna. 2021. <span>“<span class="nocase">mWISE</span>: <span>An Algorithm</span> for <span>Context-Based Annotation</span> of <span>Liquid Chromatography</span> Through <span>Diffusion</span> in <span>Graphs</span>.”</span> <em>Analytical Chemistry</em> 93 (31): 10772–78. <a href="https://doi.org/10.1021/acs.analchem.1c00238">https://doi.org/10.1021/acs.analchem.1c00238</a>.
</div>
<div class="csl-entry">
Basu, Sumanta, William Duren, Charles R. Evans, Charles F. Burant, George Michailidis, and Alla Karnovsky. 2017. <span>“Sparse Network Modeling and Metscape-Based Visualization Methods for the Analysis of Large-Scale Metabolomics Data.”</span> <em>Bioinformatics</em> 33 (10): 1545–53. <a href="https://doi.org/10.1093/bioinformatics/btx012">https://doi.org/10.1093/bioinformatics/btx012</a>.
</div>
<div class="csl-entry">
Baygi, Sadjad Fakouri, Sanjay K. Banerjee, Praloy Chakraborty, Yashwant Kumar, and Dinesh Kumar Barupal. 2022. <span>“<span>IDSL</span>.<span>UFA Assigns High-Confidence Molecular Formula Annotations</span> for <span>Untargeted LC</span>/<span>HRMS Data Sets</span> in <span>Metabolomics</span> and <span>Exposomics</span>.”</span> <em>Analytical Chemistry</em> 94 (39): 13315–22. <a href="https://doi.org/10.1021/acs.analchem.2c00563">https://doi.org/10.1021/acs.analchem.2c00563</a>.
</div>
<div class="csl-entry">
Beale, David J., Farhana R. Pinu, Konstantinos A. Kouremenos, Mahesha M. Poojary, Vinod K. Narayana, Berin A. Boughton, Komal Kanojia, Saravanan Dayalan, Oliver A. H. Jones, and Daniel A. Dias. 2018. <span>“Review of Recent Developments in <span>GC</span> Approaches to Metabolomics-Based Research.”</span> <em>Metabolomics</em> 14 (11): 152. <a href="https://doi.org/10.1007/s11306-018-1449-2">https://doi.org/10.1007/s11306-018-1449-2</a>.
</div>
<div class="csl-entry">
Begou, O., H. G. Gika, I. D. Wilson, and G. Theodoridis. 2017. <span>“Hyphenated <span class="nocase">MS-based</span> Targeted Approaches in Metabolomics.”</span> <em>Analyst</em> 142 (17): 3079–3100. <a href="https://doi.org/10.1039/C7AN00812K">https://doi.org/10.1039/C7AN00812K</a>.
</div>
<div class="csl-entry">
Benjamini, Yoav, and Yosef Hochberg. 1995. <span>“Controlling the <span>False Discovery Rate</span>: <span>A Practical</span> and <span>Powerful Approach</span> to <span>Multiple Testing</span>.”</span> <em>Journal of the Royal Statistical Society. Series B (Methodological)</em> 57 (1): 289–300.
</div>
<div class="csl-entry">
Bennett, Bryson D., Elizabeth H. Kimball, Melissa Gao, Robin Osterhout, Stephen J. Van Dien, and Joshua D. Rabinowitz. 2009. <span>“Absolute Metabolite Concentrations and Implied Enzyme Active Site Occupancy in <span>Escherichia</span> Coli.”</span> <em>Nature Chemical Biology</em> 5 (8): 593–99. <a href="https://doi.org/10.1038/nchembio.186">https://doi.org/10.1038/nchembio.186</a>.
</div>
<div class="csl-entry">
Bertsch, Andreas, Clemens Gröpl, Knut Reinert, and Oliver Kohlbacher. 2011. <span>“<span>OpenMS</span> and <span>TOPP</span>: <span>Open Source Software</span> for <span>LC-MS Data Analysis</span>.”</span> In <em>Data <span>Mining</span> in <span>Proteomics</span>: <span>From Standards</span> to <span>Applications</span></em>, edited by Michael Hamacher, Martin Eisenacher, and Christian Stephan, 353–67. Methods in <span>Molecular Biology</span>. <span>Totowa, NJ</span>: <span>Humana Press</span>. <a href="https://doi.org/10.1007/978-1-60761-987-1_23">https://doi.org/10.1007/978-1-60761-987-1_23</a>.
</div>
<div class="csl-entry">
Bijttebier, Sebastiaan, Anastasia Van der Auwera, Kenn Foubert, Stefan Voorspoels, Luc Pieters, and Sandra Apers. 2016. <span>“Bridging the Gap Between Comprehensive Extraction Protocols in Plant Metabolomics Studies and Method Validation.”</span> <em>Analytica Chimica Acta</em> 935 (September): 136–50. <a href="https://doi.org/10.1016/j.aca.2016.06.047">https://doi.org/10.1016/j.aca.2016.06.047</a>.
</div>
<div class="csl-entry">
Bilbao, Aivett, Emmanuel Varesio, Jeremy Luban, Caterina Strambio-De-Castillia, Gérard Hopfgartner, Markus Müller, and Frédérique Lisacek. 2015. <span>“Processing Strategies and Software Solutions for Data-Independent Acquisition in Mass Spectrometry.”</span> <em>PROTEOMICS</em> 15 (5-6): 964–80. <a href="https://doi.org/10.1002/pmic.201400323">https://doi.org/10.1002/pmic.201400323</a>.
</div>
<div class="csl-entry">
Bittremieux, Wout, Robin Schmid, Florian Huber, Justin J. J. van der Hooft, Mingxun Wang, and Pieter C. Dorrestein. 2022. <span>“Comparison of <span>Cosine</span>, <span>Modified Cosine</span>, and <span>Neutral Loss Based Spectrum Alignment For Discovery</span> of <span>Structurally Related Molecules</span>.”</span> <em>Journal of the American Society for Mass Spectrometry</em> 33 (9): 1733–44. <a href="https://doi.org/10.1021/jasms.2c00153">https://doi.org/10.1021/jasms.2c00153</a>.
</div>
<div class="csl-entry">
Blaise, Benjamin J. 2013. <span>“Data-<span>Driven Sample Size Determination</span> for <span>Metabolic Phenotyping Studies</span>.”</span> <em>Analytical Chemistry</em> 85 (19): 8943–50. <a href="https://doi.org/10.1021/ac4022314">https://doi.org/10.1021/ac4022314</a>.
</div>
<div class="csl-entry">
Blaise, Benjamin J., Gonçalo D. S. Correia, Gordon A. Haggart, Izabella Surowiec, Caroline Sands, Matthew R. Lewis, Jake T. M. Pearce, et al. 2021. <span>“Statistical Analysis in Metabolic Phenotyping.”</span> <em>Nature Protocols</em>, July, 1–28. <a href="https://doi.org/10.1038/s41596-021-00579-1">https://doi.org/10.1038/s41596-021-00579-1</a>.
</div>
<div class="csl-entry">
Blaise, Benjamin J., Gonçalo Correia, Adrienne Tin, J. Hunter Young, Anne-Claire Vergnaud, Matthew Lewis, Jake T. M. Pearce, et al. 2016. <span>“Power <span>Analysis</span> and <span>Sample Size Determination</span> in <span>Metabolic Phenotyping</span>.”</span> <em>Analytical Chemistry</em> 88 (10): 5179–88. <a href="https://doi.org/10.1021/acs.analchem.6b00188">https://doi.org/10.1021/acs.analchem.6b00188</a>.
</div>
<div class="csl-entry">
Blaženović, Ivana, Tobias Kind, Hrvoje Torbašinović, Slobodan Obrenović, Sajjan S. Mehta, Hiroshi Tsugawa, Tobias Wermuth, et al. 2017. <span>“Comprehensive Comparison of in Silico <span>MS</span>/<span>MS</span> Fragmentation Tools of the <span>CASMI</span> Contest: Database Boosting Is Needed to Achieve 93% Accuracy.”</span> <em>Journal of Cheminformatics</em> 9 (1): 32. <a href="https://doi.org/10.1186/s13321-017-0219-x">https://doi.org/10.1186/s13321-017-0219-x</a>.
</div>
<div class="csl-entry">
Bonini, Paolo, Tobias Kind, Hiroshi Tsugawa, Dinesh Kumar Barupal, and Oliver Fiehn. 2020. <span>“Retip: <span>Retention Time Prediction</span> for <span>Compound Annotation</span> in <span>Untargeted Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 92 (11): 7515–22. <a href="https://doi.org/10.1021/acs.analchem.9b05765">https://doi.org/10.1021/acs.analchem.9b05765</a>.
</div>
<div class="csl-entry">
Bonner, Ron, and Gérard Hopfgartner. 2018. <span>“<span>SWATH</span> Data Independent Acquisition Mass Spectrometry for Metabolomics.”</span> <em>TrAC Trends in Analytical Chemistry</em>, October. <a href="https://doi.org/10.1016/j.trac.2018.10.014">https://doi.org/10.1016/j.trac.2018.10.014</a>.
</div>
<div class="csl-entry">
Box, George E. P., J. Stuart Hunter, and William G. Hunter. 2005. <em>Statistics for <span>Experimenters</span></em>. <span>Wiley-Interscience</span>.
</div>
<div class="csl-entry">
Brereton, Richard G., and Gavin R. Lloyd. 2018. <span>“Partial Least Squares Discriminant Analysis for Chemometrics and Metabolomics: <span>How</span> Scores, Loadings, and Weights Differ According to Two Common Algorithms.”</span> <em>Journal of Chemometrics</em> 32 (4): e3028. <a href="https://doi.org/10.1002/cem.3028">https://doi.org/10.1002/cem.3028</a>.
</div>
<div class="csl-entry">
Broadhurst, David, Royston Goodacre, Stacey N. Reinke, Julia Kuligowski, Ian D. Wilson, Matthew R. Lewis, and Warwick B. Dunn. 2018. <span>“Guidelines and Considerations for the Use of System Suitability and Quality Control Samples in Mass Spectrometry Assays Applied in Untargeted Clinical Metabolomic Studies.”</span> <em>Metabolomics</em> 14 (6). <a href="https://doi.org/10.1007/s11306-018-1367-3">https://doi.org/10.1007/s11306-018-1367-3</a>.
</div>
<div class="csl-entry">
Broeckling, C. D., F. A. Afsar, S. Neumann, A. Ben-Hur, and J. E. Prenni. 2014. <span>“<span>RAMClust</span>: <span>A Novel Feature Clustering Method Enables Spectral-Matching-Based Annotation</span> for <span>Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 86 (14): 6812–17. <a href="https://doi.org/10.1021/ac501530d">https://doi.org/10.1021/ac501530d</a>.
</div>
<div class="csl-entry">
Broeckling, Corey D., Andrea Ganna, Mark Layer, Kevin Brown, Ben Sutton, Erik Ingelsson, Graham Peers, and Jessica E. Prenni. 2016. <span>“Enabling <span>Efficient</span> and <span>Confident Annotation</span> of <span>LC</span>-<span>MS Metabolomics Data</span> Through <span>MS1 Spectrum</span> and <span>Time Prediction</span>.”</span> <em>Analytical Chemistry</em> 88 (18): 9226–34. <a href="https://doi.org/10.1021/acs.analchem.6b02479">https://doi.org/10.1021/acs.analchem.6b02479</a>.
</div>
<div class="csl-entry">
Bundy, Jacob G., Matthew P. Davey, and Mark R. Viant. 2009. <span>“Environmental Metabolomics: A Critical Review and Future Perspectives.”</span> <em>Metabolomics</em> 5 (1): 3. <a href="https://doi.org/10.1007/s11306-008-0152-0">https://doi.org/10.1007/s11306-008-0152-0</a>.
</div>
<div class="csl-entry">
Cai, Jingwei, and Zhengyin Yan. 2021. <span>“Re-<span>Examining</span> the <span>Impact</span> of <span>Minimal Scans</span> in <span>Liquid Chromatography</span>.”</span> <em>Journal of the American Society for Mass Spectrometry</em>, June. <a href="https://doi.org/10.1021/jasms.1c00073">https://doi.org/10.1021/jasms.1c00073</a>.
</div>
<div class="csl-entry">
Cai, Qingpo, Jessica A. Alvarez, Jian Kang, and Tianwei Yu. 2017. <span>“Network <span>Marker Selection</span> for <span>Untargeted LC</span>.”</span> <em>Journal of Proteome Research</em> 16 (3): 1261–69. <a href="https://doi.org/10.1021/acs.jproteome.6b00861">https://doi.org/10.1021/acs.jproteome.6b00861</a>.
</div>
<div class="csl-entry">
Cajka, Tomas, and Oliver Fiehn. 2016. <span>“Toward <span>Merging Untargeted</span> and <span>Targeted Methods</span> in <span>Mass Spectrometry-Based Metabolomics</span> and <span>Lipidomics</span>.”</span> <em>Analytical Chemistry</em> 88 (1): 524–45. <a href="https://doi.org/10.1021/acs.analchem.5b04491">https://doi.org/10.1021/acs.analchem.5b04491</a>.
</div>
<div class="csl-entry">
Calbiani, F., M. Careri, L. Elviri, A. Mangia, and I. Zagnoni. 2006. <span>“Matrix Effects on Accurate Mass Measurements of Low-Molecular Weight Compounds Using Liquid Chromatography-Electrospray-Quadrupole Time-of-Flight Mass Spectrometry.”</span> <em>Journal of Mass Spectrometry</em> 41 (3): 289–94. <a href="https://doi.org/10.1002/jms.984">https://doi.org/10.1002/jms.984</a>.
</div>
<div class="csl-entry">
Carroll, Adam J., Murray R. Badger, and A. Harvey Millar. 2010. <span>“The <span>MetabolomeExpress Project</span>: Enabling Web-Based Processing, Analysis and Transparent Dissemination of <span>GC</span>/<span>MS</span> Metabolomics Datasets.”</span> <em>BMC Bioinformatics</em> 11 (1): 376. <a href="https://doi.org/10.1186/1471-2105-11-376">https://doi.org/10.1186/1471-2105-11-376</a>.
</div>
<div class="csl-entry">
Castro-Puyana, María, Raquel Pérez-Míguez, Lidia Montero, and Miguel Herrero. 2017. <span>“Application of Mass Spectrometry-Based Metabolomics Approaches for Food Safety, Quality and Traceability.”</span> <em>TrAC Trends in Analytical Chemistry</em> 93 (August): 102–18. <a href="https://doi.org/10.1016/j.trac.2017.05.004">https://doi.org/10.1016/j.trac.2017.05.004</a>.
</div>
<div class="csl-entry">
Chaker, Jade, David Møbjerg Kristensen, Thorhallur Ingi Halldorsson, Sjurdur Frodi Olsen, Christine Monfort, Cécile Chevrier, Bernard Jégou, and Arthur David. 2022. <span>“Comprehensive <span>Evaluation</span> of <span>Blood Plasma</span> and <span>Serum Sample Preparations</span> for <span>HRMS-Based Chemical Exposomics</span>: <span>Overlaps</span> and <span>Specificities</span>.”</span> <em>Analytical Chemistry</em> 94 (2): 866–74. <a href="https://doi.org/10.1021/acs.analchem.1c03638">https://doi.org/10.1021/acs.analchem.1c03638</a>.
</div>
<div class="csl-entry">
Chaleckis, Romanas, Isabel Meister, Pei Zhang, and Craig E Wheelock. 2019. <span>“Challenges, Progress and Promises of Metabolite Annotation for <span>LC</span> Metabolomics.”</span> <em>Current Opinion in Biotechnology</em>, Analytical <span>Biotechnology</span>, 55 (February): 44–50. <a href="https://doi.org/10.1016/j.copbio.2018.07.010">https://doi.org/10.1016/j.copbio.2018.07.010</a>.
</div>
<div class="csl-entry">
Chambers, Matthew C., Brendan Maclean, Robert Burke, Dario Amodei, Daniel L. Ruderman, Steffen Neumann, Laurent Gatto, et al. 2012. <span>“A Cross-Platform Toolkit for Mass Spectrometry and Proteomics.”</span> <em>Nature Biotechnology</em> 30 (October): 918–20. <a href="https://doi.org/10.1038/nbt.2377">https://doi.org/10.1038/nbt.2377</a>.
</div>
<div class="csl-entry">
Chang, Hui-Yin, Ching-Tai Chen, T. Mamie Lih, Ke-Shiuan Lynn, Chiun-Gung Juo, Wen-Lian Hsu, and Ting-Yi Sung. 2016. <span>“<span class="nocase">iMet-Q</span>: <span>A User-Friendly Tool</span> for <span>Label-Free Metabolomics Quantitation Using Dynamic Peak-Width Determination</span>.”</span> <em>PLOS ONE</em> 11 (1): e0146112. <a href="https://doi.org/10.1371/journal.pone.0146112">https://doi.org/10.1371/journal.pone.0146112</a>.
</div>
<div class="csl-entry">
Chang, Hui-Yin, Sean M. Colby, Xiuxia Du, Javier D. Gomez, Maximilian J. Helf, Katerina Kechris, Christine R. Kirkpatrick, et al. 2021. <span>“A <span>Practical Guide</span> to <span>Metabolomics Software Development</span>.”</span> <em>Analytical Chemistry</em> 93 (4): 1912–23. <a href="https://doi.org/10.1021/acs.analchem.0c03581">https://doi.org/10.1021/acs.analchem.0c03581</a>.
</div>
<div class="csl-entry">
Charbonnet, Joseph A., Carrie A. McDonough, Feng Xiao, Trever Schwichtenberg, Dunping Cao, Sarit Kaserzon, Kevin V. Thomas, et al. 2022. <span>“Communicating <span>Confidence</span> of <span>Per-</span> and <span>Polyfluoroalkyl Substance Identification</span> via <span>High-Resolution Mass Spectrometry</span>.”</span> <em>Environmental Science &amp; Technology Letters</em>, May. <a href="https://doi.org/10.1021/acs.estlett.2c00206">https://doi.org/10.1021/acs.estlett.2c00206</a>.
</div>
<div class="csl-entry">
Chen, Gengbo, Scott Walmsley, Gemmy C. M. Cheung, Liyan Chen, Ching-Yu Cheng, Roger W. Beuerman, Tien Yin Wong, Lei Zhou, and Hyungwon Choi. 2017. <span>“Customized <span>Consensus Spectral Library Building</span> for <span>Untargeted Quantitative Metabolomics Analysis</span> with <span>Data Independent Acquisition Mass Spectrometry</span> and <span>MetaboDIA Workflow</span>.”</span> <em>Analytical Chemistry</em> 89 (9): 4897–4906. <a href="https://doi.org/10.1021/acs.analchem.6b05006">https://doi.org/10.1021/acs.analchem.6b05006</a>.
</div>
<div class="csl-entry">
Chen, Li, Wenyun Lu, Lin Wang, Xi Xing, Ziyang Chen, Xin Teng, Xianfeng Zeng, et al. 2021. <span>“Metabolite Discovery Through Global Annotation of Untargeted Metabolomics Data.”</span> <em>Nature Methods</em> 18 (11): 1377–85. <a href="https://doi.org/10.1038/s41592-021-01303-3">https://doi.org/10.1038/s41592-021-01303-3</a>.
</div>
<div class="csl-entry">
Chen, Yanhua, Zhi Zhou, Wei Yang, Nan Bi, Jing Xu, Jiuming He, Ruiping Zhang, Lvhua Wang, and Zeper Abliz. 2017. <span>“Development of a <span>Data-Independent Targeted Metabolomics Method</span> for <span>Relative Quantification Using Liquid Chromatography Coupled</span> with <span>Tandem Mass Spectrometry</span>.”</span> <em>Analytical Chemistry</em> 89 (13): 6954–62. <a href="https://doi.org/10.1021/acs.analchem.6b04727">https://doi.org/10.1021/acs.analchem.6b04727</a>.
</div>
<div class="csl-entry">
Cheng, Susan, Svati H. Shah, Elizabeth J. Corwin, Oliver Fiehn, Robert L. Fitzgerald, Robert E. Gerszten, Thomas Illig, et al. 2017. <span>“Potential <span>Impact</span> and <span>Study Considerations</span> of <span>Metabolomics</span> in <span>Cardiovascular Health</span> and <span>Disease</span>: <span>A Scientific Statement From</span> the <span>American Heart Association</span>.”</span> <em>Circulation: Cardiovascular Genetics</em> 10 (2): e000032. <a href="https://doi.org/10.1161/HCG.0000000000000032">https://doi.org/10.1161/HCG.0000000000000032</a>.
</div>
<div class="csl-entry">
Choi, Meena, Ching-Yun Chang, Timothy Clough, Daniel Broudy, Trevor Killeen, Brendan MacLean, and Olga Vitek. 2014. <span>“<span>MSstats</span>: An <span>R</span> Package for Statistical Analysis of Quantitative Mass Spectrometry-Based Proteomic Experiments.”</span> <em>Bioinformatics</em> 30 (17): 2524–26. <a href="https://doi.org/10.1093/bioinformatics/btu305">https://doi.org/10.1093/bioinformatics/btu305</a>.
</div>
<div class="csl-entry">
Chokkathukalam, Achuthanunni, Andris Jankevics, Darren J. Creek, Fiona Achcar, Michael P. Barrett, and Rainer Breitling. 2013. <span>“<span class="nocase">mzMatch</span>: An <span>R</span> Tool for the Annotation and Relative Quantification of Isotope-Labelled Mass Spectrometry Data.”</span> <em>Bioinformatics</em> 29 (2): 281–83. <a href="https://doi.org/10.1093/bioinformatics/bts674">https://doi.org/10.1093/bioinformatics/bts674</a>.
</div>
<div class="csl-entry">
Chong, Jasmine, David S. Wishart, and Jianguo Xia. 2019. <span>“Using <span>MetaboAnalyst</span> 4.0 for <span>Comprehensive</span> and <span>Integrative Metabolomics Data Analysis</span>.”</span> <em>Current Protocols in Bioinformatics</em> 68 (1): e86. <a href="https://doi.org/10.1002/cpbi.86">https://doi.org/10.1002/cpbi.86</a>.
</div>
<div class="csl-entry">
Clasquin, Michelle F., Eugene Melamud, and Joshua D. Rabinowitz. 2012. <span>“<span>LC-MS Data Processing</span> with <span>MAVEN</span>: <span>A Metabolomic Analysis</span> and <span>Visualization Engine</span>.”</span> <em>Current Protocols in Bioinformatics</em> 37 (1): 14.11.1–23. <a href="https://doi.org/10.1002/0471250953.bi1411s37">https://doi.org/10.1002/0471250953.bi1411s37</a>.
</div>
<div class="csl-entry">
Climaco Pinto, Rui, Ibrahim Karaman, Matthew R. Lewis, Jenny Hällqvist, Manuja Kaluarachchi, Gonçalo Graça, Elena Chekmeneva, et al. 2022. <span>“Finding <span>Correspondence</span> Between <span>Metabolomic Features</span> in <span>Untargeted Liquid Chromatography</span>.”</span> <em>Analytical Chemistry</em> 94 (14): 5493–503. <a href="https://doi.org/10.1021/acs.analchem.1c03592">https://doi.org/10.1021/acs.analchem.1c03592</a>.
</div>
<div class="csl-entry">
Colby, Sean M., Christine H. Chang, Jessica L. Bade, Jamie R. Nunez, Madison R. Blumer, Daniel J. Orton, Kent J. Bloodsworth, et al. 2022. <span>“<span>DEIMoS</span>: <span>An Open-Source Tool</span> for <span>Processing High-Dimensional Mass Spectrometry Data</span>.”</span> <em>Analytical Chemistry</em> 94 (16): 6130–38. <a href="https://doi.org/10.1021/acs.analchem.1c05017">https://doi.org/10.1021/acs.analchem.1c05017</a>.
</div>
<div class="csl-entry">
Considine, E. C., G. Thomas, A. L. Boulesteix, A. S. Khashan, and L. C. Kenny. 2017. <span>“Critical Review of Reporting of the Data Analysis Step in Metabolomics.”</span> <em>Metabolomics</em> 14 (1): 7. <a href="https://doi.org/10.1007/s11306-017-1299-3">https://doi.org/10.1007/s11306-017-1299-3</a>.
</div>
<div class="csl-entry">
Creek, Darren J., Andris Jankevics, Karl E. V. Burgess, Rainer Breitling, and Michael P. Barrett. 2012. <span>“<span>IDEOM</span>: An <span>Excel</span> Interface for Analysis of <span>LC</span> Metabolomics Data.”</span> <em>Bioinformatics</em> 28 (7): 1048–49. <a href="https://doi.org/10.1093/bioinformatics/bts069">https://doi.org/10.1093/bioinformatics/bts069</a>.
</div>
<div class="csl-entry">
Daly, Rónán, Simon Rogers, Joe Wandy, Andris Jankevics, Karl E. V. Burgess, and Rainer Breitling. 2014. <span>“<span>MetAssign</span>: Probabilistic Annotation of Metabolites from <span>LC</span> Data Using a <span>Bayesian</span> Clustering Approach.”</span> <em>Bioinformatics</em> 30 (19): 2764–71. <a href="https://doi.org/10.1093/bioinformatics/btu370">https://doi.org/10.1093/bioinformatics/btu370</a>.
</div>
<div class="csl-entry">
De Livera, Alysha M., Daniel A. Dias, David De Souza, Thusitha Rupasinghe, James Pyke, Dedreia Tull, Ute Roessner, Malcolm McConville, and Terence P. Speed. 2012. <span>“Normalizing and <span>Integrating Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 84 (24): 10768–76. <a href="https://doi.org/10.1021/ac302748b">https://doi.org/10.1021/ac302748b</a>.
</div>
<div class="csl-entry">
Deda, Olga, Anastasia Chrysovalantou Chatziioannou, Stella Fasoula, Dimitris Palachanis, Nicolaos Raikos, Georgios A. Theodoridis, and Helen G. Gika. 2017. <span>“Sample Preparation Optimization in Fecal Metabolic Profiling.”</span> <em>Journal of Chromatography B</em>, Advances in mass spectrometry-based applications, 1047 (March): 115–23. <a href="https://doi.org/10.1016/j.jchromb.2016.06.047">https://doi.org/10.1016/j.jchromb.2016.06.047</a>.
</div>
<div class="csl-entry">
DeFelice, Brian C., Sajjan Singh Mehta, Stephanie Samra, Tomáš Čajka, Benjamin Wancewicz, Johannes F. Fahrmann, and Oliver Fiehn. 2017. <span>“Mass <span>Spectral Feature List Optimizer</span> (<span>MS-FLO</span>): <span>A Tool To Minimize False Positive Peak Reports</span> in <span>Untargeted Liquid Chromatography</span> (<span>LC-MS</span>) <span>Data Processing</span>.”</span> <em>Analytical Chemistry</em> 89 (6): 3250–55. <a href="https://doi.org/10.1021/acs.analchem.6b04372">https://doi.org/10.1021/acs.analchem.6b04372</a>.
</div>
<div class="csl-entry">
Delabriere, Alexis, Philipp Warmer, Vincenth Brennsteiner, and Nicola Zamboni. 2021. <span>“<span>SLAW</span>: <span>A Scalable</span> and <span>Self-Optimizing Processing Workflow</span> for <span>Untargeted LC-MS</span>.”</span> <em>Analytical Chemistry</em> 93 (45): 15024–32. <a href="https://doi.org/10.1021/acs.analchem.1c02687">https://doi.org/10.1021/acs.analchem.1c02687</a>.
</div>
<div class="csl-entry">
Dietrich, Christian, Arne Wick, and Thomas A. Ternes. 2022. <span>“Open-Source Feature Detection for Non-Target <span>LC</span> Analytics.”</span> <em>Rapid Communications in Mass Spectrometry</em> 36 (2): e9206. <a href="https://doi.org/10.1002/rcm.9206">https://doi.org/10.1002/rcm.9206</a>.
</div>
<div class="csl-entry">
Ding, Xian, Fen Yang, Yanhua Chen, Jing Xu, Jiuming He, Ruiping Zhang, and Zeper Abliz. 2022. <span>“Norm <span>ISWSVR</span>: <span>A Data Integration</span> and <span>Normalization Approach</span> for <span>Large-Scale Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 94 (21): 7500–7509. <a href="https://doi.org/10.1021/acs.analchem.1c05502">https://doi.org/10.1021/acs.analchem.1c05502</a>.
</div>
<div class="csl-entry">
Djoumbou Feunang, Yannick, Roman Eisner, Craig Knox, Leonid Chepelev, Janna Hastings, Gareth Owen, Eoin Fahy, et al. 2016. <span>“<span>ClassyFire</span>: Automated Chemical Classification with a Comprehensive, Computable Taxonomy.”</span> <em>Journal of Cheminformatics</em> 8 (1): 61. <a href="https://doi.org/10.1186/s13321-016-0174-y">https://doi.org/10.1186/s13321-016-0174-y</a>.
</div>
<div class="csl-entry">
Dodds, James N., Lingjue Wang, Gary J. Patti, and Erin S. Baker. 2022. <span>“Combining <span>Isotopologue Workflows</span> and <span>Simultaneous Multidimensional Separations</span> to <span>Detect</span>, <span>Identify</span>, and <span>Validate Metabolites</span> in <span>Untargeted Analyses</span>.”</span> <em>Analytical Chemistry</em> 94 (5): 2527–35. <a href="https://doi.org/10.1021/acs.analchem.1c04430">https://doi.org/10.1021/acs.analchem.1c04430</a>.
</div>
<div class="csl-entry">
Domingo-Almenara, Xavier, Jesus Brezmes, Maria Vinaixa, Sara Samino, Noelia Ramirez, Marta Ramon-Krauel, Carles Lerin, et al. 2016. <span>“<span class="nocase">eRah</span>: <span>A Computational Tool Integrating Spectral Deconvolution</span> and <span>Alignment</span> with <span>Quantification</span> and <span>Identification</span> of <span>Metabolites</span> in <span>GC</span>/<span>MS-Based Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 88 (19): 9821–29. <a href="https://doi.org/10.1021/acs.analchem.6b02927">https://doi.org/10.1021/acs.analchem.6b02927</a>.
</div>
<div class="csl-entry">
Domingo-Almenara, Xavier, J. Rafael Montenegro-Burke, H. Paul Benton, and Gary Siuzdak. 2018. <span>“Annotation: <span>A Computational Solution</span> for <span>Streamlining Metabolomics Analysis</span>.”</span> <em>Analytical Chemistry</em> 90 (1): 480–89. <a href="https://doi.org/10.1021/acs.analchem.7b03929">https://doi.org/10.1021/acs.analchem.7b03929</a>.
</div>
<div class="csl-entry">
Domingo-Almenara, Xavier, J. Rafael Montenegro-Burke, Julijana Ivanisevic, Aurelien Thomas, Jonathan Sidibé, Tony Teav, Carlos Guijas, et al. 2018. <span>“<span>XCMS-MRM</span> and <span>METLIN-MRM</span>: A Cloud Library and Public Resource for Targeted Analysis of Small Molecules.”</span> <em>Nature Methods</em> 15 (9): 681–84. <a href="https://doi.org/10.1038/s41592-018-0110-3">https://doi.org/10.1038/s41592-018-0110-3</a>.
</div>
<div class="csl-entry">
Domingo-Almenara, Xavier, and Gary Siuzdak. 2020. <span>“Metabolomics <span>Data Processing Using XCMS</span>.”</span> In <em>Computational <span>Methods</span> and <span>Data Analysis</span> for <span>Metabolomics</span></em>, edited by Shuzhao Li, 11–24. Methods in <span>Molecular Biology</span>. <span>New York, NY</span>: <span>Springer US</span>. <a href="https://doi.org/10.1007/978-1-0716-0239-3_2">https://doi.org/10.1007/978-1-0716-0239-3_2</a>.
</div>
<div class="csl-entry">
Doppler, Maria, Bernhard Kluger, Christoph Bueschl, Christina Schneider, Rudolf Krska, Sylvie Delcambre, Karsten Hiller, Marc Lemmens, and Rainer Schuhmacher. 2016. <span>“Stable <span>Isotope-Assisted Evaluation</span> of <span>Different Extraction Solvents</span> for <span>Untargeted Metabolomics</span> of <span>Plants</span>.”</span> <em>International Journal of Molecular Sciences</em> 17 (7). <a href="https://doi.org/10.3390/ijms17071017">https://doi.org/10.3390/ijms17071017</a>.
</div>
<div class="csl-entry">
Dryden, Michael D. M., Ryan Fobel, Christian Fobel, and Aaron R. Wheeler. 2017. <span>“Upon the <span>Shoulders</span> of <span>Giants</span>: <span>Open-Source Hardware</span> and <span>Software</span> in <span>Analytical Chemistry</span>.”</span> <em>Analytical Chemistry</em> 89 (8): 4330–38. <a href="https://doi.org/10.1021/acs.analchem.7b00485">https://doi.org/10.1021/acs.analchem.7b00485</a>.
</div>
<div class="csl-entry">
Du, Xinsong, Juan J. Aristizabal-Henao, Timothy J. Garrett, Mathias Brochhausen, William R. Hogan, and Dominick J. Lemas. 2022. <span>“A <span>Checklist</span> for <span>Reproducible Computational Analysis</span> in <span>Clinical Metabolomics Research</span>.”</span> <em>Metabolites</em> 12 (1): 87. <a href="https://doi.org/10.3390/metabo12010087">https://doi.org/10.3390/metabo12010087</a>.
</div>
<div class="csl-entry">
Du, Xiuxia, and Steven H Zeisel. 2013. <span>“<span>SPECTRAL DECONVOLUTION FOR GAS CHROMATOGRAPHY MASS SPECTROMETRY-BASED METABOLOMICS</span>: <span>CURRENT STATUS AND FUTURE PERSPECTIVES</span>.”</span> <em>Computational and Structural Biotechnology Journal</em> 4 (5): 1–10. <a href="https://doi.org/10.5936/csbj.201301013">https://doi.org/10.5936/csbj.201301013</a>.
</div>
<div class="csl-entry">
Dudzik, Danuta, Cecilia Barbas-Bernardos, Antonia García, and Coral Barbas. 2018. <span>“Quality Assurance Procedures for Mass Spectrometry Untargeted Metabolomics. A Review.”</span> <em>Journal of Pharmaceutical and Biomedical Analysis</em>, Review issue 2017, 147 (January): 149–73. <a href="https://doi.org/10.1016/j.jpba.2017.07.044">https://doi.org/10.1016/j.jpba.2017.07.044</a>.
</div>
<div class="csl-entry">
Dührkop, Kai, Markus Fleischauer, Marcus Ludwig, Alexander A. Aksenov, Alexey V. Melnik, Marvin Meusel, Pieter C. Dorrestein, Juho Rousu, and Sebastian Böcker. 2019. <span>“<span>SIRIUS</span> 4: A Rapid Tool for Turning Tandem Mass Spectra into Metabolite Structure Information.”</span> <em>Nature Methods</em> 16 (4): 299–302. <a href="https://doi.org/10.1038/s41592-019-0344-8">https://doi.org/10.1038/s41592-019-0344-8</a>.
</div>
<div class="csl-entry">
Dührkop, Kai, Louis-Félix Nothias, Markus Fleischauer, Raphael Reher, Marcus Ludwig, Martin A. Hoffmann, Daniel Petras, et al. 2020. <span>“Systematic Classification of Unknown Metabolites Using High-Resolution Fragmentation Mass Spectra.”</span> <em>Nature Biotechnology</em>, November, 1–10. <a href="https://doi.org/10.1038/s41587-020-0740-8">https://doi.org/10.1038/s41587-020-0740-8</a>.
</div>
<div class="csl-entry">
Dunn, Warwick B, Ian D Wilson, Andrew W Nicholls, and David Broadhurst. 2012. <span>“The Importance of Experimental Design and <span>QC</span> Samples in Large-Scale and <span class="nocase">MS-driven</span> Untargeted Metabolomic Studies of Humans.”</span> <em>Bioanalysis</em> 4 (18): 2249–64. <a href="https://doi.org/10.4155/bio.12.204">https://doi.org/10.4155/bio.12.204</a>.
</div>
<div class="csl-entry">
Edmands, William M. B., Dinesh K. Barupal, and Augustin Scalbert. 2015. <span>“<span>MetMSLine</span>: An Automated and Fully Integrated Pipeline for Rapid Processing of High-Resolution <span>LC</span> Metabolomic Datasets.”</span> <em>Bioinformatics</em> 31 (5): 788–90. <a href="https://doi.org/10.1093/bioinformatics/btu705">https://doi.org/10.1093/bioinformatics/btu705</a>.
</div>
<div class="csl-entry">
Edmands, William M. B., Josie Hayes, and Stephen M. Rappaport. 2018. <span>“<span>SimExTargId</span>: A Comprehensive Package for Real-Time <span>LC-MS</span> Data Acquisition and Analysis.”</span> <em>Bioinformatics</em> 34 (20): 3589–90. <a href="https://doi.org/10.1093/bioinformatics/bty218">https://doi.org/10.1093/bioinformatics/bty218</a>.
</div>
<div class="csl-entry">
Edmands, William M. B., Lauren Petrick, Dinesh K. Barupal, Augustin Scalbert, Mark J. Wilson, Jeffrey K. Wickliffe, and Stephen M. Rappaport. 2017. <span>“<span class="nocase">compMS2Miner</span>: <span>An Automatable Metabolite Identification</span>, <span>Visualization</span>, and <span>Data-Sharing R Package</span> for <span>High-Resolution LC</span>.”</span> <em>Analytical Chemistry</em> 89 (7): 3919–28. <a href="https://doi.org/10.1021/acs.analchem.6b02394">https://doi.org/10.1021/acs.analchem.6b02394</a>.
</div>
<div class="csl-entry">
Eilertz, Daniel, Michael Mitterer, and Joerg M. Buescher. 2022. <span>“<span class="nocase">automRm</span>: <span>An R Package</span> for <span>Fully Automatic LC-QQQ-MS Data Preprocessing Powered</span> by <span>Machine Learning</span>.”</span> <em>Analytical Chemistry</em> 94 (16): 6163–71. <a href="https://doi.org/10.1021/acs.analchem.1c05224">https://doi.org/10.1021/acs.analchem.1c05224</a>.
</div>
<div class="csl-entry">
El Abiead, Yasin, Maximilian Milford, Harald Schoeny, Mate Rusz, Reza M. Salek, and Gunda Koellensperger. 2022. <span>“Power of <span class="nocase">mzRAPP-Based Performance Assessments</span> in <span>MS1-Based Nontargeted Feature Detection</span>.”</span> <em>Analytical Chemistry</em> 94 (24): 8588–95. <a href="https://doi.org/10.1021/acs.analchem.1c05270">https://doi.org/10.1021/acs.analchem.1c05270</a>.
</div>
<div class="csl-entry">
Fenaille, François, Pierre Barbier Saint-Hilaire, Kathleen Rousseau, and Christophe Junot. 2017. <span>“Data Acquisition Workflows in Liquid Chromatography Coupled to High Resolution Mass Spectrometry-Based Metabolomics: <span>Where</span> Do We Stand?”</span> <em>Journal of Chromatography A</em> 1526 (Supplement C): 1–12. <a href="https://doi.org/10.1016/j.chroma.2017.10.043">https://doi.org/10.1016/j.chroma.2017.10.043</a>.
</div>
<div class="csl-entry">
Fernández-Albert, Francesc, Rafael Llorach, Cristina Andrés-Lacueva, and Alexandre Perera. 2014. <span>“An <span>R</span> Package to Analyse <span>LC</span>/<span>MS</span> Metabolomic Data: <span>MAIT</span> (<span>Metabolite Automatic Identification Toolkit</span>).”</span> <em>Bioinformatics</em> 30 (13): 1937–39. <a href="https://doi.org/10.1093/bioinformatics/btu136">https://doi.org/10.1093/bioinformatics/btu136</a>.
</div>
<div class="csl-entry">
Fessenden, Marissa. 2016. <span>“Metabolomics: <span>Small</span> Molecules, Single Cells.”</span> <em>Nature</em> 540 (7631): 153–55. <a href="https://doi.org/10.1038/540153a">https://doi.org/10.1038/540153a</a>.
</div>
<div class="csl-entry">
Fiehn, Oliver. 2002. <span>“Metabolomics the Link Between Genotypes and Phenotypes.”</span> <em>Plant Molecular Biology</em> 48 (1): 155–71. <a href="https://doi.org/10.1023/A:1013713905833">https://doi.org/10.1023/A:1013713905833</a>.
</div>
<div class="csl-entry">
Flasch, Mira, Veronika Fitz, Evelyn Rampler, Chibundu N. Ezekiel, Gunda Koellensperger, and Benedikt Warth. 2022. <span>“Integrated <span>Exposomics</span>/<span>Metabolomics</span> for <span>Rapid Exposure</span> and <span>Effect Analyses</span>.”</span> <em>JACS Au</em> 2 (11): 2548–60. <a href="https://doi.org/10.1021/jacsau.2c00433">https://doi.org/10.1021/jacsau.2c00433</a>.
</div>
<div class="csl-entry">
Forsberg, Erica M., Tao Huan, Duane Rinehart, H. Paul Benton, Benedikt Warth, Brian Hilmers, and Gary Siuzdak. 2018. <span>“Data Processing, Multi-Omic Pathway Mapping, and Metabolite Activity Analysis Using <span>XCMS Online</span>.”</span> <em>Nature Protocols</em> 13 (4): 633–51. <a href="https://doi.org/10.1038/nprot.2017.151">https://doi.org/10.1038/nprot.2017.151</a>.
</div>
<div class="csl-entry">
Franceschi, Pietro, Domenico Masuero, Urska Vrhovsek, Fulvio Mattivi, and Ron Wehrens. 2012. <span>“A Benchmark Spike-in Data Set for Biomarker Identification in Metabolomics.”</span> <em>Journal of Chemometrics</em> 26 (1-2): 16–24. <a href="https://doi.org/10.1002/cem.1420">https://doi.org/10.1002/cem.1420</a>.
</div>
<div class="csl-entry">
Fu, Hai-Yan, Ou Hu, Yue-Ming Zhang, Li Zhang, Jing-Jing Song, Peang Lu, Qing-Xia Zheng, et al. 2017. <span>“Mass-Spectra-Based Peak Alignment for Automatic Nontargeted Metabolic Profiling Analysis for Biomarker Screening in Plant Samples.”</span> <em>Journal of Chromatography A</em> 1513 (Supplement C): 201–9. <a href="https://doi.org/10.1016/j.chroma.2017.07.044">https://doi.org/10.1016/j.chroma.2017.07.044</a>.
</div>
<div class="csl-entry">
Fu, Jianbo, Ying Zhang, Yunxia Wang, Hongning Zhang, Jin Liu, Jing Tang, Qingxia Yang, et al. 2021. <span>“Optimization of Metabolomic Data Processing Using <span>NOREVA</span>.”</span> <em>Nature Protocols</em>, December, 1–23. <a href="https://doi.org/10.1038/s41596-021-00636-9">https://doi.org/10.1038/s41596-021-00636-9</a>.
</div>
<div class="csl-entry">
Gadara, Darshak, Katerina Coufalikova, Juraj Bosak, David Smajs, and Zdenek Spacil. 2021. <span>“Systematic <span>Feature Filtering</span> in <span>Exploratory Metabolomics</span>: <span>Application</span> Toward <span>Biomarker Discovery</span>.”</span> <em>Analytical Chemistry</em> 93 (26): 9103–10. <a href="https://doi.org/10.1021/acs.analchem.1c00816">https://doi.org/10.1021/acs.analchem.1c00816</a>.
</div>
<div class="csl-entry">
Gerlich, Michael, and Steffen Neumann. 2013. <span>“<span>MetFusion</span>: Integration of Compound Identification Strategies.”</span> <em>Journal of Mass Spectrometry</em> 48 (3): 291–98. <a href="https://doi.org/10.1002/jms.3123">https://doi.org/10.1002/jms.3123</a>.
</div>
<div class="csl-entry">
Ghaste, Manoj, Robert Mistrik, and Vladimir Shulaev. 2016. <span>“Applications of <span>Fourier Transform Ion Cyclotron Resonance</span> (<span>FT-ICR</span>) and <span>Orbitrap Based High Resolution Mass Spectrometry</span> in <span>Metabolomics</span> and <span>Lipidomics</span>.”</span> <em>International Journal of Molecular Sciences</em> 17 (6). <a href="https://doi.org/10.3390/ijms17060816">https://doi.org/10.3390/ijms17060816</a>.
</div>
<div class="csl-entry">
Ghosson, Hikmat, Yann Guitton, Amani Ben Jrad, Chandrashekhar Patil, Delphine Raviglione, Marie-Virginie Salvia, and Cédric Bertrand. 2021. <span>“Electrospray Ionization and Heterogeneous Matrix Effects in Liquid Chromatography/Mass Spectrometry Based Meta-Metabolomics: <span>A</span> Biomarker or a Suppressed Ion?”</span> <em>Rapid Communications in Mass Spectrometry</em> 35 (2): e8977. <a href="https://doi.org/10.1002/rcm.8977">https://doi.org/10.1002/rcm.8977</a>.
</div>
<div class="csl-entry">
Giacomoni, Franck, Gildas Le Corguillé, Misharl Monsoor, Marion Landi, Pierre Pericard, Mélanie Pétéra, Christophe Duperier, et al. 2015. <span>“<span>Workflow4Metabolomics</span>: A Collaborative Research Infrastructure for Computational Metabolomics.”</span> <em>Bioinformatics</em> 31 (9): 1493–95. <a href="https://doi.org/10.1093/bioinformatics/btu813">https://doi.org/10.1093/bioinformatics/btu813</a>.
</div>
<div class="csl-entry">
Giebelhaus, Ryland T., Michael D. Sorochan Armstrong, A. Paulina de la Mata, and James J. Harynuk. 2022. <span>“Untargeted Region of Interest Selection for Gas Chromatography Mass Spectrometry Data Using a Pseudo <span class="nocase">F-ratio</span> Moving Window.”</span> <em>Journal of Chromatography A</em> 1682 (October): 463499. <a href="https://doi.org/10.1016/j.chroma.2022.463499">https://doi.org/10.1016/j.chroma.2022.463499</a>.
</div>
<div class="csl-entry">
Gika, Helen G., Georgios A. Theodoridis, Robert S. Plumb, and Ian D. Wilson. 2014. <span>“Current Practice of Liquid Chromatographymass Spectrometry in Metabolomics and Metabonomics.”</span> <em>Journal of Pharmaceutical and Biomedical Analysis</em>, Review <span>Papers</span> on <span>Pharmaceutical</span> and <span>Biomedical Analysis</span> 2013, 87 (January): 12–25. <a href="https://doi.org/10.1016/j.jpba.2013.06.032">https://doi.org/10.1016/j.jpba.2013.06.032</a>.
</div>
<div class="csl-entry">
Gil, Andres, David Siegel, Hjalmar Permentier, Dirk-Jan Reijngoud, Frank Dekker, and Rainer Bischoff. 2015. <span>“Stability of Energy Metabolites Often Overlooked Issue in Metabolomics Studies: <span>A</span> Review.”</span> <em>ELECTROPHORESIS</em> 36 (18): 2156–69. <a href="https://doi.org/10.1002/elps.201500031">https://doi.org/10.1002/elps.201500031</a>.
</div>
<div class="csl-entry">
Giné, Roger, Jordi Capellades, Josep M. Badia, Dennis Vughs, Michaela Schwaiger-Haber, Theodore Alexandrov, Maria Vinaixa, Andrea M. Brunner, Gary J. Patti, and Oscar Yanes. 2021. <span>“<span>HERMES</span>: A Molecular-Formula-Oriented Method to Target the Metabolome.”</span> <em>Nature Methods</em> 18 (11): 1370–76. <a href="https://doi.org/10.1038/s41592-021-01307-z">https://doi.org/10.1038/s41592-021-01307-z</a>.
</div>
<div class="csl-entry">
Gloaguen, Yoann, Jennifer A. Kirwan, and Dieter Beule. 2022. <span>“Deep <span>Learning-Assisted Peak Curation</span> for <span>Large-Scale LC-MS Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 94 (12): 4930–37. <a href="https://doi.org/10.1021/acs.analchem.1c02220">https://doi.org/10.1021/acs.analchem.1c02220</a>.
</div>
<div class="csl-entry">
Goldansaz, Seyed Ali, An Chi Guo, Tanvir Sajed, Michael A. Steele, Graham S. Plastow, and David S. Wishart. 2017. <span>“Livestock Metabolomics and the Livestock Metabolome: <span>A</span> Systematic Review.”</span> <em>PLOS ONE</em> 12 (5): e0177675. <a href="https://doi.org/10.1371/journal.pone.0177675">https://doi.org/10.1371/journal.pone.0177675</a>.
</div>
<div class="csl-entry">
González, Oskar, Anne-Charlotte Dubbelman, and Thomas Hankemeier. 2022. <span>“Postcolumn <span>Infusion</span> as a <span>Quality Control Tool</span> for <span>LC-MS-Based Analysis</span>.”</span> <em>Postcolumn Infusion as a Quality Control Tool for LC-MS-Based Analysis</em>, April. <a href="https://doi.org/10.1021/jasms.2c00022">https://doi.org/10.1021/jasms.2c00022</a>.
</div>
<div class="csl-entry">
González-Riano, Carolina, Danuta Dudzik, Antonia Garcia, Alberto Gil-de-la-Fuente, Ana Gradillas, Joanna Godzien, Ángeles López-Gonzálvez, et al. 2020. <span>“Recent <span>Developments</span> Along the <span>Analytical Process</span> for <span>Metabolomics Workflows</span>.”</span> <em>Analytical Chemistry</em> 92 (1): 203–26. <a href="https://doi.org/10.1021/acs.analchem.9b04553">https://doi.org/10.1021/acs.analchem.9b04553</a>.
</div>
<div class="csl-entry">
Graça, Gonçalo, Yuheng Cai, Chung-Ho E. Lau, Panagiotis A. Vorkas, Matthew R. Lewis, Elizabeth J. Want, David Herrington, and Timothy M. D. Ebbels. 2022. <span>“Automated <span>Annotation</span> of <span>Untargeted All-Ion Fragmentation LC</span> with <span>MetaboAnnotatoR</span>.”</span> <em>Analytical Chemistry</em> 94 (8): 3446–55. <a href="https://doi.org/10.1021/acs.analchem.1c03032">https://doi.org/10.1021/acs.analchem.1c03032</a>.
</div>
<div class="csl-entry">
Griffiths, William J., Therese Koal, Yuqin Wang, Matthias Kohl, David P. Enot, and Hans-Peter Deigner. 2010. <span>“Targeted <span>Metabolomics</span> for <span>Biomarker Discovery</span>.”</span> <em>Angewandte Chemie International Edition</em> 49 (32): 5426–45. <a href="https://doi.org/10.1002/anie.200905579">https://doi.org/10.1002/anie.200905579</a>.
</div>
<div class="csl-entry">
Gromski, Piotr S., Howbeer Muhamadali, David I. Ellis, Yun Xu, Elon Correa, Michael L. Turner, and Royston Goodacre. 2015. <span>“A Tutorial Review: <span>Metabolomics</span> and Partial Least Squares-Discriminant Analysis a Marriage of Convenience or a Shotgun Wedding.”</span> <em>Analytica Chimica Acta</em> 879 (June): 10–23. <a href="https://doi.org/10.1016/j.aca.2015.02.012">https://doi.org/10.1016/j.aca.2015.02.012</a>.
</div>
<div class="csl-entry">
Gugisch, Ralf, Adalbert Kerber, Axel Kohnert, Reinhard Laue, Markus Meringer, Christoph Rücker, and Alfred Wassermann. 2015. <span>“Chapter 6 - <span>MOLGEN</span> 5.0, <span>A Molecular Structure Generator</span>.”</span> In <em>Advances in <span>Mathematical Chemistry</span> and <span>Applications</span></em>, edited by Subhash C. Basak, Guillermo Restrepo, and José L. Villaveces, 113–38. <span>Bentham Science Publishers</span>. <a href="https://doi.org/10.1016/B978-1-68108-198-4.50006-0">https://doi.org/10.1016/B978-1-68108-198-4.50006-0</a>.
</div>
<div class="csl-entry">
Guha, Rajarshi. 2007. <span>“Chemical <span>Informatics Functionality</span> in <span>R</span>.”</span> <em>Journal of Statistical Software</em> 18 (1): 1–16. <a href="https://doi.org/10.18637/jss.v018.i05">https://doi.org/10.18637/jss.v018.i05</a>.
</div>
<div class="csl-entry">
Guijas, Carlos, J. Rafael Montenegro-Burke, Xavier Domingo-Almenara, Amelia Palermo, Benedikt Warth, Gerrit Hermann, Gunda Koellensperger, et al. 2018. <span>“<span>METLIN</span>: <span>A Technology Platform</span> for <span>Identifying Knowns</span> and <span>Unknowns</span>.”</span> <em>Analytical Chemistry</em> 90 (5): 3156–64. <a href="https://doi.org/10.1021/acs.analchem.7b04424">https://doi.org/10.1021/acs.analchem.7b04424</a>.
</div>
<div class="csl-entry">
Guo, Jian, Sam Shen, and Tao Huan. 2022. <span>“Paramounter: <span>Direct Measurement</span> of <span>Universal Parameters To Process Metabolomics Data</span> in a <span>‘<span>White Box</span>’</span>.”</span> <em>Analytical Chemistry</em>, March. <a href="https://doi.org/10.1021/acs.analchem.1c04758">https://doi.org/10.1021/acs.analchem.1c04758</a>.
</div>
<div class="csl-entry">
Guo, Jian, Sam Shen, Shipei Xing, Huaxu Yu, and Tao Huan. 2021. <span>“<span>ISFrag</span>: <span>De Novo Recognition</span> of <span>In-Source Fragments</span> for <span>Liquid Chromatography</span>.”</span> <em>Analytical Chemistry</em>, July. <a href="https://doi.org/10.1021/acs.analchem.1c01644">https://doi.org/10.1021/acs.analchem.1c01644</a>.
</div>
<div class="csl-entry">
Habra, Hani, Maureen Kachman, Kevin Bullock, Clary Clish, Charles R. Evans, and Alla Karnovsky. 2021. <span>“<span class="nocase">metabCombiner</span>: <span>Paired Untargeted LC-HRMS Metabolomics Feature Matching</span> and <span>Concatenation</span> of <span>Disparately Acquired Data Sets</span>.”</span> <em>Analytical Chemistry</em> 93 (12): 5028–36. <a href="https://doi.org/10.1021/acs.analchem.0c03693">https://doi.org/10.1021/acs.analchem.0c03693</a>.
</div>
<div class="csl-entry">
Hansen, Rebecca L., and Young Jin Lee. 2018. <span>“High-<span>Spatial Resolution Mass Spectrometry Imaging</span>: <span>Toward Single Cell Metabolomics</span> in <span>Plant Tissues</span>.”</span> <em>The Chemical Record</em> 18 (1): 65–77. <a href="https://doi.org/10.1002/tcr.201700027">https://doi.org/10.1002/tcr.201700027</a>.
</div>
<div class="csl-entry">
Harrieder, Eva-Maria, Fleming Kretschmer, Sebastian Böcker, and Michael Witting. 2022. <span>“Current State-of-the-Art of Separation Methods Used in <span>LC-MS</span> Based Metabolomics and Lipidomics.”</span> <em>Journal of Chromatography B</em> 1188 (January): 123069. <a href="https://doi.org/10.1016/j.jchromb.2021.123069">https://doi.org/10.1016/j.jchromb.2021.123069</a>.
</div>
<div class="csl-entry">
Haug, Kenneth, Reza M Salek, and Christoph Steinbeck. 2017. <span>“Global Open Data Management in Metabolomics.”</span> <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 58–63. <a href="https://doi.org/10.1016/j.cbpa.2016.12.024">https://doi.org/10.1016/j.cbpa.2016.12.024</a>.
</div>
<div class="csl-entry">
Helmus, Rick, Thomas L. ter Laak, Annemarie P. van Wezel, Pim de Voogt, and Emma L. Schymanski. 2021. <span>“<span class="nocase">patRoon</span>: Open Source Software Platform for Environmental Mass Spectrometry Based Non-Target Screening.”</span> <em>Journal of Cheminformatics</em> 13 (1): 1. <a href="https://doi.org/10.1186/s13321-020-00477-w">https://doi.org/10.1186/s13321-020-00477-w</a>.
</div>
<div class="csl-entry">
Hernandes, Vinicius Veri, Coral Barbas, and Danuta Dudzik. 2017. <span>“A Review of Blood Sample Handling and Pre-Processing for Metabolomics Studies.”</span> <em>ELECTROPHORESIS</em> 38 (18): 2232–41. <a href="https://doi.org/10.1002/elps.201700086">https://doi.org/10.1002/elps.201700086</a>.
</div>
<div class="csl-entry">
Hiller, Karsten, Jasper Hangebrauk, Christian Jäger, Jana Spura, Kerstin Schreiber, and Dietmar Schomburg. 2009. <span>“<span>MetaboliteDetector</span>: <span>Comprehensive Analysis Tool</span> for <span>Targeted</span> and <span>Nontargeted GC</span>/<span>MS Based Metabolome Analysis</span>.”</span> <em>Analytical Chemistry</em> 81 (9): 3429–39. <a href="https://doi.org/10.1021/ac802689c">https://doi.org/10.1021/ac802689c</a>.
</div>
<div class="csl-entry">
Hites, Ronald A. 2019. <span>“Correcting for <span>Censored Environmental Measurements</span>.”</span> <em>Environmental Science &amp; Technology</em>, September. <a href="https://doi.org/10.1021/acs.est.9b05042">https://doi.org/10.1021/acs.est.9b05042</a>.
</div>
<div class="csl-entry">
Hites, Ronald A., and Karl J. Jobst. 2018. <span>“Is <span>Nontargeted Screening Reproducible</span>?”</span> <em>Environmental Science &amp; Technology</em> 52 (21): 11975–76. <a href="https://doi.org/10.1021/acs.est.8b05671">https://doi.org/10.1021/acs.est.8b05671</a>.
</div>
<div class="csl-entry">
Houriet, Joelle, Warren S. Vidar, Preston K. Manwill, Daniel A. Todd, and Nadja B. Cech. 2022. <span>“How <span>Low Can You Go</span>? <span>Selecting Intensity Thresholds</span> for <span>Untargeted Metabolomics Data Preprocessing</span>.”</span> <em>Analytical Chemistry</em> 94 (51): 17964–71. <a href="https://doi.org/10.1021/acs.analchem.2c04088">https://doi.org/10.1021/acs.analchem.2c04088</a>.
</div>
<div class="csl-entry">
Hu, Xin, Douglas I. Walker, Yongliang Liang, Matthew Ryan Smith, Michael L. Orr, Brian D. Juran, Chunyu Ma, et al. 2021. <span>“A Scalable Workflow to Characterize the Human Exposome.”</span> <em>Nature Communications</em> 12 (1): 5575. <a href="https://doi.org/10.1038/s41467-021-25840-9">https://doi.org/10.1038/s41467-021-25840-9</a>.
</div>
<div class="csl-entry">
Hu, Yaxi, Betty Cai, and Tao Huan. 2019. <span>“Enhancing <span>Metabolome Coverage</span> in <span>Data-Dependent LC</span>/<span>MS Analysis</span> Through an <span>Integrated Feature Extraction Strategy</span>.”</span> <em>Analytical Chemistry</em> 91 (22): 14433–41. <a href="https://doi.org/10.1021/acs.analchem.9b02980">https://doi.org/10.1021/acs.analchem.9b02980</a>.
</div>
<div class="csl-entry">
Huan, Tao, Erica M. Forsberg, Duane Rinehart, Caroline H. Johnson, Julijana Ivanisevic, H. Paul Benton, Mingliang Fang, et al. 2017. <span>“Systems Biology Guided by <span>XCMS Online</span> Metabolomics.”</span> <em>Nature Methods</em> 14 (5): 461–62. <a href="https://doi.org/10.1038/nmeth.4260">https://doi.org/10.1038/nmeth.4260</a>.
</div>
<div class="csl-entry">
Huang, Danning, Marcos Bouza, David A. Gaul, Franklin E. Leach, I. Jonathan Amster, Frank C. Schroeder, Arthur S. Edison, and Facundo M. Fernández. 2021. <span>“Comparison of <span>High-Resolution Fourier Transform Mass Spectrometry Platforms</span> for <span>Putative Metabolite Annotation</span>.”</span> <em>Comparison of High-Resolution Fourier Transform Mass Spectrometry Platforms for Putative Metabolite Annotation</em>, August. <a href="https://doi.org/10.1021/acs.analchem.1c02224">https://doi.org/10.1021/acs.analchem.1c02224</a>.
</div>
<div class="csl-entry">
Huber, Florian, Stefan Verhoeven, Christiaan Meijer, Hanno Spreeuw, Efraín Manuel Villanueva Castilla, Cunliang Geng, Justin J. j van der Hooft, et al. 2020. <span>“Matchms - Processing and Similarity Evaluation of Mass Spectrometry Data.”</span> <em>Journal of Open Source Software</em> 5 (52): 2411. <a href="https://doi.org/10.21105/joss.02411">https://doi.org/10.21105/joss.02411</a>.
</div>
<div class="csl-entry">
Hufsky, Franziska, Kerstin Scheubert, and Sebastian Böcker. 2014. <span>“Computational Mass Spectrometry for Small-Molecule Fragmentation.”</span> <em>TrAC Trends in Analytical Chemistry</em> 53 (January): 41–48. <a href="https://doi.org/10.1016/j.trac.2013.09.008">https://doi.org/10.1016/j.trac.2013.09.008</a>.
</div>
<div class="csl-entry">
Ibáñez, Clara, Lamia Mouhid, Guillermo Reglero, and Ana Ramírez de Molina. 2017. <span>“Lipidomics <span>Insights</span> in <span>Health</span> and <span>Nutritional Intervention Studies</span>.”</span> <em>Journal of Agricultural and Food Chemistry</em> 65 (36): 7827–42. <a href="https://doi.org/10.1021/acs.jafc.7b02643">https://doi.org/10.1021/acs.jafc.7b02643</a>.
</div>
<div class="csl-entry">
Jacyna, Julia, Marta Kordalewska, and Michał J. Markuszewski. 2019. <span>“Design of <span>Experiments</span> in Metabolomics-Related Studies: <span>An</span> Overview.”</span> <em>Journal of Pharmaceutical and Biomedical Analysis</em> 164 (February): 598–606. <a href="https://doi.org/10.1016/j.jpba.2018.11.027">https://doi.org/10.1016/j.jpba.2018.11.027</a>.
</div>
<div class="csl-entry">
Jaeger, Carsten, Friederike Hoffmann, Clemens A. Schmitt, and Jan Lisec. 2016. <span>“Automated <span>Annotation</span> and <span>Evaluation</span> of <span>In-Source Mass Spectra</span> in <span>GC</span>/<span>Atmospheric Pressure Chemical Ionization-MS-Based Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 88 (19): 9386–90. <a href="https://doi.org/10.1021/acs.analchem.6b02743">https://doi.org/10.1021/acs.analchem.6b02743</a>.
</div>
<div class="csl-entry">
Jalili, Vahid, Enis Afgan, Qiang Gu, Dave Clements, Daniel Blankenberg, Jeremy Goecks, James Taylor, and Anton Nekrutenko. 2020. <span>“The <span>Galaxy</span> Platform for Accessible, Reproducible and Collaborative Biomedical Analyses: 2020 Update.”</span> <em>Nucleic Acids Research</em> 48 (W1): W395–402. <a href="https://doi.org/10.1093/nar/gkaa434">https://doi.org/10.1093/nar/gkaa434</a>.
</div>
<div class="csl-entry">
Jang, Cholsoon, Li Chen, and Joshua D. Rabinowitz. 2018. <span>“Metabolomics and <span>Isotope Tracing</span>.”</span> <em>Cell</em> 173 (4): 822–37. <a href="https://doi.org/10.1016/j.cell.2018.03.055">https://doi.org/10.1016/j.cell.2018.03.055</a>.
</div>
<div class="csl-entry">
Jones, Dean P., Youngja Park, and Thomas R. Ziegler. 2012. <span>“Nutritional <span>Metabolomics</span>: <span>Progress</span> in <span>Addressing Complexity</span> in <span>Diet</span> and <span>Health</span>.”</span> <em>Annual Review of Nutrition</em> 32 (1): 183–202. <a href="https://doi.org/10.1146/annurev-nutr-072610-145159">https://doi.org/10.1146/annurev-nutr-072610-145159</a>.
</div>
<div class="csl-entry">
Jorge, Tiago F., Ana T. Mata, and Carla António. 2016. <span>“Mass Spectrometry as a Quantitative Tool in Plant Metabolomics.”</span> <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150370. <a href="https://doi.org/10.1098/rsta.2015.0370">https://doi.org/10.1098/rsta.2015.0370</a>.
</div>
<div class="csl-entry">
Jr, Stephen Salerno, Mahya Mehrmohamadi, Maria V. Liberti, Muting Wan, Martin T. Wells, James G. Booth, and Jason W. Locasale. 2017. <span>“<span>RRmix</span>: <span>A</span> Method for Simultaneous Batch Effect Correction and Analysis of Metabolomics Data in the Absence of Internal Standards.”</span> <em>PLOS ONE</em> 12 (6): e0179530. <a href="https://doi.org/10.1371/journal.pone.0179530">https://doi.org/10.1371/journal.pone.0179530</a>.
</div>
<div class="csl-entry">
Ju, Ran, Xinyu Liu, Fujian Zheng, Xinjie Zhao, Xin Lu, Xiaohui Lin, Zhongda Zeng, and Guowang Xu. 2020. <span>“A Graph Density-Based Strategy for Features Fusion from Different Peak Extract Software to Achieve More Metabolites in Metabolic Profiling from High-Resolution Mass Spectrometry.”</span> <em>Analytica Chimica Acta</em> 1139 (December): 8–14. <a href="https://doi.org/10.1016/j.aca.2020.09.029">https://doi.org/10.1016/j.aca.2020.09.029</a>.
</div>
<div class="csl-entry">
Kachman, Maureen, Hani Habra, William Duren, Janis Wigginton, Peter Sajjakulnukit, George Michailidis, Charles Burant, and Alla Karnovsky. 2020. <span>“Deep Annotation of Untargeted <span>LC-MS</span> Metabolomics Data with <span>Binner</span>.”</span> <em>Bioinformatics</em> 36 (6): 1801–6. <a href="https://doi.org/10.1093/bioinformatics/btz798">https://doi.org/10.1093/bioinformatics/btz798</a>.
</div>
<div class="csl-entry">
Kapoore, Rahul Vijay, and Seetharaman Vaidyanathan. 2016. <span>“Towards Quantitative Mass Spectrometry-Based Metabolomics in Microbial and Mammalian Systems.”</span> <em>Phil. Trans. R. Soc. A</em> 374 (2079): 20150363. <a href="https://doi.org/10.1098/rsta.2015.0363">https://doi.org/10.1098/rsta.2015.0363</a>.
</div>
<div class="csl-entry">
Karpievitch, Yuliya V., Sonja B. Nikolic, Richard Wilson, James E. Sharman, and Lindsay M. Edwards. 2014. <span>“Metabolomics <span>Data Normalization</span> with <span>EigenMS</span>.”</span> <em>PLOS ONE</em> 9 (12): e116221. <a href="https://doi.org/10.1371/journal.pone.0116221">https://doi.org/10.1371/journal.pone.0116221</a>.
</div>
<div class="csl-entry">
Kennedy, Adam D., Bryan M. Wittmann, Anne M. Evans, Luke A. D. Miller, Douglas R. Toal, Shaun Lonergan, Sarah H. Elsea, and Kirk L. Pappan. 2018. <span>“Metabolomics in the Clinic: <span>A</span> Review of the Shared and Unique Features of Untargeted Metabolomics for Clinical Research and Clinical Testing.”</span> <em>Journal of Mass Spectrometry</em> 53 (11): 1143–54. <a href="https://doi.org/10.1002/jms.4292">https://doi.org/10.1002/jms.4292</a>.
</div>
<div class="csl-entry">
Keshet, Uri, Tobias Kind, Xinchen Lu, Sarita Devi, and Oliver Fiehn. 2022. <span>“Acyl-<span>CoA Identification</span> in <span>Mouse Liver Samples Using</span> the <span>In Silico CoA-Blast Tandem Mass Spectral Library</span>.”</span> <em>Analytical Chemistry</em> 94 (6): 2732–39. <a href="https://doi.org/10.1021/acs.analchem.1c03272">https://doi.org/10.1021/acs.analchem.1c03272</a>.
</div>
<div class="csl-entry">
Kew, William, John W. T. Blackburn, David J. Clarke, and Dušan Uhrín. 2017. <span>“Interactive van <span>Krevelen</span> Diagrams <span>Advanced</span> Visualisation of Mass Spectrometry Data of Complex Mixtures.”</span> <em>Rapid Communications in Mass Spectrometry</em> 31 (7): 658–62. <a href="https://doi.org/10.1002/rcm.7823">https://doi.org/10.1002/rcm.7823</a>.
</div>
<div class="csl-entry">
Kim, Jungyeon, Joong Kyong Ahn, Yu Eun Cheong, Sung-Joon Lee, Hoon-Suk Cha, and Kyoung Heon Kim. 2020. <span>“Systematic Re-Evaluation of the Long-Used Standard Protocol of Urease-Dependent Metabolome Sample Preparation.”</span> <em>PloS One</em> 15 (3): e0230072. <a href="https://doi.org/10.1371/journal.pone.0230072">https://doi.org/10.1371/journal.pone.0230072</a>.
</div>
<div class="csl-entry">
Kim, Taiyun, Owen Tang, Stephen T. Vernon, Katharine A. Kott, Yen Chin Koay, John Park, David E. James, et al. 2021. <span>“A Hierarchical Approach to Removal of Unwanted Variation for Large-Scale Metabolomics Data.”</span> <em>Nature Communications</em> 12 (1): 4992. <a href="https://doi.org/10.1038/s41467-021-25210-5">https://doi.org/10.1038/s41467-021-25210-5</a>.
</div>
<div class="csl-entry">
Kind, Tobias, and Oliver Fiehn. 2007. <span>“Seven <span>Golden Rules</span> for Heuristic Filtering of Molecular Formulas Obtained by Accurate Mass Spectrometry.”</span> <em>BMC Bioinformatics</em> 8 (1): 105. <a href="https://doi.org/10.1186/1471-2105-8-105">https://doi.org/10.1186/1471-2105-8-105</a>.
</div>
<div class="csl-entry">
Kind, Tobias, Hiroshi Tsugawa, Tomas Cajka, Yan Ma, Zijuan Lai, Sajjan S. Mehta, Gert Wohlgemuth, et al. 2018. <span>“Identification of Small Molecules Using Accurate Mass <span>MS</span>/<span>MS</span> Search.”</span> <em>Mass Spectrometry Reviews</em> 37 (4): 513–32. <a href="https://doi.org/10.1002/mas.21535">https://doi.org/10.1002/mas.21535</a>.
</div>
<div class="csl-entry">
Koelmel, Jeremy P., Nicholas M. Kroeger, Candice Z. Ulmer, John A. Bowden, Rainey E. Patterson, Jason A. Cochran, Christopher W. W. Beecher, Timothy J. Garrett, and Richard A. Yost. 2017. <span>“<span>LipidMatch</span>: An Automated Workflow for Rule-Based Lipid Identification Using Untargeted High-Resolution Tandem Mass Spectrometry Data.”</span> <em>BMC Bioinformatics</em> 18 (July): 331. <a href="https://doi.org/10.1186/s12859-017-1744-3">https://doi.org/10.1186/s12859-017-1744-3</a>.
</div>
<div class="csl-entry">
Kouřil, Štěpán, Julie de Sousa, Jan Václavík, David Friedecký, and Tomáš Adam. 2020. <span>“<span>CROP</span>: Correlation-Based Reduction of Feature Multiplicities in Untargeted Metabolomic Data.”</span> <em>Bioinformatics</em> 36 (9): 2941–42. <a href="https://doi.org/10.1093/bioinformatics/btaa012">https://doi.org/10.1093/bioinformatics/btaa012</a>.
</div>
<div class="csl-entry">
Kuhl, Carsten, Ralf Tautenhahn, Christoph Böttcher, Tony R. Larson, and Steffen Neumann. 2012. <span>“<span>CAMERA</span>: <span>An Integrated Strategy</span> for <span>Compound Spectra Extraction</span> and <span>Annotation</span> of <span>Liquid Chromatography</span>/<span>Mass Spectrometry Data Sets</span>.”</span> <em>Analytical Chemistry</em> 84 (1): 283–89. <a href="https://doi.org/10.1021/ac202450g">https://doi.org/10.1021/ac202450g</a>.
</div>
<div class="csl-entry">
Kuligowski, Julia, Ángel Sánchez-Illana, Daniel Sanjuán-Herráez, Máximo Vento, and Guillermo Quintás. 2015. <span>“Intra-Batch Effect Correction in Liquid Chromatography-Mass Spectrometry Using Quality Control Samples and Support Vector Regression (<span>QC-SVRC</span>).”</span> <em>Analyst</em> 140 (22): 7810–17. <a href="https://doi.org/10.1039/C5AN01638J">https://doi.org/10.1039/C5AN01638J</a>.
</div>
<div class="csl-entry">
Kusonmano, Kanthida, Wanwipa Vongsangnak, and Pramote Chumnanpuen. 2016. <span>“Informatics for <span>Metabolomics</span>.”</span> In <em>Translational <span>Biomedical Informatics</span></em>, 91–115. Advances in <span>Experimental Medicine</span> and <span>Biology</span>. <span>Springer, Singapore</span>. <a href="https://doi.org/10.1007/978-981-10-1503-8_5">https://doi.org/10.1007/978-981-10-1503-8_5</a>.
</div>
<div class="csl-entry">
Lai, Zijuan, Hiroshi Tsugawa, Gert Wohlgemuth, Sajjan Mehta, Matthew Mueller, Yuxuan Zheng, Atsushi Ogiwara, et al. 2018. <span>“Identifying Metabolites by Integrating Metabolome Databases with Mass Spectrometry Cheminformatics.”</span> <em>Nature Methods</em> 15 (1): 53–56. <a href="https://doi.org/10.1038/nmeth.4512">https://doi.org/10.1038/nmeth.4512</a>.
</div>
<div class="csl-entry">
Lakhani, Chirag M., Braden T. Tierney, Arjun K. Manrai, Jian Yang, Peter M. Visscher, and Chirag J. Patel. 2019. <span>“Repurposing Large Health Insurance Claims Data to Estimate Genetic and Environmental Contributions in 560 Phenotypes.”</span> <em>Nature Genetics</em> 51 (2): 327–34. <a href="https://doi.org/10.1038/s41588-018-0313-7">https://doi.org/10.1038/s41588-018-0313-7</a>.
</div>
<div class="csl-entry">
Laparre, Jérôme, Zied Kaabia, Mark Mooney, Tom Buckley, Mark Sherry, Bruno Le Bizec, and Gaud Dervilly-Pinel. 2017. <span>“Impact of Storage Conditions on the Urinary Metabolomics Fingerprint.”</span> <em>Analytica Chimica Acta</em> 951 (January): 99–107. <a href="https://doi.org/10.1016/j.aca.2016.11.055">https://doi.org/10.1016/j.aca.2016.11.055</a>.
</div>
<div class="csl-entry">
Larralde, Martin, Thomas N. Lawson, Ralf J. M. Weber, Pablo Moreno, Kenneth Haug, Philippe Rocca-Serra, Mark R. Viant, Christoph Steinbeck, and Reza M. Salek. 2017. <span>“<span class="nocase">mzML2ISA</span> &amp; <span class="nocase">nmrML2ISA</span>: Generating Enriched <span>ISA-Tab</span> Metadata Files from Metabolomics <span>XML</span> Data.”</span> <em>Bioinformatics</em> 33 (16): 2598–2600. <a href="https://doi.org/10.1093/bioinformatics/btx169">https://doi.org/10.1093/bioinformatics/btx169</a>.
</div>
<div class="csl-entry">
Lassen, Johan, Kirstine Lykke Nielsen, Mogens Johannsen, and Palle Villesen. 2021. <span>“Assessment of <span>XCMS Optimization Methods</span> with <span>Machine-Learning Performance</span>.”</span> <em>Analytical Chemistry</em> 93 (40): 13459–66. <a href="https://doi.org/10.1021/acs.analchem.1c02000">https://doi.org/10.1021/acs.analchem.1c02000</a>.
</div>
<div class="csl-entry">
Lawson, Thomas N., Ralf J. M. Weber, Martin R. Jones, Andrew J. Chetwynd, Giovanny Rodrı́guez-Blanco, Riccardo Di Guida, Mark R. Viant, and Warwick B. Dunn. 2017. <span>“<span class="nocase">msPurity</span>: <span>Automated Evaluation</span> of <span>Precursor Ion Purity</span> for <span>Mass Spectrometry-Based Fragmentation</span> in <span>Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 89 (4): 2432–39. <a href="https://doi.org/10.1021/acs.analchem.6b04358">https://doi.org/10.1021/acs.analchem.6b04358</a>.
</div>
<div class="csl-entry">
Lê Cao, Kim-Anh, Simon Boitard, and Philippe Besse. 2011. <span>“Sparse <span>PLS</span> Discriminant Analysis: Biologically Relevant Feature Selection and Graphical Displays for Multiclass Problems.”</span> <em>BMC Bioinformatics</em> 12 (June): 253. <a href="https://doi.org/10.1186/1471-2105-12-253">https://doi.org/10.1186/1471-2105-12-253</a>.
</div>
<div class="csl-entry">
Leek, Jeffrey T., W. Evan Johnson, Hilary S. Parker, Andrew E. Jaffe, and John D. Storey. 2012. <span>“The Sva Package for Removing Batch Effects and Other Unwanted Variation in High-Throughput Experiments.”</span> <em>Bioinformatics</em> 28 (6): 882–83. <a href="https://doi.org/10.1093/bioinformatics/bts034">https://doi.org/10.1093/bioinformatics/bts034</a>.
</div>
<div class="csl-entry">
Leek, Jeffrey T., and John D. Storey. 2007. <span>“Capturing <span>Heterogeneity</span> in <span>Gene Expression Studies</span> by <span>Surrogate Variable Analysis</span>.”</span> <em>PLOS Genet</em> 3 (9): e161. <a href="https://doi.org/10.1371/journal.pgen.0030161">https://doi.org/10.1371/journal.pgen.0030161</a>.
</div>
<div class="csl-entry">
———. 2008. <span>“A General Framework for Multiple Testing Dependence.”</span> <em>Proceedings of the National Academy of Sciences</em> 105 (48): 18718–23. <a href="https://doi.org/10.1073/pnas.0808709105">https://doi.org/10.1073/pnas.0808709105</a>.
</div>
<div class="csl-entry">
Levy, Allison J., Nicholas R. Oranzi, Atiye Ahmadireskety, Robin H. J. Kemperman, Michael S. Wei, and Richard A. Yost. 2019. <span>“Recent Progress in Metabolomics Using Ion Mobility-Mass Spectrometry.”</span> <em>TrAC Trends in Analytical Chemistry</em> 116 (July): 274–81. <a href="https://doi.org/10.1016/j.trac.2019.05.001">https://doi.org/10.1016/j.trac.2019.05.001</a>.
</div>
<div class="csl-entry">
Li, Bo, Jing Tang, Qingxia Yang, Shuang Li, Xuejiao Cui, Yinghong Li, Yuzong Chen, Weiwei Xue, Xiaofeng Li, and Feng Zhu. 2017. <span>“<span>NOREVA</span>: Normalization and Evaluation of <span class="nocase">MS-based</span> Metabolomics Data.”</span> <em>Nucleic Acids Research</em> 45 (W1): W162–70. <a href="https://doi.org/10.1093/nar/gkx449">https://doi.org/10.1093/nar/gkx449</a>.
</div>
<div class="csl-entry">
Li, Hao, Yuping Cai, Yuan Guo, Fangfang Chen, and Zheng-Jiang Zhu. 2016. <span>“<span>MetDIA</span>: <span>Targeted Metabolite Extraction</span> of <span>Multiplexed MS</span>/<span>MS Spectra Generated</span> by <span>Data-Independent Acquisition</span>.”</span> <em>Analytical Chemistry</em> 88 (17): 8757–64. <a href="https://doi.org/10.1021/acs.analchem.6b02122">https://doi.org/10.1021/acs.analchem.6b02122</a>.
</div>
<div class="csl-entry">
Li, Liang, Ronghong Li, Jianjun Zhou, Azeret Zuniga, Avalyn E. Stanislaus, Yiman Wu, Tao Huan, et al. 2013. <span>“<span>MyCompoundID</span>: <span>Using</span> an <span>Evidence-Based Metabolome Library</span> for <span>Metabolite Identification</span>.”</span> <em>Analytical Chemistry</em> 85 (6): 3401–8. <a href="https://doi.org/10.1021/ac400099b">https://doi.org/10.1021/ac400099b</a>.
</div>
<div class="csl-entry">
Li, Lili, Weijie Ren, Hongwei Kong, Chunxia Zhao, Xinjie Zhao, Xiaohui Lin, Xin Lu, and Guowang Xu. 2017. <span>“An Alignment Algorithm for <span class="nocase">LC-MS-based</span> Metabolomics Dataset Assisted by <span>MS</span>/<span>MS</span> Information.”</span> <em>Analytica Chimica Acta</em> 990 (October): 96–102. <a href="https://doi.org/10.1016/j.aca.2017.07.058">https://doi.org/10.1016/j.aca.2017.07.058</a>.
</div>
<div class="csl-entry">
Li, Shuzhao. 2020. <em>Computational <span>Methods</span> and <span>Data Analysis</span> for <span>Metabolomics</span></em>. <span>Springer</span>.
</div>
<div class="csl-entry">
Li, Shuzhao, Youngja Park, Sai Duraisingham, Frederick H. Strobel, Nooruddin Khan, Quinlyn A. Soltow, Dean P. Jones, and Bali Pulendran. 2013. <span>“Predicting <span>Network Activity</span> from <span>High Throughput Metabolomics</span>.”</span> <em>PLOS Computational Biology</em> 9 (7): e1003123. <a href="https://doi.org/10.1371/journal.pcbi.1003123">https://doi.org/10.1371/journal.pcbi.1003123</a>.
</div>
<div class="csl-entry">
Li, Yuanyue, Tobias Kind, Jacob Folz, Arpana Vaniya, Sajjan Singh Mehta, and Oliver Fiehn. 2021. <span>“Spectral Entropy Outperforms <span>MS</span>/<span>MS</span> Dot Product Similarity for Small-Molecule Compound Identification.”</span> <em>Nature Methods</em> 18 (12): 1524–31. <a href="https://doi.org/10.1038/s41592-021-01331-z">https://doi.org/10.1038/s41592-021-01331-z</a>.
</div>
<div class="csl-entry">
Li, Zhucui, Yan Lu, Yufeng Guo, Haijie Cao, Qinhong Wang, and Wenqing Shui. 2018. <span>“Comprehensive Evaluation of Untargeted Metabolomics Data Processing Software in Feature Detection, Quantification and Discriminating Marker Selection.”</span> <em>Analytica Chimica Acta</em> 1029 (October): 50–57. <a href="https://doi.org/10.1016/j.aca.2018.05.001">https://doi.org/10.1016/j.aca.2018.05.001</a>.
</div>
<div class="csl-entry">
Libiseller, Gunnar, Michaela Dvorzak, Ulrike Kleb, Edgar Gander, Tobias Eisenberg, Frank Madeo, Steffen Neumann, et al. 2015. <span>“<span>IPO</span>: A Tool for Automated Optimization of <span>XCMS</span> Parameters.”</span> <em>BMC Bioinformatics</em> 16 (April): 118. <a href="https://doi.org/10.1186/s12859-015-0562-8">https://doi.org/10.1186/s12859-015-0562-8</a>.
</div>
<div class="csl-entry">
Lin, Ching Yu, Huifeng Wu, Ronald S. Tjeerdema, and Mark R. Viant. 2007. <span>“Evaluation of Metabolite Extraction Strategies from Tissue Samples Using <span>NMR</span> Metabolomics.”</span> <em>Metabolomics</em> 3 (1): 55–67. <a href="https://doi.org/10.1007/s11306-006-0043-1">https://doi.org/10.1007/s11306-006-0043-1</a>.
</div>
<div class="csl-entry">
Lisec, Jan, Friederike Hoffmann, Clemens Schmitt, and Carsten Jaeger. 2016. <span>“Extending the <span>Dynamic Range</span> in <span>Metabolomics Experiments</span> by <span>Automatic Correction</span> of <span>Peaks Exceeding</span> the <span>Detection Limit</span>.”</span> <em>Analytical Chemistry</em> 88 (15): 7487–92. <a href="https://doi.org/10.1021/acs.analchem.6b02515">https://doi.org/10.1021/acs.analchem.6b02515</a>.
</div>
<div class="csl-entry">
Lisitsyna, Anna, Franco Moritz, Youzhong Liu, Loubna Al Sadat, Hans Hauner, Melina Claussnitzer, Philippe Schmitt-Kopplin, and Sara Forcisi. 2022. <span>“Feature <span>Selection Pipelines</span> with <span>Classification</span> for <span class="nocase">Non-targeted Metabolomics Combining</span> the <span>Neural Network</span> and <span>Genetic Algorithm</span>.”</span> <em>Analytical Chemistry</em> 94 (14): 5474–82. <a href="https://doi.org/10.1021/acs.analchem.1c03237">https://doi.org/10.1021/acs.analchem.1c03237</a>.
</div>
<div class="csl-entry">
Liu, Qin, Douglas Walker, Karan Uppal, Zihe Liu, Chunyu Ma, ViLinh Tran, Shuzhao Li, Dean P. Jones, and Tianwei Yu. 2020. <span>“Addressing the Batch Effect Issue for <span>LC</span>/<span>MS</span> Metabolomics Data in Data Preprocessing.”</span> <em>Scientific Reports</em> 10 (1): 13856. <a href="https://doi.org/10.1038/s41598-020-70850-0">https://doi.org/10.1038/s41598-020-70850-0</a>.
</div>
<div class="csl-entry">
Liu, Xinyu, Lina Zhou, Xianzhe Shi, and Guowang Xu. 2019. <span>“New Advances in Analytical Methods for Mass Spectrometry-Based Large-Scale Metabolomics Study.”</span> <em>TrAC Trends in Analytical Chemistry</em> 121 (December): 115665. <a href="https://doi.org/10.1016/j.trac.2019.115665">https://doi.org/10.1016/j.trac.2019.115665</a>.
</div>
<div class="csl-entry">
Livera, Alysha M. De, Marko Sysi-Aho, Laurent Jacob, Johann A. Gagnon-Bartsch, Sandra Castillo, Julie A. Simpson, and Terence P. Speed. 2015. <span>“Statistical <span>Methods</span> for <span>Handling Unwanted Variation</span> in <span>Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 87 (7): 3606–15. <a href="https://doi.org/10.1021/ac502439y">https://doi.org/10.1021/ac502439y</a>.
</div>
<div class="csl-entry">
Loftfield, Erikka, Emily Vogtmann, Joshua N. Sampson, Steven C. Moore, Heidi Nelson, Rob Knight, Nicholas Chia, and Rashmi Sinha. 2016. <span>“Comparison of <span>Collection Methods</span> for <span>Fecal Samples</span> for <span>Discovery Metabolomics</span> in <span>Epidemiologic Studies</span>.”</span> <em>Cancer Epidemiology and Prevention Biomarkers</em> 25 (11): 1483–90. <a href="https://doi.org/10.1158/1055-9965.EPI-16-0409">https://doi.org/10.1158/1055-9965.EPI-16-0409</a>.
</div>
<div class="csl-entry">
Loos, Martin, and Heinz Singer. 2017. <span>“Nontargeted Homologue Series Extraction from Hyphenated High Resolution Mass Spectrometry Data.”</span> <em>Journal of Cheminformatics</em> 9 (February). <a href="https://doi.org/10.1186/s13321-017-0197-z">https://doi.org/10.1186/s13321-017-0197-z</a>.
</div>
<div class="csl-entry">
Lu, Wenyun, Bryson D. Bennett, and Joshua D. Rabinowitz. 2008. <span>“Analytical Strategies for <span>LC</span> Targeted Metabolomics.”</span> <em>Journal of Chromatography B</em>, Hyphenated <span>Techniques</span> for <span>Global Metabolite Profiling</span>, 871 (2): 236–42. <a href="https://doi.org/10.1016/j.jchromb.2008.04.031">https://doi.org/10.1016/j.jchromb.2008.04.031</a>.
</div>
<div class="csl-entry">
Lu, Wenyun, Xiaoyang Su, Matthias S. Klein, Ian A. Lewis, Oliver Fiehn, and Joshua D. Rabinowitz. 2017. <span>“Metabolite <span>Measurement</span>: <span>Pitfalls</span> to <span>Avoid</span> and <span>Practices</span> to <span>Follow</span>.”</span> <em>Annual Review of Biochemistry</em> 86 (1): 277–304. <a href="https://doi.org/10.1146/annurev-biochem-061516-044952">https://doi.org/10.1146/annurev-biochem-061516-044952</a>.
</div>
<div class="csl-entry">
Lu, Xin, and Guowang Xu. 2008. <span>“<span>LC-MS Metabonomics Methodology</span> in <span>Biomarker Discovery</span>.”</span> In <em>Biomarker <span>Methods</span> in <span>Drug Discovery</span> and <span>Development</span></em>, edited by Feng Wang, 291–315. Methods in <span>Pharmacology</span> and <span>Toxicology</span>. <span>Humana Press</span>. <a href="https://doi.org/10.1007/978-1-59745-463-6_14">https://doi.org/10.1007/978-1-59745-463-6_14</a>.
</div>
<div class="csl-entry">
Ludwig, Marcus, Louis-Félix Nothias, Kai Dührkop, Irina Koester, Markus Fleischauer, Martin A. Hoffmann, Daniel Petras, et al. 2020. <span>“Database-Independent Molecular Formula Annotation Using <span>Gibbs</span> Sampling Through <span>ZODIAC</span>.”</span> <em>Nature Machine Intelligence</em> 2 (10): 629–41. <a href="https://doi.org/10.1038/s42256-020-00234-6">https://doi.org/10.1038/s42256-020-00234-6</a>.
</div>
<div class="csl-entry">
Luo, Xian, and Liang Li. 2017. <span>“Metabolomics of <span>Small Numbers</span> of <span>Cells</span>: <span>Metabolomic Profiling</span> of 100, 1000, and 10000 <span>Human Breast Cancer Cells</span>.”</span> <em>Analytical Chemistry</em> 89 (21): 11664–71. <a href="https://doi.org/10.1021/acs.analchem.7b03100">https://doi.org/10.1021/acs.analchem.7b03100</a>.
</div>
<div class="csl-entry">
Lv, Wangjie, Zhongda Zeng, Yuqing Zhang, Qingqing Wang, Lichao Wang, Zhaoxuan Zhang, Xianzhe Shi, Xinjie Zhao, and Guowang Xu. 2022. <span>“Comprehensive Metabolite Quantitative Assay Based on Alternate Metabolomics and Lipidomics Analyses.”</span> <em>Analytica Chimica Acta</em> 1215 (July): 339979. <a href="https://doi.org/10.1016/j.aca.2022.339979">https://doi.org/10.1016/j.aca.2022.339979</a>.
</div>
<div class="csl-entry">
Ma, Yan, Tobias Kind, Dawei Yang, Carlos Leon, and Oliver Fiehn. 2014. <span>“<span>MS2Analyzer</span>: <span>A Software</span> for <span>Small Molecule Substructure Annotations</span> from <span>Accurate Tandem Mass Spectra</span>.”</span> <em>Analytical Chemistry</em> 86 (21): 10724–31. <a href="https://doi.org/10.1021/ac502818e">https://doi.org/10.1021/ac502818e</a>.
</div>
<div class="csl-entry">
Madsen, Rasmus, Torbjörn Lundstedt, and Johan Trygg. 2010. <span>“Chemometrics in Metabolomics Review in Human Disease Diagnosis.”</span> <em>Analytica Chimica Acta</em> 659 (1): 23–33. <a href="https://doi.org/10.1016/j.aca.2009.11.042">https://doi.org/10.1016/j.aca.2009.11.042</a>.
</div>
<div class="csl-entry">
Mahieu, Nathaniel G., and Gary J. Patti. 2017. <span>“Systems-<span>Level Annotation</span> of a <span>Metabolomics Data Set Reduces</span> 25 000 <span>Features</span> to <span>Fewer</span> Than 1000 <span>Unique Metabolites</span>.”</span> <em>Analytical Chemistry</em> 89 (19): 10397–406. <a href="https://doi.org/10.1021/acs.analchem.7b02380">https://doi.org/10.1021/acs.analchem.7b02380</a>.
</div>
<div class="csl-entry">
Mahieu, Nathaniel G., Jonathan L. Spalding, Susan J. Gelman, and Gary J. Patti. 2016. <span>“Defining and <span>Detecting Complex Peak Relationships</span> in <span>Mass Spectral Data</span>: <span>The Mz</span>.unity <span>Algorithm</span>.”</span> <em>Analytical Chemistry</em> 88 (18): 9037–46. <a href="https://doi.org/10.1021/acs.analchem.6b01702">https://doi.org/10.1021/acs.analchem.6b01702</a>.
</div>
<div class="csl-entry">
Mahieu, Nathaniel G., Jonathan L. Spalding, and Gary J. Patti. 2016. <span>“Warpgroup: Increased Precision of Metabolomic Data Processing by Consensus Integration Bound Analysis.”</span> <em>Bioinformatics</em> 32 (2): 268–75. <a href="https://doi.org/10.1093/bioinformatics/btv564">https://doi.org/10.1093/bioinformatics/btv564</a>.
</div>
<div class="csl-entry">
Mahmud, Iqbal, Sandi Sternberg, Michael Williams, and Timothy J. Garrett. 2017. <span>“Comparison of Global Metabolite Extraction Strategies for Soybeans Using <span>UHPLC-HRMS</span>.”</span> <em>Analytical and Bioanalytical Chemistry</em> 409 (26): 6173–80. <a href="https://doi.org/10.1007/s00216-017-0557-6">https://doi.org/10.1007/s00216-017-0557-6</a>.
</div>
<div class="csl-entry">
Maitre, Léa, Mariona Bustamante, Carles Hernández-Ferrer, Denise Thiel, Chung-Ho E. Lau, Alexandros P. Siskos, Marta Vives-Usano, et al. 2022. <span>“Multi-Omics Signatures of the Human Early Life Exposome.”</span> <em>Nature Communications</em> 13 (1): 7024. <a href="https://doi.org/10.1038/s41467-022-34422-2">https://doi.org/10.1038/s41467-022-34422-2</a>.
</div>
<div class="csl-entry">
Mangul, Serghei, Thiago Mosqueiro, Richard J. Abdill, Dat Duong, Keith Mitchell, Varuni Sarwal, Brian Hill, et al. 2019. <span>“Challenges and Recommendations to Improve the Installability and Archival Stability of Omics Computational Tools.”</span> <em>PLOS Biology</em> 17 (6): e3000333. <a href="https://doi.org/10.1371/journal.pbio.3000333">https://doi.org/10.1371/journal.pbio.3000333</a>.
</div>
<div class="csl-entry">
Mannhold, Raimund, Gennadiy I. Poda, Claude Ostermann, and Igor V. Tetko. 2009. <span>“Calculation of <span>Molecular Lipophilicity</span>: <span class="nocase">State-of-the-Art</span> and <span>Comparison</span> of <span>LogP Methods</span> on More Than 96,000 <span>Compounds</span>.”</span> <em>Journal of Pharmaceutical Sciences</em> 98 (3): 861–93. <a href="https://doi.org/10.1002/jps.21494">https://doi.org/10.1002/jps.21494</a>.
</div>
<div class="csl-entry">
Mansouri, Kamel, Chris M. Grulke, Richard S. Judson, and Antony J. Williams. 2018. <span>“<span>OPERA</span> Models for Predicting Physicochemical Properties and Environmental Fate Endpoints.”</span> <em>Journal of Cheminformatics</em> 10 (1): 10. <a href="https://doi.org/10.1186/s13321-018-0263-1">https://doi.org/10.1186/s13321-018-0263-1</a>.
</div>
<div class="csl-entry">
Martens, Jonathan, Giel Berden, Rianne E. van Outersterp, Leo A. J. Kluijtmans, Udo F. Engelke, Clara D. M. van Karnebeek, Ron A. Wevers, and Jos Oomens. 2017. <span>“Molecular Identification in Metabolomics Using Infrared Ion Spectroscopy.”</span> <em>Scientific Reports</em> 7 (June). <a href="https://doi.org/10.1038/s41598-017-03387-4">https://doi.org/10.1038/s41598-017-03387-4</a>.
</div>
<div class="csl-entry">
Matich, Eryn K., Nita G. Chavez Soria, Diana S. Aga, and G. Ekin Atilla-Gokcumen. 2019. <span>“Applications of Metabolomics in Assessing Ecological Effects of Emerging Contaminants and Pollutants on Plants.”</span> <em>Journal of Hazardous Materials</em> 373 (July): 527–35. <a href="https://doi.org/10.1016/j.jhazmat.2019.02.084">https://doi.org/10.1016/j.jhazmat.2019.02.084</a>.
</div>
<div class="csl-entry">
Matsuo, Teruko, Hiroshi Tsugawa, Hiromi Miyagawa, and Eiichiro Fukusaki. 2017. <span>“Integrated <span>Strategy</span> for <span>Unknown EI</span>, <span>Multivariate Analysis</span>, <span>EI</span>, and <span>Retention Index Prediction</span>.”</span> <em>Analytical Chemistry</em> 89 (12): 6766–73. <a href="https://doi.org/10.1021/acs.analchem.7b01010">https://doi.org/10.1021/acs.analchem.7b01010</a>.
</div>
<div class="csl-entry">
McLean, Craig, and Elizabeth B. Kujawinski. 2020. <span>“<span>AutoTuner</span>: <span>High Fidelity</span> and <span>Robust Parameter Selection</span> for <span>Metabolomics Data Processing</span>.”</span> <em>Analytical Chemistry</em> 92 (8): 5724–32. <a href="https://doi.org/10.1021/acs.analchem.9b04804">https://doi.org/10.1021/acs.analchem.9b04804</a>.
</div>
<div class="csl-entry">
Melamud, Eugene, Livia Vastag, and Joshua D. Rabinowitz. 2010. <span>“Metabolomic <span>Analysis</span> and <span>Visualization Engine</span> for <span>LC</span>-<span>MS Data</span>.”</span> <em>Analytical Chemistry</em> 82 (23): 9818–26. <a href="https://doi.org/10.1021/ac1021166">https://doi.org/10.1021/ac1021166</a>.
</div>
<div class="csl-entry">
Menikarachchi, Lochana C., Shannon Cawley, Dennis W. Hill, L. Mark Hall, Lowell Hall, Steven Lai, Janine Wilder, and David F. Grant. 2012. <span>“<span>MolFind</span>: <span>A Software Package Enabling HPLC</span>/<span>MS-Based Identification</span> of <span>Unknown Chemical Structures</span>.”</span> <em>Analytical Chemistry</em> 84 (21): 9388–94. <a href="https://doi.org/10.1021/ac302048x">https://doi.org/10.1021/ac302048x</a>.
</div>
<div class="csl-entry">
Miggiels, Paul, Bert Wouters, Gerard J. P. van Westen, Anne-Charlotte Dubbelman, and Thomas Hankemeier. 2019. <span>“Novel Technologies for Metabolomics: <span>More</span> for Less.”</span> <em>TrAC Trends in Analytical Chemistry</em> 120 (November): 115323. <a href="https://doi.org/10.1016/j.trac.2018.11.021">https://doi.org/10.1016/j.trac.2018.11.021</a>.
</div>
<div class="csl-entry">
Misra, Biswapriya B. 2018. <span>“New Tools and Resources in Metabolomics: 2016.”</span> <em>ELECTROPHORESIS</em> 39 (7): 909–23. <a href="https://doi.org/10.1002/elps.201700441">https://doi.org/10.1002/elps.201700441</a>.
</div>
<div class="csl-entry">
Misra, Biswapriya B., Johannes F. Fahrmann, and Dmitry Grapov. 2017. <span>“Review of Emerging Metabolomic Tools and Resources: 2015.”</span> <em>ELECTROPHORESIS</em> 38 (18): 2257–74. <a href="https://doi.org/10.1002/elps.201700110">https://doi.org/10.1002/elps.201700110</a>.
</div>
<div class="csl-entry">
Misra, Biswapriya B., and Justin J. J. van der Hooft. 2016. <span>“Updates in Metabolomics Tools and Resources: 2014.”</span> <em>ELECTROPHORESIS</em> 37 (1): 86–110. <a href="https://doi.org/10.1002/elps.201500417">https://doi.org/10.1002/elps.201500417</a>.
</div>
<div class="csl-entry">
Miyagawa, Hiromi, and Takeshi Bamba. 2019. <span>“Comparison of Sequential Derivatization with Concurrent Methods for <span>GC</span>/<span class="nocase">MS-based</span> Metabolomics.”</span> <em>Journal of Bioscience and Bioengineering</em> 127 (2): 160–68. <a href="https://doi.org/10.1016/j.jbiosc.2018.07.015">https://doi.org/10.1016/j.jbiosc.2018.07.015</a>.
</div>
<div class="csl-entry">
Montenegro-Burke, J. Rafael, Aries E. Aisporna, H. Paul Benton, Duane Rinehart, Mingliang Fang, Tao Huan, Benedikt Warth, et al. 2017. <span>“Data <span>Streaming</span> for <span>Metabolomics</span>: <span>Accelerating Data Processing</span> and <span>Analysis</span> from <span>Days</span> to <span>Minutes</span>.”</span> <em>Analytical Chemistry</em> 89 (2): 1254–59. <a href="https://doi.org/10.1021/acs.analchem.6b03890">https://doi.org/10.1021/acs.analchem.6b03890</a>.
</div>
<div class="csl-entry">
Müller, Manfred J., and Anja Bosy-Westphal. 2020. <span>“From a <span>‘<span>Metabolomics</span> Fashion’</span> to a Sound Application of Metabolomics in Research on Human Nutrition.”</span> <em>European Journal of Clinical Nutrition</em> 74 (12): 1619–29. <a href="https://doi.org/10.1038/s41430-020-00781-6">https://doi.org/10.1038/s41430-020-00781-6</a>.
</div>
<div class="csl-entry">
Myers, Owen D., Susan J. Sumner, Shuzhao Li, Stephen Barnes, and Xiuxia Du. 2017. <span>“Detailed <span>Investigation</span> and <span>Comparison</span> of the <span>XCMS</span> and <span>MZmine</span> 2 <span>Chromatogram Construction</span> and <span>Chromatographic Peak Detection Methods</span> for <span>Preprocessing Mass Spectrometry Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 89 (17): 8689–95. <a href="https://doi.org/10.1021/acs.analchem.7b01069">https://doi.org/10.1021/acs.analchem.7b01069</a>.
</div>
<div class="csl-entry">
Najdekr, Lukáš, David Friedecký, Ralf Tautenhahn, Tomáš Pluskal, Junhua Wang, Yingying Huang, and Tomáš Adam. 2016. <span>“Influence of <span>Mass Resolving Power</span> in <span>Orbital Ion-Trap Mass Spectrometry-Based Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 88 (23): 11429–35. <a href="https://doi.org/10.1021/acs.analchem.6b02319">https://doi.org/10.1021/acs.analchem.6b02319</a>.
</div>
<div class="csl-entry">
Nash, William J., and Warwick B. Dunn. 2019. <span>“From Mass to Metabolite in Human Untargeted Metabolomics: <span>Recent</span> Advances in Annotation of Metabolites Applying Liquid Chromatography-Mass Spectrometry Data.”</span> <em>TrAC Trends in Analytical Chemistry</em> 120 (November): 115324. <a href="https://doi.org/10.1016/j.trac.2018.11.022">https://doi.org/10.1016/j.trac.2018.11.022</a>.
</div>
<div class="csl-entry">
Ni, Yan, Mingming Su, Yunping Qiu, Wei Jia, and Xiuxia Du. 2016. <span>“<span>ADAP-GC</span> 3.0: <span>Improved Peak Detection</span> and <span>Deconvolution</span> of <span class="nocase">Co-eluting Metabolites</span> from <span>GC</span>/<span>TOF-MS Data</span> for <span>Metabolomics Studies</span>.”</span> <em>Analytical Chemistry</em> 88 (17): 8802–11. <a href="https://doi.org/10.1021/acs.analchem.6b02222">https://doi.org/10.1021/acs.analchem.6b02222</a>.
</div>
<div class="csl-entry">
Ni, Zhixu, Michele Wölk, Geoff Jukes, Karla Mendivelso Espinosa, Robert Ahrends, Lucila Aimo, Jorge Alvarez-Jarreta, et al. 2022. <span>“Guiding the Choice of Informatics Software and Tools for Lipidomics Research Applications.”</span> <em>Nature Methods</em>, December, 1–12. <a href="https://doi.org/10.1038/s41592-022-01710-0">https://doi.org/10.1038/s41592-022-01710-0</a>.
</div>
<div class="csl-entry">
Nikolskiy, Igor, Nathaniel G. Mahieu, Ying-Jr Chen, Ralf Tautenhahn, and Gary J. Patti. 2013. <span>“An <span>Untargeted Metabolomic Workflow</span> to <span>Improve Structural Characterization</span> of <span>Metabolites</span>.”</span> <em>Analytical Chemistry</em> 85 (16): 7713–19. <a href="https://doi.org/10.1021/ac400751j">https://doi.org/10.1021/ac400751j</a>.
</div>
<div class="csl-entry">
Nothias, Louis-Félix, Daniel Petras, Robin Schmid, Kai Dührkop, Johannes Rainer, Abinesh Sarvepalli, Ivan Protsyuk, et al. 2020. <span>“Feature-Based Molecular Networking in the <span>GNPS</span> Analysis Environment.”</span> <em>Nature Methods</em> 17 (9): 905–8. <a href="https://doi.org/10.1038/s41592-020-0933-6">https://doi.org/10.1038/s41592-020-0933-6</a>.
</div>
<div class="csl-entry">
Nyamundanda, Gift, Isobel Claire Gormley, Yue Fan, William M. Gallagher, and Lorraine Brennan. 2013. <span>“<span>MetSizeR</span>: Selecting the Optimal Sample Size for Metabolomic Studies Using an Analysis Based Approach.”</span> <em>BMC Bioinformatics</em> 14: 338. <a href="https://doi.org/10.1186/1471-2105-14-338">https://doi.org/10.1186/1471-2105-14-338</a>.
</div>
<div class="csl-entry">
O’Boyle, Noel M., Michael Banck, Craig A. James, Chris Morley, Tim Vandermeersch, and Geoffrey R. Hutchison. 2011. <span>“Open <span>Babel</span>: <span>An</span> Open Chemical Toolbox.”</span> <em>Journal of Cheminformatics</em> 3 (1): 33. <a href="https://doi.org/10.1186/1758-2946-3-33">https://doi.org/10.1186/1758-2946-3-33</a>.
</div>
<div class="csl-entry">
Oberg, Ann L., and Olga Vitek. 2009. <span>“Statistical <span>Design</span> of <span>Quantitative Mass Spectrometry-Based Proteomic Experiments</span>.”</span> <em>Journal of Proteome Research</em> 8 (5): 2144–56. <a href="https://doi.org/10.1021/pr8010099">https://doi.org/10.1021/pr8010099</a>.
</div>
<div class="csl-entry">
Ortmayr, Karin, Verena Charwat, Cornelia Kasper, Stephan Hann, and Gunda Koellensperger. 2016. <span>“Uncertainty Budgeting in Fold Change Determination and Implications for Non-Targeted Metabolomics Studies in Model Systems”</span> 142 (1): 80–90. <a href="https://doi.org/10.1039/C6AN01342B">https://doi.org/10.1039/C6AN01342B</a>.
</div>
<div class="csl-entry">
Osipenko, Sergey, Alexander Zherebker, Lidiia Rumiantseva, Oxana Kovaleva, Evgeny N. Nikolaev, and Yury Kostyukevich. 2022. <span>“Oxygen <span>Isotope Exchange Reaction</span> for <span>Untargeted LC</span>.”</span> <em>Journal of the American Society for Mass Spectrometry</em> 33 (2): 390–98. <a href="https://doi.org/10.1021/jasms.1c00383">https://doi.org/10.1021/jasms.1c00383</a>.
</div>
<div class="csl-entry">
Palmer, Andrew, Prasad Phapale, Ilya Chernyavsky, Regis Lavigne, Dominik Fay, Artem Tarasov, Vitaly Kovalev, et al. 2017. <span>“<span class="nocase">FDR-controlled</span> Metabolite Annotation for High-Resolution Imaging Mass Spectrometry.”</span> <em>Nature Methods</em> 14 (1): 57–60. <a href="https://doi.org/10.1038/nmeth.4072">https://doi.org/10.1038/nmeth.4072</a>.
</div>
<div class="csl-entry">
Pang, Zhiqiang, Jasmine Chong, Shuzhao Li, and Jianguo Xia. 2020. <span>“<span>MetaboAnalystR</span> 3.0: <span>Toward</span> an <span>Optimized Workflow</span> for <span>Global Metabolomics</span>.”</span> <em>Metabolites</em> 10 (5): 186. <a href="https://doi.org/10.3390/metabo10050186">https://doi.org/10.3390/metabo10050186</a>.
</div>
<div class="csl-entry">
Passos Mansoldo, Felipe Raposo, Rafael Garrett, Veronica da Silva Cardoso, Marina Amaral Alves, and Alane Beatriz Vermelho. 2022. <span>“Metabology: <span>Analysis</span> of Metabolomics Data Using Community Ecology Tools.”</span> <em>Analytica Chimica Acta</em> 1232 (November): 340469. <a href="https://doi.org/10.1016/j.aca.2022.340469">https://doi.org/10.1016/j.aca.2022.340469</a>.
</div>
<div class="csl-entry">
Patiny, Luc, and Alain Borel. 2013. <span>“<span>ChemCalc</span>: <span>A Building Block</span> for <span>Tomorrow</span>’s <span>Chemical Infrastructure</span>.”</span> <em>Journal of Chemical Information and Modeling</em> 53 (5): 1223–28. <a href="https://doi.org/10.1021/ci300563h">https://doi.org/10.1021/ci300563h</a>.
</div>
<div class="csl-entry">
Petras, Daniel, Vanessa V. Phelan, Deepa Acharya, Andrew E. Allen, Allegra T. Aron, Nuno Bandeira, Benjamin P. Bowen, et al. 2021. <span>“<span>GNPS Dashboard</span>: Collaborative Exploration of Mass Spectrometry Data in the Web Browser.”</span> <em>Nature Methods</em>, December, 1–3. <a href="https://doi.org/10.1038/s41592-021-01339-5">https://doi.org/10.1038/s41592-021-01339-5</a>.
</div>
<div class="csl-entry">
Pezzatti, Julian, Julien Boccard, Santiago Codesido, Yoric Gagnebin, Abhinav Joshi, Didier Picard, Víctor González-Ruiz, and Serge Rudaz. 2020. <span>“Implementation of Liquid Chromatographyhigh Resolution Mass Spectrometry Methods for Untargeted Metabolomic Analyses of Biological Samples: <span>A</span> Tutorial.”</span> <em>Analytica Chimica Acta</em> 1105 (April): 28–44. <a href="https://doi.org/10.1016/j.aca.2019.12.062">https://doi.org/10.1016/j.aca.2019.12.062</a>.
</div>
<div class="csl-entry">
Pfeuffer, Julianus, Timo Sachsenberg, Oliver Alka, Mathias Walzer, Alexander Fillbrunn, Lars Nilse, Oliver Schilling, Knut Reinert, and Oliver Kohlbacher. 2017. <span>“<span>OpenMS</span> <span>A</span> Platform for Reproducible Analysis of Mass Spectrometry Data.”</span> <em>Journal of Biotechnology</em>, Bioinformatics <span>Solutions</span> for <span>Big Data Analysis</span> in <span>Life Sciences</span> presented by the <span>German Network</span> for <span>Bioinformatics Infrastructure</span>, 261 (November): 142–48. <a href="https://doi.org/10.1016/j.jbiotec.2017.05.016">https://doi.org/10.1016/j.jbiotec.2017.05.016</a>.
</div>
<div class="csl-entry">
Phapale, Prasad, Vineeta Rai, Ashok Kumar Mohanty, and Sanjeeva Srivastava. 2020. <span>“Untargeted <span>Metabolomics Workshop Report</span>: <span>Quality Control Considerations</span> from <span>Sample Preparation</span> to <span>Data Analysis</span>.”</span> <em>Journal of the American Society for Mass Spectrometry</em> 31 (9): 2006–10. <a href="https://doi.org/10.1021/jasms.0c00224">https://doi.org/10.1021/jasms.0c00224</a>.
</div>
<div class="csl-entry">
Place, Benjamin J., Elin M. Ulrich, Jonathan K. Challis, Alex Chao, Bowen Du, Kristin Favela, Yong-Lai Feng, et al. 2021. <span>“An <span>Introduction</span> to the <span>Benchmarking</span> and <span>Publications</span> for <span>Non-Targeted Analysis Working Group</span>.”</span> <em>Analytical Chemistry</em> 93 (49): 16289–96. <a href="https://doi.org/10.1021/acs.analchem.1c02660">https://doi.org/10.1021/acs.analchem.1c02660</a>.
</div>
<div class="csl-entry">
Pluskal, Tomáš, Sandra Castillo, Alejandro Villar-Briones, and Matej Orešič. 2010. <span>“<span>MZmine</span> 2: <span>Modular</span> Framework for Processing, Visualizing, and Analyzing Mass Spectrometry-Based Molecular Profile Data.”</span> <em>BMC Bioinformatics</em> 11: 395. <a href="https://doi.org/10.1186/1471-2105-11-395">https://doi.org/10.1186/1471-2105-11-395</a>.
</div>
<div class="csl-entry">
Pluskal, Tomáš, Ansgar Korf, Aleksandr Smirnov, Robin Schmid, Timothy R. Fallon, Xiuxia Du, and Jing-Ke Weng. 2020. <span>“<span>CHAPTER</span> 7:<span>Metabolomics Data Analysis Using MZmine</span>.”</span> In <em>Processing <span>Metabolomics</span> and <span>Proteomics Data</span> with <span>Open Software</span></em>, 232–54. <a href="https://doi.org/10.1039/9781788019880-00232">https://doi.org/10.1039/9781788019880-00232</a>.
</div>
<div class="csl-entry">
Plyushchenko, Ivan V., Elizaveta S. Fedorova, Natalia V. Potoldykova, Konstantin A. Polyakovskiy, Alexander I. Glukhov, and Igor A. Rodin. 2022. <span>“Omics <span>Untargeted Key Script</span>: <span>R-Based Software Toolbox</span> for <span>Untargeted Metabolomics</span> with <span>Bladder Cancer Biomarkers Discovery Case Study</span>.”</span> <em>Journal of Proteome Research</em> 21 (3): 833–47. <a href="https://doi.org/10.1021/acs.jproteome.1c00392">https://doi.org/10.1021/acs.jproteome.1c00392</a>.
</div>
<div class="csl-entry">
Polderman, Tinca J. C., Beben Benyamin, Christiaan A. de Leeuw, Patrick F. Sullivan, Arjen van Bochoven, Peter M. Visscher, and Danielle Posthuma. 2015. <span>“Meta-Analysis of the Heritability of Human Traits Based on Fifty Years of Twin Studies.”</span> <em>Nature Genetics</em> 47 (7): 702–9. <a href="https://doi.org/10.1038/ng.3285">https://doi.org/10.1038/ng.3285</a>.
</div>
<div class="csl-entry">
Qiu, Feng, Dennis D. Fine, Daniel J. Wherritt, Zhentian Lei, and Lloyd W. Sumner. 2016. <span>“<span>PlantMAT</span>: <span>A Metabolomics Tool</span> for <span>Predicting</span> the <span>Specialized Metabolic Potential</span> of a <span>System</span> and for <span>Large-Scale Metabolite Identifications</span>.”</span> <em>Analytical Chemistry</em> 88 (23): 11373–83. <a href="https://doi.org/10.1021/acs.analchem.6b00906">https://doi.org/10.1021/acs.analchem.6b00906</a>.
</div>
<div class="csl-entry">
Qiu, Feng, Zhentian Lei, and Lloyd W. Sumner. 2018. <span>“<span>MetExpert</span>: <span>An</span> Expert System to Enhance Gas Chromatography-Mass Spectrometry-Based Metabolite Identifications.”</span> <em>Analytica Chimica Acta</em>, Analytical <span>Metabolomics</span>, 1037 (December): 316–26. <a href="https://doi.org/10.1016/j.aca.2018.03.052">https://doi.org/10.1016/j.aca.2018.03.052</a>.
</div>
<div class="csl-entry">
Rey-Stolle, Fernanda, Danuta Dudzik, Carolina Gonzalez-Riano, Miguel Fernández-García, Vanesa Alonso-Herranz, David Rojo, Coral Barbas, and Antonia García. 2022. <span>“Low and High Resolution Gas Chromatography-Mass Spectrometry for Untargeted Metabolomics: <span>A</span> Tutorial.”</span> <em>Analytica Chimica Acta</em> 1210 (June): 339043. <a href="https://doi.org/10.1016/j.aca.2021.339043">https://doi.org/10.1016/j.aca.2021.339043</a>.
</div>
<div class="csl-entry">
Riquelme, Gabriel, Nicolás Zabalegui, Pablo Marchi, Christina M. Jones, and María Eugenia Monge. 2020. <span>“A <span>Python-Based Pipeline</span> for <span>Preprocessing LC</span> for <span>Untargeted Metabolomics Workflows</span>.”</span> <em>Metabolites</em> 10 (10): 416. <a href="https://doi.org/10.3390/metabo10100416">https://doi.org/10.3390/metabo10100416</a>.
</div>
<div class="csl-entry">
Röst, Hannes L., Timo Sachsenberg, Stephan Aiche, Chris Bielow, Hendrik Weisser, Fabian Aicheler, Sandro Andreotti, et al. 2016. <span>“<span>OpenMS</span>: A Flexible Open-Source Software Platform for Mass Spectrometry Data Analysis.”</span> <em>Nature Methods</em> 13 (9): 741–48. <a href="https://doi.org/10.1038/nmeth.3959">https://doi.org/10.1038/nmeth.3959</a>.
</div>
<div class="csl-entry">
Röst, Hannes L., Uwe Schmitt, Ruedi Aebersold, and Lars Malmström. 2014. <span>“<span class="nocase">pyOpenMS</span>: <span class="nocase">A Python-based</span> Interface to the <span>OpenMS</span> Mass-Spectrometry Algorithm Library.”</span> <em>PROTEOMICS</em> 14 (1): 74–77. <a href="https://doi.org/10.1002/pmic.201300246">https://doi.org/10.1002/pmic.201300246</a>.
</div>
<div class="csl-entry">
Roszkowska, Anna, Miao Yu, Vincent Bessonneau, Leslie Bragg, Mark Servos, and Janusz Pawliszyn. 2018. <span>“Tissue Storage Affects Lipidome Profiling in Comparison to in Vivo Microsampling Approach.”</span> <em>Scientific Reports</em> 8 (1): 6980. <a href="https://doi.org/10.1038/s41598-018-25428-2">https://doi.org/10.1038/s41598-018-25428-2</a>.
</div>
<div class="csl-entry">
Rurik, Marc, Oliver Alka, Fabian Aicheler, and Oliver Kohlbacher. 2020. <span>“Metabolomics <span>Data Processing Using OpenMS</span>.”</span> In <em>Computational <span>Methods</span> and <span>Data Analysis</span> for <span>Metabolomics</span></em>, edited by Shuzhao Li, 49–60. Methods in <span>Molecular Biology</span>. <span>New York, NY</span>: <span>Springer US</span>. <a href="https://doi.org/10.1007/978-1-0716-0239-3_4">https://doi.org/10.1007/978-1-0716-0239-3_4</a>.
</div>
<div class="csl-entry">
Rusconi, Filippo. 2019. <span>“<span class="nocase">mineXpert</span>: <span>Biological Mass Spectrometry Data Visualization</span> and <span>Mining</span> with <span>Full JavaScript Ability</span>.”</span> <em>Journal of Proteome Research</em> 18 (5): 2254–59. <a href="https://doi.org/10.1021/acs.jproteome.9b00099">https://doi.org/10.1021/acs.jproteome.9b00099</a>.
</div>
<div class="csl-entry">
Ruttkies, Christoph, Emma L. Schymanski, Sebastian Wolf, Juliane Hollender, and Steffen Neumann. 2016. <span>“<span>MetFrag</span> Relaunched: Incorporating Strategies Beyond in Silico Fragmentation.”</span> <em>Journal of Cheminformatics</em> 8 (January): 3. <a href="https://doi.org/10.1186/s13321-016-0115-9">https://doi.org/10.1186/s13321-016-0115-9</a>.
</div>
<div class="csl-entry">
Saccenti, Edoardo, and Marieke E. Timmerman. 2016. <span>“Approaches to <span>Sample Size Determination</span> for <span>Multivariate Data</span>: <span>Applications</span> to <span>PCA</span> and <span>PLS-DA</span> of <span>Omics Data</span>.”</span> <em>Journal of Proteome Research</em> 15 (8): 2379–93. <a href="https://doi.org/10.1021/acs.jproteome.5b01029">https://doi.org/10.1021/acs.jproteome.5b01029</a>.
</div>
<div class="csl-entry">
Samanipour, Saer, Malcolm J. Reid, Kine Bæk, and Kevin V. Thomas. 2018. <span>“Combining a <span>Deconvolution</span> and a <span>Universal Library Search Algorithm</span> for the <span>Nontarget Analysis</span> of <span>Data-Independent Acquisition Mode Liquid Chromatography</span>-<span>High-Resolution Mass Spectrometry Results</span>.”</span> <em>Environmental Science &amp; Technology</em> 52 (8): 4694–4701. <a href="https://doi.org/10.1021/acs.est.8b00259">https://doi.org/10.1021/acs.est.8b00259</a>.
</div>
<div class="csl-entry">
Sarpe, Vladimir, and David C Schriemer. 2017. <span>“Supporting Metabolomics with Adaptable Software: Design Architectures for the End-User.”</span> <em>Current Opinion in Biotechnology</em>, Analytical biotechnology, 43 (February): 110–17. <a href="https://doi.org/10.1016/j.copbio.2016.11.001">https://doi.org/10.1016/j.copbio.2016.11.001</a>.
</div>
<div class="csl-entry">
Scheltema, Richard A., Andris Jankevics, Ritsert C. Jansen, Morris A. Swertz, and Rainer Breitling. 2011. <span>“<span>PeakML</span>/<span class="nocase">mzMatch</span>: <span>A File Format</span>, <span>Java Library</span>, <span>R Library</span>, and <span>Tool-Chain</span> for <span>Mass Spectrometry Data Analysis</span>.”</span> <em>Analytical Chemistry</em> 83 (7): 2786–93. <a href="https://doi.org/10.1021/ac2000994">https://doi.org/10.1021/ac2000994</a>.
</div>
<div class="csl-entry">
Scheubert, Kerstin, Franziska Hufsky, Daniel Petras, Mingxun Wang, Louis-Félix Nothias, Kai Dührkop, Nuno Bandeira, Pieter C. Dorrestein, and Sebastian Böcker. 2017. <span>“Significance Estimation for Large Scale Metabolomics Annotations by Spectral Matching.”</span> <em>Nature Communications</em> 8 (1): 1494. <a href="https://doi.org/10.1038/s41467-017-01318-5">https://doi.org/10.1038/s41467-017-01318-5</a>.
</div>
<div class="csl-entry">
Schrimpe-Rutledge, Alexandra C., Simona G. Codreanu, Stacy D. Sherrod, and John A. McLean. 2016. <span>“Untargeted <span>Metabolomics Strategies</span> and <span>Emerging Directions</span>.”</span> <em>Journal of The American Society for Mass Spectrometry</em> 27 (12): 1897–1905. <a href="https://doi.org/10.1007/s13361-016-1469-y">https://doi.org/10.1007/s13361-016-1469-y</a>.
</div>
<div class="csl-entry">
Schymanski, Emma L., and Antony J. Williams. 2017. <span>“Open <span>Science</span> for <span>Identifying</span> <span>‘<span>Known Unknown</span>’</span> <span>Chemicals</span>.”</span> <em>Environmental Science &amp; Technology</em> 51 (10): 5357–59. <a href="https://doi.org/10.1021/acs.est.7b01908">https://doi.org/10.1021/acs.est.7b01908</a>.
</div>
<div class="csl-entry">
Senan, Oriol, Antoni Aguilar-Mogas, Miriam Navarro, Jordi Capellades, Luke Noon, Deborah Burks, Oscar Yanes, Roger Guimerà, and Marta Sales-Pardo. 2019. <span>“<span>CliqueMS</span>: A Computational Tool for Annotating in-Source Metabolite Ions from <span>LC-MS</span> Untargeted Metabolomics Data Based on a Coelution Similarity Network.”</span> <em>Bioinformatics</em> 35 (20): 4089–97. <a href="https://doi.org/10.1093/bioinformatics/btz207">https://doi.org/10.1093/bioinformatics/btz207</a>.
</div>
<div class="csl-entry">
Shaffer, Justin P., Louis-Félix Nothias, Luke R. Thompson, Jon G. Sanders, Rodolfo A. Salido, Sneha P. Couvillion, Asker D. Brejnrod, et al. 2022. <span>“Standardized Multi-Omics of <span>Earth</span>’s Microbiomes Reveals Microbial and Metabolite Diversity.”</span> <em>Nature Microbiology</em> 7 (12): 2128–50. <a href="https://doi.org/10.1038/s41564-022-01266-x">https://doi.org/10.1038/s41564-022-01266-x</a>.
</div>
<div class="csl-entry">
Shen, Xiaotao, Ruohong Wang, Xin Xiong, Yandong Yin, Yuping Cai, Zaijun Ma, Nan Liu, and Zheng-Jiang Zhu. 2019. <span>“Metabolic Reaction Network-Based Recursive Metabolite Annotation for Untargeted Metabolomics.”</span> <em>Nature Communications</em> 10 (1): 1–14. <a href="https://doi.org/10.1038/s41467-019-09550-x">https://doi.org/10.1038/s41467-019-09550-x</a>.
</div>
<div class="csl-entry">
Shen, Xiaotao, Hong Yan, Chuchu Wang, Peng Gao, Caroline H. Johnson, and Michael P. Snyder. 2022. <span>“<span>TidyMass</span> an Object-Oriented Reproducible Analysis Framework for <span>LC</span> Data.”</span> <em>Nature Communications</em> 13 (1): 4365. <a href="https://doi.org/10.1038/s41467-022-32155-w">https://doi.org/10.1038/s41467-022-32155-w</a>.
</div>
<div class="csl-entry">
Silva, Ricardo R. da, Mingxun Wang, Louis-Félix Nothias, Justin J. J. van der Hooft, Andrés Mauricio Caraballo-Rodríguez, Evan Fox, Marcy J. Balunas, Jonathan L. Klassen, Norberto Peporine Lopes, and Pieter C. Dorrestein. 2018. <span>“Propagating Annotations of Molecular Networks Using in Silico Fragmentation.”</span> <em>PLOS Computational Biology</em> 14 (4): e1006089. <a href="https://doi.org/10.1371/journal.pcbi.1006089">https://doi.org/10.1371/journal.pcbi.1006089</a>.
</div>
<div class="csl-entry">
Silva, Ricardo R., Fabien Jourdan, Diego M. Salvanha, Fabien Letisse, Emilien L. Jamin, Simone Guidetti-Gonzalez, Carlos A. Labate, and Ricardo Z. N. Vêncio. 2014. <span>“<span>ProbMetab</span>: An <span>R</span> Package for <span>Bayesian</span> Probabilistic Annotation of <span>LC</span> Metabolomics.”</span> <em>Bioinformatics</em> 30 (9): 1336–37. <a href="https://doi.org/10.1093/bioinformatics/btu019">https://doi.org/10.1093/bioinformatics/btu019</a>.
</div>
<div class="csl-entry">
Sindelar, Miriam, and Gary J. Patti. 2020. <span>“Chemical <span>Discovery</span> in the <span>Era</span> of <span>Metabolomics</span>.”</span> <em>Journal of the American Chemical Society</em>, April. <a href="https://doi.org/10.1021/jacs.9b13198">https://doi.org/10.1021/jacs.9b13198</a>.
</div>
<div class="csl-entry">
Siskos, Alexandros P., Pooja Jain, Werner Römisch-Margl, Mark Bennett, David Achaintre, Yasmin Asad, Luke Marney, et al. 2017. <span>“Interlaboratory <span>Reproducibility</span> of a <span>Targeted Metabolomics Platform</span> for <span>Analysis</span> of <span>Human Serum</span> and <span>Plasma</span>.”</span> <em>Analytical Chemistry</em> 89 (1): 656–65. <a href="https://doi.org/10.1021/acs.analchem.6b02930">https://doi.org/10.1021/acs.analchem.6b02930</a>.
</div>
<div class="csl-entry">
Sitnikov, Dmitri G., Cian S. Monnin, and Dajana Vuckovic. 2016. <span>“Systematic <span>Assessment</span> of <span>Seven Solvent</span> and <span>Solid-Phase Extraction Methods</span> for <span>Metabolomics Analysis</span> of <span>Human Plasma</span> by <span>LC-MS</span>.”</span> <em>Scientific Reports</em> 6 (December). <a href="https://doi.org/10.1038/srep38885">https://doi.org/10.1038/srep38885</a>.
</div>
<div class="csl-entry">
Smirnov, Kirill S., Sara Forcisi, Franco Moritz, Marianna Lucio, and Philippe Schmitt-Kopplin. 2019. <span>“Mass <span>Difference Maps</span> and <span>Their Application</span> for the <span>Recalibration</span> of <span>Mass Spectrometric Data</span> in <span>Nontargeted Metabolomics</span>.”</span> <em>Analytical Chemistry</em> 91 (5): 3350–58. <a href="https://doi.org/10.1021/acs.analchem.8b04555">https://doi.org/10.1021/acs.analchem.8b04555</a>.
</div>
<div class="csl-entry">
Smirnov, Kirill S., Tanja V. Maier, Alesia Walker, Silke S. Heinzmann, Sara Forcisi, Inés Martinez, Jens Walter, and Philippe Schmitt-Kopplin. 2016. <span>“Challenges of Metabolomics in Human Gut Microbiota Research.”</span> <em>International Journal of Medical Microbiology</em>, Intestinal microbiota - a microbial ecosystem at the edge between immune homeostasis and inflammation, 306 (5): 266–79. <a href="https://doi.org/10.1016/j.ijmm.2016.03.006">https://doi.org/10.1016/j.ijmm.2016.03.006</a>.
</div>
<div class="csl-entry">
Smith, Colin A., Elizabeth J. Want, Grace O’Maille, Ruben Abagyan, and Gary Siuzdak. 2006. <span>“<span>XCMS</span>:  <span>Processing Mass Spectrometry Data</span> for <span>Metabolite Profiling Using Nonlinear Peak Alignment</span>, <span>Matching</span>, and <span>Identification</span>.”</span> <em>Analytical Chemistry</em> 78 (3): 779–87. <a href="https://doi.org/10.1021/ac051437y">https://doi.org/10.1021/ac051437y</a>.
</div>
<div class="csl-entry">
Spalding, Jonathan L., Kevin Cho, Nathaniel G. Mahieu, Igor Nikolskiy, Elizabeth M. Llufrio, Stephen L. Johnson, and Gary J. Patti. 2016. <span>“Bar <span>Coding MS2 Spectra</span> for <span>Metabolite Identification</span>.”</span> <em>Analytical Chemistry</em> 88 (5): 2538–42. <a href="https://doi.org/10.1021/acs.analchem.5b04925">https://doi.org/10.1021/acs.analchem.5b04925</a>.
</div>
<div class="csl-entry">
Spicer, Rachel, Reza M. Salek, Pablo Moreno, Daniel Cañueto, and Christoph Steinbeck. 2017. <span>“Navigating Freely-Available Software Tools for Metabolomics Analysis.”</span> <em>Metabolomics</em> 13 (9). <a href="https://doi.org/10.1007/s11306-017-1242-7">https://doi.org/10.1007/s11306-017-1242-7</a>.
</div>
<div class="csl-entry">
Spratlin, Jennifer L., Natalie J. Serkova, and S. Gail Eckhardt. 2009. <span>“Clinical <span>Applications</span> of <span>Metabolomics</span> in <span>Oncology</span>: <span>A Review</span>.”</span> <em>Clinical Cancer Research</em> 15 (2): 431–40. <a href="https://doi.org/10.1158/1078-0432.CCR-08-1059">https://doi.org/10.1158/1078-0432.CCR-08-1059</a>.
</div>
<div class="csl-entry">
Stancliffe, Ethan, Michaela Schwaiger-Haber, Miriam Sindelar, Matthew J. Murphy, Mette Soerensen, and Gary J. Patti. 2022. <span>“An <span>Untargeted Metabolomics Workflow</span> That <span>Scales</span> to <span>Thousands</span> of <span>Samples</span> for <span>Population-Based Studies</span>.”</span> <em>Analytical Chemistry</em>, December. <a href="https://doi.org/10.1021/acs.analchem.2c01270">https://doi.org/10.1021/acs.analchem.2c01270</a>.
</div>
<div class="csl-entry">
Styczynski, Mark P., Joel F. Moxley, Lily V. Tong, Jason L. Walther, Kyle L. Jensen, and Gregory N. Stephanopoulos. 2007. <span>“Systematic <span>Identification</span> of <span>Conserved Metabolites</span> in <span>GC</span>/<span>MS Data</span> for <span>Metabolomics</span> and <span>Biomarker Discovery</span>.”</span> <em>Analytical Chemistry</em> 79 (3): 966–73. <a href="https://doi.org/10.1021/ac0614846">https://doi.org/10.1021/ac0614846</a>.
</div>
<div class="csl-entry">
Sumner, Lloyd W., Alexander Amberg, Dave Barrett, Michael H. Beale, Richard Beger, Clare A. Daykin, Teresa W.-M. Fan, et al. 2007. <span>“Proposed Minimum Reporting Standards for Chemical Analysis <span>Chemical Analysis Working Group</span> (<span>CAWG</span>) <span>Metabolomics Standards Initiative</span> (<span>MSI</span>).”</span> <em>Metabolomics : Official Journal of the Metabolomic Society</em> 3 (3): 211–21. <a href="https://doi.org/10.1007/s11306-007-0082-2">https://doi.org/10.1007/s11306-007-0082-2</a>.
</div>
<div class="csl-entry">
Sumner, Lloyd W, Pedro Mendes, and Richard A Dixon. 2003. <span>“Plant Metabolomics: Large-Scale Phytochemistry in the Functional Genomics Era.”</span> <em>Phytochemistry</em>, Plant <span>Metabolomics</span>, 62 (6): 817–36. <a href="https://doi.org/10.1016/S0031-9422(02)00708-2">https://doi.org/10.1016/S0031-9422(02)00708-2</a>.
</div>
<div class="csl-entry">
Sysi-Aho, Marko, Mikko Katajamaa, Laxman Yetukuri, and Matej Orešič. 2007. <span>“Normalization Method for Metabolomics Data Using Optimal Selection of Multiple Internal Standards.”</span> <em>BMC Bioinformatics</em> 8 (March): 93. <a href="https://doi.org/10.1186/1471-2105-8-93">https://doi.org/10.1186/1471-2105-8-93</a>.
</div>
<div class="csl-entry">
Tang, Yanan, Caley B. Craven, Nicholas J. P. Wawryk, Junlang Qiu, Feng Li, and Xing-Fang Li. 2020. <span>“Advances in Mass Spectrometry-Based Omics Analysis of Trace Organics in Water.”</span> <em>TrAC Trends in Analytical Chemistry</em> 128 (July): 115918. <a href="https://doi.org/10.1016/j.trac.2020.115918">https://doi.org/10.1016/j.trac.2020.115918</a>.
</div>
<div class="csl-entry">
Tautenhahn, Ralf, Christoph Böttcher, and Steffen Neumann. 2008. <span>“Highly Sensitive Feature Detection for High Resolution <span>LC</span>/<span>MS</span>.”</span> <em>BMC Bioinformatics</em> 9: 504. <a href="https://doi.org/10.1186/1471-2105-9-504">https://doi.org/10.1186/1471-2105-9-504</a>.
</div>
<div class="csl-entry">
Tautenhahn, Ralf, Kevin Cho, Winnie Uritboonthai, Zhengjiang Zhu, Gary J. Patti, and Gary Siuzdak. 2012. <span>“An Accelerated Workflow for Untargeted Metabolomics Using the <span>METLIN</span> Database.”</span> <em>Nature Biotechnology</em> 30 (9): 826–28. <a href="https://doi.org/10.1038/nbt.2348">https://doi.org/10.1038/nbt.2348</a>.
</div>
<div class="csl-entry">
Theodoridis, Georgios A., Helen G. Gika, Elizabeth J. Want, and Ian D. Wilson. 2012. <span>“Liquid Chromatographymass Spectrometry Based Global Metabolite Profiling: <span>A</span> Review.”</span> <em>Analytica Chimica Acta</em> 711 (January): 7–16. <a href="https://doi.org/10.1016/j.aca.2011.09.042">https://doi.org/10.1016/j.aca.2011.09.042</a>.
</div>
<div class="csl-entry">
Thonusin, Chanisa, Heidi B. IglayReger, Tanu Soni, Amy E. Rothberg, Charles F. Burant, and Charles R. Evans. 2017. <span>“Evaluation of Intensity Drift Correction Strategies Using <span>MetaboDrift</span>, a Normalization Tool for Multi-Batch Metabolomics Data.”</span> <em>Journal of Chromatography A</em>, Pushing the <span>Boundaries</span> of <span>Chromatography</span> and <span>Electrophoresis</span>, 1523 (Supplement C): 265–74. <a href="https://doi.org/10.1016/j.chroma.2017.09.023">https://doi.org/10.1016/j.chroma.2017.09.023</a>.
</div>
<div class="csl-entry">
Tian, Leqi, Zhenjiang Li, Guoxuan Ma, Xiaoyue Zhang, Ziyin Tang, Siheng Wang, Jian Kang, Donghai Liang, and Tianwei Yu. 2022. <span>“Metapone: A <span>Bioconductor</span> Package for Joint Pathway Testing for Untargeted Metabolomics Data.”</span> <em>Bioinformatics</em> 38 (14): 3662–64. <a href="https://doi.org/10.1093/bioinformatics/btac364">https://doi.org/10.1093/bioinformatics/btac364</a>.
</div>
<div class="csl-entry">
Tian, Tze-Feng, San-Yuan Wang, Tien-Chueh Kuo, Cheng-En Tan, Guan-Yuan Chen, Ching-Hua Kuo, Chi-Hsin Sally Chen, Chang-Chuan Chan, Olivia A. Lin, and Y. Jane Tseng. 2016. <span>“Web <span>Server</span> for <span>Peak Detection</span>, <span>Baseline Correction</span>, and <span>Alignment</span> in <span>Two-Dimensional Gas Chromatography Mass Spectrometry-Based Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 88 (21): 10395–403. <a href="https://doi.org/10.1021/acs.analchem.6b00755">https://doi.org/10.1021/acs.analchem.6b00755</a>.
</div>
<div class="csl-entry">
Treutler, Hendrik, and Steffen Neumann. 2016. <span>“Prediction, <span>Detection</span>, and <span>Validation</span> of <span>Isotope Clusters</span> in <span>Mass Spectrometry Data</span>.”</span> <em>Metabolites</em> 6 (4): 37. <a href="https://doi.org/10.3390/metabo6040037">https://doi.org/10.3390/metabo6040037</a>.
</div>
<div class="csl-entry">
Treutler, Hendrik, Hiroshi Tsugawa, Andrea Porzel, Karin Gorzolka, Alain Tissier, Steffen Neumann, and Gerd Ulrich Balcke. 2016. <span>“Discovering <span>Regulated Metabolite Families</span> in <span>Untargeted Metabolomics Studies</span>.”</span> <em>Analytical Chemistry</em> 88 (16): 8082–90. <a href="https://doi.org/10.1021/acs.analchem.6b01569">https://doi.org/10.1021/acs.analchem.6b01569</a>.
</div>
<div class="csl-entry">
Tsou, Chih-Chiang, Dmitry Avtonomov, Brett Larsen, Monika Tucholska, Hyungwon Choi, Anne-Claude Gingras, and Alexey I. Nesvizhskii. 2015. <span>“<span>DIA-Umpire</span>: Comprehensive Computational Framework for Data-Independent Acquisition Proteomics.”</span> <em>Nature Methods</em> 12 (3): 258–64. <a href="https://doi.org/10.1038/nmeth.3255">https://doi.org/10.1038/nmeth.3255</a>.
</div>
<div class="csl-entry">
Tsugawa, Hiroshi, Tomas Cajka, Tobias Kind, Yan Ma, Brendan Higgins, Kazutaka Ikeda, Mitsuhiro Kanazawa, Jean VanderGheynst, Oliver Fiehn, and Masanori Arita. 2015. <span>“<span>MS-DIAL</span>: Data-Independent <span>MS</span>/<span>MS</span> Deconvolution for Comprehensive Metabolome Analysis.”</span> <em>Nature Methods</em> 12 (6): 523–26. <a href="https://doi.org/10.1038/nmeth.3393">https://doi.org/10.1038/nmeth.3393</a>.
</div>
<div class="csl-entry">
Tsugawa, Hiroshi, Tobias Kind, Ryo Nakabayashi, Daichi Yukihira, Wataru Tanaka, Tomas Cajka, Kazuki Saito, Oliver Fiehn, and Masanori Arita. 2016. <span>“Hydrogen <span>Rearrangement Rules</span>: <span>Computational MS</span>/<span>MS Fragmentation</span> and <span>Structure Elucidation Using MS-FINDER Software</span>.”</span> <em>Analytical Chemistry</em> 88 (16): 7946–58. <a href="https://doi.org/10.1021/acs.analchem.6b00770">https://doi.org/10.1021/acs.analchem.6b00770</a>.
</div>
<div class="csl-entry">
Uchino, Haruki, Hiroshi Tsugawa, Hidenori Takahashi, and Makoto Arita. 2022. <span>“Computational Mass Spectrometry Accelerates <span>C</span> = <span>C</span> Position-Resolved Untargeted Lipidomics Using Oxygen Attachment Dissociation.”</span> <em>Communications Chemistry</em> 5 (1): 1–13. <a href="https://doi.org/10.1038/s42004-022-00778-1">https://doi.org/10.1038/s42004-022-00778-1</a>.
</div>
<div class="csl-entry">
Uppal, Karan, Quinlyn A. Soltow, Frederick H. Strobel, W. Stephen Pittard, Kim M. Gernert, Tianwei Yu, and Dean P. Jones. 2013. <span>“<span class="nocase">xMSanalyzer</span>: Automated Pipeline for Improved Feature Detection and Downstream Analysis of Large-Scale, Non-Targeted Metabolomics Data.”</span> <em>BMC Bioinformatics</em> 14 (1): 15. <a href="https://doi.org/10.1186/1471-2105-14-15">https://doi.org/10.1186/1471-2105-14-15</a>.
</div>
<div class="csl-entry">
Uppal, Karan, Douglas I. Walker, and Dean P. Jones. 2017. <span>“<span class="nocase">xMSannotator</span>: <span>An R Package</span> for <span>Network-Based Annotation</span> of <span>High-Resolution Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 89 (2): 1063–67. <a href="https://doi.org/10.1021/acs.analchem.6b01214">https://doi.org/10.1021/acs.analchem.6b01214</a>.
</div>
<div class="csl-entry">
Uppal, Karan, Douglas I. Walker, Ken Liu, Shuzhao Li, Young-Mi Go, and Dean P. Jones. 2016. <span>“Computational <span>Metabolomics</span>: <span>A Framework</span> for the <span>Million Metabolome</span>.”</span> <em>Chemical Research in Toxicology</em> 29 (12): 1956–75. <a href="https://doi.org/10.1021/acs.chemrestox.6b00179">https://doi.org/10.1021/acs.chemrestox.6b00179</a>.
</div>
<div class="csl-entry">
van der Kloet, Frans M., Ivana Bobeldijk, Elwin R. Verheij, and Renger H. Jellema. 2009. <span>“Analytical <span>Error Reduction Using Single Point Calibration</span> for <span>Accurate</span> and <span>Precise Metabolomic Phenotyping</span>.”</span> <em>Journal of Proteome Research</em> 8 (11): 5132–41. <a href="https://doi.org/10.1021/pr900499r">https://doi.org/10.1021/pr900499r</a>.
</div>
<div class="csl-entry">
Verhoeven, Aswin, Martin Giera, and Oleg A. Mayboroda. 2020. <span>“Scientific Workflow Managers in Metabolomics: An Overview.”</span> <em>Analyst</em> 145 (11): 3801–8. <a href="https://doi.org/10.1039/D0AN00272K">https://doi.org/10.1039/D0AN00272K</a>.
</div>
<div class="csl-entry">
Viant, Mark R., Timothy M. D. Ebbels, Richard D. Beger, Drew R. Ekman, David J. T. Epps, Hennicke Kamp, Pim E. G. Leonards, et al. 2019. <span>“Use Cases, Best Practice and Reporting Standards for Metabolomics in Regulatory Toxicology.”</span> <em>Nature Communications</em> 10 (1): 3041. <a href="https://doi.org/10.1038/s41467-019-10900-y">https://doi.org/10.1038/s41467-019-10900-y</a>.
</div>
<div class="csl-entry">
Viant, Mark R, Irwin J Kurland, Martin R Jones, and Warwick B Dunn. 2017. <span>“How Close Are We to Complete Annotation of Metabolomes?”</span> <em>Current Opinion in Chemical Biology</em>, Omics, 36 (February): 64–69. <a href="https://doi.org/10.1016/j.cbpa.2017.01.001">https://doi.org/10.1016/j.cbpa.2017.01.001</a>.
</div>
<div class="csl-entry">
Vinaixa, Maria, Emma L. Schymanski, Steffen Neumann, Miriam Navarro, Reza M. Salek, and Oscar Yanes. 2016. <span>“Mass Spectral Databases for <span>LC</span>/<span>MS-</span> and <span>GC</span>/<span class="nocase">MS-based</span> Metabolomics: <span>State</span> of the Field and Future Prospects.”</span> <em>TrAC Trends in Analytical Chemistry</em> 78 (April): 23–35. <a href="https://doi.org/10.1016/j.trac.2015.09.005">https://doi.org/10.1016/j.trac.2015.09.005</a>.
</div>
<div class="csl-entry">
Vitale, Chiara Maria, Arjen Lommen, Carolin Huber, Kevin Wagner, Borja Garlito Molina, Rosalie Nijssen, Elliott James Price, et al. 2022. <span>“Harmonized <span>Quality Assurance</span>/<span>Quality Control Provisions</span> for <span>Nontargeted Measurement</span> of <span>Urinary Pesticide Biomarkers</span> in the <span>HBM4EU Multisite SPECIMEn Study</span>.”</span> <em>Analytical Chemistry</em> 94 (22): 7833–43. <a href="https://doi.org/10.1021/acs.analchem.2c00061">https://doi.org/10.1021/acs.analchem.2c00061</a>.
</div>
<div class="csl-entry">
Wallach, Joshua D., Kevin W. Boyack, and John P. A. Ioannidis. 2018. <span>“Reproducible Research Practices, Transparency, and Open Access Data in the Biomedical Literature, 2015.”</span> <em>PLOS Biology</em> 16 (11): e2006930. <a href="https://doi.org/10.1371/journal.pbio.2006930">https://doi.org/10.1371/journal.pbio.2006930</a>.
</div>
<div class="csl-entry">
Wandro, Stephen, Lisa Carmody, Tara Gallagher, John J. LiPuma, and Katrine Whiteson. 2017. <span>“Making <span>It Last</span>: <span>Storage Time</span> and <span>Temperature Have Differential Impacts</span> on <span>Metabolite Profiles</span> of <span>Airway Samples</span> from <span>Cystic Fibrosis Patients</span>.”</span> <em>mSystems</em> 2 (6). <a href="https://doi.org/10.1128/mSystems.00100-17">https://doi.org/10.1128/mSystems.00100-17</a>.
</div>
<div class="csl-entry">
Wang, Mingxun, Jeremy J. Carver, Vanessa V. Phelan, Laura M. Sanchez, Neha Garg, Yao Peng, Don Duy Nguyen, et al. 2016. <span>“Sharing and Community Curation of Mass Spectrometry Data with <span>Global Natural Products Social Molecular Networking</span>.”</span> <em>Nature Biotechnology</em> 34 (8): 828–37. <a href="https://doi.org/10.1038/nbt.3597">https://doi.org/10.1038/nbt.3597</a>.
</div>
<div class="csl-entry">
Wang, Ruohong, Yandong Yin, and Zheng-Jiang Zhu. 2019. <span>“Advancing Untargeted Metabolomics Using Data-Independent Acquisition Mass Spectrometry Technology.”</span> <em>Analytical and Bioanalytical Chemistry</em> 411 (19): 4349–57. <a href="https://doi.org/10.1007/s00216-019-01709-1">https://doi.org/10.1007/s00216-019-01709-1</a>.
</div>
<div class="csl-entry">
Wang, San-Yuan, Ching-Hua Kuo, and Yufeng J. Tseng. 2013. <span>“Batch <span>Normalizer</span>: <span>A Fast Total Abundance Regression Calibration Method</span> to <span>Simultaneously Adjust Batch</span> and <span>Injection Order Effects</span> in <span>Liquid Chromatography</span>/<span class="nocase">Time-of-Flight Mass Spectrometry-Based Metabolomics Data</span> and <span>Comparison</span> with <span>Current Calibration Methods</span>.”</span> <em>Analytical Chemistry</em> 85 (2): 1037–46. <a href="https://doi.org/10.1021/ac302877x">https://doi.org/10.1021/ac302877x</a>.
</div>
<div class="csl-entry">
Wang, Suping, Xiaojuan Jiang, Rong Ding, Binbin Chen, Haiyan Lyu, Junyang Liu, Chunyan Zhu, et al. 2022. <span>“<span>MS-IDF</span>: <span>A Software Tool</span> for <span>Nontargeted Identification</span> of <span>Endogenous Metabolites</span> After <span>Chemical Isotope Labeling Based</span> on a <span>Narrow Mass Defect Filter</span>.”</span> <em>Analytical Chemistry</em> 94 (7): 3194–3202. <a href="https://doi.org/10.1021/acs.analchem.1c04719">https://doi.org/10.1021/acs.analchem.1c04719</a>.
</div>
<div class="csl-entry">
Wang, Yang, Fang Liu, Peng Li, Chengwei He, Ruibing Wang, Huanxing Su, and Jian-Bo Wan. 2016. <span>“An Improved Pseudotargeted Metabolomics Approach Using Multiple Ion Monitoring with Time-Staggered Ion Lists Based on Ultra-High Performance Liquid Chromatography/Quadrupole Time-of-Flight Mass Spectrometry.”</span> <em>Analytica Chimica Acta</em> 927 (July): 82–88. <a href="https://doi.org/10.1016/j.aca.2016.05.008">https://doi.org/10.1016/j.aca.2016.05.008</a>.
</div>
<div class="csl-entry">
Warth, Benedikt, Scott Spangler, Mingliang Fang, Caroline H. Johnson, Erica M. Forsberg, Ana Granados, Richard L. Martin, et al. 2017. <span>“Exposome-<span>Scale Investigations Guided</span> by <span>Global Metabolomics</span>, <span>Pathway Analysis</span>, and <span>Cognitive Computing</span>.”</span> <em>Analytical Chemistry</em> 89 (21): 11505–13. <a href="https://doi.org/10.1021/acs.analchem.7b02759">https://doi.org/10.1021/acs.analchem.7b02759</a>.
</div>
<div class="csl-entry">
Weber, Ralf J. M., Thomas N. Lawson, Reza M. Salek, Timothy M. D. Ebbels, Robert C. Glen, Royston Goodacre, Julian L. Griffin, et al. 2017. <span>“Computational Tools and Workflows in Metabolomics: <span>An</span> International Survey Highlights the Opportunity for Harmonisation Through <span>Galaxy</span>.”</span> <em>Metabolomics</em> 13 (2). <a href="https://doi.org/10.1007/s11306-016-1147-x">https://doi.org/10.1007/s11306-016-1147-x</a>.
</div>
<div class="csl-entry">
Weber, Ralf J. M., and Mark R. Viant. 2010. <span>“<span>MI-Pack</span>: <span>Increased</span> Confidence of Metabolite Identification in Mass Spectra by Integrating Accurate Masses and Metabolic Pathways.”</span> <em>Chemometrics and Intelligent Laboratory Systems</em>, <span>OMICS</span>, 104 (1): 75–82. <a href="https://doi.org/10.1016/j.chemolab.2010.04.010">https://doi.org/10.1016/j.chemolab.2010.04.010</a>.
</div>
<div class="csl-entry">
Wehrens, Ron, Tom G. Bloemberg, and Paul H. C. Eilers. 2015. <span>“Fast Parametric Time Warping of Peak Lists.”</span> <em>Bioinformatics</em> 31 (18): 3063–65. <a href="https://doi.org/10.1093/bioinformatics/btv299">https://doi.org/10.1093/bioinformatics/btv299</a>.
</div>
<div class="csl-entry">
Wei, Runmin, Jingye Wang, Mingming Su, Erik Jia, Shaoqiu Chen, Tianlu Chen, and Yan Ni. 2018. <span>“Missing <span>Value Imputation Approach</span> for <span class="nocase">Mass Spectrometry-based Metabolomics Data</span>.”</span> <em>Scientific Reports</em> 8 (1): 663. <a href="https://doi.org/10.1038/s41598-017-19120-0">https://doi.org/10.1038/s41598-017-19120-0</a>.
</div>
<div class="csl-entry">
Weljie, Aalim M., Jack Newton, Pascal Mercier, Erin Carlson, and Carolyn M. Slupsky. 2006. <span>“Targeted <span>Profiling</span>:  <span>Quantitative Analysis</span> of <span>1H NMR Metabolomics Data</span>.”</span> <em>Analytical Chemistry</em> 78 (13): 4430–42. <a href="https://doi.org/10.1021/ac060209g">https://doi.org/10.1021/ac060209g</a>.
</div>
<div class="csl-entry">
Wen, Bo, Zhanlong Mei, Chunwei Zeng, and Siqi Liu. 2017. <span>“<span class="nocase">metaX</span>: A Flexible and Comprehensive Software for Processing Metabolomics Data.”</span> <em>BMC Bioinformatics</em> 18 (March): 183. <a href="https://doi.org/10.1186/s12859-017-1579-y">https://doi.org/10.1186/s12859-017-1579-y</a>.
</div>
<div class="csl-entry">
Wiklund, Susanne, Erik Johansson, Lina Sjöström, Ewa J. Mellerowicz, Ulf Edlund, John P. Shockcor, Johan Gottfries, Thomas Moritz, and Johan Trygg. 2008. <span>“Visualization of <span>GC</span>/<span>TOF-MS-Based Metabolomics Data</span> for <span>Identification</span> of <span>Biochemically Interesting Compounds Using OPLS Class Models</span>.”</span> <em>Analytical Chemistry</em> 80 (1): 115–22. <a href="https://doi.org/10.1021/ac0713510">https://doi.org/10.1021/ac0713510</a>.
</div>
<div class="csl-entry">
Wise, Stephen A. 2022. <span>“What If Using Certified Reference Materials (<span>CRMs</span>) Was a Requirement to Publish in Analytical/Bioanalytical Chemistry Journals?”</span> <em>Analytical and Bioanalytical Chemistry</em> 414 (24): 7015–22. <a href="https://doi.org/10.1007/s00216-022-04163-8">https://doi.org/10.1007/s00216-022-04163-8</a>.
</div>
<div class="csl-entry">
Wishart, David S. 2016. <span>“Emerging Applications of Metabolomics in Drug Discovery and Precision Medicine.”</span> <em>Nature Reviews Drug Discovery</em> 15 (7): 473–84. <a href="https://doi.org/10.1038/nrd.2016.32">https://doi.org/10.1038/nrd.2016.32</a>.
</div>
<div class="csl-entry">
Witting, Michael, Christoph Ruttkies, Steffen Neumann, and Philippe Schmitt-Kopplin. 2017. <span>“<span>LipidFrag</span>: <span>Improving</span> Reliability of in Silico Fragmentation of Lipids and Application to the <span>Caenorhabditis</span> Elegans Lipidome.”</span> <em>PLOS ONE</em> 12 (3): e0172311. <a href="https://doi.org/10.1371/journal.pone.0172311">https://doi.org/10.1371/journal.pone.0172311</a>.
</div>
<div class="csl-entry">
Wolf, Sebastian, Stephan Schmidt, Matthias Müller-Hannemann, and Steffen Neumann. 2010. <span>“In Silico Fragmentation for Computer Assisted Identification of Metabolite Mass Spectra.”</span> <em>BMC Bioinformatics</em> 11 (March): 148. <a href="https://doi.org/10.1186/1471-2105-11-148">https://doi.org/10.1186/1471-2105-11-148</a>.
</div>
<div class="csl-entry">
Wolfender, Jean-Luc, Guillaume Marti, Aurélien Thomas, and Samuel Bertrand. 2015. <span>“Current Approaches and Challenges for the Metabolite Profiling of Complex Natural Extracts.”</span> <em>Journal of Chromatography A</em>, Editors’ <span>Choice IX</span>, 1382 (February): 136–64. <a href="https://doi.org/10.1016/j.chroma.2014.10.091">https://doi.org/10.1016/j.chroma.2014.10.091</a>.
</div>
<div class="csl-entry">
Wright, Elliott J., Daniel G. Beach, and Pearse McCarron. 2022. <span>“Non-Target Analysis and Stability Assessment of Reference Materials Using Liquid Chromatography-High-Resolution Mass Spectrometry.”</span> <em>Analytica Chimica Acta</em> 1201 (April): 339622. <a href="https://doi.org/10.1016/j.aca.2022.339622">https://doi.org/10.1016/j.aca.2022.339622</a>.
</div>
<div class="csl-entry">
Wu, Yiman, and Liang Li. 2016. <span>“Sample Normalization Methods in Quantitative Metabolomics.”</span> <em>Journal of Chromatography A</em>, Editors’ <span>Choice X</span>, 1430 (January): 80–95. <a href="https://doi.org/10.1016/j.chroma.2015.12.007">https://doi.org/10.1016/j.chroma.2015.12.007</a>.
</div>
<div class="csl-entry">
Xu, Yi-Fan, Wenyun Lu, and Joshua D. Rabinowitz. 2015. <span>“Avoiding <span>Misannotation</span> of <span>In-Source Fragmentation Products</span> as <span>Cellular Metabolites</span> in <span>Liquid Chromatography</span>.”</span> <em>Analytical Chemistry</em> 87 (4): 2273–81. <a href="https://doi.org/10.1021/ac504118y">https://doi.org/10.1021/ac504118y</a>.
</div>
<div class="csl-entry">
Xue, Jingchuan, Rico J. E. Derks, Bill Webb, Elizabeth M. Billings, Aries Aisporna, Martin Giera, and Gary Siuzdak. 2021. <span>“Single <span>Quadrupole Multiple Fragment Ion Monitoring Quantitative Mass Spectrometry</span>.”</span> <em>Analytical Chemistry</em> 93 (31): 10879–89. <a href="https://doi.org/10.1021/acs.analchem.1c01246">https://doi.org/10.1021/acs.analchem.1c01246</a>.
</div>
<div class="csl-entry">
Xue, Jingchuan, Xavier Domingo-Almenara, Carlos Guijas, Amelia Palermo, Markus M. Rinschen, John Isbell, H. Paul Benton, and Gary Siuzdak. 2020. <span>“Enhanced in-<span>Source Fragmentation Annotation Enables Novel Data Independent Acquisition</span> and <span>Autonomous METLIN Molecular Identification</span>.”</span> <em>Analytical Chemistry</em> 92 (8): 6051–59. <a href="https://doi.org/10.1021/acs.analchem.0c00409">https://doi.org/10.1021/acs.analchem.0c00409</a>.
</div>
<div class="csl-entry">
Xue, Jingchuan, Carlos Guijas, H. Paul Benton, Benedikt Warth, and Gary Siuzdak. 2020. <span>“<span>METLIN MS</span> 2 Molecular Standards Database: A Broad Chemical and Biological Resource.”</span> <em>Nature Methods</em> 17 (10): 953–54. <a href="https://doi.org/10.1038/s41592-020-0942-5">https://doi.org/10.1038/s41592-020-0942-5</a>.
</div>
<div class="csl-entry">
Yamamoto, Hiroyuki, Tamaki Fujimori, Hajime Sato, Gen Ishikawa, Kenjiro Kami, and Yoshiaki Ohashi. 2014. <span>“Statistical Hypothesis Testing of Factor Loading in Principal Component Analysis and Its Application to Metabolite Set Enrichment Analysis.”</span> <em>BMC Bioinformatics</em> 15 (February): 51. <a href="https://doi.org/10.1186/1471-2105-15-51">https://doi.org/10.1186/1471-2105-15-51</a>.
</div>
<div class="csl-entry">
Yang, Qingxia, Yunxia Wang, Ying Zhang, Fengcheng Li, Weiqi Xia, Ying Zhou, Yunqing Qiu, Honglin Li, and Feng Zhu. 2020. <span>“<span>NOREVA</span>: Enhanced Normalization and Evaluation of Time-Course and Multi-Class Metabolomic Data.”</span> <em>Nucleic Acids Research</em> 48 (W1): W436–48. <a href="https://doi.org/10.1093/nar/gkaa258">https://doi.org/10.1093/nar/gkaa258</a>.
</div>
<div class="csl-entry">
Yang, Qin, Shan-Shan Lin, Jiang-Tao Yang, Li-Juan Tang, and Ru-Qin Yu. 2017. <span>“Detection of Inborn Errors of Metabolism Utilizing <span>GC-MS</span> Urinary Metabolomics Coupled with a Modified Orthogonal Partial Least Squares Discriminant Analysis.”</span> <em>Talanta</em> 165 (April): 545–52. <a href="https://doi.org/10.1016/j.talanta.2017.01.018">https://doi.org/10.1016/j.talanta.2017.01.018</a>.
</div>
<div class="csl-entry">
Yang, Ruochen, Xi Chen, and Idoia Ochoa. 2019. <span>“<span>MassComp</span>, a Lossless Compressor for Mass Spectrometry Data.”</span> <em>BMC Bioinformatics</em> 20 (1): 368. <a href="https://doi.org/10.1186/s12859-019-2962-7">https://doi.org/10.1186/s12859-019-2962-7</a>.
</div>
<div class="csl-entry">
Yates Iii, John R. 2011. <span>“A Century of Mass Spectrometry: From Atoms to Proteomes.”</span> <em>Nature Methods</em> 8 (8): 633–37. <a href="https://doi.org/10.1038/nmeth.1659">https://doi.org/10.1038/nmeth.1659</a>.
</div>
<div class="csl-entry">
Yu, Miao, Georgia Dolios, and Lauren Petrick. 2022. <span>“Reproducible Untargeted Metabolomics Workflow for Exhaustive <span>MS2</span> Data Acquisition of <span>MS1</span> Features.”</span> <em>Journal of Cheminformatics</em> 14 (1): 6. <a href="https://doi.org/10.1186/s13321-022-00586-8">https://doi.org/10.1186/s13321-022-00586-8</a>.
</div>
<div class="csl-entry">
Yu, Miao, Sofia Lendor, Anna Roszkowska, Mariola Olkowicz, Leslie Bragg, Mark Servos, and Janusz Pawliszyn. 2020. <span>“Metabolic Profile of Fish Muscle Tissue Changes with Sampling Method, Storage Strategy and Time.”</span> <em>Analytica Chimica Acta</em> 1136 (November): 42–50. <a href="https://doi.org/10.1016/j.aca.2020.08.050">https://doi.org/10.1016/j.aca.2020.08.050</a>.
</div>
<div class="csl-entry">
Yu, Miao, Mariola Olkowicz, and Janusz Pawliszyn. 2019. <span>“Structure/Reaction Directed Analysis for <span>LC-MS</span> Based Untargeted Analysis.”</span> <em>Analytica Chimica Acta</em> 1050 (March): 16–24. <a href="https://doi.org/10.1016/j.aca.2018.10.062">https://doi.org/10.1016/j.aca.2018.10.062</a>.
</div>
<div class="csl-entry">
Yu, Miao, Susan L. Teitelbaum, Georgia Dolios, Lam-Ha T. Dang, Peijun Tu, Mary S. Wolff, and Lauren M. Petrick. 2022. <span>“Molecular <span>Gatekeeper Discovery</span>: <span>Workflow</span> for <span>Linking Multiple Exposure Biomarkers</span> to <span>Metabolomics</span>.”</span> <em>Environmental Science &amp; Technology</em>, February. <a href="https://doi.org/10.1021/acs.est.1c04039">https://doi.org/10.1021/acs.est.1c04039</a>.
</div>
<div class="csl-entry">
Yu, Tianwei, Youngja Park, Jennifer M. Johnson, and Dean P. Jones. 2009. <span>“<span class="nocase">apLCMS</span>adaptive Processing of High-Resolution <span>LC</span>/<span>MS</span> Data.”</span> <em>Bioinformatics</em> 25 (15): 1930–36. <a href="https://doi.org/10.1093/bioinformatics/btp291">https://doi.org/10.1093/bioinformatics/btp291</a>.
</div>
<div class="csl-entry">
Yu, Yong-Jie, Qing-Xia Zheng, Yue-Ming Zhang, Qian Zhang, Yu-Ying Zhang, Ping-Ping Liu, Peng Lu, et al. 2019. <span>“Automatic Data Analysis Workflow for Ultra-High Performance Liquid Chromatography-High Resolution Mass Spectrometry-Based Metabolomics.”</span> <em>Journal of Chromatography A</em> 1585 (January): 172–81. <a href="https://doi.org/10.1016/j.chroma.2018.11.070">https://doi.org/10.1016/j.chroma.2018.11.070</a>.
</div>
<div class="csl-entry">
Yu, Zhihao, Haylea C. Miller, Geoffrey J. Puzon, and Brian H. Clowers. 2017. <span>“Development of <span>Untargeted Metabolomics Methods</span> for the <span>Rapid Detection</span> of <span>Pathogenic Naegleria</span> Fowleri.”</span> <em>Environmental Science &amp; Technology</em> 51 (8): 4210–19. <a href="https://doi.org/10.1021/acs.est.6b05969">https://doi.org/10.1021/acs.est.6b05969</a>.
</div>
<div class="csl-entry">
Yuan, Min, Susanne B. Breitkopf, Xuemei Yang, and John M. Asara. 2012. <span>“A Positive/Negative Ionswitching, Targeted Mass Spectrometrybased Metabolomics Platform for Bodily Fluids, Cells, and Fresh and Fixed Tissue.”</span> <em>Nature Protocols</em> 7 (5): 872–81. <a href="https://doi.org/10.1038/nprot.2012.024">https://doi.org/10.1038/nprot.2012.024</a>.
</div>
<div class="csl-entry">
Zenobi, R. 2013. <span>“Single-<span>Cell Metabolomics</span>: <span>Analytical</span> and <span>Biological Perspectives</span>.”</span> <em>Science</em> 342 (6163): 1243259. <a href="https://doi.org/10.1126/science.1243259">https://doi.org/10.1126/science.1243259</a>.
</div>
<div class="csl-entry">
Zha, Haihong, Yuping Cai, Yandong Yin, Zhuozhong Wang, Kang Li, and Zheng-Jiang Zhu. 2018. <span>“<span>SWATHtoMRM</span>: <span>Development</span> of <span>High-Coverage Targeted Metabolomics Method Using SWATH Technology</span> for <span>Biomarker Discovery</span>.”</span> <em>Analytical Chemistry</em> 90 (6): 4062–70. <a href="https://doi.org/10.1021/acs.analchem.7b05318">https://doi.org/10.1021/acs.analchem.7b05318</a>.
</div>
<div class="csl-entry">
Zhang, Aihua, Hui Sun, Ping Wang, Ying Han, and Xijun Wang. 2012. <span>“Modern Analytical Techniques in Metabolomics Analysis.”</span> <em>The Analyst</em> 137 (2): 293–300. <a href="https://doi.org/10.1039/C1AN15605E">https://doi.org/10.1039/C1AN15605E</a>.
</div>
<div class="csl-entry">
Zhang, Yu-Ying, Qian Zhang, Yue-Ming Zhang, Wei-Wei Wang, Li Zhang, Yong-Jie Yu, Chang-Cai Bai, Ji-Zhao Guo, Hai-Yan Fu, and Yuanbin She. 2020. <span>“A Comprehensive Automatic Data Analysis Strategy for Gas Chromatography-Mass Spectrometry Based Untargeted Metabolomics.”</span> <em>Journal of Chromatography A</em> 1616 (April): 460787. <a href="https://doi.org/10.1016/j.chroma.2019.460787">https://doi.org/10.1016/j.chroma.2019.460787</a>.
</div>
<div class="csl-entry">
Zhao, Fan, Shuai Huang, and Xiaozhe Zhang. 2021. <span>“High Sensitivity and Specificity Feature Detection in Liquid Chromatographymass Spectrometry Data: <span>A</span> Deep Learning Framework.”</span> <em>Talanta</em> 222 (January): 121580. <a href="https://doi.org/10.1016/j.talanta.2020.121580">https://doi.org/10.1016/j.talanta.2020.121580</a>.
</div>
<div class="csl-entry">
Zhao, Shuang, and Liang Li. 2020. <span>“Chemical Derivatization in <span class="nocase">LC-MS-based</span> Metabolomics Study.”</span> <em>TrAC Trends in Analytical Chemistry</em> 131 (October): 115988. <a href="https://doi.org/10.1016/j.trac.2020.115988">https://doi.org/10.1016/j.trac.2020.115988</a>.
</div>
<div class="csl-entry">
Zheng, Fujian, Lei You, Wangshu Qin, Runze Ouyang, Wangjie Lv, Lei Guo, Xin Lu, Enyou Li, Xinjie Zhao, and Guowang Xu. 2022. <span>“<span>MetEx</span>: <span>A Targeted Extraction Strategy</span> for <span>Improving</span> the <span>Coverage</span> and <span>Accuracy</span> of <span>Metabolite Annotation</span> in <span>Liquid Chromatography</span>.”</span> <em>Analytical Chemistry</em> 94 (24): 8561–69. <a href="https://doi.org/10.1021/acs.analchem.1c04783">https://doi.org/10.1021/acs.analchem.1c04783</a>.
</div>
<div class="csl-entry">
Zheng, Fujian, Xinjie Zhao, Zhongda Zeng, Lichao Wang, Wangjie Lv, Qingqing Wang, and Guowang Xu. 2020. <span>“Development of a Plasma Pseudotargeted Metabolomics Method Based on Ultra-High-Performance Liquid Chromatographymass Spectrometry.”</span> <em>Nature Protocols</em> 15 (8): 2519–37. <a href="https://doi.org/10.1038/s41596-020-0341-5">https://doi.org/10.1038/s41596-020-0341-5</a>.
</div>
<div class="csl-entry">
Zhou, Juntuo, and Yuxin Yin. 2016. <span>“Strategies for Large-Scale Targeted Metabolomics Quantification by Liquid Chromatography-Mass Spectrometry.”</span> <em>Analyst</em> 141 (23): 6362–73. <a href="https://doi.org/10.1039/C6AN01753C">https://doi.org/10.1039/C6AN01753C</a>.
</div>
<div class="csl-entry">
Zhou, Zhiwei, Mingdu Luo, Haosong Zhang, Yandong Yin, Yuping Cai, and Zheng-Jiang Zhu. 2022. <span>“Metabolite Annotation from Knowns to Unknowns Through Knowledge-Guided Multi-Layer Metabolic Networking.”</span> <em>Nature Communications</em> 13 (1): 6656. <a href="https://doi.org/10.1038/s41467-022-34537-6">https://doi.org/10.1038/s41467-022-34537-6</a>.
</div>
<div class="csl-entry">
Zhu, Xiaochun, Yuping Chen, and Raju Subramanian. 2014. <span>“Comparison of <span>Information-Dependent Acquisition</span>, <span>SWATH</span>, and <span>MSAll Techniques</span> in <span>Metabolite Identification Study Employing Ultrahigh-Performance Liquid Chromatography</span>.”</span> <em>Analytical Chemistry</em> 86 (2): 1202–9. <a href="https://doi.org/10.1021/ac403385y">https://doi.org/10.1021/ac403385y</a>.
</div>
<div class="csl-entry">
Zubeldia-Varela, Elisa, Domingo Barber, Coral Barbas, Marina Perez-Gordo, and David Rojo. 2020. <span>“Sample Pre-Treatment Procedures for the Omics Analysis of Human Gut Microbiota: <span>Turning</span> Points, Tips and Tricks for Gene Sequencing and Metabolomics.”</span> <em>Journal of Pharmaceutical and Biomedical Analysis</em> 191 (November): 113592. <a href="https://doi.org/10.1016/j.jpba.2020.113592">https://doi.org/10.1016/j.jpba.2020.113592</a>.
</div>
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="exposome.html" class="navigation navigation-prev navigation-unique" aria-label="Previous page"><i class="fa fa-angle-left"></i></a>

    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": false,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": {
"link": "https://github.com/yufree/metaworkflow/edit/master/references.Rmd",
"text": "Edit"
},
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": ["Metabolomics.pub"],
"search": {
"engine": "fuse",
"options": null
},
"toc": {
"collapse": "subsection"
}
});
});
</script>

<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    var src = "true";
    if (src === "" || src === "true") src = "https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.9/latest.js?config=TeX-MML-AM_CHTML";
    if (location.protocol !== "file:")
      if (/^https?:/.test(src))
        src = src.replace(/^https?:/, '');
    script.src = src;
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>
</body>

</html>
